{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [],
   "source": [
    "import pandas as pd\n",
    "data_df = pd.read_csv('../../data/out/clef16_caption_classes.csv')\n",
    "\n",
    "labels_dict = data_df.set_index('ID').T.to_dict('list')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>FIG_NAME</th>\n",
       "      <th>CLASS</th>\n",
       "      <th>CAPTION</th>\n",
       "      <th>ID</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>11373_2007_9226_Fig1_HTML-10</td>\n",
       "      <td>DMFL</td>\n",
       "      <td>Colocalization of hNopp140, pol I and rDNA rep...</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>11373_2007_9226_Fig1_HTML-11</td>\n",
       "      <td>DMFL</td>\n",
       "      <td>Colocalization of hNopp140, pol I and rDNA rep...</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>11373_2007_9226_Fig1_HTML-12</td>\n",
       "      <td>DMFL</td>\n",
       "      <td>Colocalization of hNopp140, pol I and rDNA rep...</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>11373_2007_9226_Fig1_HTML-13</td>\n",
       "      <td>DMFL</td>\n",
       "      <td>Colocalization of hNopp140, pol I and rDNA rep...</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>11373_2007_9226_Fig1_HTML-14</td>\n",
       "      <td>DMFL</td>\n",
       "      <td>Colocalization of hNopp140, pol I and rDNA rep...</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                       FIG_NAME CLASS  \\\n",
       "0  11373_2007_9226_Fig1_HTML-10  DMFL   \n",
       "1  11373_2007_9226_Fig1_HTML-11  DMFL   \n",
       "2  11373_2007_9226_Fig1_HTML-12  DMFL   \n",
       "3  11373_2007_9226_Fig1_HTML-13  DMFL   \n",
       "4  11373_2007_9226_Fig1_HTML-14  DMFL   \n",
       "\n",
       "                                             CAPTION  ID  \n",
       "0  Colocalization of hNopp140, pol I and rDNA rep...   0  \n",
       "1  Colocalization of hNopp140, pol I and rDNA rep...   1  \n",
       "2  Colocalization of hNopp140, pol I and rDNA rep...   2  \n",
       "3  Colocalization of hNopp140, pol I and rDNA rep...   3  \n",
       "4  Colocalization of hNopp140, pol I and rDNA rep...   4  "
      ]
     },
     "execution_count": 2,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "data_df.head()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{0: ['11373_2007_9226_Fig1_HTML-10',\n",
       "  'DMFL',\n",
       "  'Colocalization of hNopp140, pol I and rDNA repeats. ( a ) Distribution of hNopp140 and pol I. The rounded-up mitotic cells were collected from HeLa culture by mechanical shake-off (upper panel; arrows for anaphase, arrowheads for telophase). The attached cells were at interphase (lower panel). Cells were subjected to double-fluorescent immunostaining of pol I and hNopp140. The DNA was counterstained with Hoechst 33258. ( b ) Distribution of hNopp140 and rDNA. HeLa cells at interphase were visualized in FISH experiments with a biotin labeled rDNA probe and a detector, avidin ï¿½ï¿½ rhodamin conjugate. The same cells were also immunofluorescence-stained with anti-hNopp140 mAb. Arrow, Cajal body. ( c ) rDNA colocalized with the ectopically expressed hNopp140. HeLa cells transfected with Flag-tagged full-length hNopp140 were processed for detecting the rDNA and the Flag-tagged proteins. Bars, 10 ï¿½ Î¼ m'],\n",
       " 1: ['11373_2007_9226_Fig1_HTML-11',\n",
       "  'DMFL',\n",
       "  'Colocalization of hNopp140, pol I and rDNA repeats. ( a ) Distribution of hNopp140 and pol I. The rounded-up mitotic cells were collected from HeLa culture by mechanical shake-off (upper panel; arrows for anaphase, arrowheads for telophase). The attached cells were at interphase (lower panel). Cells were subjected to double-fluorescent immunostaining of pol I and hNopp140. The DNA was counterstained with Hoechst 33258. ( b ) Distribution of hNopp140 and rDNA. HeLa cells at interphase were visualized in FISH experiments with a biotin labeled rDNA probe and a detector, avidin ï¿½ï¿½ rhodamin conjugate. The same cells were also immunofluorescence-stained with anti-hNopp140 mAb. Arrow, Cajal body. ( c ) rDNA colocalized with the ectopically expressed hNopp140. HeLa cells transfected with Flag-tagged full-length hNopp140 were processed for detecting the rDNA and the Flag-tagged proteins. Bars, 10 ï¿½ Î¼ m'],\n",
       " 2: ['11373_2007_9226_Fig1_HTML-12',\n",
       "  'DMFL',\n",
       "  'Colocalization of hNopp140, pol I and rDNA repeats. ( a ) Distribution of hNopp140 and pol I. The rounded-up mitotic cells were collected from HeLa culture by mechanical shake-off (upper panel; arrows for anaphase, arrowheads for telophase). The attached cells were at interphase (lower panel). Cells were subjected to double-fluorescent immunostaining of pol I and hNopp140. The DNA was counterstained with Hoechst 33258. ( b ) Distribution of hNopp140 and rDNA. HeLa cells at interphase were visualized in FISH experiments with a biotin labeled rDNA probe and a detector, avidin ï¿½ï¿½ rhodamin conjugate. The same cells were also immunofluorescence-stained with anti-hNopp140 mAb. Arrow, Cajal body. ( c ) rDNA colocalized with the ectopically expressed hNopp140. HeLa cells transfected with Flag-tagged full-length hNopp140 were processed for detecting the rDNA and the Flag-tagged proteins. Bars, 10 ï¿½ Î¼ m'],\n",
       " 3: ['11373_2007_9226_Fig1_HTML-13',\n",
       "  'DMFL',\n",
       "  'Colocalization of hNopp140, pol I and rDNA repeats. ( a ) Distribution of hNopp140 and pol I. The rounded-up mitotic cells were collected from HeLa culture by mechanical shake-off (upper panel; arrows for anaphase, arrowheads for telophase). The attached cells were at interphase (lower panel). Cells were subjected to double-fluorescent immunostaining of pol I and hNopp140. The DNA was counterstained with Hoechst 33258. ( b ) Distribution of hNopp140 and rDNA. HeLa cells at interphase were visualized in FISH experiments with a biotin labeled rDNA probe and a detector, avidin ï¿½ï¿½ rhodamin conjugate. The same cells were also immunofluorescence-stained with anti-hNopp140 mAb. Arrow, Cajal body. ( c ) rDNA colocalized with the ectopically expressed hNopp140. HeLa cells transfected with Flag-tagged full-length hNopp140 were processed for detecting the rDNA and the Flag-tagged proteins. Bars, 10 ï¿½ Î¼ m'],\n",
       " 4: ['11373_2007_9226_Fig1_HTML-14',\n",
       "  'DMFL',\n",
       "  'Colocalization of hNopp140, pol I and rDNA repeats. ( a ) Distribution of hNopp140 and pol I. The rounded-up mitotic cells were collected from HeLa culture by mechanical shake-off (upper panel; arrows for anaphase, arrowheads for telophase). The attached cells were at interphase (lower panel). Cells were subjected to double-fluorescent immunostaining of pol I and hNopp140. The DNA was counterstained with Hoechst 33258. ( b ) Distribution of hNopp140 and rDNA. HeLa cells at interphase were visualized in FISH experiments with a biotin labeled rDNA probe and a detector, avidin ï¿½ï¿½ rhodamin conjugate. The same cells were also immunofluorescence-stained with anti-hNopp140 mAb. Arrow, Cajal body. ( c ) rDNA colocalized with the ectopically expressed hNopp140. HeLa cells transfected with Flag-tagged full-length hNopp140 were processed for detecting the rDNA and the Flag-tagged proteins. Bars, 10 ï¿½ Î¼ m'],\n",
       " 5: ['11373_2007_9226_Fig1_HTML-15',\n",
       "  'DMFL',\n",
       "  'Colocalization of hNopp140, pol I and rDNA repeats. ( a ) Distribution of hNopp140 and pol I. The rounded-up mitotic cells were collected from HeLa culture by mechanical shake-off (upper panel; arrows for anaphase, arrowheads for telophase). The attached cells were at interphase (lower panel). Cells were subjected to double-fluorescent immunostaining of pol I and hNopp140. The DNA was counterstained with Hoechst 33258. ( b ) Distribution of hNopp140 and rDNA. HeLa cells at interphase were visualized in FISH experiments with a biotin labeled rDNA probe and a detector, avidin ï¿½ï¿½ rhodamin conjugate. The same cells were also immunofluorescence-stained with anti-hNopp140 mAb. Arrow, Cajal body. ( c ) rDNA colocalized with the ectopically expressed hNopp140. HeLa cells transfected with Flag-tagged full-length hNopp140 were processed for detecting the rDNA and the Flag-tagged proteins. Bars, 10 ï¿½ Î¼ m'],\n",
       " 6: ['11373_2007_9226_Fig1_HTML-16',\n",
       "  'DMFL',\n",
       "  'Colocalization of hNopp140, pol I and rDNA repeats. ( a ) Distribution of hNopp140 and pol I. The rounded-up mitotic cells were collected from HeLa culture by mechanical shake-off (upper panel; arrows for anaphase, arrowheads for telophase). The attached cells were at interphase (lower panel). Cells were subjected to double-fluorescent immunostaining of pol I and hNopp140. The DNA was counterstained with Hoechst 33258. ( b ) Distribution of hNopp140 and rDNA. HeLa cells at interphase were visualized in FISH experiments with a biotin labeled rDNA probe and a detector, avidin ï¿½ï¿½ rhodamin conjugate. The same cells were also immunofluorescence-stained with anti-hNopp140 mAb. Arrow, Cajal body. ( c ) rDNA colocalized with the ectopically expressed hNopp140. HeLa cells transfected with Flag-tagged full-length hNopp140 were processed for detecting the rDNA and the Flag-tagged proteins. Bars, 10 ï¿½ Î¼ m'],\n",
       " 7: ['11373_2007_9226_Fig1_HTML-17',\n",
       "  'DMFL',\n",
       "  'Colocalization of hNopp140, pol I and rDNA repeats. ( a ) Distribution of hNopp140 and pol I. The rounded-up mitotic cells were collected from HeLa culture by mechanical shake-off (upper panel; arrows for anaphase, arrowheads for telophase). The attached cells were at interphase (lower panel). Cells were subjected to double-fluorescent immunostaining of pol I and hNopp140. The DNA was counterstained with Hoechst 33258. ( b ) Distribution of hNopp140 and rDNA. HeLa cells at interphase were visualized in FISH experiments with a biotin labeled rDNA probe and a detector, avidin ï¿½ï¿½ rhodamin conjugate. The same cells were also immunofluorescence-stained with anti-hNopp140 mAb. Arrow, Cajal body. ( c ) rDNA colocalized with the ectopically expressed hNopp140. HeLa cells transfected with Flag-tagged full-length hNopp140 were processed for detecting the rDNA and the Flag-tagged proteins. Bars, 10 ï¿½ Î¼ m'],\n",
       " 8: ['11373_2007_9226_Fig1_HTML-18',\n",
       "  'DMFL',\n",
       "  'Colocalization of hNopp140, pol I and rDNA repeats. ( a ) Distribution of hNopp140 and pol I. The rounded-up mitotic cells were collected from HeLa culture by mechanical shake-off (upper panel; arrows for anaphase, arrowheads for telophase). The attached cells were at interphase (lower panel). Cells were subjected to double-fluorescent immunostaining of pol I and hNopp140. The DNA was counterstained with Hoechst 33258. ( b ) Distribution of hNopp140 and rDNA. HeLa cells at interphase were visualized in FISH experiments with a biotin labeled rDNA probe and a detector, avidin ï¿½ï¿½ rhodamin conjugate. The same cells were also immunofluorescence-stained with anti-hNopp140 mAb. Arrow, Cajal body. ( c ) rDNA colocalized with the ectopically expressed hNopp140. HeLa cells transfected with Flag-tagged full-length hNopp140 were processed for detecting the rDNA and the Flag-tagged proteins. Bars, 10 ï¿½ Î¼ m'],\n",
       " 9: ['11373_2007_9226_Fig1_HTML-19',\n",
       "  'DMFL',\n",
       "  'Colocalization of hNopp140, pol I and rDNA repeats. ( a ) Distribution of hNopp140 and pol I. The rounded-up mitotic cells were collected from HeLa culture by mechanical shake-off (upper panel; arrows for anaphase, arrowheads for telophase). The attached cells were at interphase (lower panel). Cells were subjected to double-fluorescent immunostaining of pol I and hNopp140. The DNA was counterstained with Hoechst 33258. ( b ) Distribution of hNopp140 and rDNA. HeLa cells at interphase were visualized in FISH experiments with a biotin labeled rDNA probe and a detector, avidin ï¿½ï¿½ rhodamin conjugate. The same cells were also immunofluorescence-stained with anti-hNopp140 mAb. Arrow, Cajal body. ( c ) rDNA colocalized with the ectopically expressed hNopp140. HeLa cells transfected with Flag-tagged full-length hNopp140 were processed for detecting the rDNA and the Flag-tagged proteins. Bars, 10 ï¿½ Î¼ m'],\n",
       " 10: ['11373_2007_9226_Fig1_HTML-20',\n",
       "  'DMFL',\n",
       "  'Colocalization of hNopp140, pol I and rDNA repeats. ( a ) Distribution of hNopp140 and pol I. The rounded-up mitotic cells were collected from HeLa culture by mechanical shake-off (upper panel; arrows for anaphase, arrowheads for telophase). The attached cells were at interphase (lower panel). Cells were subjected to double-fluorescent immunostaining of pol I and hNopp140. The DNA was counterstained with Hoechst 33258. ( b ) Distribution of hNopp140 and rDNA. HeLa cells at interphase were visualized in FISH experiments with a biotin labeled rDNA probe and a detector, avidin ï¿½ï¿½ rhodamin conjugate. The same cells were also immunofluorescence-stained with anti-hNopp140 mAb. Arrow, Cajal body. ( c ) rDNA colocalized with the ectopically expressed hNopp140. HeLa cells transfected with Flag-tagged full-length hNopp140 were processed for detecting the rDNA and the Flag-tagged proteins. Bars, 10 ï¿½ Î¼ m'],\n",
       " 11: ['11373_2007_9226_Fig1_HTML-2',\n",
       "  'DMFL',\n",
       "  'Colocalization of hNopp140, pol I and rDNA repeats. ( a ) Distribution of hNopp140 and pol I. The rounded-up mitotic cells were collected from HeLa culture by mechanical shake-off (upper panel; arrows for anaphase, arrowheads for telophase). The attached cells were at interphase (lower panel). Cells were subjected to double-fluorescent immunostaining of pol I and hNopp140. The DNA was counterstained with Hoechst 33258. ( b ) Distribution of hNopp140 and rDNA. HeLa cells at interphase were visualized in FISH experiments with a biotin labeled rDNA probe and a detector, avidin ï¿½ï¿½ rhodamin conjugate. The same cells were also immunofluorescence-stained with anti-hNopp140 mAb. Arrow, Cajal body. ( c ) rDNA colocalized with the ectopically expressed hNopp140. HeLa cells transfected with Flag-tagged full-length hNopp140 were processed for detecting the rDNA and the Flag-tagged proteins. Bars, 10 ï¿½ Î¼ m'],\n",
       " 12: ['11373_2007_9226_Fig1_HTML-3',\n",
       "  'DMFL',\n",
       "  'Colocalization of hNopp140, pol I and rDNA repeats. ( a ) Distribution of hNopp140 and pol I. The rounded-up mitotic cells were collected from HeLa culture by mechanical shake-off (upper panel; arrows for anaphase, arrowheads for telophase). The attached cells were at interphase (lower panel). Cells were subjected to double-fluorescent immunostaining of pol I and hNopp140. The DNA was counterstained with Hoechst 33258. ( b ) Distribution of hNopp140 and rDNA. HeLa cells at interphase were visualized in FISH experiments with a biotin labeled rDNA probe and a detector, avidin ï¿½ï¿½ rhodamin conjugate. The same cells were also immunofluorescence-stained with anti-hNopp140 mAb. Arrow, Cajal body. ( c ) rDNA colocalized with the ectopically expressed hNopp140. HeLa cells transfected with Flag-tagged full-length hNopp140 were processed for detecting the rDNA and the Flag-tagged proteins. Bars, 10 ï¿½ Î¼ m'],\n",
       " 13: ['11373_2007_9226_Fig1_HTML-4',\n",
       "  'DMFL',\n",
       "  'Colocalization of hNopp140, pol I and rDNA repeats. ( a ) Distribution of hNopp140 and pol I. The rounded-up mitotic cells were collected from HeLa culture by mechanical shake-off (upper panel; arrows for anaphase, arrowheads for telophase). The attached cells were at interphase (lower panel). Cells were subjected to double-fluorescent immunostaining of pol I and hNopp140. The DNA was counterstained with Hoechst 33258. ( b ) Distribution of hNopp140 and rDNA. HeLa cells at interphase were visualized in FISH experiments with a biotin labeled rDNA probe and a detector, avidin ï¿½ï¿½ rhodamin conjugate. The same cells were also immunofluorescence-stained with anti-hNopp140 mAb. Arrow, Cajal body. ( c ) rDNA colocalized with the ectopically expressed hNopp140. HeLa cells transfected with Flag-tagged full-length hNopp140 were processed for detecting the rDNA and the Flag-tagged proteins. Bars, 10 ï¿½ Î¼ m'],\n",
       " 14: ['11373_2007_9226_Fig1_HTML-5',\n",
       "  'DMFL',\n",
       "  'Colocalization of hNopp140, pol I and rDNA repeats. ( a ) Distribution of hNopp140 and pol I. The rounded-up mitotic cells were collected from HeLa culture by mechanical shake-off (upper panel; arrows for anaphase, arrowheads for telophase). The attached cells were at interphase (lower panel). Cells were subjected to double-fluorescent immunostaining of pol I and hNopp140. The DNA was counterstained with Hoechst 33258. ( b ) Distribution of hNopp140 and rDNA. HeLa cells at interphase were visualized in FISH experiments with a biotin labeled rDNA probe and a detector, avidin ï¿½ï¿½ rhodamin conjugate. The same cells were also immunofluorescence-stained with anti-hNopp140 mAb. Arrow, Cajal body. ( c ) rDNA colocalized with the ectopically expressed hNopp140. HeLa cells transfected with Flag-tagged full-length hNopp140 were processed for detecting the rDNA and the Flag-tagged proteins. Bars, 10 ï¿½ Î¼ m'],\n",
       " 15: ['11373_2007_9226_Fig1_HTML-6',\n",
       "  'DMFL',\n",
       "  'Colocalization of hNopp140, pol I and rDNA repeats. ( a ) Distribution of hNopp140 and pol I. The rounded-up mitotic cells were collected from HeLa culture by mechanical shake-off (upper panel; arrows for anaphase, arrowheads for telophase). The attached cells were at interphase (lower panel). Cells were subjected to double-fluorescent immunostaining of pol I and hNopp140. The DNA was counterstained with Hoechst 33258. ( b ) Distribution of hNopp140 and rDNA. HeLa cells at interphase were visualized in FISH experiments with a biotin labeled rDNA probe and a detector, avidin ï¿½ï¿½ rhodamin conjugate. The same cells were also immunofluorescence-stained with anti-hNopp140 mAb. Arrow, Cajal body. ( c ) rDNA colocalized with the ectopically expressed hNopp140. HeLa cells transfected with Flag-tagged full-length hNopp140 were processed for detecting the rDNA and the Flag-tagged proteins. Bars, 10 ï¿½ Î¼ m'],\n",
       " 16: ['11373_2007_9226_Fig1_HTML-7',\n",
       "  'DMFL',\n",
       "  'Colocalization of hNopp140, pol I and rDNA repeats. ( a ) Distribution of hNopp140 and pol I. The rounded-up mitotic cells were collected from HeLa culture by mechanical shake-off (upper panel; arrows for anaphase, arrowheads for telophase). The attached cells were at interphase (lower panel). Cells were subjected to double-fluorescent immunostaining of pol I and hNopp140. The DNA was counterstained with Hoechst 33258. ( b ) Distribution of hNopp140 and rDNA. HeLa cells at interphase were visualized in FISH experiments with a biotin labeled rDNA probe and a detector, avidin ï¿½ï¿½ rhodamin conjugate. The same cells were also immunofluorescence-stained with anti-hNopp140 mAb. Arrow, Cajal body. ( c ) rDNA colocalized with the ectopically expressed hNopp140. HeLa cells transfected with Flag-tagged full-length hNopp140 were processed for detecting the rDNA and the Flag-tagged proteins. Bars, 10 ï¿½ Î¼ m'],\n",
       " 17: ['11373_2007_9226_Fig1_HTML-8',\n",
       "  'DMFL',\n",
       "  'Colocalization of hNopp140, pol I and rDNA repeats. ( a ) Distribution of hNopp140 and pol I. The rounded-up mitotic cells were collected from HeLa culture by mechanical shake-off (upper panel; arrows for anaphase, arrowheads for telophase). The attached cells were at interphase (lower panel). Cells were subjected to double-fluorescent immunostaining of pol I and hNopp140. The DNA was counterstained with Hoechst 33258. ( b ) Distribution of hNopp140 and rDNA. HeLa cells at interphase were visualized in FISH experiments with a biotin labeled rDNA probe and a detector, avidin ï¿½ï¿½ rhodamin conjugate. The same cells were also immunofluorescence-stained with anti-hNopp140 mAb. Arrow, Cajal body. ( c ) rDNA colocalized with the ectopically expressed hNopp140. HeLa cells transfected with Flag-tagged full-length hNopp140 were processed for detecting the rDNA and the Flag-tagged proteins. Bars, 10 ï¿½ Î¼ m'],\n",
       " 18: ['11373_2007_9226_Fig1_HTML-9',\n",
       "  'DMFL',\n",
       "  'Colocalization of hNopp140, pol I and rDNA repeats. ( a ) Distribution of hNopp140 and pol I. The rounded-up mitotic cells were collected from HeLa culture by mechanical shake-off (upper panel; arrows for anaphase, arrowheads for telophase). The attached cells were at interphase (lower panel). Cells were subjected to double-fluorescent immunostaining of pol I and hNopp140. The DNA was counterstained with Hoechst 33258. ( b ) Distribution of hNopp140 and rDNA. HeLa cells at interphase were visualized in FISH experiments with a biotin labeled rDNA probe and a detector, avidin ï¿½ï¿½ rhodamin conjugate. The same cells were also immunofluorescence-stained with anti-hNopp140 mAb. Arrow, Cajal body. ( c ) rDNA colocalized with the ectopically expressed hNopp140. HeLa cells transfected with Flag-tagged full-length hNopp140 were processed for detecting the rDNA and the Flag-tagged proteins. Bars, 10 ï¿½ Î¼ m'],\n",
       " 19: ['12178_2007_9007_Fig1_HTML-1', 'GHDR', 'Steps of kyphoplasty,'],\n",
       " 20: ['12178_2007_9007_Fig1_HTML-2', 'DMTR', 'Steps of kyphoplasty,'],\n",
       " 21: ['12178_2007_9007_Fig1_HTML-3', 'GHDR', 'Steps of kyphoplasty,'],\n",
       " 22: ['12178_2007_9007_Fig1_HTML-4', 'GHDR', 'Steps of kyphoplasty,'],\n",
       " 23: ['12178_2008_9027_Fig4_HTML-1',\n",
       "  'DRXR',\n",
       "  '( a ) This anteroposterior radiograph demonstrates a minimally displaced type II supracondylar fracture. ( b ) The lateral view shows slight posterior angulation of the distal fracture fragment. ( c ) After 4 ï¿½ weeks in a cast, there has been interval healing and elbow range of motion may be started'],\n",
       " 24: ['12178_2008_9027_Fig4_HTML-2',\n",
       "  'DRXR',\n",
       "  '( a ) This anteroposterior radiograph demonstrates a minimally displaced type II supracondylar fracture. ( b ) The lateral view shows slight posterior angulation of the distal fracture fragment. ( c ) After 4 ï¿½ weeks in a cast, there has been interval healing and elbow range of motion may be started'],\n",
       " 25: ['12178_2008_9027_Fig4_HTML-3',\n",
       "  'DRXR',\n",
       "  '( a ) This anteroposterior radiograph demonstrates a minimally displaced type II supracondylar fracture. ( b ) The lateral view shows slight posterior angulation of the distal fracture fragment. ( c ) After 4 ï¿½ weeks in a cast, there has been interval healing and elbow range of motion may be started'],\n",
       " 26: ['12178_2009_9056_Fig9_HTML-1',\n",
       "  'DRUS',\n",
       "  'CTS patient #4. Longitudinal view of the median nerve pre-stress ( a ) and during stress ( b ). The median nerve is the darker, hypoechoic linear (or tubular) structure bordered by a thin hyperechoic rim (epineurium) traversing from the left to the right side of the image. The brighter, hyperechoic tubular structure with a fibrillar pattern just below the nerve is the flexor tendon. Note that during stress ( b ) there is compressive narrowing, seen as a depression or notching of the median nerve at the  ï¿½ï¿½ A ï¿½ï¿½  markers (centrally and slightly to the right), measured as a diameter decrease to 0.13 ï¿½ cm, from a pre-stress diameter of 0.20 ï¿½ cm. It appears that during contraction the thenar muscle mass located ventrally (above the nerve in this image) is bulging dorsally into the tunnel against the median nerve, and a portion of the flexor tendon has tightened and is bulging upward (ventrally) into the nerve, effectively  ï¿½ï¿½ sandwiching ï¿½ï¿½  the nerve between the muscle and tendon'],\n",
       " 27: ['12178_2009_9056_Fig9_HTML-2',\n",
       "  'DRUS',\n",
       "  'CTS patient #4. Longitudinal view of the median nerve pre-stress ( a ) and during stress ( b ). The median nerve is the darker, hypoechoic linear (or tubular) structure bordered by a thin hyperechoic rim (epineurium) traversing from the left to the right side of the image. The brighter, hyperechoic tubular structure with a fibrillar pattern just below the nerve is the flexor tendon. Note that during stress ( b ) there is compressive narrowing, seen as a depression or notching of the median nerve at the  ï¿½ï¿½ A ï¿½ï¿½  markers (centrally and slightly to the right), measured as a diameter decrease to 0.13 ï¿½ cm, from a pre-stress diameter of 0.20 ï¿½ cm. It appears that during contraction the thenar muscle mass located ventrally (above the nerve in this image) is bulging dorsally into the tunnel against the median nerve, and a portion of the flexor tendon has tightened and is bulging upward (ventrally) into the nerve, effectively  ï¿½ï¿½ sandwiching ï¿½ï¿½  the nerve between the muscle and tendon'],\n",
       " 28: ['1297-9686-42-10-3-1',\n",
       "  'GFIG',\n",
       "  'Accuracy of QTL-EBV and total EBV as a function of marker distance for genotyped males and ungenotyped females . Accuracy of QTL-EBV and total EBV for marker-assisted BLUP with neighboring marker (NM), 2-marker haplotype (HAP2) and 4-marker haplotype (HAP4), gene-assisted BLUP (GABLUP) when all animals are genotyped and conventional BLUP (CONBLUP); panels A and B: accuracy of QTL-EBV; panels C and D accuracy of total EBV; for MABLUP, sires and males in the last generation were genotyped, the rest was not genotyped, heritability is 0.30, the QTL explains 15% of the genetic variance, results are averages of 200 replicates.'],\n",
       " 29: ['1297-9686-42-10-3-2',\n",
       "  'GFIG',\n",
       "  'Accuracy of QTL-EBV and total EBV as a function of marker distance for genotyped males and ungenotyped females . Accuracy of QTL-EBV and total EBV for marker-assisted BLUP with neighboring marker (NM), 2-marker haplotype (HAP2) and 4-marker haplotype (HAP4), gene-assisted BLUP (GABLUP) when all animals are genotyped and conventional BLUP (CONBLUP); panels A and B: accuracy of QTL-EBV; panels C and D accuracy of total EBV; for MABLUP, sires and males in the last generation were genotyped, the rest was not genotyped, heritability is 0.30, the QTL explains 15% of the genetic variance, results are averages of 200 replicates.'],\n",
       " 30: ['1297-9686-42-10-3-3',\n",
       "  'GFIG',\n",
       "  'Accuracy of QTL-EBV and total EBV as a function of marker distance for genotyped males and ungenotyped females . Accuracy of QTL-EBV and total EBV for marker-assisted BLUP with neighboring marker (NM), 2-marker haplotype (HAP2) and 4-marker haplotype (HAP4), gene-assisted BLUP (GABLUP) when all animals are genotyped and conventional BLUP (CONBLUP); panels A and B: accuracy of QTL-EBV; panels C and D accuracy of total EBV; for MABLUP, sires and males in the last generation were genotyped, the rest was not genotyped, heritability is 0.30, the QTL explains 15% of the genetic variance, results are averages of 200 replicates.'],\n",
       " 31: ['1297-9686-42-10-3-4',\n",
       "  'GFIG',\n",
       "  'Accuracy of QTL-EBV and total EBV as a function of marker distance for genotyped males and ungenotyped females . Accuracy of QTL-EBV and total EBV for marker-assisted BLUP with neighboring marker (NM), 2-marker haplotype (HAP2) and 4-marker haplotype (HAP4), gene-assisted BLUP (GABLUP) when all animals are genotyped and conventional BLUP (CONBLUP); panels A and B: accuracy of QTL-EBV; panels C and D accuracy of total EBV; for MABLUP, sires and males in the last generation were genotyped, the rest was not genotyped, heritability is 0.30, the QTL explains 15% of the genetic variance, results are averages of 200 replicates.'],\n",
       " 32: ['1423-0127-16-109-3-1',\n",
       "  'D3DR',\n",
       "  'A. Three dimensional model of class C protein  (PDB code  2QZ6 )  showing group C specific motif identified from present study in red colour and class C specific PROSITE motif in orange . The active site region is highlighted with dotted surface while the known two of the three conserved regions playing a role in activity are shown in purple. Third conserved region overlapped with motif identified in current study, hence shown in red. B. Three dimensional model of class A beta lactamase (PDC code 1W7F) showing group A specific motif identified in present study (pink) and PROSITE motif (yellow). Dotted surface highlights the active site.'],\n",
       " 33: ['1423-0127-16-109-3-2',\n",
       "  'D3DR',\n",
       "  'A. Three dimensional model of class C protein  (PDB code  2QZ6 )  showing group C specific motif identified from present study in red colour and class C specific PROSITE motif in orange . The active site region is highlighted with dotted surface while the known two of the three conserved regions playing a role in activity are shown in purple. Third conserved region overlapped with motif identified in current study, hence shown in red. B. Three dimensional model of class A beta lactamase (PDC code 1W7F) showing group A specific motif identified in present study (pink) and PROSITE motif (yellow). Dotted surface highlights the active site.'],\n",
       " 34: ['1423-0127-16-6-3-1',\n",
       "  'GGEL',\n",
       "  'Kinetic analysis of human EndoG . (A) Electrophoresis analysis. EndoG (0.05 pmol) was mixed with various amounts of supercoiled pUC18 dsDNA and incubated at 37 Â° C for 3 min. The resulting products were then analyzed by 1.2% agarose gels. Arrowheads denote the different topological forms of pUC18 plasmids. OC: open circular; L: linear; SC: supercoiled. (B) Initial velocity of DNA cleavage as a function of substrate concentration. Steady-state kinetic parameters were determined by curve fitting of these values to the Michaelis-Menten equation.,'],\n",
       " 35: ['1423-0127-16-6-3-2',\n",
       "  'GFIG',\n",
       "  'Kinetic analysis of human EndoG . (A) Electrophoresis analysis. EndoG (0.05 pmol) was mixed with various amounts of supercoiled pUC18 dsDNA and incubated at 37 Â° C for 3 min. The resulting products were then analyzed by 1.2% agarose gels. Arrowheads denote the different topological forms of pUC18 plasmids. OC: open circular; L: linear; SC: supercoiled. (B) Initial velocity of DNA cleavage as a function of substrate concentration. Steady-state kinetic parameters were determined by curve fitting of these values to the Michaelis-Menten equation.,'],\n",
       " 36: ['1423-0127-16-6-5-1',\n",
       "  'GFIG',\n",
       "  \"Cleavage specificities of human EndoG variants . (A)Sequence-specific cleavage assay. The structure of  Eco R I-linearized pKJH20 DNA is shown at the top, with the open box designating the GC-rich sequence region (0.3 kb).  Xba  I site is located 1.6 kb from the  Eco R I site at the 3' end of the linear pKJH20 DNA. In the cleavage assay, pKJH20 was cleaved by  Eco R I and  Xba  I to generate the 2.8-kb and 1.6-kb substrates. Enzymes cleaving within the GC-rich sequence are expected to generate both the 2.8-kb and the 1.3-kb product fragments. (B) Electrophoresis analysis. The reaction mixtures containing 0.2  Î¼ g  Eco R I/ Xba  I-treated pKJH20 and EndoG variants were incubated at 37 Â° C for various periods. The resulting products were analyzed by 1.2% agarose gels. Arrowheads denote the substrate and product fragments.,\"],\n",
       " 37: ['1423-0127-16-6-5-2',\n",
       "  'GGEL',\n",
       "  \"Cleavage specificities of human EndoG variants . (A)Sequence-specific cleavage assay. The structure of  Eco R I-linearized pKJH20 DNA is shown at the top, with the open box designating the GC-rich sequence region (0.3 kb).  Xba  I site is located 1.6 kb from the  Eco R I site at the 3' end of the linear pKJH20 DNA. In the cleavage assay, pKJH20 was cleaved by  Eco R I and  Xba  I to generate the 2.8-kb and 1.6-kb substrates. Enzymes cleaving within the GC-rich sequence are expected to generate both the 2.8-kb and the 1.3-kb product fragments. (B) Electrophoresis analysis. The reaction mixtures containing 0.2  Î¼ g  Eco R I/ Xba  I-treated pKJH20 and EndoG variants were incubated at 37 Â° C for various periods. The resulting products were analyzed by 1.2% agarose gels. Arrowheads denote the substrate and product fragments.,\"],\n",
       " 38: ['1423-0127-16-6-5-3',\n",
       "  'GGEL',\n",
       "  \"Cleavage specificities of human EndoG variants . (A)Sequence-specific cleavage assay. The structure of  Eco R I-linearized pKJH20 DNA is shown at the top, with the open box designating the GC-rich sequence region (0.3 kb).  Xba  I site is located 1.6 kb from the  Eco R I site at the 3' end of the linear pKJH20 DNA. In the cleavage assay, pKJH20 was cleaved by  Eco R I and  Xba  I to generate the 2.8-kb and 1.6-kb substrates. Enzymes cleaving within the GC-rich sequence are expected to generate both the 2.8-kb and the 1.3-kb product fragments. (B) Electrophoresis analysis. The reaction mixtures containing 0.2  Î¼ g  Eco R I/ Xba  I-treated pKJH20 and EndoG variants were incubated at 37 Â° C for various periods. The resulting products were analyzed by 1.2% agarose gels. Arrowheads denote the substrate and product fragments.,\"],\n",
       " 39: ['1423-0127-16-7-1-1',\n",
       "  'DMEL',\n",
       "  'Scanning electron microscope images of the TiO 2  nanoparticles . (A) C150, (B) C200, (C) EDS elemental spectrum of C200, (D)  S. aureus  and UV100, (E)  S. aureus  and C150, and (F)  S. aureus  and C200. Scale bars: 100 nm.'],\n",
       " 40: ['1423-0127-16-7-1-2',\n",
       "  'DMEL',\n",
       "  'Scanning electron microscope images of the TiO 2  nanoparticles . (A) C150, (B) C200, (C) EDS elemental spectrum of C200, (D)  S. aureus  and UV100, (E)  S. aureus  and C150, and (F)  S. aureus  and C200. Scale bars: 100 nm.'],\n",
       " 41: ['1423-0127-16-7-1-3',\n",
       "  'GFIG',\n",
       "  'Scanning electron microscope images of the TiO 2  nanoparticles . (A) C150, (B) C200, (C) EDS elemental spectrum of C200, (D)  S. aureus  and UV100, (E)  S. aureus  and C150, and (F)  S. aureus  and C200. Scale bars: 100 nm.'],\n",
       " 42: ['1423-0127-16-7-1-4',\n",
       "  'DMEL',\n",
       "  'Scanning electron microscope images of the TiO 2  nanoparticles . (A) C150, (B) C200, (C) EDS elemental spectrum of C200, (D)  S. aureus  and UV100, (E)  S. aureus  and C150, and (F)  S. aureus  and C200. Scale bars: 100 nm.'],\n",
       " 43: ['1423-0127-16-7-1-5',\n",
       "  'DMEL',\n",
       "  'Scanning electron microscope images of the TiO 2  nanoparticles . (A) C150, (B) C200, (C) EDS elemental spectrum of C200, (D)  S. aureus  and UV100, (E)  S. aureus  and C150, and (F)  S. aureus  and C200. Scale bars: 100 nm.'],\n",
       " 44: ['1423-0127-16-7-1-6',\n",
       "  'DMEL',\n",
       "  'Scanning electron microscope images of the TiO 2  nanoparticles . (A) C150, (B) C200, (C) EDS elemental spectrum of C200, (D)  S. aureus  and UV100, (E)  S. aureus  and C150, and (F)  S. aureus  and C200. Scale bars: 100 nm.'],\n",
       " 45: ['1423-0127-16-88-1-1',\n",
       "  'DMEL',\n",
       "  'Trophoblast differentiation from embryoid bodies on the biomimetic platform . Panel (a) shows a non-cystic early EB. Panel (b) shows a late cytic EB with clear fluid filled cavity (red arrow) and a small mass of cells similar to inner cell mass (ICM) of an embryos (green arrowhead) pushed towards the top. Panel (c) shows a cystic EB outgrowing on the biomimetic platform on day 16. Panels (d) and (e) immunolocalization of trophoblast markers cytokeratin 8/Troma1 (red) and SSEA1 (green) in cytic EB outgrowths at early day 8. Blue (DAPI stain) represent the nuclei. This shows the evidences for the first signs for clear and distinct trophoblast cell differentiation from cystic EBs as early as day 8.,'],\n",
       " 46: ['1423-0127-16-88-1-2',\n",
       "  'DRMR',\n",
       "  'Trophoblast differentiation from embryoid bodies on the biomimetic platform . Panel (a) shows a non-cystic early EB. Panel (b) shows a late cytic EB with clear fluid filled cavity (red arrow) and a small mass of cells similar to inner cell mass (ICM) of an embryos (green arrowhead) pushed towards the top. Panel (c) shows a cystic EB outgrowing on the biomimetic platform on day 16. Panels (d) and (e) immunolocalization of trophoblast markers cytokeratin 8/Troma1 (red) and SSEA1 (green) in cytic EB outgrowths at early day 8. Blue (DAPI stain) represent the nuclei. This shows the evidences for the first signs for clear and distinct trophoblast cell differentiation from cystic EBs as early as day 8.,'],\n",
       " 47: ['1423-0127-16-88-1-3',\n",
       "  'DMLI',\n",
       "  'Trophoblast differentiation from embryoid bodies on the biomimetic platform . Panel (a) shows a non-cystic early EB. Panel (b) shows a late cytic EB with clear fluid filled cavity (red arrow) and a small mass of cells similar to inner cell mass (ICM) of an embryos (green arrowhead) pushed towards the top. Panel (c) shows a cystic EB outgrowing on the biomimetic platform on day 16. Panels (d) and (e) immunolocalization of trophoblast markers cytokeratin 8/Troma1 (red) and SSEA1 (green) in cytic EB outgrowths at early day 8. Blue (DAPI stain) represent the nuclei. This shows the evidences for the first signs for clear and distinct trophoblast cell differentiation from cystic EBs as early as day 8.,'],\n",
       " 48: ['1423-0127-16-88-1-4',\n",
       "  'DMFL',\n",
       "  'Trophoblast differentiation from embryoid bodies on the biomimetic platform . Panel (a) shows a non-cystic early EB. Panel (b) shows a late cytic EB with clear fluid filled cavity (red arrow) and a small mass of cells similar to inner cell mass (ICM) of an embryos (green arrowhead) pushed towards the top. Panel (c) shows a cystic EB outgrowing on the biomimetic platform on day 16. Panels (d) and (e) immunolocalization of trophoblast markers cytokeratin 8/Troma1 (red) and SSEA1 (green) in cytic EB outgrowths at early day 8. Blue (DAPI stain) represent the nuclei. This shows the evidences for the first signs for clear and distinct trophoblast cell differentiation from cystic EBs as early as day 8.,'],\n",
       " 49: ['1423-0127-16-88-1-5',\n",
       "  'DMFL',\n",
       "  'Trophoblast differentiation from embryoid bodies on the biomimetic platform . Panel (a) shows a non-cystic early EB. Panel (b) shows a late cytic EB with clear fluid filled cavity (red arrow) and a small mass of cells similar to inner cell mass (ICM) of an embryos (green arrowhead) pushed towards the top. Panel (c) shows a cystic EB outgrowing on the biomimetic platform on day 16. Panels (d) and (e) immunolocalization of trophoblast markers cytokeratin 8/Troma1 (red) and SSEA1 (green) in cytic EB outgrowths at early day 8. Blue (DAPI stain) represent the nuclei. This shows the evidences for the first signs for clear and distinct trophoblast cell differentiation from cystic EBs as early as day 8.,'],\n",
       " 50: ['1423-0127-17-22-2-1',\n",
       "  'GGEL',\n",
       "  'MBP-pulldown and mammalian two hybrid assay indicated homo-interaction character of LMO2-L and -S isoform . (A) Coomassie blue R-250 staining of MBP-pulldown samples after SDS-PAGE. NC, amylose resin without recombinant protein and incubated with K562/Jurkat total protein; MAL- Î² -galactase, sample of purified MAL- Î² -galactase incubated with K562/Jurkat total protein used as a control; MAL-LMO2-L/-S, sample of purified MAL-LMO2-L/-S incubated with K562/Jurkat total protein. (B) Western blot analysis of relative samples corresponding to (A) using anti-LMO2 antibody. (C) Mammalian two hybrid assay of LMO2-L and LMO2-S. pACT, activation domain fusion protein expression plasmid; pBIND, GAL4 binding domain fusion protein expression plasmid. +, cells transfected with such component; -, cells transfected without such component.'],\n",
       " 51: ['1423-0127-17-22-2-2',\n",
       "  'GGEL',\n",
       "  'MBP-pulldown and mammalian two hybrid assay indicated homo-interaction character of LMO2-L and -S isoform . (A) Coomassie blue R-250 staining of MBP-pulldown samples after SDS-PAGE. NC, amylose resin without recombinant protein and incubated with K562/Jurkat total protein; MAL- Î² -galactase, sample of purified MAL- Î² -galactase incubated with K562/Jurkat total protein used as a control; MAL-LMO2-L/-S, sample of purified MAL-LMO2-L/-S incubated with K562/Jurkat total protein. (B) Western blot analysis of relative samples corresponding to (A) using anti-LMO2 antibody. (C) Mammalian two hybrid assay of LMO2-L and LMO2-S. pACT, activation domain fusion protein expression plasmid; pBIND, GAL4 binding domain fusion protein expression plasmid. +, cells transfected with such component; -, cells transfected without such component.'],\n",
       " 52: ['1423-0127-17-22-2-3',\n",
       "  'GFIG',\n",
       "  'MBP-pulldown and mammalian two hybrid assay indicated homo-interaction character of LMO2-L and -S isoform . (A) Coomassie blue R-250 staining of MBP-pulldown samples after SDS-PAGE. NC, amylose resin without recombinant protein and incubated with K562/Jurkat total protein; MAL- Î² -galactase, sample of purified MAL- Î² -galactase incubated with K562/Jurkat total protein used as a control; MAL-LMO2-L/-S, sample of purified MAL-LMO2-L/-S incubated with K562/Jurkat total protein. (B) Western blot analysis of relative samples corresponding to (A) using anti-LMO2 antibody. (C) Mammalian two hybrid assay of LMO2-L and LMO2-S. pACT, activation domain fusion protein expression plasmid; pBIND, GAL4 binding domain fusion protein expression plasmid. +, cells transfected with such component; -, cells transfected without such component.'],\n",
       " 53: ['1423-0127-17-22-2-4',\n",
       "  'GTAB',\n",
       "  'MBP-pulldown and mammalian two hybrid assay indicated homo-interaction character of LMO2-L and -S isoform . (A) Coomassie blue R-250 staining of MBP-pulldown samples after SDS-PAGE. NC, amylose resin without recombinant protein and incubated with K562/Jurkat total protein; MAL- Î² -galactase, sample of purified MAL- Î² -galactase incubated with K562/Jurkat total protein used as a control; MAL-LMO2-L/-S, sample of purified MAL-LMO2-L/-S incubated with K562/Jurkat total protein. (B) Western blot analysis of relative samples corresponding to (A) using anti-LMO2 antibody. (C) Mammalian two hybrid assay of LMO2-L and LMO2-S. pACT, activation domain fusion protein expression plasmid; pBIND, GAL4 binding domain fusion protein expression plasmid. +, cells transfected with such component; -, cells transfected without such component.'],\n",
       " 54: ['1423-0127-17-34-3-1',\n",
       "  'D3DR',\n",
       "  'Three domains of AAG A (or C)-chain: (a) domain  Î³ 1, (b) domain  Î³ 2, (c) domain  Î³ 3 . These figures were generated by O program (Jones et al., 1991).'],\n",
       " 55: ['1423-0127-17-34-3-2',\n",
       "  'D3DR',\n",
       "  'Three domains of AAG A (or C)-chain: (a) domain  Î³ 1, (b) domain  Î³ 2, (c) domain  Î³ 3 . These figures were generated by O program (Jones et al., 1991).'],\n",
       " 56: ['1423-0127-17-34-3-3',\n",
       "  'D3DR',\n",
       "  'Three domains of AAG A (or C)-chain: (a) domain  Î³ 1, (b) domain  Î³ 2, (c) domain  Î³ 3 . These figures were generated by O program (Jones et al., 1991).'],\n",
       " 57: ['1423-0127-17-34-5-1',\n",
       "  'D3DR',\n",
       "  'Two domains of AAG B (or D)-chain: (a) domain  Î´ 1, and (b) domain  Î´ 2 . These figures were generated by O program (Jones et al., 1991).,'],\n",
       " 58: ['1423-0127-17-34-5-2',\n",
       "  'D3DR',\n",
       "  'Two domains of AAG B (or D)-chain: (a) domain  Î´ 1, and (b) domain  Î´ 2 . These figures were generated by O program (Jones et al., 1991).,'],\n",
       " 59: ['1423-0127-17-34-6-1',\n",
       "  'GHDR',\n",
       "  'Electron density of the NAG (red) near B 100Asn using the (2Fo - Fc) map contoured at 2.0 F . This figure was generated by O program (Jones et al., 1991).'],\n",
       " 60: ['1423-0127-17-34-6-2',\n",
       "  'GHDR',\n",
       "  'Electron density of the NAG (red) near B 100Asn using the (2Fo - Fc) map contoured at 2.0 F . This figure was generated by O program (Jones et al., 1991).'],\n",
       " 61: ['1423-0127-17-77-3-1',\n",
       "  'DMEL',\n",
       "  'Transfection of SUNE1 cell with  LMP1  induces apoptotic cell death . SUNE1 cells were transiently transfected with pTracer vector (A and B) or  LMP1  expression plasmid, pTracer-LMP1 (C and D). Cell morphology was monitored under bright-field microscopy (A and C) as well as dark field microscopy (B and D). Expression of the green fluorescence protein, GFP, is observed as green colored cells.'],\n",
       " 62: ['1423-0127-17-77-3-2',\n",
       "  'DMLI',\n",
       "  'Transfection of SUNE1 cell with  LMP1  induces apoptotic cell death . SUNE1 cells were transiently transfected with pTracer vector (A and B) or  LMP1  expression plasmid, pTracer-LMP1 (C and D). Cell morphology was monitored under bright-field microscopy (A and C) as well as dark field microscopy (B and D). Expression of the green fluorescence protein, GFP, is observed as green colored cells.'],\n",
       " 63: ['1423-0127-17-77-3-3',\n",
       "  'DMFL',\n",
       "  'Transfection of SUNE1 cell with  LMP1  induces apoptotic cell death . SUNE1 cells were transiently transfected with pTracer vector (A and B) or  LMP1  expression plasmid, pTracer-LMP1 (C and D). Cell morphology was monitored under bright-field microscopy (A and C) as well as dark field microscopy (B and D). Expression of the green fluorescence protein, GFP, is observed as green colored cells.'],\n",
       " 64: ['1423-0127-17-77-3-4',\n",
       "  'DMFL',\n",
       "  'Transfection of SUNE1 cell with  LMP1  induces apoptotic cell death . SUNE1 cells were transiently transfected with pTracer vector (A and B) or  LMP1  expression plasmid, pTracer-LMP1 (C and D). Cell morphology was monitored under bright-field microscopy (A and C) as well as dark field microscopy (B and D). Expression of the green fluorescence protein, GFP, is observed as green colored cells.'],\n",
       " 65: ['1423-0127-17-80-4-1',\n",
       "  'GFIG',\n",
       "  'Effects of HO-1-MSCs transplantation on ventricular function . (A) Hemodynamic assessment of cardiac function at 4 weeks after transplantation. LVSP: left ventricle systolic pressure; LVEDP: left ventricle end-diastolic pressure; + dP/dtmax and -dP/dtmax: rate of rise and fall of ventricular pressure, respectively. means  Â±  SD of data from 6 separate experiments, *P  <  0.05 compared with the hearts treated with PBS,  # P  <  0.05 compared with the hearts treated with Null-MSCs. Lane a, hearts treated with PBS; Lane b, hearts treated with Null-MSCs; Lane c, hearts treated with HO-1-MSCs.,'],\n",
       " 66: ['1423-0127-17-80-4-2',\n",
       "  'GFIG',\n",
       "  'Effects of HO-1-MSCs transplantation on ventricular function . (A) Hemodynamic assessment of cardiac function at 4 weeks after transplantation. LVSP: left ventricle systolic pressure; LVEDP: left ventricle end-diastolic pressure; + dP/dtmax and -dP/dtmax: rate of rise and fall of ventricular pressure, respectively. means  Â±  SD of data from 6 separate experiments, *P  <  0.05 compared with the hearts treated with PBS,  # P  <  0.05 compared with the hearts treated with Null-MSCs. Lane a, hearts treated with PBS; Lane b, hearts treated with Null-MSCs; Lane c, hearts treated with HO-1-MSCs.,'],\n",
       " 67: ['1423-0127-17-80-4-3',\n",
       "  'GFIG',\n",
       "  'Effects of HO-1-MSCs transplantation on ventricular function . (A) Hemodynamic assessment of cardiac function at 4 weeks after transplantation. LVSP: left ventricle systolic pressure; LVEDP: left ventricle end-diastolic pressure; + dP/dtmax and -dP/dtmax: rate of rise and fall of ventricular pressure, respectively. means  Â±  SD of data from 6 separate experiments, *P  <  0.05 compared with the hearts treated with PBS,  # P  <  0.05 compared with the hearts treated with Null-MSCs. Lane a, hearts treated with PBS; Lane b, hearts treated with Null-MSCs; Lane c, hearts treated with HO-1-MSCs.,'],\n",
       " 68: ['1423-0127-17-80-4-4',\n",
       "  'GFIG',\n",
       "  'Effects of HO-1-MSCs transplantation on ventricular function . (A) Hemodynamic assessment of cardiac function at 4 weeks after transplantation. LVSP: left ventricle systolic pressure; LVEDP: left ventricle end-diastolic pressure; + dP/dtmax and -dP/dtmax: rate of rise and fall of ventricular pressure, respectively. means  Â±  SD of data from 6 separate experiments, *P  <  0.05 compared with the hearts treated with PBS,  # P  <  0.05 compared with the hearts treated with Null-MSCs. Lane a, hearts treated with PBS; Lane b, hearts treated with Null-MSCs; Lane c, hearts treated with HO-1-MSCs.,'],\n",
       " 69: ['1423-0127-18-32-6-1',\n",
       "  'DMFL',\n",
       "  'Co-localization of PKA and nNOS proteins in NTS . Confocal microscopic images of NTS showing immunofluorescence staining for nNOS (A, Alexa Fluor 546), PKA (B, Alexa Fluor 488), or cell nuclei (C, DAPI). The merged image (D) showing single staining for PKA (arrowhead) or double immunofluorescence staining (yellow color) for PKA and nNOS (arrows). Scale bar: 25  Î¼ m.'],\n",
       " 70: ['1423-0127-18-32-6-2',\n",
       "  'DMFL',\n",
       "  'Co-localization of PKA and nNOS proteins in NTS . Confocal microscopic images of NTS showing immunofluorescence staining for nNOS (A, Alexa Fluor 546), PKA (B, Alexa Fluor 488), or cell nuclei (C, DAPI). The merged image (D) showing single staining for PKA (arrowhead) or double immunofluorescence staining (yellow color) for PKA and nNOS (arrows). Scale bar: 25  Î¼ m.'],\n",
       " 71: ['1423-0127-18-32-6-3',\n",
       "  'DMFL',\n",
       "  'Co-localization of PKA and nNOS proteins in NTS . Confocal microscopic images of NTS showing immunofluorescence staining for nNOS (A, Alexa Fluor 546), PKA (B, Alexa Fluor 488), or cell nuclei (C, DAPI). The merged image (D) showing single staining for PKA (arrowhead) or double immunofluorescence staining (yellow color) for PKA and nNOS (arrows). Scale bar: 25  Î¼ m.'],\n",
       " 72: ['1423-0127-18-32-6-4',\n",
       "  'DMFL',\n",
       "  'Co-localization of PKA and nNOS proteins in NTS . Confocal microscopic images of NTS showing immunofluorescence staining for nNOS (A, Alexa Fluor 546), PKA (B, Alexa Fluor 488), or cell nuclei (C, DAPI). The merged image (D) showing single staining for PKA (arrowhead) or double immunofluorescence staining (yellow color) for PKA and nNOS (arrows). Scale bar: 25  Î¼ m.'],\n",
       " 73: ['1465-9921-10-20-1-1',\n",
       "  'GFIG',\n",
       "  'The separation of intensity values of COPD and control sera is exemplarily shown for an arbitrary antigen  A . A: Intensity values of each single COPD (1) and control (0) serum for antigen  A  are exemplarily shown. The position of the threshold (vertical green bar (2)) determines the number of true positives (TP), true negatives (TN), false positives (FP) and false negatives (FN). The vertical bars 1 and 3 indicate the minimal and maximal thresholds. B: The two curves represent density estimations of intensity values of COPD patients (red curve) and controls (black curve) for antigen  A  corresponding to Figure 1A. C: The specificity (TN/(TN+FP)) and sensitivity (TP/(TP+FN)) of a test are visualized by the receiver operator characteristics (ROC) curve. The performance of the test can be represented by the area under the ROC curve (AUC). Here, the threshold is represented by the green circle. The values for sensitivity and specificity can be modified by moving the threshold.'],\n",
       " 74: ['1465-9921-10-20-1-2',\n",
       "  'GFIG',\n",
       "  'The separation of intensity values of COPD and control sera is exemplarily shown for an arbitrary antigen  A . A: Intensity values of each single COPD (1) and control (0) serum for antigen  A  are exemplarily shown. The position of the threshold (vertical green bar (2)) determines the number of true positives (TP), true negatives (TN), false positives (FP) and false negatives (FN). The vertical bars 1 and 3 indicate the minimal and maximal thresholds. B: The two curves represent density estimations of intensity values of COPD patients (red curve) and controls (black curve) for antigen  A  corresponding to Figure 1A. C: The specificity (TN/(TN+FP)) and sensitivity (TP/(TP+FN)) of a test are visualized by the receiver operator characteristics (ROC) curve. The performance of the test can be represented by the area under the ROC curve (AUC). Here, the threshold is represented by the green circle. The values for sensitivity and specificity can be modified by moving the threshold.'],\n",
       " 75: ['1465-9921-10-20-1-3',\n",
       "  'GFIG',\n",
       "  'The separation of intensity values of COPD and control sera is exemplarily shown for an arbitrary antigen  A . A: Intensity values of each single COPD (1) and control (0) serum for antigen  A  are exemplarily shown. The position of the threshold (vertical green bar (2)) determines the number of true positives (TP), true negatives (TN), false positives (FP) and false negatives (FN). The vertical bars 1 and 3 indicate the minimal and maximal thresholds. B: The two curves represent density estimations of intensity values of COPD patients (red curve) and controls (black curve) for antigen  A  corresponding to Figure 1A. C: The specificity (TN/(TN+FP)) and sensitivity (TP/(TP+FN)) of a test are visualized by the receiver operator characteristics (ROC) curve. The performance of the test can be represented by the area under the ROC curve (AUC). Here, the threshold is represented by the green circle. The values for sensitivity and specificity can be modified by moving the threshold.'],\n",
       " 76: ['1465-9921-10-22-7-1',\n",
       "  'GFIG',\n",
       "  'Intracellular particle number and surface area upon exposure to the same particle number . Cells were exposed to the same number concentration of particles (3    10 7  particles per well) and the number of intracellular particles was counted by LSM. From the number of intracellular particles, the total particle surface area taken up by the cells was calculated. There was a greater number (A) and surface (B) of 1  Î¼ m particles inside the cells than of 0.05  Î¼ m particles. Due to the small sample size (n = 3) and the use of the Mann Whitney u-test, these obvious differences failed to reach statistical significance (p = 0.1). Note the logarithmic scale on the y-axis in B.,'],\n",
       " 77: ['1465-9921-10-22-7-2',\n",
       "  'GFIG',\n",
       "  'Intracellular particle number and surface area upon exposure to the same particle number . Cells were exposed to the same number concentration of particles (3    10 7  particles per well) and the number of intracellular particles was counted by LSM. From the number of intracellular particles, the total particle surface area taken up by the cells was calculated. There was a greater number (A) and surface (B) of 1  Î¼ m particles inside the cells than of 0.05  Î¼ m particles. Due to the small sample size (n = 3) and the use of the Mann Whitney u-test, these obvious differences failed to reach statistical significance (p = 0.1). Note the logarithmic scale on the y-axis in B.,'],\n",
       " 78: ['1465-9921-10-22-8-1',\n",
       "  'GFIG',\n",
       "  'Intracellular particle number and surface area upon exposure to the same particle surface area . Cells were exposed to the same total surface area concentration of particles (3.5    10 7  Î¼ m 2  per well) and the number of intracellular particles was counted by LSM. From the number of intracellular particles, the total particle surface area taken up by the cells was calculated. At the same surface area concentration, the number of 0.05  Î¼ m particles exceeded the number of 1  Î¼ m particles taken up by the cells (A), however, the fine particles accounted for a greater intracellular particle surface area (B). Due to the small sample size (n = 3) and the use of the Mann Whitney u-test, these obvious differences failed to reach statistical significance (p = 0.1). Note the logarithmic scale on the y-axis in B.'],\n",
       " 79: ['1465-9921-10-22-8-2',\n",
       "  'GFIG',\n",
       "  'Intracellular particle number and surface area upon exposure to the same particle surface area . Cells were exposed to the same total surface area concentration of particles (3.5    10 7  Î¼ m 2  per well) and the number of intracellular particles was counted by LSM. From the number of intracellular particles, the total particle surface area taken up by the cells was calculated. At the same surface area concentration, the number of 0.05  Î¼ m particles exceeded the number of 1  Î¼ m particles taken up by the cells (A), however, the fine particles accounted for a greater intracellular particle surface area (B). Due to the small sample size (n = 3) and the use of the Mann Whitney u-test, these obvious differences failed to reach statistical significance (p = 0.1). Note the logarithmic scale on the y-axis in B.'],\n",
       " 80: ['1465-9921-10-26-1-1',\n",
       "  'GFIG',\n",
       "  'CFTR expression in bone marrow derived dendritic cells (DC) . RNA and protein were extracted from wild type (WT) mouse lung and DC. CFTR expression was measured by real-time RT-PCR and Western analysis. A. Real-time RT-PCR. WT mouse lung tissue was used as a positive control and calibration. The  y -axis represents CFTR cDNA transcription level in terms of relative quantity value (RQ). B. Western analysis of CFTR protein in DC compared to the WT lung tissue. C. Quantification of CFTR protein by image intensity analysis. Images were scanned and analyzed by software Image J normalized to GAPDH loading control. Shown is the mean  Â±  SEM of three pairs of independent samples. **denotes p  <  0.01.,'],\n",
       " 81: ['1465-9921-10-26-1-2',\n",
       "  'GGEL',\n",
       "  'CFTR expression in bone marrow derived dendritic cells (DC) . RNA and protein were extracted from wild type (WT) mouse lung and DC. CFTR expression was measured by real-time RT-PCR and Western analysis. A. Real-time RT-PCR. WT mouse lung tissue was used as a positive control and calibration. The  y -axis represents CFTR cDNA transcription level in terms of relative quantity value (RQ). B. Western analysis of CFTR protein in DC compared to the WT lung tissue. C. Quantification of CFTR protein by image intensity analysis. Images were scanned and analyzed by software Image J normalized to GAPDH loading control. Shown is the mean  Â±  SEM of three pairs of independent samples. **denotes p  <  0.01.,'],\n",
       " 82: ['1465-9921-10-26-1-3',\n",
       "  'GGEL',\n",
       "  'CFTR expression in bone marrow derived dendritic cells (DC) . RNA and protein were extracted from wild type (WT) mouse lung and DC. CFTR expression was measured by real-time RT-PCR and Western analysis. A. Real-time RT-PCR. WT mouse lung tissue was used as a positive control and calibration. The  y -axis represents CFTR cDNA transcription level in terms of relative quantity value (RQ). B. Western analysis of CFTR protein in DC compared to the WT lung tissue. C. Quantification of CFTR protein by image intensity analysis. Images were scanned and analyzed by software Image J normalized to GAPDH loading control. Shown is the mean  Â±  SEM of three pairs of independent samples. **denotes p  <  0.01.,'],\n",
       " 83: ['1465-9921-10-26-1-4',\n",
       "  'GFIG',\n",
       "  'CFTR expression in bone marrow derived dendritic cells (DC) . RNA and protein were extracted from wild type (WT) mouse lung and DC. CFTR expression was measured by real-time RT-PCR and Western analysis. A. Real-time RT-PCR. WT mouse lung tissue was used as a positive control and calibration. The  y -axis represents CFTR cDNA transcription level in terms of relative quantity value (RQ). B. Western analysis of CFTR protein in DC compared to the WT lung tissue. C. Quantification of CFTR protein by image intensity analysis. Images were scanned and analyzed by software Image J normalized to GAPDH loading control. Shown is the mean  Â±  SEM of three pairs of independent samples. **denotes p  <  0.01.,'],\n",
       " 84: ['1465-9921-10-26-2-1',\n",
       "  'GFIG',\n",
       "  'CFTR expression in DC from  Cftr tm1UNC  mice . RNA was extracted from WT and  Cftr tm1UNC  (CF) DC. CFTR expression was measured by real-time RT-PCR and reverse-transcription PCR. A. Real-time RT-PCR. Relative expression levels in the samples were calculated using the    Ct method, using GAPDH as internal normalization control. The  y -axis represents CFTR cDNA transcription level in terms of relative quantity value (RQ). B. Reverse-transcription PCR of CFTR in DC from WT and CF mice. Lung from WT mice were used as positive control. Primers were designed to detect WT CFTR cDNA but not mutant CFTR. GAPDH was used as endogenous PCR control. Shown is the mean  Â±  SEM of three different samples. *denotes p  <  0.05.'],\n",
       " 85: ['1465-9921-10-26-2-2',\n",
       "  'GGEL',\n",
       "  'CFTR expression in DC from  Cftr tm1UNC  mice . RNA was extracted from WT and  Cftr tm1UNC  (CF) DC. CFTR expression was measured by real-time RT-PCR and reverse-transcription PCR. A. Real-time RT-PCR. Relative expression levels in the samples were calculated using the    Ct method, using GAPDH as internal normalization control. The  y -axis represents CFTR cDNA transcription level in terms of relative quantity value (RQ). B. Reverse-transcription PCR of CFTR in DC from WT and CF mice. Lung from WT mice were used as positive control. Primers were designed to detect WT CFTR cDNA but not mutant CFTR. GAPDH was used as endogenous PCR control. Shown is the mean  Â±  SEM of three different samples. *denotes p  <  0.05.'],\n",
       " 86: ['1465-9921-10-26-2-3',\n",
       "  'GGEL',\n",
       "  'CFTR expression in DC from  Cftr tm1UNC  mice . RNA was extracted from WT and  Cftr tm1UNC  (CF) DC. CFTR expression was measured by real-time RT-PCR and reverse-transcription PCR. A. Real-time RT-PCR. Relative expression levels in the samples were calculated using the    Ct method, using GAPDH as internal normalization control. The  y -axis represents CFTR cDNA transcription level in terms of relative quantity value (RQ). B. Reverse-transcription PCR of CFTR in DC from WT and CF mice. Lung from WT mice were used as positive control. Primers were designed to detect WT CFTR cDNA but not mutant CFTR. GAPDH was used as endogenous PCR control. Shown is the mean  Â±  SEM of three different samples. *denotes p  <  0.05.'],\n",
       " 87: ['1465-9921-11-121-1-1',\n",
       "  'GFIG',\n",
       "  'Total BAL cell expression of the M1 markers IL-12p35 (a), IL-23p19 (b) and CCR7 (c), and the M2 markers IL-10 (d) and CCR2 (e) mRNA, in healthy controls (HC) and sarcoidosis patients (Sarc), cultured in medium (24 h), or stimulated with LPS (24 h) . Horizontal lines depict median values. (n: number of individuals analyzed.)'],\n",
       " 88: ['1465-9921-11-121-1-2',\n",
       "  'GFIG',\n",
       "  'Total BAL cell expression of the M1 markers IL-12p35 (a), IL-23p19 (b) and CCR7 (c), and the M2 markers IL-10 (d) and CCR2 (e) mRNA, in healthy controls (HC) and sarcoidosis patients (Sarc), cultured in medium (24 h), or stimulated with LPS (24 h) . Horizontal lines depict median values. (n: number of individuals analyzed.)'],\n",
       " 89: ['1465-9921-11-121-1-3',\n",
       "  'GFIG',\n",
       "  'Total BAL cell expression of the M1 markers IL-12p35 (a), IL-23p19 (b) and CCR7 (c), and the M2 markers IL-10 (d) and CCR2 (e) mRNA, in healthy controls (HC) and sarcoidosis patients (Sarc), cultured in medium (24 h), or stimulated with LPS (24 h) . Horizontal lines depict median values. (n: number of individuals analyzed.)'],\n",
       " 90: ['1465-9921-11-121-1-4',\n",
       "  'GFIG',\n",
       "  'Total BAL cell expression of the M1 markers IL-12p35 (a), IL-23p19 (b) and CCR7 (c), and the M2 markers IL-10 (d) and CCR2 (e) mRNA, in healthy controls (HC) and sarcoidosis patients (Sarc), cultured in medium (24 h), or stimulated with LPS (24 h) . Horizontal lines depict median values. (n: number of individuals analyzed.)'],\n",
       " 91: ['1465-9921-11-121-1-5',\n",
       "  'GFIG',\n",
       "  'Total BAL cell expression of the M1 markers IL-12p35 (a), IL-23p19 (b) and CCR7 (c), and the M2 markers IL-10 (d) and CCR2 (e) mRNA, in healthy controls (HC) and sarcoidosis patients (Sarc), cultured in medium (24 h), or stimulated with LPS (24 h) . Horizontal lines depict median values. (n: number of individuals analyzed.)'],\n",
       " 92: ['1465-9921-11-121-2-1',\n",
       "  'GFIG',\n",
       "  'Intra-individual comparisons of relative gene expression of the M1 markers IL-12p35 (a, b), IL-23p19 (c, d) and CCR7 (e, f) in total BAL cells cultured in medium (four or 24 h), or stimulated with LPS (24 h), in sarcoidosis patients (left columns) and healthy controls (right columns) .,'],\n",
       " 93: ['1465-9921-11-121-2-2',\n",
       "  'GFIG',\n",
       "  'Intra-individual comparisons of relative gene expression of the M1 markers IL-12p35 (a, b), IL-23p19 (c, d) and CCR7 (e, f) in total BAL cells cultured in medium (four or 24 h), or stimulated with LPS (24 h), in sarcoidosis patients (left columns) and healthy controls (right columns) .,'],\n",
       " 94: ['1465-9921-11-121-2-3',\n",
       "  'GFIG',\n",
       "  'Intra-individual comparisons of relative gene expression of the M1 markers IL-12p35 (a, b), IL-23p19 (c, d) and CCR7 (e, f) in total BAL cells cultured in medium (four or 24 h), or stimulated with LPS (24 h), in sarcoidosis patients (left columns) and healthy controls (right columns) .,'],\n",
       " 95: ['1465-9921-11-121-2-4',\n",
       "  'GFIG',\n",
       "  'Intra-individual comparisons of relative gene expression of the M1 markers IL-12p35 (a, b), IL-23p19 (c, d) and CCR7 (e, f) in total BAL cells cultured in medium (four or 24 h), or stimulated with LPS (24 h), in sarcoidosis patients (left columns) and healthy controls (right columns) .,'],\n",
       " 96: ['1465-9921-11-121-2-5',\n",
       "  'GFIG',\n",
       "  'Intra-individual comparisons of relative gene expression of the M1 markers IL-12p35 (a, b), IL-23p19 (c, d) and CCR7 (e, f) in total BAL cells cultured in medium (four or 24 h), or stimulated with LPS (24 h), in sarcoidosis patients (left columns) and healthy controls (right columns) .,'],\n",
       " 97: ['1465-9921-11-121-2-6',\n",
       "  'GFIG',\n",
       "  'Intra-individual comparisons of relative gene expression of the M1 markers IL-12p35 (a, b), IL-23p19 (c, d) and CCR7 (e, f) in total BAL cells cultured in medium (four or 24 h), or stimulated with LPS (24 h), in sarcoidosis patients (left columns) and healthy controls (right columns) .,'],\n",
       " 98: ['1465-9921-11-121-3-1',\n",
       "  'GFIG',\n",
       "  'Intra-individual comparisons of relative gene expression of the M2 markers IL-10 (a, b) and CCR2 (c, d) in total BAL cells cultured in medium (four or 24 h), or stimulated with LPS (24 h), in sarcoidosis patients (left columns) and healthy controls (right columns) .,'],\n",
       " 99: ['1465-9921-11-121-3-2',\n",
       "  'GFIG',\n",
       "  'Intra-individual comparisons of relative gene expression of the M2 markers IL-10 (a, b) and CCR2 (c, d) in total BAL cells cultured in medium (four or 24 h), or stimulated with LPS (24 h), in sarcoidosis patients (left columns) and healthy controls (right columns) .,'],\n",
       " 100: ['1465-9921-11-121-3-3',\n",
       "  'GFIG',\n",
       "  'Intra-individual comparisons of relative gene expression of the M2 markers IL-10 (a, b) and CCR2 (c, d) in total BAL cells cultured in medium (four or 24 h), or stimulated with LPS (24 h), in sarcoidosis patients (left columns) and healthy controls (right columns) .,'],\n",
       " 101: ['1465-9921-11-121-3-4',\n",
       "  'GFIG',\n",
       "  'Intra-individual comparisons of relative gene expression of the M2 markers IL-10 (a, b) and CCR2 (c, d) in total BAL cells cultured in medium (four or 24 h), or stimulated with LPS (24 h), in sarcoidosis patients (left columns) and healthy controls (right columns) .,'],\n",
       " 102: ['1465-9921-11-122-3-1',\n",
       "  'GFIG',\n",
       "  'Relationship between age (panel A) and cumulative smoking exposure (panel B) at entry into the study and degree of airflow limitation in patients with COPD . For further explanations see text.,'],\n",
       " 103: ['1465-9921-11-122-3-2',\n",
       "  'GFIG',\n",
       "  'Relationship between age (panel A) and cumulative smoking exposure (panel B) at entry into the study and degree of airflow limitation in patients with COPD . For further explanations see text.,'],\n",
       " 104: ['1465-9921-11-124-1-1',\n",
       "  'DMLI',\n",
       "  'Morphology and CD90 staining . M Î¦  visualization by Pappenheim staining (A) and electron microscopy (B). Images are representative for cell preparations from at least two different donors. C: Comparison of M Î¦  sizes by forward scatter as measured by flow cytometry. Light grey line: IM; filled/dark grey: AM. D: CD90 staining of AM and IM. Filled/dark grey: isotype control; light grey line: antibody staining. MFI values are given within graphs. Data show one representative out of three independent experiments with cells obtained from different donors.,'],\n",
       " 105: ['1465-9921-11-124-1-2',\n",
       "  'DMLI',\n",
       "  'Morphology and CD90 staining . M Î¦  visualization by Pappenheim staining (A) and electron microscopy (B). Images are representative for cell preparations from at least two different donors. C: Comparison of M Î¦  sizes by forward scatter as measured by flow cytometry. Light grey line: IM; filled/dark grey: AM. D: CD90 staining of AM and IM. Filled/dark grey: isotype control; light grey line: antibody staining. MFI values are given within graphs. Data show one representative out of three independent experiments with cells obtained from different donors.,'],\n",
       " 106: ['1465-9921-11-124-1-3',\n",
       "  'DMEL',\n",
       "  'Morphology and CD90 staining . M Î¦  visualization by Pappenheim staining (A) and electron microscopy (B). Images are representative for cell preparations from at least two different donors. C: Comparison of M Î¦  sizes by forward scatter as measured by flow cytometry. Light grey line: IM; filled/dark grey: AM. D: CD90 staining of AM and IM. Filled/dark grey: isotype control; light grey line: antibody staining. MFI values are given within graphs. Data show one representative out of three independent experiments with cells obtained from different donors.,'],\n",
       " 107: ['1465-9921-11-124-1-4',\n",
       "  'DMEL',\n",
       "  'Morphology and CD90 staining . M Î¦  visualization by Pappenheim staining (A) and electron microscopy (B). Images are representative for cell preparations from at least two different donors. C: Comparison of M Î¦  sizes by forward scatter as measured by flow cytometry. Light grey line: IM; filled/dark grey: AM. D: CD90 staining of AM and IM. Filled/dark grey: isotype control; light grey line: antibody staining. MFI values are given within graphs. Data show one representative out of three independent experiments with cells obtained from different donors.,'],\n",
       " 108: ['1465-9921-11-124-1-5',\n",
       "  'GFIG',\n",
       "  'Morphology and CD90 staining . M Î¦  visualization by Pappenheim staining (A) and electron microscopy (B). Images are representative for cell preparations from at least two different donors. C: Comparison of M Î¦  sizes by forward scatter as measured by flow cytometry. Light grey line: IM; filled/dark grey: AM. D: CD90 staining of AM and IM. Filled/dark grey: isotype control; light grey line: antibody staining. MFI values are given within graphs. Data show one representative out of three independent experiments with cells obtained from different donors.,'],\n",
       " 109: ['1465-9921-11-124-1-6',\n",
       "  'GFIG',\n",
       "  'Morphology and CD90 staining . M Î¦  visualization by Pappenheim staining (A) and electron microscopy (B). Images are representative for cell preparations from at least two different donors. C: Comparison of M Î¦  sizes by forward scatter as measured by flow cytometry. Light grey line: IM; filled/dark grey: AM. D: CD90 staining of AM and IM. Filled/dark grey: isotype control; light grey line: antibody staining. MFI values are given within graphs. Data show one representative out of three independent experiments with cells obtained from different donors.,'],\n",
       " 110: ['1465-9921-11-124-1-7',\n",
       "  'GFIG',\n",
       "  'Morphology and CD90 staining . M Î¦  visualization by Pappenheim staining (A) and electron microscopy (B). Images are representative for cell preparations from at least two different donors. C: Comparison of M Î¦  sizes by forward scatter as measured by flow cytometry. Light grey line: IM; filled/dark grey: AM. D: CD90 staining of AM and IM. Filled/dark grey: isotype control; light grey line: antibody staining. MFI values are given within graphs. Data show one representative out of three independent experiments with cells obtained from different donors.,'],\n",
       " 111: ['1465-9921-11-139-3-1',\n",
       "  'GFIG',\n",
       "  'Desensitization of tachykinin NK 1  (A and B) and NK 2  (C and D) receptors . Human bronchi were pre-treated with SR 48968 (10 -7  M) (A and B) or SR 140333 (10 -7  M) (C and D) before cumulative or non cumulative additions of hHK-1 (B and D,  n  = 5), [Sar 9 ,Met(O 2 ) 11 ] SP (A,  n  = 10) or NKA (C,  n  = 5). Experiments were performed in the presence of phosphoramidon (10 -6  M). Values are expressed in percentage (mean  Â±  s.e.m.) of maximal contraction obtained with ACh 3 mM.'],\n",
       " 112: ['1465-9921-11-139-3-2',\n",
       "  'GFIG',\n",
       "  'Desensitization of tachykinin NK 1  (A and B) and NK 2  (C and D) receptors . Human bronchi were pre-treated with SR 48968 (10 -7  M) (A and B) or SR 140333 (10 -7  M) (C and D) before cumulative or non cumulative additions of hHK-1 (B and D,  n  = 5), [Sar 9 ,Met(O 2 ) 11 ] SP (A,  n  = 10) or NKA (C,  n  = 5). Experiments were performed in the presence of phosphoramidon (10 -6  M). Values are expressed in percentage (mean  Â±  s.e.m.) of maximal contraction obtained with ACh 3 mM.'],\n",
       " 113: ['1465-9921-11-139-3-3',\n",
       "  'GFIG',\n",
       "  'Desensitization of tachykinin NK 1  (A and B) and NK 2  (C and D) receptors . Human bronchi were pre-treated with SR 48968 (10 -7  M) (A and B) or SR 140333 (10 -7  M) (C and D) before cumulative or non cumulative additions of hHK-1 (B and D,  n  = 5), [Sar 9 ,Met(O 2 ) 11 ] SP (A,  n  = 10) or NKA (C,  n  = 5). Experiments were performed in the presence of phosphoramidon (10 -6  M). Values are expressed in percentage (mean  Â±  s.e.m.) of maximal contraction obtained with ACh 3 mM.'],\n",
       " 114: ['1465-9921-11-139-3-4',\n",
       "  'GFIG',\n",
       "  'Desensitization of tachykinin NK 1  (A and B) and NK 2  (C and D) receptors . Human bronchi were pre-treated with SR 48968 (10 -7  M) (A and B) or SR 140333 (10 -7  M) (C and D) before cumulative or non cumulative additions of hHK-1 (B and D,  n  = 5), [Sar 9 ,Met(O 2 ) 11 ] SP (A,  n  = 10) or NKA (C,  n  = 5). Experiments were performed in the presence of phosphoramidon (10 -6  M). Values are expressed in percentage (mean  Â±  s.e.m.) of maximal contraction obtained with ACh 3 mM.'],\n",
       " 115: ['1465-9921-11-139-6-1',\n",
       "  'GFIG',\n",
       "  'Concentration-response curves to (A) cumulative additions of hHK-1, EKA/B, EKC, EKD, SP, NKA and specific NK 1  ([Sar 9 ,Met(O 2 ) 11 ]SP) and NK 2  ([ Î² -Ala 8 ]-NKA (4-10)) receptor agonists on guinea pig trachea ( n  = 6-12); (B) cumulative additions of hHK-1, EKA/B, EKC, SP and [Sar 9 ,Met(O 2 ) 11 ]SP on guinea pig main bronchi ( n  = 6-7) . Experiments were performed in the presence of phosphoramidon (10 -6  M). Values are expressed in percentage (mean  Â±  s.e.m.) of maximal contraction obtained with ACh 3 mM.,'],\n",
       " 116: ['1465-9921-11-139-6-2',\n",
       "  'GFIG',\n",
       "  'Concentration-response curves to (A) cumulative additions of hHK-1, EKA/B, EKC, EKD, SP, NKA and specific NK 1  ([Sar 9 ,Met(O 2 ) 11 ]SP) and NK 2  ([ Î² -Ala 8 ]-NKA (4-10)) receptor agonists on guinea pig trachea ( n  = 6-12); (B) cumulative additions of hHK-1, EKA/B, EKC, SP and [Sar 9 ,Met(O 2 ) 11 ]SP on guinea pig main bronchi ( n  = 6-7) . Experiments were performed in the presence of phosphoramidon (10 -6  M). Values are expressed in percentage (mean  Â±  s.e.m.) of maximal contraction obtained with ACh 3 mM.,'],\n",
       " 117: ['1465-9921-11-139-7-1',\n",
       "  'GFIG',\n",
       "  'Cumulative concentration-response curves to hHK-1 on guinea pig trachea (A and B) and main bronchi (C) pre-treated with NK 1  or NK 2  receptor antagonists . (A) Cumulative additions of hHK-1 on guinea pig trachea in the absence (control) and presence of various concentrations of NK 1  receptor antagonist SR 140333 (10 -9  to 10 -7  M) ( n  = 4-11). (B) Cumulative additions of hHK-1 on guinea pig trachea in the absence (control) and presence of NK 2  receptor antagonist SR 48968 (10 -7  M) ( n  = 6). (C) Cumulative additions of hHK-1 on guinea pig main bronchi in the absence (control) and presence of NK 1  or NK 2  receptor antagonists SR 140333 and SR 48968 (10 -7  M) ( n  = 5-7). Experiments were performed in the presence of phosphoramidon (10 -6  M). Values are expressed in percentage (mean  Â±  s.e.m.) of maximal contraction obtained with ACh 3 mM. Statistical analysis was performed with two-way ANOVA for repeated measures followed with Bonferroni post-test. *  p  <  0.05, **  p  <  0.01 and ***  p  <  0.001  versus  paired control.'],\n",
       " 118: ['1465-9921-11-139-7-2',\n",
       "  'GFIG',\n",
       "  'Cumulative concentration-response curves to hHK-1 on guinea pig trachea (A and B) and main bronchi (C) pre-treated with NK 1  or NK 2  receptor antagonists . (A) Cumulative additions of hHK-1 on guinea pig trachea in the absence (control) and presence of various concentrations of NK 1  receptor antagonist SR 140333 (10 -9  to 10 -7  M) ( n  = 4-11). (B) Cumulative additions of hHK-1 on guinea pig trachea in the absence (control) and presence of NK 2  receptor antagonist SR 48968 (10 -7  M) ( n  = 6). (C) Cumulative additions of hHK-1 on guinea pig main bronchi in the absence (control) and presence of NK 1  or NK 2  receptor antagonists SR 140333 and SR 48968 (10 -7  M) ( n  = 5-7). Experiments were performed in the presence of phosphoramidon (10 -6  M). Values are expressed in percentage (mean  Â±  s.e.m.) of maximal contraction obtained with ACh 3 mM. Statistical analysis was performed with two-way ANOVA for repeated measures followed with Bonferroni post-test. *  p  <  0.05, **  p  <  0.01 and ***  p  <  0.001  versus  paired control.'],\n",
       " 119: ['1465-9921-11-139-7-3',\n",
       "  'GFIG',\n",
       "  'Cumulative concentration-response curves to hHK-1 on guinea pig trachea (A and B) and main bronchi (C) pre-treated with NK 1  or NK 2  receptor antagonists . (A) Cumulative additions of hHK-1 on guinea pig trachea in the absence (control) and presence of various concentrations of NK 1  receptor antagonist SR 140333 (10 -9  to 10 -7  M) ( n  = 4-11). (B) Cumulative additions of hHK-1 on guinea pig trachea in the absence (control) and presence of NK 2  receptor antagonist SR 48968 (10 -7  M) ( n  = 6). (C) Cumulative additions of hHK-1 on guinea pig main bronchi in the absence (control) and presence of NK 1  or NK 2  receptor antagonists SR 140333 and SR 48968 (10 -7  M) ( n  = 5-7). Experiments were performed in the presence of phosphoramidon (10 -6  M). Values are expressed in percentage (mean  Â±  s.e.m.) of maximal contraction obtained with ACh 3 mM. Statistical analysis was performed with two-way ANOVA for repeated measures followed with Bonferroni post-test. *  p  <  0.05, **  p  <  0.01 and ***  p  <  0.001  versus  paired control.'],\n",
       " 120: ['1465-9921-11-154-2-1',\n",
       "  'GFIG',\n",
       "  'Levels of mucus expression in Siglec-F deficient vs WT mice . Different groups of Siglec-F deficient or WT mice were subjected to chronic OVA challenge. Non-OVA challenged mice served as a control. The level of mucus expression was quantitated in lung sections by PAS staining using a light microscope objective at 20   (Fig 2 A-E). Chronic OVA challenge in WT mice induced a significant increase in the number of PAS+ mucus cells (p = 0.0001*)(WT no OVA vs WT OVA)(Fig 2A; Fig 2B vs Fig 2D). Levels of mucus expression were significantly increased in OVA challenged Siglec-F deficient mice compared to OVA challenged WT mice (p = 0.0001 # )(Fig 2A; Fig 2E vs Fig 2D)(n = 16 mice/group).,'],\n",
       " 121: ['1465-9921-11-154-2-2',\n",
       "  'DMLI',\n",
       "  'Levels of mucus expression in Siglec-F deficient vs WT mice . Different groups of Siglec-F deficient or WT mice were subjected to chronic OVA challenge. Non-OVA challenged mice served as a control. The level of mucus expression was quantitated in lung sections by PAS staining using a light microscope objective at 20   (Fig 2 A-E). Chronic OVA challenge in WT mice induced a significant increase in the number of PAS+ mucus cells (p = 0.0001*)(WT no OVA vs WT OVA)(Fig 2A; Fig 2B vs Fig 2D). Levels of mucus expression were significantly increased in OVA challenged Siglec-F deficient mice compared to OVA challenged WT mice (p = 0.0001 # )(Fig 2A; Fig 2E vs Fig 2D)(n = 16 mice/group).,'],\n",
       " 122: ['1465-9921-11-154-2-3',\n",
       "  'DMLI',\n",
       "  'Levels of mucus expression in Siglec-F deficient vs WT mice . Different groups of Siglec-F deficient or WT mice were subjected to chronic OVA challenge. Non-OVA challenged mice served as a control. The level of mucus expression was quantitated in lung sections by PAS staining using a light microscope objective at 20   (Fig 2 A-E). Chronic OVA challenge in WT mice induced a significant increase in the number of PAS+ mucus cells (p = 0.0001*)(WT no OVA vs WT OVA)(Fig 2A; Fig 2B vs Fig 2D). Levels of mucus expression were significantly increased in OVA challenged Siglec-F deficient mice compared to OVA challenged WT mice (p = 0.0001 # )(Fig 2A; Fig 2E vs Fig 2D)(n = 16 mice/group).,'],\n",
       " 123: ['1465-9921-11-154-2-4',\n",
       "  'DMLI',\n",
       "  'Levels of mucus expression in Siglec-F deficient vs WT mice . Different groups of Siglec-F deficient or WT mice were subjected to chronic OVA challenge. Non-OVA challenged mice served as a control. The level of mucus expression was quantitated in lung sections by PAS staining using a light microscope objective at 20   (Fig 2 A-E). Chronic OVA challenge in WT mice induced a significant increase in the number of PAS+ mucus cells (p = 0.0001*)(WT no OVA vs WT OVA)(Fig 2A; Fig 2B vs Fig 2D). Levels of mucus expression were significantly increased in OVA challenged Siglec-F deficient mice compared to OVA challenged WT mice (p = 0.0001 # )(Fig 2A; Fig 2E vs Fig 2D)(n = 16 mice/group).,'],\n",
       " 124: ['1465-9921-11-154-2-5',\n",
       "  'DMLI',\n",
       "  'Levels of mucus expression in Siglec-F deficient vs WT mice . Different groups of Siglec-F deficient or WT mice were subjected to chronic OVA challenge. Non-OVA challenged mice served as a control. The level of mucus expression was quantitated in lung sections by PAS staining using a light microscope objective at 20   (Fig 2 A-E). Chronic OVA challenge in WT mice induced a significant increase in the number of PAS+ mucus cells (p = 0.0001*)(WT no OVA vs WT OVA)(Fig 2A; Fig 2B vs Fig 2D). Levels of mucus expression were significantly increased in OVA challenged Siglec-F deficient mice compared to OVA challenged WT mice (p = 0.0001 # )(Fig 2A; Fig 2E vs Fig 2D)(n = 16 mice/group).,'],\n",
       " 125: ['1465-9921-11-154-7-1',\n",
       "  'DMLI',\n",
       "  'Quantitation of Siglec-F ligands in airway epithelial cells and peribronchial inflammatory cells in WT and Siglec-F deficient mice challenged with IL-4, IL-13 or TNF- Î± . WT or Siglec-F deficient mice were administered either IL-4, IL-13, TNF- Î± , or PBS diluent control. Twenty four hours after each individual cytokine or diluent challenge, the mice were sacrificed. BAL was obtained for determination of eosinophil and neutrophil cell counts, and the lungs were processed for immunohistology to detect Siglec-F ligand expression and MBP+ peribronchial eosinophils using a light microscope objective at 20  . Administration of either IL-4 (p  <  0.001, WT; p  <  0.001, Siglec-F deficient) or IL-13 (p  <  0.001, WT; p  <  0.001, Siglec-F deficient) induced a similar significant increase in levels of Siglec-F ligand expression by peribronchial cells (Fig 7A-C, 7E). In contrast, administration of TNF- Î±  induced a small increase in peribronchial Siglec-F ligands (p  <  0.01)(Fig 7E), but did not significantly increase Siglec-F ligand expression by airway epithelium in either WT or Siglec-F deficient mice (p = ns) (Fig 7 D, 7F). Administration of either IL-4 (p  <  0.001 WT; p  <  0.001 Siglec-F deficient) or IL-13 (p  <  0.001 WT; p  <  0.001 Siglec-F deficient) induced significantly increased levels of Siglec-F ligand expression by airway epithelium (Fig 7A-C, 7F). Although both IL-4 and IL-13 induced strong upregulation of Siglec-F ligand expression by airway epithelium (Fig 7F), levels of Siglec-F ligands in airway epithelium were slightly lower in Siglec-F deficient vs WT mice induced by IL-13 (p = 0.02) but not IL-4 (p = 0.10) (Fig 7F). IL-4 induced the strongest eosinophil response in BAL (WT p  <  0.03; Siglec-F deficient p = 0.02)(Fig 7G) and lung (WT p  <  0.001; Siglec-F deficient p  <  0.001)(Fig 7H), while IL-13 and TNF- Î±  induced a weaker eosinophil response in BAL (Fig 7G) and lung (Fig 7H). TNF- Î± , but not IL-4 or IL-13, induced a strong neutrophil response in BAL in both WT (p = 0.03) and Siglec-F deficient mice (p = 0.03) (Fig 7I)(n = 3 mice/group).'],\n",
       " 126: ['1465-9921-11-154-7-2',\n",
       "  'DMLI',\n",
       "  'Quantitation of Siglec-F ligands in airway epithelial cells and peribronchial inflammatory cells in WT and Siglec-F deficient mice challenged with IL-4, IL-13 or TNF- Î± . WT or Siglec-F deficient mice were administered either IL-4, IL-13, TNF- Î± , or PBS diluent control. Twenty four hours after each individual cytokine or diluent challenge, the mice were sacrificed. BAL was obtained for determination of eosinophil and neutrophil cell counts, and the lungs were processed for immunohistology to detect Siglec-F ligand expression and MBP+ peribronchial eosinophils using a light microscope objective at 20  . Administration of either IL-4 (p  <  0.001, WT; p  <  0.001, Siglec-F deficient) or IL-13 (p  <  0.001, WT; p  <  0.001, Siglec-F deficient) induced a similar significant increase in levels of Siglec-F ligand expression by peribronchial cells (Fig 7A-C, 7E). In contrast, administration of TNF- Î±  induced a small increase in peribronchial Siglec-F ligands (p  <  0.01)(Fig 7E), but did not significantly increase Siglec-F ligand expression by airway epithelium in either WT or Siglec-F deficient mice (p = ns) (Fig 7 D, 7F). Administration of either IL-4 (p  <  0.001 WT; p  <  0.001 Siglec-F deficient) or IL-13 (p  <  0.001 WT; p  <  0.001 Siglec-F deficient) induced significantly increased levels of Siglec-F ligand expression by airway epithelium (Fig 7A-C, 7F). Although both IL-4 and IL-13 induced strong upregulation of Siglec-F ligand expression by airway epithelium (Fig 7F), levels of Siglec-F ligands in airway epithelium were slightly lower in Siglec-F deficient vs WT mice induced by IL-13 (p = 0.02) but not IL-4 (p = 0.10) (Fig 7F). IL-4 induced the strongest eosinophil response in BAL (WT p  <  0.03; Siglec-F deficient p = 0.02)(Fig 7G) and lung (WT p  <  0.001; Siglec-F deficient p  <  0.001)(Fig 7H), while IL-13 and TNF- Î±  induced a weaker eosinophil response in BAL (Fig 7G) and lung (Fig 7H). TNF- Î± , but not IL-4 or IL-13, induced a strong neutrophil response in BAL in both WT (p = 0.03) and Siglec-F deficient mice (p = 0.03) (Fig 7I)(n = 3 mice/group).'],\n",
       " 127: ['1465-9921-11-154-7-3',\n",
       "  'DMLI',\n",
       "  'Quantitation of Siglec-F ligands in airway epithelial cells and peribronchial inflammatory cells in WT and Siglec-F deficient mice challenged with IL-4, IL-13 or TNF- Î± . WT or Siglec-F deficient mice were administered either IL-4, IL-13, TNF- Î± , or PBS diluent control. Twenty four hours after each individual cytokine or diluent challenge, the mice were sacrificed. BAL was obtained for determination of eosinophil and neutrophil cell counts, and the lungs were processed for immunohistology to detect Siglec-F ligand expression and MBP+ peribronchial eosinophils using a light microscope objective at 20  . Administration of either IL-4 (p  <  0.001, WT; p  <  0.001, Siglec-F deficient) or IL-13 (p  <  0.001, WT; p  <  0.001, Siglec-F deficient) induced a similar significant increase in levels of Siglec-F ligand expression by peribronchial cells (Fig 7A-C, 7E). In contrast, administration of TNF- Î±  induced a small increase in peribronchial Siglec-F ligands (p  <  0.01)(Fig 7E), but did not significantly increase Siglec-F ligand expression by airway epithelium in either WT or Siglec-F deficient mice (p = ns) (Fig 7 D, 7F). Administration of either IL-4 (p  <  0.001 WT; p  <  0.001 Siglec-F deficient) or IL-13 (p  <  0.001 WT; p  <  0.001 Siglec-F deficient) induced significantly increased levels of Siglec-F ligand expression by airway epithelium (Fig 7A-C, 7F). Although both IL-4 and IL-13 induced strong upregulation of Siglec-F ligand expression by airway epithelium (Fig 7F), levels of Siglec-F ligands in airway epithelium were slightly lower in Siglec-F deficient vs WT mice induced by IL-13 (p = 0.02) but not IL-4 (p = 0.10) (Fig 7F). IL-4 induced the strongest eosinophil response in BAL (WT p  <  0.03; Siglec-F deficient p = 0.02)(Fig 7G) and lung (WT p  <  0.001; Siglec-F deficient p  <  0.001)(Fig 7H), while IL-13 and TNF- Î±  induced a weaker eosinophil response in BAL (Fig 7G) and lung (Fig 7H). TNF- Î± , but not IL-4 or IL-13, induced a strong neutrophil response in BAL in both WT (p = 0.03) and Siglec-F deficient mice (p = 0.03) (Fig 7I)(n = 3 mice/group).'],\n",
       " 128: ['1465-9921-11-154-7-4',\n",
       "  'DMLI',\n",
       "  'Quantitation of Siglec-F ligands in airway epithelial cells and peribronchial inflammatory cells in WT and Siglec-F deficient mice challenged with IL-4, IL-13 or TNF- Î± . WT or Siglec-F deficient mice were administered either IL-4, IL-13, TNF- Î± , or PBS diluent control. Twenty four hours after each individual cytokine or diluent challenge, the mice were sacrificed. BAL was obtained for determination of eosinophil and neutrophil cell counts, and the lungs were processed for immunohistology to detect Siglec-F ligand expression and MBP+ peribronchial eosinophils using a light microscope objective at 20  . Administration of either IL-4 (p  <  0.001, WT; p  <  0.001, Siglec-F deficient) or IL-13 (p  <  0.001, WT; p  <  0.001, Siglec-F deficient) induced a similar significant increase in levels of Siglec-F ligand expression by peribronchial cells (Fig 7A-C, 7E). In contrast, administration of TNF- Î±  induced a small increase in peribronchial Siglec-F ligands (p  <  0.01)(Fig 7E), but did not significantly increase Siglec-F ligand expression by airway epithelium in either WT or Siglec-F deficient mice (p = ns) (Fig 7 D, 7F). Administration of either IL-4 (p  <  0.001 WT; p  <  0.001 Siglec-F deficient) or IL-13 (p  <  0.001 WT; p  <  0.001 Siglec-F deficient) induced significantly increased levels of Siglec-F ligand expression by airway epithelium (Fig 7A-C, 7F). Although both IL-4 and IL-13 induced strong upregulation of Siglec-F ligand expression by airway epithelium (Fig 7F), levels of Siglec-F ligands in airway epithelium were slightly lower in Siglec-F deficient vs WT mice induced by IL-13 (p = 0.02) but not IL-4 (p = 0.10) (Fig 7F). IL-4 induced the strongest eosinophil response in BAL (WT p  <  0.03; Siglec-F deficient p = 0.02)(Fig 7G) and lung (WT p  <  0.001; Siglec-F deficient p  <  0.001)(Fig 7H), while IL-13 and TNF- Î±  induced a weaker eosinophil response in BAL (Fig 7G) and lung (Fig 7H). TNF- Î± , but not IL-4 or IL-13, induced a strong neutrophil response in BAL in both WT (p = 0.03) and Siglec-F deficient mice (p = 0.03) (Fig 7I)(n = 3 mice/group).'],\n",
       " 129: ['1465-9921-11-154-7-5',\n",
       "  'GFIG',\n",
       "  'Quantitation of Siglec-F ligands in airway epithelial cells and peribronchial inflammatory cells in WT and Siglec-F deficient mice challenged with IL-4, IL-13 or TNF- Î± . WT or Siglec-F deficient mice were administered either IL-4, IL-13, TNF- Î± , or PBS diluent control. Twenty four hours after each individual cytokine or diluent challenge, the mice were sacrificed. BAL was obtained for determination of eosinophil and neutrophil cell counts, and the lungs were processed for immunohistology to detect Siglec-F ligand expression and MBP+ peribronchial eosinophils using a light microscope objective at 20  . Administration of either IL-4 (p  <  0.001, WT; p  <  0.001, Siglec-F deficient) or IL-13 (p  <  0.001, WT; p  <  0.001, Siglec-F deficient) induced a similar significant increase in levels of Siglec-F ligand expression by peribronchial cells (Fig 7A-C, 7E). In contrast, administration of TNF- Î±  induced a small increase in peribronchial Siglec-F ligands (p  <  0.01)(Fig 7E), but did not significantly increase Siglec-F ligand expression by airway epithelium in either WT or Siglec-F deficient mice (p = ns) (Fig 7 D, 7F). Administration of either IL-4 (p  <  0.001 WT; p  <  0.001 Siglec-F deficient) or IL-13 (p  <  0.001 WT; p  <  0.001 Siglec-F deficient) induced significantly increased levels of Siglec-F ligand expression by airway epithelium (Fig 7A-C, 7F). Although both IL-4 and IL-13 induced strong upregulation of Siglec-F ligand expression by airway epithelium (Fig 7F), levels of Siglec-F ligands in airway epithelium were slightly lower in Siglec-F deficient vs WT mice induced by IL-13 (p = 0.02) but not IL-4 (p = 0.10) (Fig 7F). IL-4 induced the strongest eosinophil response in BAL (WT p  <  0.03; Siglec-F deficient p = 0.02)(Fig 7G) and lung (WT p  <  0.001; Siglec-F deficient p  <  0.001)(Fig 7H), while IL-13 and TNF- Î±  induced a weaker eosinophil response in BAL (Fig 7G) and lung (Fig 7H). TNF- Î± , but not IL-4 or IL-13, induced a strong neutrophil response in BAL in both WT (p = 0.03) and Siglec-F deficient mice (p = 0.03) (Fig 7I)(n = 3 mice/group).'],\n",
       " 130: ['1465-9921-11-154-7-6',\n",
       "  'GFIG',\n",
       "  'Quantitation of Siglec-F ligands in airway epithelial cells and peribronchial inflammatory cells in WT and Siglec-F deficient mice challenged with IL-4, IL-13 or TNF- Î± . WT or Siglec-F deficient mice were administered either IL-4, IL-13, TNF- Î± , or PBS diluent control. Twenty four hours after each individual cytokine or diluent challenge, the mice were sacrificed. BAL was obtained for determination of eosinophil and neutrophil cell counts, and the lungs were processed for immunohistology to detect Siglec-F ligand expression and MBP+ peribronchial eosinophils using a light microscope objective at 20  . Administration of either IL-4 (p  <  0.001, WT; p  <  0.001, Siglec-F deficient) or IL-13 (p  <  0.001, WT; p  <  0.001, Siglec-F deficient) induced a similar significant increase in levels of Siglec-F ligand expression by peribronchial cells (Fig 7A-C, 7E). In contrast, administration of TNF- Î±  induced a small increase in peribronchial Siglec-F ligands (p  <  0.01)(Fig 7E), but did not significantly increase Siglec-F ligand expression by airway epithelium in either WT or Siglec-F deficient mice (p = ns) (Fig 7 D, 7F). Administration of either IL-4 (p  <  0.001 WT; p  <  0.001 Siglec-F deficient) or IL-13 (p  <  0.001 WT; p  <  0.001 Siglec-F deficient) induced significantly increased levels of Siglec-F ligand expression by airway epithelium (Fig 7A-C, 7F). Although both IL-4 and IL-13 induced strong upregulation of Siglec-F ligand expression by airway epithelium (Fig 7F), levels of Siglec-F ligands in airway epithelium were slightly lower in Siglec-F deficient vs WT mice induced by IL-13 (p = 0.02) but not IL-4 (p = 0.10) (Fig 7F). IL-4 induced the strongest eosinophil response in BAL (WT p  <  0.03; Siglec-F deficient p = 0.02)(Fig 7G) and lung (WT p  <  0.001; Siglec-F deficient p  <  0.001)(Fig 7H), while IL-13 and TNF- Î±  induced a weaker eosinophil response in BAL (Fig 7G) and lung (Fig 7H). TNF- Î± , but not IL-4 or IL-13, induced a strong neutrophil response in BAL in both WT (p = 0.03) and Siglec-F deficient mice (p = 0.03) (Fig 7I)(n = 3 mice/group).'],\n",
       " 131: ['1465-9921-11-154-7-7',\n",
       "  'GFIG',\n",
       "  'Quantitation of Siglec-F ligands in airway epithelial cells and peribronchial inflammatory cells in WT and Siglec-F deficient mice challenged with IL-4, IL-13 or TNF- Î± . WT or Siglec-F deficient mice were administered either IL-4, IL-13, TNF- Î± , or PBS diluent control. Twenty four hours after each individual cytokine or diluent challenge, the mice were sacrificed. BAL was obtained for determination of eosinophil and neutrophil cell counts, and the lungs were processed for immunohistology to detect Siglec-F ligand expression and MBP+ peribronchial eosinophils using a light microscope objective at 20  . Administration of either IL-4 (p  <  0.001, WT; p  <  0.001, Siglec-F deficient) or IL-13 (p  <  0.001, WT; p  <  0.001, Siglec-F deficient) induced a similar significant increase in levels of Siglec-F ligand expression by peribronchial cells (Fig 7A-C, 7E). In contrast, administration of TNF- Î±  induced a small increase in peribronchial Siglec-F ligands (p  <  0.01)(Fig 7E), but did not significantly increase Siglec-F ligand expression by airway epithelium in either WT or Siglec-F deficient mice (p = ns) (Fig 7 D, 7F). Administration of either IL-4 (p  <  0.001 WT; p  <  0.001 Siglec-F deficient) or IL-13 (p  <  0.001 WT; p  <  0.001 Siglec-F deficient) induced significantly increased levels of Siglec-F ligand expression by airway epithelium (Fig 7A-C, 7F). Although both IL-4 and IL-13 induced strong upregulation of Siglec-F ligand expression by airway epithelium (Fig 7F), levels of Siglec-F ligands in airway epithelium were slightly lower in Siglec-F deficient vs WT mice induced by IL-13 (p = 0.02) but not IL-4 (p = 0.10) (Fig 7F). IL-4 induced the strongest eosinophil response in BAL (WT p  <  0.03; Siglec-F deficient p = 0.02)(Fig 7G) and lung (WT p  <  0.001; Siglec-F deficient p  <  0.001)(Fig 7H), while IL-13 and TNF- Î±  induced a weaker eosinophil response in BAL (Fig 7G) and lung (Fig 7H). TNF- Î± , but not IL-4 or IL-13, induced a strong neutrophil response in BAL in both WT (p = 0.03) and Siglec-F deficient mice (p = 0.03) (Fig 7I)(n = 3 mice/group).'],\n",
       " 132: ['1465-9921-11-154-7-8',\n",
       "  'GFIG',\n",
       "  'Quantitation of Siglec-F ligands in airway epithelial cells and peribronchial inflammatory cells in WT and Siglec-F deficient mice challenged with IL-4, IL-13 or TNF- Î± . WT or Siglec-F deficient mice were administered either IL-4, IL-13, TNF- Î± , or PBS diluent control. Twenty four hours after each individual cytokine or diluent challenge, the mice were sacrificed. BAL was obtained for determination of eosinophil and neutrophil cell counts, and the lungs were processed for immunohistology to detect Siglec-F ligand expression and MBP+ peribronchial eosinophils using a light microscope objective at 20  . Administration of either IL-4 (p  <  0.001, WT; p  <  0.001, Siglec-F deficient) or IL-13 (p  <  0.001, WT; p  <  0.001, Siglec-F deficient) induced a similar significant increase in levels of Siglec-F ligand expression by peribronchial cells (Fig 7A-C, 7E). In contrast, administration of TNF- Î±  induced a small increase in peribronchial Siglec-F ligands (p  <  0.01)(Fig 7E), but did not significantly increase Siglec-F ligand expression by airway epithelium in either WT or Siglec-F deficient mice (p = ns) (Fig 7 D, 7F). Administration of either IL-4 (p  <  0.001 WT; p  <  0.001 Siglec-F deficient) or IL-13 (p  <  0.001 WT; p  <  0.001 Siglec-F deficient) induced significantly increased levels of Siglec-F ligand expression by airway epithelium (Fig 7A-C, 7F). Although both IL-4 and IL-13 induced strong upregulation of Siglec-F ligand expression by airway epithelium (Fig 7F), levels of Siglec-F ligands in airway epithelium were slightly lower in Siglec-F deficient vs WT mice induced by IL-13 (p = 0.02) but not IL-4 (p = 0.10) (Fig 7F). IL-4 induced the strongest eosinophil response in BAL (WT p  <  0.03; Siglec-F deficient p = 0.02)(Fig 7G) and lung (WT p  <  0.001; Siglec-F deficient p  <  0.001)(Fig 7H), while IL-13 and TNF- Î±  induced a weaker eosinophil response in BAL (Fig 7G) and lung (Fig 7H). TNF- Î± , but not IL-4 or IL-13, induced a strong neutrophil response in BAL in both WT (p = 0.03) and Siglec-F deficient mice (p = 0.03) (Fig 7I)(n = 3 mice/group).'],\n",
       " 133: ['1465-9921-11-154-7-9',\n",
       "  'GFIG',\n",
       "  'Quantitation of Siglec-F ligands in airway epithelial cells and peribronchial inflammatory cells in WT and Siglec-F deficient mice challenged with IL-4, IL-13 or TNF- Î± . WT or Siglec-F deficient mice were administered either IL-4, IL-13, TNF- Î± , or PBS diluent control. Twenty four hours after each individual cytokine or diluent challenge, the mice were sacrificed. BAL was obtained for determination of eosinophil and neutrophil cell counts, and the lungs were processed for immunohistology to detect Siglec-F ligand expression and MBP+ peribronchial eosinophils using a light microscope objective at 20  . Administration of either IL-4 (p  <  0.001, WT; p  <  0.001, Siglec-F deficient) or IL-13 (p  <  0.001, WT; p  <  0.001, Siglec-F deficient) induced a similar significant increase in levels of Siglec-F ligand expression by peribronchial cells (Fig 7A-C, 7E). In contrast, administration of TNF- Î±  induced a small increase in peribronchial Siglec-F ligands (p  <  0.01)(Fig 7E), but did not significantly increase Siglec-F ligand expression by airway epithelium in either WT or Siglec-F deficient mice (p = ns) (Fig 7 D, 7F). Administration of either IL-4 (p  <  0.001 WT; p  <  0.001 Siglec-F deficient) or IL-13 (p  <  0.001 WT; p  <  0.001 Siglec-F deficient) induced significantly increased levels of Siglec-F ligand expression by airway epithelium (Fig 7A-C, 7F). Although both IL-4 and IL-13 induced strong upregulation of Siglec-F ligand expression by airway epithelium (Fig 7F), levels of Siglec-F ligands in airway epithelium were slightly lower in Siglec-F deficient vs WT mice induced by IL-13 (p = 0.02) but not IL-4 (p = 0.10) (Fig 7F). IL-4 induced the strongest eosinophil response in BAL (WT p  <  0.03; Siglec-F deficient p = 0.02)(Fig 7G) and lung (WT p  <  0.001; Siglec-F deficient p  <  0.001)(Fig 7H), while IL-13 and TNF- Î±  induced a weaker eosinophil response in BAL (Fig 7G) and lung (Fig 7H). TNF- Î± , but not IL-4 or IL-13, induced a strong neutrophil response in BAL in both WT (p = 0.03) and Siglec-F deficient mice (p = 0.03) (Fig 7I)(n = 3 mice/group).'],\n",
       " 134: ['1465-9921-11-21-6-1',\n",
       "  'DMEL',\n",
       "  'Regulation of TGF- Î² 1-induced changes in cell morphology by rosiglitazone . A549 cells were incubated with vehicle (0.1% DMSO), TGF- Î² 1 (0.25 ng/ml), TGF- Î² 1 (2.5 ng/ml) or TGF- Î² 1 (2.5 ng/ml) + RGZ (10  Î¼ M) for 72 hr and photographed at 100   magnification. The images are representative of 4 separate experiments.,'],\n",
       " 135: ['1465-9921-11-21-6-2',\n",
       "  'DMEL',\n",
       "  'Regulation of TGF- Î² 1-induced changes in cell morphology by rosiglitazone . A549 cells were incubated with vehicle (0.1% DMSO), TGF- Î² 1 (0.25 ng/ml), TGF- Î² 1 (2.5 ng/ml) or TGF- Î² 1 (2.5 ng/ml) + RGZ (10  Î¼ M) for 72 hr and photographed at 100   magnification. The images are representative of 4 separate experiments.,'],\n",
       " 136: ['1465-9921-11-21-6-3',\n",
       "  'DMEL',\n",
       "  'Regulation of TGF- Î² 1-induced changes in cell morphology by rosiglitazone . A549 cells were incubated with vehicle (0.1% DMSO), TGF- Î² 1 (0.25 ng/ml), TGF- Î² 1 (2.5 ng/ml) or TGF- Î² 1 (2.5 ng/ml) + RGZ (10  Î¼ M) for 72 hr and photographed at 100   magnification. The images are representative of 4 separate experiments.,'],\n",
       " 137: ['1465-9921-11-21-6-4',\n",
       "  'DMEL',\n",
       "  'Regulation of TGF- Î² 1-induced changes in cell morphology by rosiglitazone . A549 cells were incubated with vehicle (0.1% DMSO), TGF- Î² 1 (0.25 ng/ml), TGF- Î² 1 (2.5 ng/ml) or TGF- Î² 1 (2.5 ng/ml) + RGZ (10  Î¼ M) for 72 hr and photographed at 100   magnification. The images are representative of 4 separate experiments.,'],\n",
       " 138: ['1465-9921-11-99-5-1',\n",
       "  'DMLI',\n",
       "  'Pigmented macrophage accumulation in the lung tissue before and after smoking cessation . Representative photomicrographs of hematoxylin and eosin stained lung tissue of air-exposed mice (A), smoke-exposed mice (B), smoke-exposed mice 8 weeks after smoking cessation (C). n = 4-5 animals per group. Magnification,400'],\n",
       " 139: ['1465-9921-11-99-5-2',\n",
       "  'DMLI',\n",
       "  'Pigmented macrophage accumulation in the lung tissue before and after smoking cessation . Representative photomicrographs of hematoxylin and eosin stained lung tissue of air-exposed mice (A), smoke-exposed mice (B), smoke-exposed mice 8 weeks after smoking cessation (C). n = 4-5 animals per group. Magnification,400'],\n",
       " 140: ['1465-9921-11-99-5-3',\n",
       "  'DMLI',\n",
       "  'Pigmented macrophage accumulation in the lung tissue before and after smoking cessation . Representative photomicrographs of hematoxylin and eosin stained lung tissue of air-exposed mice (A), smoke-exposed mice (B), smoke-exposed mice 8 weeks after smoking cessation (C). n = 4-5 animals per group. Magnification,400'],\n",
       " 141: ['1465-9921-12-111-1-1',\n",
       "  'DRCT',\n",
       "  'High resolution CT scan in patient no. 1 shows severe pleural and subpleural thickening with moderate fibrotic changes in the marginal parenchyma . Traction bronchiectasis and honeycombing are also seen.,'],\n",
       " 142: ['1465-9921-12-111-1-2',\n",
       "  'DRCT',\n",
       "  'High resolution CT scan in patient no. 1 shows severe pleural and subpleural thickening with moderate fibrotic changes in the marginal parenchyma . Traction bronchiectasis and honeycombing are also seen.,'],\n",
       " 143: ['1465-9921-12-111-1-3',\n",
       "  'DRCT',\n",
       "  'High resolution CT scan in patient no. 1 shows severe pleural and subpleural thickening with moderate fibrotic changes in the marginal parenchyma . Traction bronchiectasis and honeycombing are also seen.,'],\n",
       " 144: ['1465-9921-12-111-1-4',\n",
       "  'DRCT',\n",
       "  'High resolution CT scan in patient no. 1 shows severe pleural and subpleural thickening with moderate fibrotic changes in the marginal parenchyma . Traction bronchiectasis and honeycombing are also seen.,'],\n",
       " 145: ['1465-9921-12-111-2-1',\n",
       "  'DMTR',\n",
       "  'Histological features of patient no. 1 . Marked subpleural thickening characterized by abnormal increase of elastic fibers and abrupt transition to normal parenchyma is evident at low magnification (2A and B). A number of residual alveolar structures are entrapped within the thickened pleural wall together with numerous lymphatic vessels (2c-e, podoplanin immunohistochemistry).'],\n",
       " 146: ['1465-9921-12-111-2-2',\n",
       "  'DMLI',\n",
       "  'Histological features of patient no. 1 . Marked subpleural thickening characterized by abnormal increase of elastic fibers and abrupt transition to normal parenchyma is evident at low magnification (2A and B). A number of residual alveolar structures are entrapped within the thickened pleural wall together with numerous lymphatic vessels (2c-e, podoplanin immunohistochemistry).'],\n",
       " 147: ['1465-9921-12-111-2-3',\n",
       "  'DMLI',\n",
       "  'Histological features of patient no. 1 . Marked subpleural thickening characterized by abnormal increase of elastic fibers and abrupt transition to normal parenchyma is evident at low magnification (2A and B). A number of residual alveolar structures are entrapped within the thickened pleural wall together with numerous lymphatic vessels (2c-e, podoplanin immunohistochemistry).'],\n",
       " 148: ['1465-9921-12-111-2-4',\n",
       "  'DMLI',\n",
       "  'Histological features of patient no. 1 . Marked subpleural thickening characterized by abnormal increase of elastic fibers and abrupt transition to normal parenchyma is evident at low magnification (2A and B). A number of residual alveolar structures are entrapped within the thickened pleural wall together with numerous lymphatic vessels (2c-e, podoplanin immunohistochemistry).'],\n",
       " 149: ['1465-9921-12-111-2-5',\n",
       "  'DMLI',\n",
       "  'Histological features of patient no. 1 . Marked subpleural thickening characterized by abnormal increase of elastic fibers and abrupt transition to normal parenchyma is evident at low magnification (2A and B). A number of residual alveolar structures are entrapped within the thickened pleural wall together with numerous lymphatic vessels (2c-e, podoplanin immunohistochemistry).'],\n",
       " 150: ['1465-9921-12-142-3-1',\n",
       "  'GFIG',\n",
       "  'CD4 + CD25 + Foxp3 +  Tregs increased in the peripheral blood of COPD by SFC therapy . Flow cytometry analysis of Foxp3 + Tregs in COPD patients before and after SFC treatment. (A) Lymphocytes (R1) were identified based on their characteristic properties shown in the forward scatter (FSC) and sideward scatter (SSC). (B) A representative gating was set for CD4 +  T cells from blood lymphocytes (R2). (C) A representative dot plots showing expression of CD25 + Foxp3 +  Tregs in blood CD4 +  T cells of COPD before SFC therapy (Baseline). (D) In COPD after SFC therapy, a representative dot plots showing expression of CD25 + Foxp3 +  Tregs in blood CD4 +  T cells. (E) In healthy controls, a representative dot plots showing expression of CD25 + Foxp3 +  Tregs in blood CD4 +  T cells. (F) Comparison with the percentages of CD25 + Foxp3 + Tregs (Foxp3 + Tregs) in COPD blood before and after SFC treatment by histogram. p  <  0.001, compared with that in COPD patients before SFC therapy. p  <  0.01, compared with that in healthy controls.,'],\n",
       " 151: ['1465-9921-12-142-3-2',\n",
       "  'GFIG',\n",
       "  'CD4 + CD25 + Foxp3 +  Tregs increased in the peripheral blood of COPD by SFC therapy . Flow cytometry analysis of Foxp3 + Tregs in COPD patients before and after SFC treatment. (A) Lymphocytes (R1) were identified based on their characteristic properties shown in the forward scatter (FSC) and sideward scatter (SSC). (B) A representative gating was set for CD4 +  T cells from blood lymphocytes (R2). (C) A representative dot plots showing expression of CD25 + Foxp3 +  Tregs in blood CD4 +  T cells of COPD before SFC therapy (Baseline). (D) In COPD after SFC therapy, a representative dot plots showing expression of CD25 + Foxp3 +  Tregs in blood CD4 +  T cells. (E) In healthy controls, a representative dot plots showing expression of CD25 + Foxp3 +  Tregs in blood CD4 +  T cells. (F) Comparison with the percentages of CD25 + Foxp3 + Tregs (Foxp3 + Tregs) in COPD blood before and after SFC treatment by histogram. p  <  0.001, compared with that in COPD patients before SFC therapy. p  <  0.01, compared with that in healthy controls.,'],\n",
       " 152: ['1465-9921-12-142-3-3',\n",
       "  'GFIG',\n",
       "  'CD4 + CD25 + Foxp3 +  Tregs increased in the peripheral blood of COPD by SFC therapy . Flow cytometry analysis of Foxp3 + Tregs in COPD patients before and after SFC treatment. (A) Lymphocytes (R1) were identified based on their characteristic properties shown in the forward scatter (FSC) and sideward scatter (SSC). (B) A representative gating was set for CD4 +  T cells from blood lymphocytes (R2). (C) A representative dot plots showing expression of CD25 + Foxp3 +  Tregs in blood CD4 +  T cells of COPD before SFC therapy (Baseline). (D) In COPD after SFC therapy, a representative dot plots showing expression of CD25 + Foxp3 +  Tregs in blood CD4 +  T cells. (E) In healthy controls, a representative dot plots showing expression of CD25 + Foxp3 +  Tregs in blood CD4 +  T cells. (F) Comparison with the percentages of CD25 + Foxp3 + Tregs (Foxp3 + Tregs) in COPD blood before and after SFC treatment by histogram. p  <  0.001, compared with that in COPD patients before SFC therapy. p  <  0.01, compared with that in healthy controls.,'],\n",
       " 153: ['1465-9921-12-142-3-4',\n",
       "  'GFIG',\n",
       "  'CD4 + CD25 + Foxp3 +  Tregs increased in the peripheral blood of COPD by SFC therapy . Flow cytometry analysis of Foxp3 + Tregs in COPD patients before and after SFC treatment. (A) Lymphocytes (R1) were identified based on their characteristic properties shown in the forward scatter (FSC) and sideward scatter (SSC). (B) A representative gating was set for CD4 +  T cells from blood lymphocytes (R2). (C) A representative dot plots showing expression of CD25 + Foxp3 +  Tregs in blood CD4 +  T cells of COPD before SFC therapy (Baseline). (D) In COPD after SFC therapy, a representative dot plots showing expression of CD25 + Foxp3 +  Tregs in blood CD4 +  T cells. (E) In healthy controls, a representative dot plots showing expression of CD25 + Foxp3 +  Tregs in blood CD4 +  T cells. (F) Comparison with the percentages of CD25 + Foxp3 + Tregs (Foxp3 + Tregs) in COPD blood before and after SFC treatment by histogram. p  <  0.001, compared with that in COPD patients before SFC therapy. p  <  0.01, compared with that in healthy controls.,'],\n",
       " 154: ['1465-9921-12-142-3-5',\n",
       "  'GFIG',\n",
       "  'CD4 + CD25 + Foxp3 +  Tregs increased in the peripheral blood of COPD by SFC therapy . Flow cytometry analysis of Foxp3 + Tregs in COPD patients before and after SFC treatment. (A) Lymphocytes (R1) were identified based on their characteristic properties shown in the forward scatter (FSC) and sideward scatter (SSC). (B) A representative gating was set for CD4 +  T cells from blood lymphocytes (R2). (C) A representative dot plots showing expression of CD25 + Foxp3 +  Tregs in blood CD4 +  T cells of COPD before SFC therapy (Baseline). (D) In COPD after SFC therapy, a representative dot plots showing expression of CD25 + Foxp3 +  Tregs in blood CD4 +  T cells. (E) In healthy controls, a representative dot plots showing expression of CD25 + Foxp3 +  Tregs in blood CD4 +  T cells. (F) Comparison with the percentages of CD25 + Foxp3 + Tregs (Foxp3 + Tregs) in COPD blood before and after SFC treatment by histogram. p  <  0.001, compared with that in COPD patients before SFC therapy. p  <  0.01, compared with that in healthy controls.,'],\n",
       " 155: ['1465-9921-12-142-3-6',\n",
       "  'GFIG',\n",
       "  'CD4 + CD25 + Foxp3 +  Tregs increased in the peripheral blood of COPD by SFC therapy . Flow cytometry analysis of Foxp3 + Tregs in COPD patients before and after SFC treatment. (A) Lymphocytes (R1) were identified based on their characteristic properties shown in the forward scatter (FSC) and sideward scatter (SSC). (B) A representative gating was set for CD4 +  T cells from blood lymphocytes (R2). (C) A representative dot plots showing expression of CD25 + Foxp3 +  Tregs in blood CD4 +  T cells of COPD before SFC therapy (Baseline). (D) In COPD after SFC therapy, a representative dot plots showing expression of CD25 + Foxp3 +  Tregs in blood CD4 +  T cells. (E) In healthy controls, a representative dot plots showing expression of CD25 + Foxp3 +  Tregs in blood CD4 +  T cells. (F) Comparison with the percentages of CD25 + Foxp3 + Tregs (Foxp3 + Tregs) in COPD blood before and after SFC treatment by histogram. p  <  0.001, compared with that in COPD patients before SFC therapy. p  <  0.01, compared with that in healthy controls.,'],\n",
       " 156: ['1465-9921-12-61-2-1',\n",
       "  'GFIG',\n",
       "  'Expression signals for strongly correlated genes in the lung . Numbers indicate BXD strains, and the parental C57BL/6J and DBA/2J strains as well as F1 individuals are presented. Expression signals of (A)  Klra3  and  Gzma  (p  <  10 -16 ) and (B)  Klra3  versus  Il18rap  (p  <  10 -11 ) were strongly correlated. (C) List of nine genes highly correlated with the first principal component of the expression of  Gzma ,  Klrg1 ,  Klra3  and  Il18rap . (D) Strong correlation between the first principal component and  Prf1  (p  <  10 -16 ). X- and Y-axis of the plots show the names of genes used for the analysis.'],\n",
       " 157: ['1465-9921-12-61-2-2',\n",
       "  'GFIG',\n",
       "  'Expression signals for strongly correlated genes in the lung . Numbers indicate BXD strains, and the parental C57BL/6J and DBA/2J strains as well as F1 individuals are presented. Expression signals of (A)  Klra3  and  Gzma  (p  <  10 -16 ) and (B)  Klra3  versus  Il18rap  (p  <  10 -11 ) were strongly correlated. (C) List of nine genes highly correlated with the first principal component of the expression of  Gzma ,  Klrg1 ,  Klra3  and  Il18rap . (D) Strong correlation between the first principal component and  Prf1  (p  <  10 -16 ). X- and Y-axis of the plots show the names of genes used for the analysis.'],\n",
       " 158: ['1465-9921-12-61-2-3',\n",
       "  'GSCR',\n",
       "  'Expression signals for strongly correlated genes in the lung . Numbers indicate BXD strains, and the parental C57BL/6J and DBA/2J strains as well as F1 individuals are presented. Expression signals of (A)  Klra3  and  Gzma  (p  <  10 -16 ) and (B)  Klra3  versus  Il18rap  (p  <  10 -11 ) were strongly correlated. (C) List of nine genes highly correlated with the first principal component of the expression of  Gzma ,  Klrg1 ,  Klra3  and  Il18rap . (D) Strong correlation between the first principal component and  Prf1  (p  <  10 -16 ). X- and Y-axis of the plots show the names of genes used for the analysis.'],\n",
       " 159: ['1465-9921-12-61-2-4',\n",
       "  'GFIG',\n",
       "  'Expression signals for strongly correlated genes in the lung . Numbers indicate BXD strains, and the parental C57BL/6J and DBA/2J strains as well as F1 individuals are presented. Expression signals of (A)  Klra3  and  Gzma  (p  <  10 -16 ) and (B)  Klra3  versus  Il18rap  (p  <  10 -11 ) were strongly correlated. (C) List of nine genes highly correlated with the first principal component of the expression of  Gzma ,  Klrg1 ,  Klra3  and  Il18rap . (D) Strong correlation between the first principal component and  Prf1  (p  <  10 -16 ). X- and Y-axis of the plots show the names of genes used for the analysis.'],\n",
       " 160: ['1465-9921-12-61-7-1',\n",
       "  'GFIG',\n",
       "  'Examples for variations of expression levels in different BXD and inbred mouse strains and ge-nome-wide analysis of  cis-  and  trans -eQTLs . (A) Log 2  expression levels for  Ang1  and (B) corresponding  cis -eQTL signal. The numbers at the top are chromosomes. The blue line represents the significance level of the QTL expressed as LRS score (likelihood ratio statistic). A positive additive coefficient (green line) indicates that DBA/2J alleles increased trait values. A negative additive coefficient (red line) indicates that C57BL/6J alleles increased trait values. The two horizontal lines mark the genome-wide significance levels at p  <  0.05 (red line) and p  <  0.63 (gray line).  Ang1  is located on Chr 14 (triangle) and the QTL peak is at the same location as the gene. (C) Log 2  expression levels for  Cyp1a1  and (D) corresponding  trans  eQTL peak.  Cyp1a1  is on chromosome 9 (triangle) and the QTL was found on chromosome 12.,'],\n",
       " 161: ['1465-9921-12-61-7-2',\n",
       "  'GFIG',\n",
       "  'Examples for variations of expression levels in different BXD and inbred mouse strains and ge-nome-wide analysis of  cis-  and  trans -eQTLs . (A) Log 2  expression levels for  Ang1  and (B) corresponding  cis -eQTL signal. The numbers at the top are chromosomes. The blue line represents the significance level of the QTL expressed as LRS score (likelihood ratio statistic). A positive additive coefficient (green line) indicates that DBA/2J alleles increased trait values. A negative additive coefficient (red line) indicates that C57BL/6J alleles increased trait values. The two horizontal lines mark the genome-wide significance levels at p  <  0.05 (red line) and p  <  0.63 (gray line).  Ang1  is located on Chr 14 (triangle) and the QTL peak is at the same location as the gene. (C) Log 2  expression levels for  Cyp1a1  and (D) corresponding  trans  eQTL peak.  Cyp1a1  is on chromosome 9 (triangle) and the QTL was found on chromosome 12.,'],\n",
       " 162: ['1465-9921-12-61-7-3',\n",
       "  'GFIG',\n",
       "  'Examples for variations of expression levels in different BXD and inbred mouse strains and ge-nome-wide analysis of  cis-  and  trans -eQTLs . (A) Log 2  expression levels for  Ang1  and (B) corresponding  cis -eQTL signal. The numbers at the top are chromosomes. The blue line represents the significance level of the QTL expressed as LRS score (likelihood ratio statistic). A positive additive coefficient (green line) indicates that DBA/2J alleles increased trait values. A negative additive coefficient (red line) indicates that C57BL/6J alleles increased trait values. The two horizontal lines mark the genome-wide significance levels at p  <  0.05 (red line) and p  <  0.63 (gray line).  Ang1  is located on Chr 14 (triangle) and the QTL peak is at the same location as the gene. (C) Log 2  expression levels for  Cyp1a1  and (D) corresponding  trans  eQTL peak.  Cyp1a1  is on chromosome 9 (triangle) and the QTL was found on chromosome 12.,'],\n",
       " 163: ['1465-9921-12-61-7-4',\n",
       "  'GGEN',\n",
       "  'Examples for variations of expression levels in different BXD and inbred mouse strains and ge-nome-wide analysis of  cis-  and  trans -eQTLs . (A) Log 2  expression levels for  Ang1  and (B) corresponding  cis -eQTL signal. The numbers at the top are chromosomes. The blue line represents the significance level of the QTL expressed as LRS score (likelihood ratio statistic). A positive additive coefficient (green line) indicates that DBA/2J alleles increased trait values. A negative additive coefficient (red line) indicates that C57BL/6J alleles increased trait values. The two horizontal lines mark the genome-wide significance levels at p  <  0.05 (red line) and p  <  0.63 (gray line).  Ang1  is located on Chr 14 (triangle) and the QTL peak is at the same location as the gene. (C) Log 2  expression levels for  Cyp1a1  and (D) corresponding  trans  eQTL peak.  Cyp1a1  is on chromosome 9 (triangle) and the QTL was found on chromosome 12.,'],\n",
       " 164: ['1465-9921-12-77-1-1',\n",
       "  'GFIG',\n",
       "  \"Both CD39 +  regulatory T (Tregs) cells and Th17 cells increased in malignant pleural effusion (MPE) . (A) Lymphocytes were identified based on their characteristic properties shown in the forward scatter (FSC) and sideward scatter (SSC). (B) A representative gating was set for CD4 +  T cells from pleural lymphocytes. (C) A representative dot plots showing expression of CD39 and IL-17 in pleural CD4 +  T cells. Comparisons of percentages of CD39 + Tregs (D), Th17 cells (E), and ratios of CD39 + Tregs/Th17 cells (F) in MPE and blood from patients with lung cancer (n = 16). The percentages of CD39 + Tregs and Th17 cells were determined by flow cytometry. Horizontal bars indicate medians. Comparison was made using a Wilcoxon signed-rank test. (G) The percentages of Th17 cells correlated with CD39 + Tregs cells in MPE. Correlations were determined by Spearman's rank correlation coefficients.,\"],\n",
       " 165: ['1465-9921-12-77-1-2',\n",
       "  'GFIG',\n",
       "  \"Both CD39 +  regulatory T (Tregs) cells and Th17 cells increased in malignant pleural effusion (MPE) . (A) Lymphocytes were identified based on their characteristic properties shown in the forward scatter (FSC) and sideward scatter (SSC). (B) A representative gating was set for CD4 +  T cells from pleural lymphocytes. (C) A representative dot plots showing expression of CD39 and IL-17 in pleural CD4 +  T cells. Comparisons of percentages of CD39 + Tregs (D), Th17 cells (E), and ratios of CD39 + Tregs/Th17 cells (F) in MPE and blood from patients with lung cancer (n = 16). The percentages of CD39 + Tregs and Th17 cells were determined by flow cytometry. Horizontal bars indicate medians. Comparison was made using a Wilcoxon signed-rank test. (G) The percentages of Th17 cells correlated with CD39 + Tregs cells in MPE. Correlations were determined by Spearman's rank correlation coefficients.,\"],\n",
       " 166: ['1465-9921-12-77-1-3',\n",
       "  'GFIG',\n",
       "  \"Both CD39 +  regulatory T (Tregs) cells and Th17 cells increased in malignant pleural effusion (MPE) . (A) Lymphocytes were identified based on their characteristic properties shown in the forward scatter (FSC) and sideward scatter (SSC). (B) A representative gating was set for CD4 +  T cells from pleural lymphocytes. (C) A representative dot plots showing expression of CD39 and IL-17 in pleural CD4 +  T cells. Comparisons of percentages of CD39 + Tregs (D), Th17 cells (E), and ratios of CD39 + Tregs/Th17 cells (F) in MPE and blood from patients with lung cancer (n = 16). The percentages of CD39 + Tregs and Th17 cells were determined by flow cytometry. Horizontal bars indicate medians. Comparison was made using a Wilcoxon signed-rank test. (G) The percentages of Th17 cells correlated with CD39 + Tregs cells in MPE. Correlations were determined by Spearman's rank correlation coefficients.,\"],\n",
       " 167: ['1465-9921-12-77-1-4',\n",
       "  'GFIG',\n",
       "  \"Both CD39 +  regulatory T (Tregs) cells and Th17 cells increased in malignant pleural effusion (MPE) . (A) Lymphocytes were identified based on their characteristic properties shown in the forward scatter (FSC) and sideward scatter (SSC). (B) A representative gating was set for CD4 +  T cells from pleural lymphocytes. (C) A representative dot plots showing expression of CD39 and IL-17 in pleural CD4 +  T cells. Comparisons of percentages of CD39 + Tregs (D), Th17 cells (E), and ratios of CD39 + Tregs/Th17 cells (F) in MPE and blood from patients with lung cancer (n = 16). The percentages of CD39 + Tregs and Th17 cells were determined by flow cytometry. Horizontal bars indicate medians. Comparison was made using a Wilcoxon signed-rank test. (G) The percentages of Th17 cells correlated with CD39 + Tregs cells in MPE. Correlations were determined by Spearman's rank correlation coefficients.,\"],\n",
       " 168: ['1465-9921-12-77-1-5',\n",
       "  'GFIG',\n",
       "  \"Both CD39 +  regulatory T (Tregs) cells and Th17 cells increased in malignant pleural effusion (MPE) . (A) Lymphocytes were identified based on their characteristic properties shown in the forward scatter (FSC) and sideward scatter (SSC). (B) A representative gating was set for CD4 +  T cells from pleural lymphocytes. (C) A representative dot plots showing expression of CD39 and IL-17 in pleural CD4 +  T cells. Comparisons of percentages of CD39 + Tregs (D), Th17 cells (E), and ratios of CD39 + Tregs/Th17 cells (F) in MPE and blood from patients with lung cancer (n = 16). The percentages of CD39 + Tregs and Th17 cells were determined by flow cytometry. Horizontal bars indicate medians. Comparison was made using a Wilcoxon signed-rank test. (G) The percentages of Th17 cells correlated with CD39 + Tregs cells in MPE. Correlations were determined by Spearman's rank correlation coefficients.,\"],\n",
       " 169: ['1465-9921-12-77-1-6',\n",
       "  'GFIG',\n",
       "  \"Both CD39 +  regulatory T (Tregs) cells and Th17 cells increased in malignant pleural effusion (MPE) . (A) Lymphocytes were identified based on their characteristic properties shown in the forward scatter (FSC) and sideward scatter (SSC). (B) A representative gating was set for CD4 +  T cells from pleural lymphocytes. (C) A representative dot plots showing expression of CD39 and IL-17 in pleural CD4 +  T cells. Comparisons of percentages of CD39 + Tregs (D), Th17 cells (E), and ratios of CD39 + Tregs/Th17 cells (F) in MPE and blood from patients with lung cancer (n = 16). The percentages of CD39 + Tregs and Th17 cells were determined by flow cytometry. Horizontal bars indicate medians. Comparison was made using a Wilcoxon signed-rank test. (G) The percentages of Th17 cells correlated with CD39 + Tregs cells in MPE. Correlations were determined by Spearman's rank correlation coefficients.,\"],\n",
       " 170: ['1465-9921-12-77-1-7',\n",
       "  'GFIG',\n",
       "  \"Both CD39 +  regulatory T (Tregs) cells and Th17 cells increased in malignant pleural effusion (MPE) . (A) Lymphocytes were identified based on their characteristic properties shown in the forward scatter (FSC) and sideward scatter (SSC). (B) A representative gating was set for CD4 +  T cells from pleural lymphocytes. (C) A representative dot plots showing expression of CD39 and IL-17 in pleural CD4 +  T cells. Comparisons of percentages of CD39 + Tregs (D), Th17 cells (E), and ratios of CD39 + Tregs/Th17 cells (F) in MPE and blood from patients with lung cancer (n = 16). The percentages of CD39 + Tregs and Th17 cells were determined by flow cytometry. Horizontal bars indicate medians. Comparison was made using a Wilcoxon signed-rank test. (G) The percentages of Th17 cells correlated with CD39 + Tregs cells in MPE. Correlations were determined by Spearman's rank correlation coefficients.,\"],\n",
       " 171: ['1465-9921-5-25-3-1',\n",
       "  'GFIG',\n",
       "  'Effect of supplemental oxygen on exercise capacity.  The forest plot shows the results from five trials comparing respiratory rehabilitation with and without oxygen. Walking tests, incremental and constant work rate exercise tests were used to assess the additional effect of supplemental oxygen during exercise. The x-axis represents the difference in change scores between study groups with negative values favoring exercise on room air and positive values favoring exercise with supplemental oxygen. A difference of 0 means that both study groups changed to the same amount. Boxes with 95% confidence intervals represent point estimates for the difference between the walking distance and maximum exercise capacity change scores (from baseline to follow-up) of the study groups. Dotted lines represent the minimal important difference of the six-minute walking distance (53 meters). On the right of the forest plot, point estimates for differences between groups and 95% confidence intervals are shown.,'],\n",
       " 172: ['1465-9921-5-25-3-2',\n",
       "  'GFIG',\n",
       "  'Effect of supplemental oxygen on exercise capacity.  The forest plot shows the results from five trials comparing respiratory rehabilitation with and without oxygen. Walking tests, incremental and constant work rate exercise tests were used to assess the additional effect of supplemental oxygen during exercise. The x-axis represents the difference in change scores between study groups with negative values favoring exercise on room air and positive values favoring exercise with supplemental oxygen. A difference of 0 means that both study groups changed to the same amount. Boxes with 95% confidence intervals represent point estimates for the difference between the walking distance and maximum exercise capacity change scores (from baseline to follow-up) of the study groups. Dotted lines represent the minimal important difference of the six-minute walking distance (53 meters). On the right of the forest plot, point estimates for differences between groups and 95% confidence intervals are shown.,'],\n",
       " 173: ['1465-9921-5-25-3-3',\n",
       "  'GFIG',\n",
       "  'Effect of supplemental oxygen on exercise capacity.  The forest plot shows the results from five trials comparing respiratory rehabilitation with and without oxygen. Walking tests, incremental and constant work rate exercise tests were used to assess the additional effect of supplemental oxygen during exercise. The x-axis represents the difference in change scores between study groups with negative values favoring exercise on room air and positive values favoring exercise with supplemental oxygen. A difference of 0 means that both study groups changed to the same amount. Boxes with 95% confidence intervals represent point estimates for the difference between the walking distance and maximum exercise capacity change scores (from baseline to follow-up) of the study groups. Dotted lines represent the minimal important difference of the six-minute walking distance (53 meters). On the right of the forest plot, point estimates for differences between groups and 95% confidence intervals are shown.,'],\n",
       " 174: ['1465-9921-6-60-3-1',\n",
       "  'DMTR',\n",
       "  'Intact tubular myelin immunolabelled for SP-A . Immunolabelling for SP-A on ultrastructurally intact tubular myelin (TM) lattices A) in the control, B) after ischemia and reperfusion following preservation with either Celsior or C) EuroCollins. Alveolar lumen (AL), epithelium (EPI).,'],\n",
       " 175: ['1465-9921-6-60-3-2',\n",
       "  'DMTR',\n",
       "  'Intact tubular myelin immunolabelled for SP-A . Immunolabelling for SP-A on ultrastructurally intact tubular myelin (TM) lattices A) in the control, B) after ischemia and reperfusion following preservation with either Celsior or C) EuroCollins. Alveolar lumen (AL), epithelium (EPI).,'],\n",
       " 176: ['1465-9921-6-60-3-3',\n",
       "  'DMTR',\n",
       "  'Intact tubular myelin immunolabelled for SP-A . Immunolabelling for SP-A on ultrastructurally intact tubular myelin (TM) lattices A) in the control, B) after ischemia and reperfusion following preservation with either Celsior or C) EuroCollins. Alveolar lumen (AL), epithelium (EPI).,'],\n",
       " 177: ['1465-9921-6-83-11-1',\n",
       "  'DMLI',\n",
       "  'Histological appearance of  DBA/2  lungs 4 and 6 months after chronic cigarette smoke exposure.  Lung parenchyma from smoke-exposed  DBA/2  mice, at 4 (A) and 6 months (B). The first foci of subpleural fibrosis are seen from 4 months of smoke exposure (A). After 6 months of cigarette smoke exposure, disseminated foci of severe emphysema and evident areas of subpleural fibrosis are present (B). (A) and (B): Hematoxylin-eosin stain, Scale bars represent 25  Î¼ m and 400  Î¼ m. respectively.'],\n",
       " 178: ['1465-9921-6-83-11-2',\n",
       "  'DMLI',\n",
       "  'Histological appearance of  DBA/2  lungs 4 and 6 months after chronic cigarette smoke exposure.  Lung parenchyma from smoke-exposed  DBA/2  mice, at 4 (A) and 6 months (B). The first foci of subpleural fibrosis are seen from 4 months of smoke exposure (A). After 6 months of cigarette smoke exposure, disseminated foci of severe emphysema and evident areas of subpleural fibrosis are present (B). (A) and (B): Hematoxylin-eosin stain, Scale bars represent 25  Î¼ m and 400  Î¼ m. respectively.'],\n",
       " 179: ['1465-9921-7-112-1-1',\n",
       "  'GCHE',\n",
       "  'Chemical structure of chitosan-thioglycolic acid conjugate  (A)  and transmission electron micrograph of chitosan nanoparticles following ionic crosslinking  (B) . Adsorption kinetics of mucin I-S with unmodified and thiolated chitosan nanoparticles  (C) . The value of mucin adsorbed represents the amount of mucin per 2 mg of chitosan nanoparticles. The experiments were repeated twice and results are expressed as mean  Â±  S.D (** P  <  0.01, * P  <  0.05 relative to 1 h).'],\n",
       " 180: ['1465-9921-7-112-1-2',\n",
       "  'DMTR',\n",
       "  'Chemical structure of chitosan-thioglycolic acid conjugate  (A)  and transmission electron micrograph of chitosan nanoparticles following ionic crosslinking  (B) . Adsorption kinetics of mucin I-S with unmodified and thiolated chitosan nanoparticles  (C) . The value of mucin adsorbed represents the amount of mucin per 2 mg of chitosan nanoparticles. The experiments were repeated twice and results are expressed as mean  Â±  S.D (** P  <  0.01, * P  <  0.05 relative to 1 h).'],\n",
       " 181: ['1465-9921-7-112-1-3',\n",
       "  'GFIG',\n",
       "  'Chemical structure of chitosan-thioglycolic acid conjugate  (A)  and transmission electron micrograph of chitosan nanoparticles following ionic crosslinking  (B) . Adsorption kinetics of mucin I-S with unmodified and thiolated chitosan nanoparticles  (C) . The value of mucin adsorbed represents the amount of mucin per 2 mg of chitosan nanoparticles. The experiments were repeated twice and results are expressed as mean  Â±  S.D (** P  <  0.01, * P  <  0.05 relative to 1 h).'],\n",
       " 182: ['1465-9921-7-116-4-1',\n",
       "  'GFIG',\n",
       "  'Levels of IL-6 and IL-8 in nasal lavage. (A) Time course of IL-6. (B) Time course of IL-8. (C) Mean levels of IL-6 in nasal lavage when stable and at exacerbation. There was an increase in IL-6 at exacerbation but this was not significant (p = 0.054). (D) Mean levels of IL-8 when stable and at exacerbation. There was a significant increase in IL-8 at exacerbation (p = 0.046).,'],\n",
       " 183: ['1465-9921-7-116-4-2',\n",
       "  'GFIG',\n",
       "  'Levels of IL-6 and IL-8 in nasal lavage. (A) Time course of IL-6. (B) Time course of IL-8. (C) Mean levels of IL-6 in nasal lavage when stable and at exacerbation. There was an increase in IL-6 at exacerbation but this was not significant (p = 0.054). (D) Mean levels of IL-8 when stable and at exacerbation. There was a significant increase in IL-8 at exacerbation (p = 0.046).,'],\n",
       " 184: ['1465-9921-7-116-4-3',\n",
       "  'GFIG',\n",
       "  'Levels of IL-6 and IL-8 in nasal lavage. (A) Time course of IL-6. (B) Time course of IL-8. (C) Mean levels of IL-6 in nasal lavage when stable and at exacerbation. There was an increase in IL-6 at exacerbation but this was not significant (p = 0.054). (D) Mean levels of IL-8 when stable and at exacerbation. There was a significant increase in IL-8 at exacerbation (p = 0.046).,'],\n",
       " 185: ['1465-9921-7-116-4-4',\n",
       "  'GFIG',\n",
       "  'Levels of IL-6 and IL-8 in nasal lavage. (A) Time course of IL-6. (B) Time course of IL-8. (C) Mean levels of IL-6 in nasal lavage when stable and at exacerbation. There was an increase in IL-6 at exacerbation but this was not significant (p = 0.054). (D) Mean levels of IL-8 when stable and at exacerbation. There was a significant increase in IL-8 at exacerbation (p = 0.046).,'],\n",
       " 186: ['1465-9921-7-24-6-1',\n",
       "  'DMTR',\n",
       "  '(A) Transmission electron micrograph of a bronchiole specimen from mice exposed to filtered air showing normal epithelium, smooth muscle, and capillaries. (B ï¿½ï¿½ D) Transmission electron micrographs of bronchiole specimens from mice exposed to 2 ppm ozone for 4,8, and 12 weeks showing (B) hypertrophied smooth muscle cells (sm), a few infiltrating lymphocytes (lym), and myofibroblasts (arrow); (C) myofibroblasts (arrow), interstitial deposition of collagen fibers, and increased smooth muscle cell hypertrophy; and (D) disorganized smooth muscle cells, increased deposition of collagen fiber, unmyelinated nerve fiber (arrow), and myofibroblasts. Original magnification,   5000. Scale bar = 5  Î¼ m.,'],\n",
       " 187: ['1465-9921-7-24-6-2',\n",
       "  'DMTR',\n",
       "  '(A) Transmission electron micrograph of a bronchiole specimen from mice exposed to filtered air showing normal epithelium, smooth muscle, and capillaries. (B ï¿½ï¿½ D) Transmission electron micrographs of bronchiole specimens from mice exposed to 2 ppm ozone for 4,8, and 12 weeks showing (B) hypertrophied smooth muscle cells (sm), a few infiltrating lymphocytes (lym), and myofibroblasts (arrow); (C) myofibroblasts (arrow), interstitial deposition of collagen fibers, and increased smooth muscle cell hypertrophy; and (D) disorganized smooth muscle cells, increased deposition of collagen fiber, unmyelinated nerve fiber (arrow), and myofibroblasts. Original magnification,   5000. Scale bar = 5  Î¼ m.,'],\n",
       " 188: ['1465-9921-7-24-6-3',\n",
       "  'DMTR',\n",
       "  '(A) Transmission electron micrograph of a bronchiole specimen from mice exposed to filtered air showing normal epithelium, smooth muscle, and capillaries. (B ï¿½ï¿½ D) Transmission electron micrographs of bronchiole specimens from mice exposed to 2 ppm ozone for 4,8, and 12 weeks showing (B) hypertrophied smooth muscle cells (sm), a few infiltrating lymphocytes (lym), and myofibroblasts (arrow); (C) myofibroblasts (arrow), interstitial deposition of collagen fibers, and increased smooth muscle cell hypertrophy; and (D) disorganized smooth muscle cells, increased deposition of collagen fiber, unmyelinated nerve fiber (arrow), and myofibroblasts. Original magnification,   5000. Scale bar = 5  Î¼ m.,'],\n",
       " 189: ['1465-9921-7-24-6-4',\n",
       "  'DMTR',\n",
       "  '(A) Transmission electron micrograph of a bronchiole specimen from mice exposed to filtered air showing normal epithelium, smooth muscle, and capillaries. (B ï¿½ï¿½ D) Transmission electron micrographs of bronchiole specimens from mice exposed to 2 ppm ozone for 4,8, and 12 weeks showing (B) hypertrophied smooth muscle cells (sm), a few infiltrating lymphocytes (lym), and myofibroblasts (arrow); (C) myofibroblasts (arrow), interstitial deposition of collagen fibers, and increased smooth muscle cell hypertrophy; and (D) disorganized smooth muscle cells, increased deposition of collagen fiber, unmyelinated nerve fiber (arrow), and myofibroblasts. Original magnification,   5000. Scale bar = 5  Î¼ m.,'],\n",
       " 190: ['1465-9921-7-50-3-1',\n",
       "  'DMFL',\n",
       "  'Fibrocytes in patients with mild asthma express collagen. Bronchial biopsies were stained using immunofluorescent antibodies against prolyl-4-hydroxylase (green) CD45RO (red) and nuclei (blue) (A). Squares indicate the same cluster area as shown in A. The merged picture describes the positive cells within the square. Cells were further stained for CD34 (green), procollagen I (red), and nuclei (blue) (B), and were subjected to confocal microscopy. The merged picture describes the positive cell within the square.  Definition of abbreviations : BM = Basement membrane'],\n",
       " 191: ['1465-9921-7-50-3-2',\n",
       "  'DMFL',\n",
       "  'Fibrocytes in patients with mild asthma express collagen. Bronchial biopsies were stained using immunofluorescent antibodies against prolyl-4-hydroxylase (green) CD45RO (red) and nuclei (blue) (A). Squares indicate the same cluster area as shown in A. The merged picture describes the positive cells within the square. Cells were further stained for CD34 (green), procollagen I (red), and nuclei (blue) (B), and were subjected to confocal microscopy. The merged picture describes the positive cell within the square.  Definition of abbreviations : BM = Basement membrane'],\n",
       " 192: ['1465-9921-7-50-3-3',\n",
       "  'DMFL',\n",
       "  'Fibrocytes in patients with mild asthma express collagen. Bronchial biopsies were stained using immunofluorescent antibodies against prolyl-4-hydroxylase (green) CD45RO (red) and nuclei (blue) (A). Squares indicate the same cluster area as shown in A. The merged picture describes the positive cells within the square. Cells were further stained for CD34 (green), procollagen I (red), and nuclei (blue) (B), and were subjected to confocal microscopy. The merged picture describes the positive cell within the square.  Definition of abbreviations : BM = Basement membrane'],\n",
       " 193: ['1465-9921-7-50-3-4',\n",
       "  'DMFL',\n",
       "  'Fibrocytes in patients with mild asthma express collagen. Bronchial biopsies were stained using immunofluorescent antibodies against prolyl-4-hydroxylase (green) CD45RO (red) and nuclei (blue) (A). Squares indicate the same cluster area as shown in A. The merged picture describes the positive cells within the square. Cells were further stained for CD34 (green), procollagen I (red), and nuclei (blue) (B), and were subjected to confocal microscopy. The merged picture describes the positive cell within the square.  Definition of abbreviations : BM = Basement membrane'],\n",
       " 194: ['1465-9921-7-50-3-5',\n",
       "  'DMFL',\n",
       "  'Fibrocytes in patients with mild asthma express collagen. Bronchial biopsies were stained using immunofluorescent antibodies against prolyl-4-hydroxylase (green) CD45RO (red) and nuclei (blue) (A). Squares indicate the same cluster area as shown in A. The merged picture describes the positive cells within the square. Cells were further stained for CD34 (green), procollagen I (red), and nuclei (blue) (B), and were subjected to confocal microscopy. The merged picture describes the positive cell within the square.  Definition of abbreviations : BM = Basement membrane'],\n",
       " 195: ['1465-9921-7-50-3-6',\n",
       "  'DMFL',\n",
       "  'Fibrocytes in patients with mild asthma express collagen. Bronchial biopsies were stained using immunofluorescent antibodies against prolyl-4-hydroxylase (green) CD45RO (red) and nuclei (blue) (A). Squares indicate the same cluster area as shown in A. The merged picture describes the positive cells within the square. Cells were further stained for CD34 (green), procollagen I (red), and nuclei (blue) (B), and were subjected to confocal microscopy. The merged picture describes the positive cell within the square.  Definition of abbreviations : BM = Basement membrane'],\n",
       " 196: ['1465-9921-7-50-3-7',\n",
       "  'DMFL',\n",
       "  'Fibrocytes in patients with mild asthma express collagen. Bronchial biopsies were stained using immunofluorescent antibodies against prolyl-4-hydroxylase (green) CD45RO (red) and nuclei (blue) (A). Squares indicate the same cluster area as shown in A. The merged picture describes the positive cells within the square. Cells were further stained for CD34 (green), procollagen I (red), and nuclei (blue) (B), and were subjected to confocal microscopy. The merged picture describes the positive cell within the square.  Definition of abbreviations : BM = Basement membrane'],\n",
       " 197: ['1465-9921-7-50-3-8',\n",
       "  'DMFL',\n",
       "  'Fibrocytes in patients with mild asthma express collagen. Bronchial biopsies were stained using immunofluorescent antibodies against prolyl-4-hydroxylase (green) CD45RO (red) and nuclei (blue) (A). Squares indicate the same cluster area as shown in A. The merged picture describes the positive cells within the square. Cells were further stained for CD34 (green), procollagen I (red), and nuclei (blue) (B), and were subjected to confocal microscopy. The merged picture describes the positive cell within the square.  Definition of abbreviations : BM = Basement membrane'],\n",
       " 198: ['1465-9921-7-50-6-1',\n",
       "  'DMLI',\n",
       "  'BALF fibroblasts in patients with mild asthma express the fibrocyte markers CD34, CD45RO, and  Î± -SMA. BALF fibroblasts were cultured as described in the method section and observed after approximately 30 days in culture (A). The BALF fibroblasts were fixed in 4% paraformaldehyde and stained with antibodies against CD34 (green), CD45RO (red) and  Î± -SMA (blue) (B). These pictures were merged together with a transmission picture.'],\n",
       " 199: ['1465-9921-7-50-6-2',\n",
       "  'DMLI',\n",
       "  'BALF fibroblasts in patients with mild asthma express the fibrocyte markers CD34, CD45RO, and  Î± -SMA. BALF fibroblasts were cultured as described in the method section and observed after approximately 30 days in culture (A). The BALF fibroblasts were fixed in 4% paraformaldehyde and stained with antibodies against CD34 (green), CD45RO (red) and  Î± -SMA (blue) (B). These pictures were merged together with a transmission picture.'],\n",
       " 200: ['1465-9921-7-50-6-3',\n",
       "  'DMLI',\n",
       "  'BALF fibroblasts in patients with mild asthma express the fibrocyte markers CD34, CD45RO, and  Î± -SMA. BALF fibroblasts were cultured as described in the method section and observed after approximately 30 days in culture (A). The BALF fibroblasts were fixed in 4% paraformaldehyde and stained with antibodies against CD34 (green), CD45RO (red) and  Î± -SMA (blue) (B). These pictures were merged together with a transmission picture.'],\n",
       " 201: ['1465-9921-7-50-6-4',\n",
       "  'DMFL',\n",
       "  'BALF fibroblasts in patients with mild asthma express the fibrocyte markers CD34, CD45RO, and  Î± -SMA. BALF fibroblasts were cultured as described in the method section and observed after approximately 30 days in culture (A). The BALF fibroblasts were fixed in 4% paraformaldehyde and stained with antibodies against CD34 (green), CD45RO (red) and  Î± -SMA (blue) (B). These pictures were merged together with a transmission picture.'],\n",
       " 202: ['1465-9921-7-50-6-5',\n",
       "  'DMFL',\n",
       "  'BALF fibroblasts in patients with mild asthma express the fibrocyte markers CD34, CD45RO, and  Î± -SMA. BALF fibroblasts were cultured as described in the method section and observed after approximately 30 days in culture (A). The BALF fibroblasts were fixed in 4% paraformaldehyde and stained with antibodies against CD34 (green), CD45RO (red) and  Î± -SMA (blue) (B). These pictures were merged together with a transmission picture.'],\n",
       " 203: ['1465-9921-7-50-6-6',\n",
       "  'DMFL',\n",
       "  'BALF fibroblasts in patients with mild asthma express the fibrocyte markers CD34, CD45RO, and  Î± -SMA. BALF fibroblasts were cultured as described in the method section and observed after approximately 30 days in culture (A). The BALF fibroblasts were fixed in 4% paraformaldehyde and stained with antibodies against CD34 (green), CD45RO (red) and  Î± -SMA (blue) (B). These pictures were merged together with a transmission picture.'],\n",
       " 204: ['1465-9921-7-50-6-7',\n",
       "  'DMFL',\n",
       "  'BALF fibroblasts in patients with mild asthma express the fibrocyte markers CD34, CD45RO, and  Î± -SMA. BALF fibroblasts were cultured as described in the method section and observed after approximately 30 days in culture (A). The BALF fibroblasts were fixed in 4% paraformaldehyde and stained with antibodies against CD34 (green), CD45RO (red) and  Î± -SMA (blue) (B). These pictures were merged together with a transmission picture.'],\n",
       " 205: ['1465-9921-7-50-6-8',\n",
       "  'DMEL',\n",
       "  'BALF fibroblasts in patients with mild asthma express the fibrocyte markers CD34, CD45RO, and  Î± -SMA. BALF fibroblasts were cultured as described in the method section and observed after approximately 30 days in culture (A). The BALF fibroblasts were fixed in 4% paraformaldehyde and stained with antibodies against CD34 (green), CD45RO (red) and  Î± -SMA (blue) (B). These pictures were merged together with a transmission picture.'],\n",
       " 206: ['1465-9921-7-68-1-1',\n",
       "  'GFIG',\n",
       "  'Phosphorylation of MAP kinase family proteins at baseline. Phosphorylation of the MAP kinases was detected by immunoblot on cell lysates using antibodies specific to the phosphorylated forms of the kinases (right panels). Equal loading of cell lysates was determined by immunoblotting with antibodies raised against unphosphorylated forms or actin. Left panels show the quantification of (A) ERK1/2, (B) p38 and (C) JNK. The values for phosphorylated forms of each MAP kinase were normalized according to the values for the respective, total MAP kinase or actin (JNK). The data represent mean  Â±  SE. AF, fibroblasts from asthmatic patients; NF, fibroblast from normal controls. *: p  <  0.05,'],\n",
       " 207: ['1465-9921-7-68-1-2',\n",
       "  'GGEL',\n",
       "  'Phosphorylation of MAP kinase family proteins at baseline. Phosphorylation of the MAP kinases was detected by immunoblot on cell lysates using antibodies specific to the phosphorylated forms of the kinases (right panels). Equal loading of cell lysates was determined by immunoblotting with antibodies raised against unphosphorylated forms or actin. Left panels show the quantification of (A) ERK1/2, (B) p38 and (C) JNK. The values for phosphorylated forms of each MAP kinase were normalized according to the values for the respective, total MAP kinase or actin (JNK). The data represent mean  Â±  SE. AF, fibroblasts from asthmatic patients; NF, fibroblast from normal controls. *: p  <  0.05,'],\n",
       " 208: ['1465-9921-7-68-1-3',\n",
       "  'GFIG',\n",
       "  'Phosphorylation of MAP kinase family proteins at baseline. Phosphorylation of the MAP kinases was detected by immunoblot on cell lysates using antibodies specific to the phosphorylated forms of the kinases (right panels). Equal loading of cell lysates was determined by immunoblotting with antibodies raised against unphosphorylated forms or actin. Left panels show the quantification of (A) ERK1/2, (B) p38 and (C) JNK. The values for phosphorylated forms of each MAP kinase were normalized according to the values for the respective, total MAP kinase or actin (JNK). The data represent mean  Â±  SE. AF, fibroblasts from asthmatic patients; NF, fibroblast from normal controls. *: p  <  0.05,'],\n",
       " 209: ['1465-9921-7-68-1-4',\n",
       "  'GGEL',\n",
       "  'Phosphorylation of MAP kinase family proteins at baseline. Phosphorylation of the MAP kinases was detected by immunoblot on cell lysates using antibodies specific to the phosphorylated forms of the kinases (right panels). Equal loading of cell lysates was determined by immunoblotting with antibodies raised against unphosphorylated forms or actin. Left panels show the quantification of (A) ERK1/2, (B) p38 and (C) JNK. The values for phosphorylated forms of each MAP kinase were normalized according to the values for the respective, total MAP kinase or actin (JNK). The data represent mean  Â±  SE. AF, fibroblasts from asthmatic patients; NF, fibroblast from normal controls. *: p  <  0.05,'],\n",
       " 210: ['1465-9921-7-68-1-5',\n",
       "  'GFIG',\n",
       "  'Phosphorylation of MAP kinase family proteins at baseline. Phosphorylation of the MAP kinases was detected by immunoblot on cell lysates using antibodies specific to the phosphorylated forms of the kinases (right panels). Equal loading of cell lysates was determined by immunoblotting with antibodies raised against unphosphorylated forms or actin. Left panels show the quantification of (A) ERK1/2, (B) p38 and (C) JNK. The values for phosphorylated forms of each MAP kinase were normalized according to the values for the respective, total MAP kinase or actin (JNK). The data represent mean  Â±  SE. AF, fibroblasts from asthmatic patients; NF, fibroblast from normal controls. *: p  <  0.05,'],\n",
       " 211: ['1465-9921-7-68-1-6',\n",
       "  'GGEL',\n",
       "  'Phosphorylation of MAP kinase family proteins at baseline. Phosphorylation of the MAP kinases was detected by immunoblot on cell lysates using antibodies specific to the phosphorylated forms of the kinases (right panels). Equal loading of cell lysates was determined by immunoblotting with antibodies raised against unphosphorylated forms or actin. Left panels show the quantification of (A) ERK1/2, (B) p38 and (C) JNK. The values for phosphorylated forms of each MAP kinase were normalized according to the values for the respective, total MAP kinase or actin (JNK). The data represent mean  Â±  SE. AF, fibroblasts from asthmatic patients; NF, fibroblast from normal controls. *: p  <  0.05,'],\n",
       " 212: ['1465-9921-7-9-4-1',\n",
       "  'GFIG',\n",
       "  'Anti-mitogenic effects of PDE4 inhibition in PASMCs . Effects of PDE4 (cilomilast, rolipram, roflumilast) and PDE3 (cilostamide) inhibitors on PDGF-BB (5 ng/ml) stimulated DNA synthesis (A). Concentration-dependent effect of roflumilast, combined with a sub-maximal concentration of cicaprost, on [methyl- 3 H]-thymidine incorporation (B). Effect of roflumilast (10 -6  M, open squares) on serum-stimulated (5% FBS) cell growth (closed squares) (C) and combined inhibitory effect of iloprost (10 -7  M) after 10 days serum-stimulated growth (D). *, P  <  0.05; **, P  <  0.01; ***, P  <  0.001 versus control cells stimulated with either PDGF-BB (A), cicaprost (B) or serum (C-D). Data represent mean  Â±  SEM from four-six distinct isolates (A-B, D) and four replicates (C).,'],\n",
       " 213: ['1465-9921-7-9-4-2',\n",
       "  'GFIG',\n",
       "  'Anti-mitogenic effects of PDE4 inhibition in PASMCs . Effects of PDE4 (cilomilast, rolipram, roflumilast) and PDE3 (cilostamide) inhibitors on PDGF-BB (5 ng/ml) stimulated DNA synthesis (A). Concentration-dependent effect of roflumilast, combined with a sub-maximal concentration of cicaprost, on [methyl- 3 H]-thymidine incorporation (B). Effect of roflumilast (10 -6  M, open squares) on serum-stimulated (5% FBS) cell growth (closed squares) (C) and combined inhibitory effect of iloprost (10 -7  M) after 10 days serum-stimulated growth (D). *, P  <  0.05; **, P  <  0.01; ***, P  <  0.001 versus control cells stimulated with either PDGF-BB (A), cicaprost (B) or serum (C-D). Data represent mean  Â±  SEM from four-six distinct isolates (A-B, D) and four replicates (C).,'],\n",
       " 214: ['1465-9921-7-9-4-3',\n",
       "  'GFIG',\n",
       "  'Anti-mitogenic effects of PDE4 inhibition in PASMCs . Effects of PDE4 (cilomilast, rolipram, roflumilast) and PDE3 (cilostamide) inhibitors on PDGF-BB (5 ng/ml) stimulated DNA synthesis (A). Concentration-dependent effect of roflumilast, combined with a sub-maximal concentration of cicaprost, on [methyl- 3 H]-thymidine incorporation (B). Effect of roflumilast (10 -6  M, open squares) on serum-stimulated (5% FBS) cell growth (closed squares) (C) and combined inhibitory effect of iloprost (10 -7  M) after 10 days serum-stimulated growth (D). *, P  <  0.05; **, P  <  0.01; ***, P  <  0.001 versus control cells stimulated with either PDGF-BB (A), cicaprost (B) or serum (C-D). Data represent mean  Â±  SEM from four-six distinct isolates (A-B, D) and four replicates (C).,'],\n",
       " 215: ['1465-9921-7-9-4-4',\n",
       "  'GFIG',\n",
       "  'Anti-mitogenic effects of PDE4 inhibition in PASMCs . Effects of PDE4 (cilomilast, rolipram, roflumilast) and PDE3 (cilostamide) inhibitors on PDGF-BB (5 ng/ml) stimulated DNA synthesis (A). Concentration-dependent effect of roflumilast, combined with a sub-maximal concentration of cicaprost, on [methyl- 3 H]-thymidine incorporation (B). Effect of roflumilast (10 -6  M, open squares) on serum-stimulated (5% FBS) cell growth (closed squares) (C) and combined inhibitory effect of iloprost (10 -7  M) after 10 days serum-stimulated growth (D). *, P  <  0.05; **, P  <  0.01; ***, P  <  0.001 versus control cells stimulated with either PDGF-BB (A), cicaprost (B) or serum (C-D). Data represent mean  Â±  SEM from four-six distinct isolates (A-B, D) and four replicates (C).,'],\n",
       " 216: ['1465-9921-8-3-2-10',\n",
       "  'DMLI',\n",
       "  'Apoptosis in lung tissues of mice infected with  S. pneumoniae  D39 serotype 2 . (A) Distribution patterns of PLY and apoptosis in representative lung sections from mice intranasally infected with  S. pneumoniae  D39 serotype 2. PLY was established by staining with anti-PLY rabbit antibodies. Apoptosis was assessed by active caspase-9 staining and  in situ  TUNEL assay. No staining was observed in lung tissues from uninfected mice. At 12 h post-infection, resident alveolar macrophages were positively stained with anti-PLY, anti-caspase-9, and TUNEL (arrow heads). Infiltrating leukocytes (24 h post-infection) and bronchial epithelium (48 h post-infection) were stained with anti-PLY, anti-caspase-9, and TUNEL, respectively (arrow heads). Note non-stained vascular endothelium. Blood vessel (v), alveolar space (a), bronchiole (b). Scale bars 50  Î¼ m. (B) Apoptosis and PLY in lung tissues from untreated mice during pneumococcal pneumonia. Apoptosis was identified by immunohistochemical detection of active caspase-9 and by  in situ  TUNEL assay. PLY was stained with anti-PLY rabbit antibodies. Adjacent sections were co-stained for co-localization of PLY, TUNEL, and caspase-9. Five sections were analyzed in each time point. Statistical differences were not found for a comparison of number of PLY, caspase-9, and TUNEL positive cells as determined by two-way ANOVA followed by the Bonferroni test. (C) Comparison of caspase-9 positive cells in lung tissues from anti-PLY IgG-, control IgG-, and PBS-treated mice. Percentage of caspase-9 stained cells was calculated with respect to total cells counted in random areas of lung tissue sections. Results are means  Â±  SD of 3 mice and are representative of three independent experiments. *,  P  <  0.05 for a comparison of anti-PLY IgG-treated mice with PBS-treated mice, and +, P  <  0.05 for a comparison of anti-PLY IgG with control IgG-treated mice, as determined by two-way ANOVA followed by the Bonferroni test.,'],\n",
       " 217: ['1465-9921-8-3-2-11',\n",
       "  'DMLI',\n",
       "  'Apoptosis in lung tissues of mice infected with  S. pneumoniae  D39 serotype 2 . (A) Distribution patterns of PLY and apoptosis in representative lung sections from mice intranasally infected with  S. pneumoniae  D39 serotype 2. PLY was established by staining with anti-PLY rabbit antibodies. Apoptosis was assessed by active caspase-9 staining and  in situ  TUNEL assay. No staining was observed in lung tissues from uninfected mice. At 12 h post-infection, resident alveolar macrophages were positively stained with anti-PLY, anti-caspase-9, and TUNEL (arrow heads). Infiltrating leukocytes (24 h post-infection) and bronchial epithelium (48 h post-infection) were stained with anti-PLY, anti-caspase-9, and TUNEL, respectively (arrow heads). Note non-stained vascular endothelium. Blood vessel (v), alveolar space (a), bronchiole (b). Scale bars 50  Î¼ m. (B) Apoptosis and PLY in lung tissues from untreated mice during pneumococcal pneumonia. Apoptosis was identified by immunohistochemical detection of active caspase-9 and by  in situ  TUNEL assay. PLY was stained with anti-PLY rabbit antibodies. Adjacent sections were co-stained for co-localization of PLY, TUNEL, and caspase-9. Five sections were analyzed in each time point. Statistical differences were not found for a comparison of number of PLY, caspase-9, and TUNEL positive cells as determined by two-way ANOVA followed by the Bonferroni test. (C) Comparison of caspase-9 positive cells in lung tissues from anti-PLY IgG-, control IgG-, and PBS-treated mice. Percentage of caspase-9 stained cells was calculated with respect to total cells counted in random areas of lung tissue sections. Results are means  Â±  SD of 3 mice and are representative of three independent experiments. *,  P  <  0.05 for a comparison of anti-PLY IgG-treated mice with PBS-treated mice, and +, P  <  0.05 for a comparison of anti-PLY IgG with control IgG-treated mice, as determined by two-way ANOVA followed by the Bonferroni test.,'],\n",
       " 218: ['1465-9921-8-3-2-12',\n",
       "  'DMLI',\n",
       "  'Apoptosis in lung tissues of mice infected with  S. pneumoniae  D39 serotype 2 . (A) Distribution patterns of PLY and apoptosis in representative lung sections from mice intranasally infected with  S. pneumoniae  D39 serotype 2. PLY was established by staining with anti-PLY rabbit antibodies. Apoptosis was assessed by active caspase-9 staining and  in situ  TUNEL assay. No staining was observed in lung tissues from uninfected mice. At 12 h post-infection, resident alveolar macrophages were positively stained with anti-PLY, anti-caspase-9, and TUNEL (arrow heads). Infiltrating leukocytes (24 h post-infection) and bronchial epithelium (48 h post-infection) were stained with anti-PLY, anti-caspase-9, and TUNEL, respectively (arrow heads). Note non-stained vascular endothelium. Blood vessel (v), alveolar space (a), bronchiole (b). Scale bars 50  Î¼ m. (B) Apoptosis and PLY in lung tissues from untreated mice during pneumococcal pneumonia. Apoptosis was identified by immunohistochemical detection of active caspase-9 and by  in situ  TUNEL assay. PLY was stained with anti-PLY rabbit antibodies. Adjacent sections were co-stained for co-localization of PLY, TUNEL, and caspase-9. Five sections were analyzed in each time point. Statistical differences were not found for a comparison of number of PLY, caspase-9, and TUNEL positive cells as determined by two-way ANOVA followed by the Bonferroni test. (C) Comparison of caspase-9 positive cells in lung tissues from anti-PLY IgG-, control IgG-, and PBS-treated mice. Percentage of caspase-9 stained cells was calculated with respect to total cells counted in random areas of lung tissue sections. Results are means  Â±  SD of 3 mice and are representative of three independent experiments. *,  P  <  0.05 for a comparison of anti-PLY IgG-treated mice with PBS-treated mice, and +, P  <  0.05 for a comparison of anti-PLY IgG with control IgG-treated mice, as determined by two-way ANOVA followed by the Bonferroni test.,'],\n",
       " 219: ['1465-9921-8-3-2-13',\n",
       "  'GFIG',\n",
       "  'Apoptosis in lung tissues of mice infected with  S. pneumoniae  D39 serotype 2 . (A) Distribution patterns of PLY and apoptosis in representative lung sections from mice intranasally infected with  S. pneumoniae  D39 serotype 2. PLY was established by staining with anti-PLY rabbit antibodies. Apoptosis was assessed by active caspase-9 staining and  in situ  TUNEL assay. No staining was observed in lung tissues from uninfected mice. At 12 h post-infection, resident alveolar macrophages were positively stained with anti-PLY, anti-caspase-9, and TUNEL (arrow heads). Infiltrating leukocytes (24 h post-infection) and bronchial epithelium (48 h post-infection) were stained with anti-PLY, anti-caspase-9, and TUNEL, respectively (arrow heads). Note non-stained vascular endothelium. Blood vessel (v), alveolar space (a), bronchiole (b). Scale bars 50  Î¼ m. (B) Apoptosis and PLY in lung tissues from untreated mice during pneumococcal pneumonia. Apoptosis was identified by immunohistochemical detection of active caspase-9 and by  in situ  TUNEL assay. PLY was stained with anti-PLY rabbit antibodies. Adjacent sections were co-stained for co-localization of PLY, TUNEL, and caspase-9. Five sections were analyzed in each time point. Statistical differences were not found for a comparison of number of PLY, caspase-9, and TUNEL positive cells as determined by two-way ANOVA followed by the Bonferroni test. (C) Comparison of caspase-9 positive cells in lung tissues from anti-PLY IgG-, control IgG-, and PBS-treated mice. Percentage of caspase-9 stained cells was calculated with respect to total cells counted in random areas of lung tissue sections. Results are means  Â±  SD of 3 mice and are representative of three independent experiments. *,  P  <  0.05 for a comparison of anti-PLY IgG-treated mice with PBS-treated mice, and +, P  <  0.05 for a comparison of anti-PLY IgG with control IgG-treated mice, as determined by two-way ANOVA followed by the Bonferroni test.,'],\n",
       " 220: ['1465-9921-8-3-2-14',\n",
       "  'GFIG',\n",
       "  'Apoptosis in lung tissues of mice infected with  S. pneumoniae  D39 serotype 2 . (A) Distribution patterns of PLY and apoptosis in representative lung sections from mice intranasally infected with  S. pneumoniae  D39 serotype 2. PLY was established by staining with anti-PLY rabbit antibodies. Apoptosis was assessed by active caspase-9 staining and  in situ  TUNEL assay. No staining was observed in lung tissues from uninfected mice. At 12 h post-infection, resident alveolar macrophages were positively stained with anti-PLY, anti-caspase-9, and TUNEL (arrow heads). Infiltrating leukocytes (24 h post-infection) and bronchial epithelium (48 h post-infection) were stained with anti-PLY, anti-caspase-9, and TUNEL, respectively (arrow heads). Note non-stained vascular endothelium. Blood vessel (v), alveolar space (a), bronchiole (b). Scale bars 50  Î¼ m. (B) Apoptosis and PLY in lung tissues from untreated mice during pneumococcal pneumonia. Apoptosis was identified by immunohistochemical detection of active caspase-9 and by  in situ  TUNEL assay. PLY was stained with anti-PLY rabbit antibodies. Adjacent sections were co-stained for co-localization of PLY, TUNEL, and caspase-9. Five sections were analyzed in each time point. Statistical differences were not found for a comparison of number of PLY, caspase-9, and TUNEL positive cells as determined by two-way ANOVA followed by the Bonferroni test. (C) Comparison of caspase-9 positive cells in lung tissues from anti-PLY IgG-, control IgG-, and PBS-treated mice. Percentage of caspase-9 stained cells was calculated with respect to total cells counted in random areas of lung tissue sections. Results are means  Â±  SD of 3 mice and are representative of three independent experiments. *,  P  <  0.05 for a comparison of anti-PLY IgG-treated mice with PBS-treated mice, and +, P  <  0.05 for a comparison of anti-PLY IgG with control IgG-treated mice, as determined by two-way ANOVA followed by the Bonferroni test.,'],\n",
       " 221: ['1465-9921-8-3-2-1',\n",
       "  'DMLI',\n",
       "  'Apoptosis in lung tissues of mice infected with  S. pneumoniae  D39 serotype 2 . (A) Distribution patterns of PLY and apoptosis in representative lung sections from mice intranasally infected with  S. pneumoniae  D39 serotype 2. PLY was established by staining with anti-PLY rabbit antibodies. Apoptosis was assessed by active caspase-9 staining and  in situ  TUNEL assay. No staining was observed in lung tissues from uninfected mice. At 12 h post-infection, resident alveolar macrophages were positively stained with anti-PLY, anti-caspase-9, and TUNEL (arrow heads). Infiltrating leukocytes (24 h post-infection) and bronchial epithelium (48 h post-infection) were stained with anti-PLY, anti-caspase-9, and TUNEL, respectively (arrow heads). Note non-stained vascular endothelium. Blood vessel (v), alveolar space (a), bronchiole (b). Scale bars 50  Î¼ m. (B) Apoptosis and PLY in lung tissues from untreated mice during pneumococcal pneumonia. Apoptosis was identified by immunohistochemical detection of active caspase-9 and by  in situ  TUNEL assay. PLY was stained with anti-PLY rabbit antibodies. Adjacent sections were co-stained for co-localization of PLY, TUNEL, and caspase-9. Five sections were analyzed in each time point. Statistical differences were not found for a comparison of number of PLY, caspase-9, and TUNEL positive cells as determined by two-way ANOVA followed by the Bonferroni test. (C) Comparison of caspase-9 positive cells in lung tissues from anti-PLY IgG-, control IgG-, and PBS-treated mice. Percentage of caspase-9 stained cells was calculated with respect to total cells counted in random areas of lung tissue sections. Results are means  Â±  SD of 3 mice and are representative of three independent experiments. *,  P  <  0.05 for a comparison of anti-PLY IgG-treated mice with PBS-treated mice, and +, P  <  0.05 for a comparison of anti-PLY IgG with control IgG-treated mice, as determined by two-way ANOVA followed by the Bonferroni test.,'],\n",
       " 222: ['1465-9921-8-3-2-2',\n",
       "  'DMLI',\n",
       "  'Apoptosis in lung tissues of mice infected with  S. pneumoniae  D39 serotype 2 . (A) Distribution patterns of PLY and apoptosis in representative lung sections from mice intranasally infected with  S. pneumoniae  D39 serotype 2. PLY was established by staining with anti-PLY rabbit antibodies. Apoptosis was assessed by active caspase-9 staining and  in situ  TUNEL assay. No staining was observed in lung tissues from uninfected mice. At 12 h post-infection, resident alveolar macrophages were positively stained with anti-PLY, anti-caspase-9, and TUNEL (arrow heads). Infiltrating leukocytes (24 h post-infection) and bronchial epithelium (48 h post-infection) were stained with anti-PLY, anti-caspase-9, and TUNEL, respectively (arrow heads). Note non-stained vascular endothelium. Blood vessel (v), alveolar space (a), bronchiole (b). Scale bars 50  Î¼ m. (B) Apoptosis and PLY in lung tissues from untreated mice during pneumococcal pneumonia. Apoptosis was identified by immunohistochemical detection of active caspase-9 and by  in situ  TUNEL assay. PLY was stained with anti-PLY rabbit antibodies. Adjacent sections were co-stained for co-localization of PLY, TUNEL, and caspase-9. Five sections were analyzed in each time point. Statistical differences were not found for a comparison of number of PLY, caspase-9, and TUNEL positive cells as determined by two-way ANOVA followed by the Bonferroni test. (C) Comparison of caspase-9 positive cells in lung tissues from anti-PLY IgG-, control IgG-, and PBS-treated mice. Percentage of caspase-9 stained cells was calculated with respect to total cells counted in random areas of lung tissue sections. Results are means  Â±  SD of 3 mice and are representative of three independent experiments. *,  P  <  0.05 for a comparison of anti-PLY IgG-treated mice with PBS-treated mice, and +, P  <  0.05 for a comparison of anti-PLY IgG with control IgG-treated mice, as determined by two-way ANOVA followed by the Bonferroni test.,'],\n",
       " 223: ['1465-9921-8-3-2-3',\n",
       "  'DMLI',\n",
       "  'Apoptosis in lung tissues of mice infected with  S. pneumoniae  D39 serotype 2 . (A) Distribution patterns of PLY and apoptosis in representative lung sections from mice intranasally infected with  S. pneumoniae  D39 serotype 2. PLY was established by staining with anti-PLY rabbit antibodies. Apoptosis was assessed by active caspase-9 staining and  in situ  TUNEL assay. No staining was observed in lung tissues from uninfected mice. At 12 h post-infection, resident alveolar macrophages were positively stained with anti-PLY, anti-caspase-9, and TUNEL (arrow heads). Infiltrating leukocytes (24 h post-infection) and bronchial epithelium (48 h post-infection) were stained with anti-PLY, anti-caspase-9, and TUNEL, respectively (arrow heads). Note non-stained vascular endothelium. Blood vessel (v), alveolar space (a), bronchiole (b). Scale bars 50  Î¼ m. (B) Apoptosis and PLY in lung tissues from untreated mice during pneumococcal pneumonia. Apoptosis was identified by immunohistochemical detection of active caspase-9 and by  in situ  TUNEL assay. PLY was stained with anti-PLY rabbit antibodies. Adjacent sections were co-stained for co-localization of PLY, TUNEL, and caspase-9. Five sections were analyzed in each time point. Statistical differences were not found for a comparison of number of PLY, caspase-9, and TUNEL positive cells as determined by two-way ANOVA followed by the Bonferroni test. (C) Comparison of caspase-9 positive cells in lung tissues from anti-PLY IgG-, control IgG-, and PBS-treated mice. Percentage of caspase-9 stained cells was calculated with respect to total cells counted in random areas of lung tissue sections. Results are means  Â±  SD of 3 mice and are representative of three independent experiments. *,  P  <  0.05 for a comparison of anti-PLY IgG-treated mice with PBS-treated mice, and +, P  <  0.05 for a comparison of anti-PLY IgG with control IgG-treated mice, as determined by two-way ANOVA followed by the Bonferroni test.,'],\n",
       " 224: ['1465-9921-8-3-2-4',\n",
       "  'DMLI',\n",
       "  'Apoptosis in lung tissues of mice infected with  S. pneumoniae  D39 serotype 2 . (A) Distribution patterns of PLY and apoptosis in representative lung sections from mice intranasally infected with  S. pneumoniae  D39 serotype 2. PLY was established by staining with anti-PLY rabbit antibodies. Apoptosis was assessed by active caspase-9 staining and  in situ  TUNEL assay. No staining was observed in lung tissues from uninfected mice. At 12 h post-infection, resident alveolar macrophages were positively stained with anti-PLY, anti-caspase-9, and TUNEL (arrow heads). Infiltrating leukocytes (24 h post-infection) and bronchial epithelium (48 h post-infection) were stained with anti-PLY, anti-caspase-9, and TUNEL, respectively (arrow heads). Note non-stained vascular endothelium. Blood vessel (v), alveolar space (a), bronchiole (b). Scale bars 50  Î¼ m. (B) Apoptosis and PLY in lung tissues from untreated mice during pneumococcal pneumonia. Apoptosis was identified by immunohistochemical detection of active caspase-9 and by  in situ  TUNEL assay. PLY was stained with anti-PLY rabbit antibodies. Adjacent sections were co-stained for co-localization of PLY, TUNEL, and caspase-9. Five sections were analyzed in each time point. Statistical differences were not found for a comparison of number of PLY, caspase-9, and TUNEL positive cells as determined by two-way ANOVA followed by the Bonferroni test. (C) Comparison of caspase-9 positive cells in lung tissues from anti-PLY IgG-, control IgG-, and PBS-treated mice. Percentage of caspase-9 stained cells was calculated with respect to total cells counted in random areas of lung tissue sections. Results are means  Â±  SD of 3 mice and are representative of three independent experiments. *,  P  <  0.05 for a comparison of anti-PLY IgG-treated mice with PBS-treated mice, and +, P  <  0.05 for a comparison of anti-PLY IgG with control IgG-treated mice, as determined by two-way ANOVA followed by the Bonferroni test.,'],\n",
       " 225: ['1465-9921-8-3-2-5',\n",
       "  'DMLI',\n",
       "  'Apoptosis in lung tissues of mice infected with  S. pneumoniae  D39 serotype 2 . (A) Distribution patterns of PLY and apoptosis in representative lung sections from mice intranasally infected with  S. pneumoniae  D39 serotype 2. PLY was established by staining with anti-PLY rabbit antibodies. Apoptosis was assessed by active caspase-9 staining and  in situ  TUNEL assay. No staining was observed in lung tissues from uninfected mice. At 12 h post-infection, resident alveolar macrophages were positively stained with anti-PLY, anti-caspase-9, and TUNEL (arrow heads). Infiltrating leukocytes (24 h post-infection) and bronchial epithelium (48 h post-infection) were stained with anti-PLY, anti-caspase-9, and TUNEL, respectively (arrow heads). Note non-stained vascular endothelium. Blood vessel (v), alveolar space (a), bronchiole (b). Scale bars 50  Î¼ m. (B) Apoptosis and PLY in lung tissues from untreated mice during pneumococcal pneumonia. Apoptosis was identified by immunohistochemical detection of active caspase-9 and by  in situ  TUNEL assay. PLY was stained with anti-PLY rabbit antibodies. Adjacent sections were co-stained for co-localization of PLY, TUNEL, and caspase-9. Five sections were analyzed in each time point. Statistical differences were not found for a comparison of number of PLY, caspase-9, and TUNEL positive cells as determined by two-way ANOVA followed by the Bonferroni test. (C) Comparison of caspase-9 positive cells in lung tissues from anti-PLY IgG-, control IgG-, and PBS-treated mice. Percentage of caspase-9 stained cells was calculated with respect to total cells counted in random areas of lung tissue sections. Results are means  Â±  SD of 3 mice and are representative of three independent experiments. *,  P  <  0.05 for a comparison of anti-PLY IgG-treated mice with PBS-treated mice, and +, P  <  0.05 for a comparison of anti-PLY IgG with control IgG-treated mice, as determined by two-way ANOVA followed by the Bonferroni test.,'],\n",
       " 226: ['1465-9921-8-3-2-6',\n",
       "  'DMLI',\n",
       "  'Apoptosis in lung tissues of mice infected with  S. pneumoniae  D39 serotype 2 . (A) Distribution patterns of PLY and apoptosis in representative lung sections from mice intranasally infected with  S. pneumoniae  D39 serotype 2. PLY was established by staining with anti-PLY rabbit antibodies. Apoptosis was assessed by active caspase-9 staining and  in situ  TUNEL assay. No staining was observed in lung tissues from uninfected mice. At 12 h post-infection, resident alveolar macrophages were positively stained with anti-PLY, anti-caspase-9, and TUNEL (arrow heads). Infiltrating leukocytes (24 h post-infection) and bronchial epithelium (48 h post-infection) were stained with anti-PLY, anti-caspase-9, and TUNEL, respectively (arrow heads). Note non-stained vascular endothelium. Blood vessel (v), alveolar space (a), bronchiole (b). Scale bars 50  Î¼ m. (B) Apoptosis and PLY in lung tissues from untreated mice during pneumococcal pneumonia. Apoptosis was identified by immunohistochemical detection of active caspase-9 and by  in situ  TUNEL assay. PLY was stained with anti-PLY rabbit antibodies. Adjacent sections were co-stained for co-localization of PLY, TUNEL, and caspase-9. Five sections were analyzed in each time point. Statistical differences were not found for a comparison of number of PLY, caspase-9, and TUNEL positive cells as determined by two-way ANOVA followed by the Bonferroni test. (C) Comparison of caspase-9 positive cells in lung tissues from anti-PLY IgG-, control IgG-, and PBS-treated mice. Percentage of caspase-9 stained cells was calculated with respect to total cells counted in random areas of lung tissue sections. Results are means  Â±  SD of 3 mice and are representative of three independent experiments. *,  P  <  0.05 for a comparison of anti-PLY IgG-treated mice with PBS-treated mice, and +, P  <  0.05 for a comparison of anti-PLY IgG with control IgG-treated mice, as determined by two-way ANOVA followed by the Bonferroni test.,'],\n",
       " 227: ['1465-9921-8-3-2-7',\n",
       "  'DMLI',\n",
       "  'Apoptosis in lung tissues of mice infected with  S. pneumoniae  D39 serotype 2 . (A) Distribution patterns of PLY and apoptosis in representative lung sections from mice intranasally infected with  S. pneumoniae  D39 serotype 2. PLY was established by staining with anti-PLY rabbit antibodies. Apoptosis was assessed by active caspase-9 staining and  in situ  TUNEL assay. No staining was observed in lung tissues from uninfected mice. At 12 h post-infection, resident alveolar macrophages were positively stained with anti-PLY, anti-caspase-9, and TUNEL (arrow heads). Infiltrating leukocytes (24 h post-infection) and bronchial epithelium (48 h post-infection) were stained with anti-PLY, anti-caspase-9, and TUNEL, respectively (arrow heads). Note non-stained vascular endothelium. Blood vessel (v), alveolar space (a), bronchiole (b). Scale bars 50  Î¼ m. (B) Apoptosis and PLY in lung tissues from untreated mice during pneumococcal pneumonia. Apoptosis was identified by immunohistochemical detection of active caspase-9 and by  in situ  TUNEL assay. PLY was stained with anti-PLY rabbit antibodies. Adjacent sections were co-stained for co-localization of PLY, TUNEL, and caspase-9. Five sections were analyzed in each time point. Statistical differences were not found for a comparison of number of PLY, caspase-9, and TUNEL positive cells as determined by two-way ANOVA followed by the Bonferroni test. (C) Comparison of caspase-9 positive cells in lung tissues from anti-PLY IgG-, control IgG-, and PBS-treated mice. Percentage of caspase-9 stained cells was calculated with respect to total cells counted in random areas of lung tissue sections. Results are means  Â±  SD of 3 mice and are representative of three independent experiments. *,  P  <  0.05 for a comparison of anti-PLY IgG-treated mice with PBS-treated mice, and +, P  <  0.05 for a comparison of anti-PLY IgG with control IgG-treated mice, as determined by two-way ANOVA followed by the Bonferroni test.,'],\n",
       " 228: ['1465-9921-8-3-2-8',\n",
       "  'DMLI',\n",
       "  'Apoptosis in lung tissues of mice infected with  S. pneumoniae  D39 serotype 2 . (A) Distribution patterns of PLY and apoptosis in representative lung sections from mice intranasally infected with  S. pneumoniae  D39 serotype 2. PLY was established by staining with anti-PLY rabbit antibodies. Apoptosis was assessed by active caspase-9 staining and  in situ  TUNEL assay. No staining was observed in lung tissues from uninfected mice. At 12 h post-infection, resident alveolar macrophages were positively stained with anti-PLY, anti-caspase-9, and TUNEL (arrow heads). Infiltrating leukocytes (24 h post-infection) and bronchial epithelium (48 h post-infection) were stained with anti-PLY, anti-caspase-9, and TUNEL, respectively (arrow heads). Note non-stained vascular endothelium. Blood vessel (v), alveolar space (a), bronchiole (b). Scale bars 50  Î¼ m. (B) Apoptosis and PLY in lung tissues from untreated mice during pneumococcal pneumonia. Apoptosis was identified by immunohistochemical detection of active caspase-9 and by  in situ  TUNEL assay. PLY was stained with anti-PLY rabbit antibodies. Adjacent sections were co-stained for co-localization of PLY, TUNEL, and caspase-9. Five sections were analyzed in each time point. Statistical differences were not found for a comparison of number of PLY, caspase-9, and TUNEL positive cells as determined by two-way ANOVA followed by the Bonferroni test. (C) Comparison of caspase-9 positive cells in lung tissues from anti-PLY IgG-, control IgG-, and PBS-treated mice. Percentage of caspase-9 stained cells was calculated with respect to total cells counted in random areas of lung tissue sections. Results are means  Â±  SD of 3 mice and are representative of three independent experiments. *,  P  <  0.05 for a comparison of anti-PLY IgG-treated mice with PBS-treated mice, and +, P  <  0.05 for a comparison of anti-PLY IgG with control IgG-treated mice, as determined by two-way ANOVA followed by the Bonferroni test.,'],\n",
       " 229: ['1465-9921-8-3-2-9',\n",
       "  'DMLI',\n",
       "  'Apoptosis in lung tissues of mice infected with  S. pneumoniae  D39 serotype 2 . (A) Distribution patterns of PLY and apoptosis in representative lung sections from mice intranasally infected with  S. pneumoniae  D39 serotype 2. PLY was established by staining with anti-PLY rabbit antibodies. Apoptosis was assessed by active caspase-9 staining and  in situ  TUNEL assay. No staining was observed in lung tissues from uninfected mice. At 12 h post-infection, resident alveolar macrophages were positively stained with anti-PLY, anti-caspase-9, and TUNEL (arrow heads). Infiltrating leukocytes (24 h post-infection) and bronchial epithelium (48 h post-infection) were stained with anti-PLY, anti-caspase-9, and TUNEL, respectively (arrow heads). Note non-stained vascular endothelium. Blood vessel (v), alveolar space (a), bronchiole (b). Scale bars 50  Î¼ m. (B) Apoptosis and PLY in lung tissues from untreated mice during pneumococcal pneumonia. Apoptosis was identified by immunohistochemical detection of active caspase-9 and by  in situ  TUNEL assay. PLY was stained with anti-PLY rabbit antibodies. Adjacent sections were co-stained for co-localization of PLY, TUNEL, and caspase-9. Five sections were analyzed in each time point. Statistical differences were not found for a comparison of number of PLY, caspase-9, and TUNEL positive cells as determined by two-way ANOVA followed by the Bonferroni test. (C) Comparison of caspase-9 positive cells in lung tissues from anti-PLY IgG-, control IgG-, and PBS-treated mice. Percentage of caspase-9 stained cells was calculated with respect to total cells counted in random areas of lung tissue sections. Results are means  Â±  SD of 3 mice and are representative of three independent experiments. *,  P  <  0.05 for a comparison of anti-PLY IgG-treated mice with PBS-treated mice, and +, P  <  0.05 for a comparison of anti-PLY IgG with control IgG-treated mice, as determined by two-way ANOVA followed by the Bonferroni test.,'],\n",
       " 230: ['1465-9921-9-28-8-1',\n",
       "  'DMLI',\n",
       "  'Histological examples of atelectasis (10*) on HE slides. Severe atelectasis was demonstrated up to day 90 after LIRI. HE = Haematoxylin and Eosin staining; LIRI = Lung Ischemia-Reperfusion Injury.,'],\n",
       " 231: ['1465-9921-9-28-8-2',\n",
       "  'DMLI',\n",
       "  'Histological examples of atelectasis (10*) on HE slides. Severe atelectasis was demonstrated up to day 90 after LIRI. HE = Haematoxylin and Eosin staining; LIRI = Lung Ischemia-Reperfusion Injury.,'],\n",
       " 232: ['1465-9921-9-28-8-3',\n",
       "  'DMLI',\n",
       "  'Histological examples of atelectasis (10*) on HE slides. Severe atelectasis was demonstrated up to day 90 after LIRI. HE = Haematoxylin and Eosin staining; LIRI = Lung Ischemia-Reperfusion Injury.,'],\n",
       " 233: ['1465-9921-9-28-8-4',\n",
       "  'DMLI',\n",
       "  'Histological examples of atelectasis (10*) on HE slides. Severe atelectasis was demonstrated up to day 90 after LIRI. HE = Haematoxylin and Eosin staining; LIRI = Lung Ischemia-Reperfusion Injury.,'],\n",
       " 234: ['1465-9921-9-28-8-5',\n",
       "  'DMLI',\n",
       "  'Histological examples of atelectasis (10*) on HE slides. Severe atelectasis was demonstrated up to day 90 after LIRI. HE = Haematoxylin and Eosin staining; LIRI = Lung Ischemia-Reperfusion Injury.,'],\n",
       " 235: ['1465-9921-9-28-8-6',\n",
       "  'DMLI',\n",
       "  'Histological examples of atelectasis (10*) on HE slides. Severe atelectasis was demonstrated up to day 90 after LIRI. HE = Haematoxylin and Eosin staining; LIRI = Lung Ischemia-Reperfusion Injury.,'],\n",
       " 236: ['1467-4866-10-4-9-1',\n",
       "  'GFIG',\n",
       "  'pH breakthrough curves in response to CO 2  = 10 3  t/yr (0.63 t/m 2 /yr) in developed aquifers as a function pumping rate and distance of the monitoring well from the CO 2  leak .,'],\n",
       " 237: ['1467-4866-10-4-9-2',\n",
       "  'GFIG',\n",
       "  'pH breakthrough curves in response to CO 2  = 10 3  t/yr (0.63 t/m 2 /yr) in developed aquifers as a function pumping rate and distance of the monitoring well from the CO 2  leak .,'],\n",
       " 238: ['1467-4866-10-4-9-3',\n",
       "  'GFIG',\n",
       "  'pH breakthrough curves in response to CO 2  = 10 3  t/yr (0.63 t/m 2 /yr) in developed aquifers as a function pumping rate and distance of the monitoring well from the CO 2  leak .,'],\n",
       " 239: ['1467-4866-10-4-9-4',\n",
       "  'GFIG',\n",
       "  'pH breakthrough curves in response to CO 2  = 10 3  t/yr (0.63 t/m 2 /yr) in developed aquifers as a function pumping rate and distance of the monitoring well from the CO 2  leak .,'],\n",
       " 240: ['1467-4866-12-6-1-1',\n",
       "  'DMLI',\n",
       "  'ESEM spot analyses of potential catalyst sites on the olivine surfaces used in the incubation experiments . Relative proportions of elements in weight percent is presented in each picture. The high carbon content in picture B and D is due to carbon coating of the sample. A) SEM image of a chrome-spinel crystal taken using optical microscopy, B) SEM image of a Cr and Fe-rich phase C) SEM image of a Fe-rich phase and D) SEM image of Fe, Ni and Cu-rich phases.,'],\n",
       " 241: ['1467-4866-12-6-1-2',\n",
       "  'DMEL',\n",
       "  'ESEM spot analyses of potential catalyst sites on the olivine surfaces used in the incubation experiments . Relative proportions of elements in weight percent is presented in each picture. The high carbon content in picture B and D is due to carbon coating of the sample. A) SEM image of a chrome-spinel crystal taken using optical microscopy, B) SEM image of a Cr and Fe-rich phase C) SEM image of a Fe-rich phase and D) SEM image of Fe, Ni and Cu-rich phases.,'],\n",
       " 242: ['1467-4866-12-6-1-3',\n",
       "  'DMEL',\n",
       "  'ESEM spot analyses of potential catalyst sites on the olivine surfaces used in the incubation experiments . Relative proportions of elements in weight percent is presented in each picture. The high carbon content in picture B and D is due to carbon coating of the sample. A) SEM image of a chrome-spinel crystal taken using optical microscopy, B) SEM image of a Cr and Fe-rich phase C) SEM image of a Fe-rich phase and D) SEM image of Fe, Ni and Cu-rich phases.,'],\n",
       " 243: ['1467-4866-12-6-1-4',\n",
       "  'DMTR',\n",
       "  'ESEM spot analyses of potential catalyst sites on the olivine surfaces used in the incubation experiments . Relative proportions of elements in weight percent is presented in each picture. The high carbon content in picture B and D is due to carbon coating of the sample. A) SEM image of a chrome-spinel crystal taken using optical microscopy, B) SEM image of a Cr and Fe-rich phase C) SEM image of a Fe-rich phase and D) SEM image of Fe, Ni and Cu-rich phases.,'],\n",
       " 244: ['1467-4866-7-5-2-1',\n",
       "  'DMLI',\n",
       "  'Photos showing microfossils from opposite sides. An example of the advantages of using both sides of a thin section. A and B showing a tubular filament attached to volcanic glass from opposite sides. C and D showing a twisted filament where the same twisted features are shown from both sides indicating that it is a three-dimensional morphological feature. E and F showing a filament blocked by another filament. In E the whole filament is visible but in F parts of the filament is blocked. The scale bars are 10  Î¼ m.,'],\n",
       " 245: ['1467-4866-7-5-2-2',\n",
       "  'DMLI',\n",
       "  'Photos showing microfossils from opposite sides. An example of the advantages of using both sides of a thin section. A and B showing a tubular filament attached to volcanic glass from opposite sides. C and D showing a twisted filament where the same twisted features are shown from both sides indicating that it is a three-dimensional morphological feature. E and F showing a filament blocked by another filament. In E the whole filament is visible but in F parts of the filament is blocked. The scale bars are 10  Î¼ m.,'],\n",
       " 246: ['1467-4866-7-5-2-3',\n",
       "  'DMLI',\n",
       "  'Photos showing microfossils from opposite sides. An example of the advantages of using both sides of a thin section. A and B showing a tubular filament attached to volcanic glass from opposite sides. C and D showing a twisted filament where the same twisted features are shown from both sides indicating that it is a three-dimensional morphological feature. E and F showing a filament blocked by another filament. In E the whole filament is visible but in F parts of the filament is blocked. The scale bars are 10  Î¼ m.,'],\n",
       " 247: ['1467-4866-7-5-2-4',\n",
       "  'DMLI',\n",
       "  'Photos showing microfossils from opposite sides. An example of the advantages of using both sides of a thin section. A and B showing a tubular filament attached to volcanic glass from opposite sides. C and D showing a twisted filament where the same twisted features are shown from both sides indicating that it is a three-dimensional morphological feature. E and F showing a filament blocked by another filament. In E the whole filament is visible but in F parts of the filament is blocked. The scale bars are 10  Î¼ m.,'],\n",
       " 248: ['1467-4866-7-5-2-5',\n",
       "  'DMLI',\n",
       "  'Photos showing microfossils from opposite sides. An example of the advantages of using both sides of a thin section. A and B showing a tubular filament attached to volcanic glass from opposite sides. C and D showing a twisted filament where the same twisted features are shown from both sides indicating that it is a three-dimensional morphological feature. E and F showing a filament blocked by another filament. In E the whole filament is visible but in F parts of the filament is blocked. The scale bars are 10  Î¼ m.,'],\n",
       " 249: ['1467-4866-7-5-2-6',\n",
       "  'DMLI',\n",
       "  'Photos showing microfossils from opposite sides. An example of the advantages of using both sides of a thin section. A and B showing a tubular filament attached to volcanic glass from opposite sides. C and D showing a twisted filament where the same twisted features are shown from both sides indicating that it is a three-dimensional morphological feature. E and F showing a filament blocked by another filament. In E the whole filament is visible but in F parts of the filament is blocked. The scale bars are 10  Î¼ m.,'],\n",
       " 250: ['1471-2091-10-22-6-1',\n",
       "  'GFIG',\n",
       "  'A: Schematic diagrams of default PSRPK and mutant PSRPK compared with PSRPK . B: Distribution of RFP-PSRPK d  and RFP-PSRPK m  expression in L929 cells. PSRPK d , default PSRPK without  318 PKKGDK 323 ; PSRPK m , mutant PSRPK with the K 320    T 320 mutation. 1 and 2: Distribution of RFP-PSRPK d  and RFP-PSRPK m  expressions in L929 cells observed under a confocal microscope (TCS SP2, Leica) compared with their position in the nucleus in DAPI-stained cells observed under a BX51 fluorescent microscope (Olympus, 400  ).,'],\n",
       " 251: ['1471-2091-10-22-6-2',\n",
       "  'DMFL',\n",
       "  'A: Schematic diagrams of default PSRPK and mutant PSRPK compared with PSRPK . B: Distribution of RFP-PSRPK d  and RFP-PSRPK m  expression in L929 cells. PSRPK d , default PSRPK without  318 PKKGDK 323 ; PSRPK m , mutant PSRPK with the K 320    T 320 mutation. 1 and 2: Distribution of RFP-PSRPK d  and RFP-PSRPK m  expressions in L929 cells observed under a confocal microscope (TCS SP2, Leica) compared with their position in the nucleus in DAPI-stained cells observed under a BX51 fluorescent microscope (Olympus, 400  ).,'],\n",
       " 252: ['1471-2091-10-22-6-3',\n",
       "  'DMFL',\n",
       "  'A: Schematic diagrams of default PSRPK and mutant PSRPK compared with PSRPK . B: Distribution of RFP-PSRPK d  and RFP-PSRPK m  expression in L929 cells. PSRPK d , default PSRPK without  318 PKKGDK 323 ; PSRPK m , mutant PSRPK with the K 320    T 320 mutation. 1 and 2: Distribution of RFP-PSRPK d  and RFP-PSRPK m  expressions in L929 cells observed under a confocal microscope (TCS SP2, Leica) compared with their position in the nucleus in DAPI-stained cells observed under a BX51 fluorescent microscope (Olympus, 400  ).,'],\n",
       " 253: ['1471-2091-10-22-6-4',\n",
       "  'DMFL',\n",
       "  'A: Schematic diagrams of default PSRPK and mutant PSRPK compared with PSRPK . B: Distribution of RFP-PSRPK d  and RFP-PSRPK m  expression in L929 cells. PSRPK d , default PSRPK without  318 PKKGDK 323 ; PSRPK m , mutant PSRPK with the K 320    T 320 mutation. 1 and 2: Distribution of RFP-PSRPK d  and RFP-PSRPK m  expressions in L929 cells observed under a confocal microscope (TCS SP2, Leica) compared with their position in the nucleus in DAPI-stained cells observed under a BX51 fluorescent microscope (Olympus, 400  ).,'],\n",
       " 254: ['1471-2091-10-22-6-5',\n",
       "  'DMFL',\n",
       "  'A: Schematic diagrams of default PSRPK and mutant PSRPK compared with PSRPK . B: Distribution of RFP-PSRPK d  and RFP-PSRPK m  expression in L929 cells. PSRPK d , default PSRPK without  318 PKKGDK 323 ; PSRPK m , mutant PSRPK with the K 320    T 320 mutation. 1 and 2: Distribution of RFP-PSRPK d  and RFP-PSRPK m  expressions in L929 cells observed under a confocal microscope (TCS SP2, Leica) compared with their position in the nucleus in DAPI-stained cells observed under a BX51 fluorescent microscope (Olympus, 400  ).,'],\n",
       " 255: ['1471-2091-10-22-6-6',\n",
       "  'DMFL',\n",
       "  'A: Schematic diagrams of default PSRPK and mutant PSRPK compared with PSRPK . B: Distribution of RFP-PSRPK d  and RFP-PSRPK m  expression in L929 cells. PSRPK d , default PSRPK without  318 PKKGDK 323 ; PSRPK m , mutant PSRPK with the K 320    T 320 mutation. 1 and 2: Distribution of RFP-PSRPK d  and RFP-PSRPK m  expressions in L929 cells observed under a confocal microscope (TCS SP2, Leica) compared with their position in the nucleus in DAPI-stained cells observed under a BX51 fluorescent microscope (Olympus, 400  ).,'],\n",
       " 256: ['1471-2091-10-22-6-7',\n",
       "  'DMFL',\n",
       "  'A: Schematic diagrams of default PSRPK and mutant PSRPK compared with PSRPK . B: Distribution of RFP-PSRPK d  and RFP-PSRPK m  expression in L929 cells. PSRPK d , default PSRPK without  318 PKKGDK 323 ; PSRPK m , mutant PSRPK with the K 320    T 320 mutation. 1 and 2: Distribution of RFP-PSRPK d  and RFP-PSRPK m  expressions in L929 cells observed under a confocal microscope (TCS SP2, Leica) compared with their position in the nucleus in DAPI-stained cells observed under a BX51 fluorescent microscope (Olympus, 400  ).,'],\n",
       " 257: ['1471-2091-10-23-5-1',\n",
       "  'GGEN',\n",
       "  'Preferential cleavage sites for cysteine cathepsins within the thyroglobulin sequence at distinct redox- and pH-conditions . Characteristic Tg-fragments obtained by  in vitro  degradation at extracellular or intra-endo-lysosomal conditions were N-terminally sequenced. Newly generated N-termini are highlighted (A); box colors represent the respective condition at which fragments were obtained (c.f. Figure 1). Hormogenic sites, i.e. pre-formed thyroid hormones tri-iodothyronine (T 3 ) and thyroxine (T 4 ), are tagged in blue and green; the signal peptide is highlighted in grey. (B-D) Schematic drawings sketch the Tg molecule that comprises Tg Type I domains (green), Type II (blue), Type III A (light grey), Type III B repeats (dark grey), Thyropins (red), and CXXC-motifs (see [ 6 ] for details). Arrowheads indicate relative positions of identified cleavage sites utilized in extracellular- (B) or endosomal-like Tg processing (C). Cleavage sites of cathepsins B, D, and L were determined by N-terminal sequencing of Tg fragments derived from incubation of rabbit Tg with lysosomal extracts or cathepsins isolated from human thyrocytes (D; according to [ 36 , 37 ]). This scheme (D) is shown for comparison, only, and shaded to indicate that it is adopted (see [ 6 ]) from the results of studies performed by Dunn and co-workers [ 36 , 37 ]. Corresponding cysteine cathepsins or combinations thereof are given below the arrowheads; the respective neo-N-termini are specified above the representative schematic drawing. Arrows in B and D indicate similar cleavage sites under extracellular conditions and within lysosomes, respectively.,'],\n",
       " 258: ['1471-2091-10-23-5-2',\n",
       "  'GGEN',\n",
       "  'Preferential cleavage sites for cysteine cathepsins within the thyroglobulin sequence at distinct redox- and pH-conditions . Characteristic Tg-fragments obtained by  in vitro  degradation at extracellular or intra-endo-lysosomal conditions were N-terminally sequenced. Newly generated N-termini are highlighted (A); box colors represent the respective condition at which fragments were obtained (c.f. Figure 1). Hormogenic sites, i.e. pre-formed thyroid hormones tri-iodothyronine (T 3 ) and thyroxine (T 4 ), are tagged in blue and green; the signal peptide is highlighted in grey. (B-D) Schematic drawings sketch the Tg molecule that comprises Tg Type I domains (green), Type II (blue), Type III A (light grey), Type III B repeats (dark grey), Thyropins (red), and CXXC-motifs (see [ 6 ] for details). Arrowheads indicate relative positions of identified cleavage sites utilized in extracellular- (B) or endosomal-like Tg processing (C). Cleavage sites of cathepsins B, D, and L were determined by N-terminal sequencing of Tg fragments derived from incubation of rabbit Tg with lysosomal extracts or cathepsins isolated from human thyrocytes (D; according to [ 36 , 37 ]). This scheme (D) is shown for comparison, only, and shaded to indicate that it is adopted (see [ 6 ]) from the results of studies performed by Dunn and co-workers [ 36 , 37 ]. Corresponding cysteine cathepsins or combinations thereof are given below the arrowheads; the respective neo-N-termini are specified above the representative schematic drawing. Arrows in B and D indicate similar cleavage sites under extracellular conditions and within lysosomes, respectively.,'],\n",
       " 259: ['1471-2091-10-3-3-1',\n",
       "  'GFIG',\n",
       "  'Root Mean Square Deviation (RMSD) analysis . Root Mean Square Deviation (RMSD) for the entire protein (A), N-terminal (B) and C-terminal (C) regions, and (D) radius of gyration (template SufU is presented in black and  E. faecalis  SufU in red).,'],\n",
       " 260: ['1471-2091-10-3-3-2',\n",
       "  'GFIG',\n",
       "  'Root Mean Square Deviation (RMSD) analysis . Root Mean Square Deviation (RMSD) for the entire protein (A), N-terminal (B) and C-terminal (C) regions, and (D) radius of gyration (template SufU is presented in black and  E. faecalis  SufU in red).,'],\n",
       " 261: ['1471-2091-10-3-3-3',\n",
       "  'GFIG',\n",
       "  'Root Mean Square Deviation (RMSD) analysis . Root Mean Square Deviation (RMSD) for the entire protein (A), N-terminal (B) and C-terminal (C) regions, and (D) radius of gyration (template SufU is presented in black and  E. faecalis  SufU in red).,'],\n",
       " 262: ['1471-2091-10-3-3-4',\n",
       "  'GFIG',\n",
       "  'Root Mean Square Deviation (RMSD) analysis . Root Mean Square Deviation (RMSD) for the entire protein (A), N-terminal (B) and C-terminal (C) regions, and (D) radius of gyration (template SufU is presented in black and  E. faecalis  SufU in red).,'],\n",
       " 263: ['1471-2091-11-14-3-1',\n",
       "  'DMTR',\n",
       "  'Co-detection of Cx43 and phospho-PKC Î´  at the plasma membrane . Serum starved MC3T3 cells were treated with FGF2 for 10 min prior to fixation and subsequent staining with (i) mouse anti-Cx43 (red), (ii) rabbit anti-phospho-PKC Î´  (Thr505) (green), (iii) DAPI (blue). A merged image is also shown (iv). Arrows indicate the same position on each panel and correspond to areas where both Cx43 and phospho-PKC Î´  appear to be closely co-detected in their respective panels. A representative field of view is shown. The white scale bar is ~10  Î¼ m.,'],\n",
       " 264: ['1471-2091-11-14-3-2',\n",
       "  'DMFL',\n",
       "  'Co-detection of Cx43 and phospho-PKC Î´  at the plasma membrane . Serum starved MC3T3 cells were treated with FGF2 for 10 min prior to fixation and subsequent staining with (i) mouse anti-Cx43 (red), (ii) rabbit anti-phospho-PKC Î´  (Thr505) (green), (iii) DAPI (blue). A merged image is also shown (iv). Arrows indicate the same position on each panel and correspond to areas where both Cx43 and phospho-PKC Î´  appear to be closely co-detected in their respective panels. A representative field of view is shown. The white scale bar is ~10  Î¼ m.,'],\n",
       " 265: ['1471-2091-11-14-3-3',\n",
       "  'DMFL',\n",
       "  'Co-detection of Cx43 and phospho-PKC Î´  at the plasma membrane . Serum starved MC3T3 cells were treated with FGF2 for 10 min prior to fixation and subsequent staining with (i) mouse anti-Cx43 (red), (ii) rabbit anti-phospho-PKC Î´  (Thr505) (green), (iii) DAPI (blue). A merged image is also shown (iv). Arrows indicate the same position on each panel and correspond to areas where both Cx43 and phospho-PKC Î´  appear to be closely co-detected in their respective panels. A representative field of view is shown. The white scale bar is ~10  Î¼ m.,'],\n",
       " 266: ['1471-2091-11-14-3-4',\n",
       "  'DMFL',\n",
       "  'Co-detection of Cx43 and phospho-PKC Î´  at the plasma membrane . Serum starved MC3T3 cells were treated with FGF2 for 10 min prior to fixation and subsequent staining with (i) mouse anti-Cx43 (red), (ii) rabbit anti-phospho-PKC Î´  (Thr505) (green), (iii) DAPI (blue). A merged image is also shown (iv). Arrows indicate the same position on each panel and correspond to areas where both Cx43 and phospho-PKC Î´  appear to be closely co-detected in their respective panels. A representative field of view is shown. The white scale bar is ~10  Î¼ m.,'],\n",
       " 267: ['1471-2091-11-27-6-1',\n",
       "  'GFIG',\n",
       "  'Measurement of the interaction between AgsA and   C11 . (A) Equal amounts (50  Î¼ M) AgsA and   C11 were mixed and immediately (Black line) or after 1 h incubation at 25 Â° C (red line) analysed using size exclusion chromatography at 25 Â° C. The retention volume of the protein molecular weight standards is shown above the panel. (B) Fluorescein-labelled AgsA and rhodamine-labelled AgsA (solid circles), fluorescein-labelled   C11 and rhodamine-labelled AgsA (open diamond), fluorescein-labelled AgsA and rhodamine-labelled   C11 (open triangles), and fluorescein-labelled   C11 and rhodamine-labelled   C11 (solid squares) were mixed and incubated for the subunit exchange reaction at 25 Â° C. The rate of subunit exchange is reflected by the time-dependent change for the emission fluorescence intensity of fluorescein recorded at 525 nm [F 525 nm (t)/F 525 nm (0)].'],\n",
       " 268: ['1471-2091-11-27-6-2',\n",
       "  'GFIG',\n",
       "  'Measurement of the interaction between AgsA and   C11 . (A) Equal amounts (50  Î¼ M) AgsA and   C11 were mixed and immediately (Black line) or after 1 h incubation at 25 Â° C (red line) analysed using size exclusion chromatography at 25 Â° C. The retention volume of the protein molecular weight standards is shown above the panel. (B) Fluorescein-labelled AgsA and rhodamine-labelled AgsA (solid circles), fluorescein-labelled   C11 and rhodamine-labelled AgsA (open diamond), fluorescein-labelled AgsA and rhodamine-labelled   C11 (open triangles), and fluorescein-labelled   C11 and rhodamine-labelled   C11 (solid squares) were mixed and incubated for the subunit exchange reaction at 25 Â° C. The rate of subunit exchange is reflected by the time-dependent change for the emission fluorescence intensity of fluorescein recorded at 525 nm [F 525 nm (t)/F 525 nm (0)].'],\n",
       " 269: ['1471-2091-11-44-3-1',\n",
       "  'GFIG',\n",
       "  'Quantitative analysis of proinsulin peptide binding to DnaK .  A : For DnaK-binding peptides of the proinsulin A-,\" B-chain or C- peptide the amount of DnaK released during the three electroelution cycles was determined in order to assess the progress of DnaK dissociation from the peptides. For this analysis the strength of the signals obtained from the high affinity DnaK ligand C3 after each elution cycle (cycle 1: 134 LU; cycle 2: 46 LU; cycle 3: 33 LU) was set as \"\"1\"\" and used as reference for the signal strengths obtained from the investigated peptides after the corresponding cycles of elution. For each peptide\", the resulting three data points (corresponding to the three elution cycles) were subjected to linear regression analysis and for better comparison of the progress of dissociation a line was plotted against the elution cycles. The bold line in the left panel indicates the release of the B-chain peptide 18. In a fluid phase competition assay a-NR (200 nM) and DnaK (1  Î¼ M) were incubated in the presence of increasing concentrations of peptide B11-23 (solid circles) or B18-30 (open circles)  (B)  or increasing concentrations of proinsulin (solid circles) or insulin (open circles)  (C) . The difference in fluorescence emission intensity at 500 nm was plotted as a function of the concentrations of peptides or proteins.,'],\n",
       " 270: ['1471-2091-11-44-3-2',\n",
       "  'GFIG',\n",
       "  'Quantitative analysis of proinsulin peptide binding to DnaK .  A : For DnaK-binding peptides of the proinsulin A-,\" B-chain or C- peptide the amount of DnaK released during the three electroelution cycles was determined in order to assess the progress of DnaK dissociation from the peptides. For this analysis the strength of the signals obtained from the high affinity DnaK ligand C3 after each elution cycle (cycle 1: 134 LU; cycle 2: 46 LU; cycle 3: 33 LU) was set as \"\"1\"\" and used as reference for the signal strengths obtained from the investigated peptides after the corresponding cycles of elution. For each peptide\", the resulting three data points (corresponding to the three elution cycles) were subjected to linear regression analysis and for better comparison of the progress of dissociation a line was plotted against the elution cycles. The bold line in the left panel indicates the release of the B-chain peptide 18. In a fluid phase competition assay a-NR (200 nM) and DnaK (1  Î¼ M) were incubated in the presence of increasing concentrations of peptide B11-23 (solid circles) or B18-30 (open circles)  (B)  or increasing concentrations of proinsulin (solid circles) or insulin (open circles)  (C) . The difference in fluorescence emission intensity at 500 nm was plotted as a function of the concentrations of peptides or proteins.,'],\n",
       " 271: ['1471-2091-11-44-3-3',\n",
       "  'GFIG',\n",
       "  'Quantitative analysis of proinsulin peptide binding to DnaK .  A : For DnaK-binding peptides of the proinsulin A-,\" B-chain or C- peptide the amount of DnaK released during the three electroelution cycles was determined in order to assess the progress of DnaK dissociation from the peptides. For this analysis the strength of the signals obtained from the high affinity DnaK ligand C3 after each elution cycle (cycle 1: 134 LU; cycle 2: 46 LU; cycle 3: 33 LU) was set as \"\"1\"\" and used as reference for the signal strengths obtained from the investigated peptides after the corresponding cycles of elution. For each peptide\", the resulting three data points (corresponding to the three elution cycles) were subjected to linear regression analysis and for better comparison of the progress of dissociation a line was plotted against the elution cycles. The bold line in the left panel indicates the release of the B-chain peptide 18. In a fluid phase competition assay a-NR (200 nM) and DnaK (1  Î¼ M) were incubated in the presence of increasing concentrations of peptide B11-23 (solid circles) or B18-30 (open circles)  (B)  or increasing concentrations of proinsulin (solid circles) or insulin (open circles)  (C) . The difference in fluorescence emission intensity at 500 nm was plotted as a function of the concentrations of peptides or proteins.,'],\n",
       " 272: ['1471-2091-11-44-3-4',\n",
       "  'GFIG',\n",
       "  'Quantitative analysis of proinsulin peptide binding to DnaK .  A : For DnaK-binding peptides of the proinsulin A-,\" B-chain or C- peptide the amount of DnaK released during the three electroelution cycles was determined in order to assess the progress of DnaK dissociation from the peptides. For this analysis the strength of the signals obtained from the high affinity DnaK ligand C3 after each elution cycle (cycle 1: 134 LU; cycle 2: 46 LU; cycle 3: 33 LU) was set as \"\"1\"\" and used as reference for the signal strengths obtained from the investigated peptides after the corresponding cycles of elution. For each peptide\", the resulting three data points (corresponding to the three elution cycles) were subjected to linear regression analysis and for better comparison of the progress of dissociation a line was plotted against the elution cycles. The bold line in the left panel indicates the release of the B-chain peptide 18. In a fluid phase competition assay a-NR (200 nM) and DnaK (1  Î¼ M) were incubated in the presence of increasing concentrations of peptide B11-23 (solid circles) or B18-30 (open circles)  (B)  or increasing concentrations of proinsulin (solid circles) or insulin (open circles)  (C) . The difference in fluorescence emission intensity at 500 nm was plotted as a function of the concentrations of peptides or proteins.,'],\n",
       " 273: ['1471-2091-11-44-3-5',\n",
       "  'GFIG',\n",
       "  'Quantitative analysis of proinsulin peptide binding to DnaK .  A : For DnaK-binding peptides of the proinsulin A-,\" B-chain or C- peptide the amount of DnaK released during the three electroelution cycles was determined in order to assess the progress of DnaK dissociation from the peptides. For this analysis the strength of the signals obtained from the high affinity DnaK ligand C3 after each elution cycle (cycle 1: 134 LU; cycle 2: 46 LU; cycle 3: 33 LU) was set as \"\"1\"\" and used as reference for the signal strengths obtained from the investigated peptides after the corresponding cycles of elution. For each peptide\", the resulting three data points (corresponding to the three elution cycles) were subjected to linear regression analysis and for better comparison of the progress of dissociation a line was plotted against the elution cycles. The bold line in the left panel indicates the release of the B-chain peptide 18. In a fluid phase competition assay a-NR (200 nM) and DnaK (1  Î¼ M) were incubated in the presence of increasing concentrations of peptide B11-23 (solid circles) or B18-30 (open circles)  (B)  or increasing concentrations of proinsulin (solid circles) or insulin (open circles)  (C) . The difference in fluorescence emission intensity at 500 nm was plotted as a function of the concentrations of peptides or proteins.,'],\n",
       " 274: ['1471-2091-12-1-3-1',\n",
       "  'GFIG',\n",
       "  'Comparison of the size-exclusion chromatography profiles of the wild-type and mutant BCCL2 proteins .  A . Purified wild-type (wt) and mutant BCCL2 proteins were loaded onto a gel-filtration column and eluted at a flow rate of 0.3 ml/min. The protein peaks 1 (P1) and 2 (P2) are indicated. The positions at which the globular proteins standards thyroglobulin (670 kD) and bovine gamma-globulin (158 kD) were eluted in a parallel run are indicated.  B . Fractions from the gel-filtration column were separated on a 12% SDS-polyacrylamide gel, which was stained with Coomassie Blue (bottom panel). The 25- and 37-kD molecular weight markers (M) are shown in the left-most lanes. An aliquot of the LLWL protein loaded on the gel-filtration column was also analyzed (Load). The positions of peaks P1 and P2 are noted.'],\n",
       " 275: ['1471-2091-12-1-3-2',\n",
       "  'GGEL',\n",
       "  'Comparison of the size-exclusion chromatography profiles of the wild-type and mutant BCCL2 proteins .  A . Purified wild-type (wt) and mutant BCCL2 proteins were loaded onto a gel-filtration column and eluted at a flow rate of 0.3 ml/min. The protein peaks 1 (P1) and 2 (P2) are indicated. The positions at which the globular proteins standards thyroglobulin (670 kD) and bovine gamma-globulin (158 kD) were eluted in a parallel run are indicated.  B . Fractions from the gel-filtration column were separated on a 12% SDS-polyacrylamide gel, which was stained with Coomassie Blue (bottom panel). The 25- and 37-kD molecular weight markers (M) are shown in the left-most lanes. An aliquot of the LLWL protein loaded on the gel-filtration column was also analyzed (Load). The positions of peaks P1 and P2 are noted.'],\n",
       " 276: ['1471-2091-4-6-1-1',\n",
       "  'GFIG',\n",
       "  'Purification of p48  The p48-containing fraction was purified as described in Materials and Methods. A, Q-Sepharose column chromatography: 35 ï¿½ï¿½ 80 % ammonium sulfate fraction was applied to a Q-Sepharose column (2.6    10 cm). Proteins were eluted with a linear gradient of NaCl and the 0.3 ï¿½ï¿½ 0.5 M eluate was collected in 10 ml fractions. B, SP-Sepharose column chromatography: p48-containing fractions from a Q-Sepharose column were pooled and applied to a SP-Sepharose column (1.6    10 cm). Proteins were eluted with a linear gradient of NaCl and the 0 ï¿½ï¿½ 0.25 M eluate was collected in 10 ml fractions. Fractions were assessed by immunoblotting using anti- Xenopus  20S proteasome. Protein bands of p48 and subunits of 20S proteasome are indicated by arrows and square brackets, respectively. Proteasome activity of the fractions toward a fluorogenic peptide substrate (Suc-Leu-Leu-Val-Tyr-MCA) was determined in the presence (  ) and absence (  ) of 0.05% SDS as described (21). Elution profile was monitored by absorbance at 280 nm ( ï¿½ï¿½ ).'],\n",
       " 277: ['1471-2091-4-6-1-2',\n",
       "  'GFIG',\n",
       "  'Purification of p48  The p48-containing fraction was purified as described in Materials and Methods. A, Q-Sepharose column chromatography: 35 ï¿½ï¿½ 80 % ammonium sulfate fraction was applied to a Q-Sepharose column (2.6    10 cm). Proteins were eluted with a linear gradient of NaCl and the 0.3 ï¿½ï¿½ 0.5 M eluate was collected in 10 ml fractions. B, SP-Sepharose column chromatography: p48-containing fractions from a Q-Sepharose column were pooled and applied to a SP-Sepharose column (1.6    10 cm). Proteins were eluted with a linear gradient of NaCl and the 0 ï¿½ï¿½ 0.25 M eluate was collected in 10 ml fractions. Fractions were assessed by immunoblotting using anti- Xenopus  20S proteasome. Protein bands of p48 and subunits of 20S proteasome are indicated by arrows and square brackets, respectively. Proteasome activity of the fractions toward a fluorogenic peptide substrate (Suc-Leu-Leu-Val-Tyr-MCA) was determined in the presence (  ) and absence (  ) of 0.05% SDS as described (21). Elution profile was monitored by absorbance at 280 nm ( ï¿½ï¿½ ).'],\n",
       " 278: ['1471-2091-4-6-1-3',\n",
       "  'GGEL',\n",
       "  'Purification of p48  The p48-containing fraction was purified as described in Materials and Methods. A, Q-Sepharose column chromatography: 35 ï¿½ï¿½ 80 % ammonium sulfate fraction was applied to a Q-Sepharose column (2.6    10 cm). Proteins were eluted with a linear gradient of NaCl and the 0.3 ï¿½ï¿½ 0.5 M eluate was collected in 10 ml fractions. B, SP-Sepharose column chromatography: p48-containing fractions from a Q-Sepharose column were pooled and applied to a SP-Sepharose column (1.6    10 cm). Proteins were eluted with a linear gradient of NaCl and the 0 ï¿½ï¿½ 0.25 M eluate was collected in 10 ml fractions. Fractions were assessed by immunoblotting using anti- Xenopus  20S proteasome. Protein bands of p48 and subunits of 20S proteasome are indicated by arrows and square brackets, respectively. Proteasome activity of the fractions toward a fluorogenic peptide substrate (Suc-Leu-Leu-Val-Tyr-MCA) was determined in the presence (  ) and absence (  ) of 0.05% SDS as described (21). Elution profile was monitored by absorbance at 280 nm ( ï¿½ï¿½ ).'],\n",
       " 279: ['1471-2091-4-6-1-4',\n",
       "  'GGEL',\n",
       "  'Purification of p48  The p48-containing fraction was purified as described in Materials and Methods. A, Q-Sepharose column chromatography: 35 ï¿½ï¿½ 80 % ammonium sulfate fraction was applied to a Q-Sepharose column (2.6    10 cm). Proteins were eluted with a linear gradient of NaCl and the 0.3 ï¿½ï¿½ 0.5 M eluate was collected in 10 ml fractions. B, SP-Sepharose column chromatography: p48-containing fractions from a Q-Sepharose column were pooled and applied to a SP-Sepharose column (1.6    10 cm). Proteins were eluted with a linear gradient of NaCl and the 0 ï¿½ï¿½ 0.25 M eluate was collected in 10 ml fractions. Fractions were assessed by immunoblotting using anti- Xenopus  20S proteasome. Protein bands of p48 and subunits of 20S proteasome are indicated by arrows and square brackets, respectively. Proteasome activity of the fractions toward a fluorogenic peptide substrate (Suc-Leu-Leu-Val-Tyr-MCA) was determined in the presence (  ) and absence (  ) of 0.05% SDS as described (21). Elution profile was monitored by absorbance at 280 nm ( ï¿½ï¿½ ).'],\n",
       " 280: ['1471-2091-4-6-5-1',\n",
       "  'GGEL',\n",
       "  'Phosphorylation of EF-1 Î³  bound to the 26S proteasome  (A) Phosphorylation of p48 complex and the 26S proteasome by MPF. Purified p48 complex and the 26S proteasome from immature oocytes were treated with or without MPF as indicated in the presence of [ Î³ - 32 P]-ATP. After incubation at 30 Â° C for 1 hour the reaction was stopped by addition of SDS-PAGE sample buffer.  32 P-labeled proteins were resolved by SDS-PAGE followed by autoradiography on Imaging plates (Fuji Film). (B) Immunoprecipitation of p48 complex by anti-EF-1 complex. The 26S proteasome fractions were treated with affinity purified anti-EF-1 complex (Anti) or control IgG (Cont) and EF-1 components were immunoprecipitated as described previously [ 37 ]. Supernatants (S) and precipitates (P) were resolved by SDS-PAGE followed by autoradiography. Arrows indicate  32 P-labeled 48, 37 and 30 kDa protein bands. Image analysis was performed using a Molecular Imager FX (BioRad).'],\n",
       " 281: ['1471-2091-4-6-5-2',\n",
       "  'GGEL',\n",
       "  'Phosphorylation of EF-1 Î³  bound to the 26S proteasome  (A) Phosphorylation of p48 complex and the 26S proteasome by MPF. Purified p48 complex and the 26S proteasome from immature oocytes were treated with or without MPF as indicated in the presence of [ Î³ - 32 P]-ATP. After incubation at 30 Â° C for 1 hour the reaction was stopped by addition of SDS-PAGE sample buffer.  32 P-labeled proteins were resolved by SDS-PAGE followed by autoradiography on Imaging plates (Fuji Film). (B) Immunoprecipitation of p48 complex by anti-EF-1 complex. The 26S proteasome fractions were treated with affinity purified anti-EF-1 complex (Anti) or control IgG (Cont) and EF-1 components were immunoprecipitated as described previously [ 37 ]. Supernatants (S) and precipitates (P) were resolved by SDS-PAGE followed by autoradiography. Arrows indicate  32 P-labeled 48, 37 and 30 kDa protein bands. Image analysis was performed using a Molecular Imager FX (BioRad).'],\n",
       " 282: ['1471-2091-5-12-1-1',\n",
       "  'GGEL',\n",
       "  'Expression of HA-S1P 4 -G Î± i1  and HA-S1P 4 (E 122 Q)-G Î± i1 (C 351 I) in CHO-K1 cells.  Membranes from untransfected CHO-K1 cells (lane 2) and CHO-K1 cells stably expressing HA-S1P 4 -G Î± i1 (C 351 I) (A) or HA-S1P 4 (E 122 Q)-G Î± i1 (C 351 I) (B) were analysed by Western blotting using anti-HA (panel I) or anti-G Î± i1  (panel II) antibodies. Visualisation of immunoreactive proteins was achieved using chemiluminescence after incubation of the blot with appropriate HRP-conjugated secondary antibodies. The position of each HA-S1P 4  fusion protein is indicated by an arrow. Cell-surface expressed HA-S1P 4  receptor was detected by FACS analysis (panel III) using a Fluorescein conjugate of the anti-HA antibody (blue line). Cells were also stained with an isotype matched control antibody (red line). No staining of untransfected CHO-K1 cells was observed using the Fluorescein conjugate of the anti-HA antibody (not shown). Data are presented as overlay histograms and are representative of at least five independent experiments.,'],\n",
       " 283: ['1471-2091-5-12-1-2',\n",
       "  'GGEL',\n",
       "  'Expression of HA-S1P 4 -G Î± i1  and HA-S1P 4 (E 122 Q)-G Î± i1 (C 351 I) in CHO-K1 cells.  Membranes from untransfected CHO-K1 cells (lane 2) and CHO-K1 cells stably expressing HA-S1P 4 -G Î± i1 (C 351 I) (A) or HA-S1P 4 (E 122 Q)-G Î± i1 (C 351 I) (B) were analysed by Western blotting using anti-HA (panel I) or anti-G Î± i1  (panel II) antibodies. Visualisation of immunoreactive proteins was achieved using chemiluminescence after incubation of the blot with appropriate HRP-conjugated secondary antibodies. The position of each HA-S1P 4  fusion protein is indicated by an arrow. Cell-surface expressed HA-S1P 4  receptor was detected by FACS analysis (panel III) using a Fluorescein conjugate of the anti-HA antibody (blue line). Cells were also stained with an isotype matched control antibody (red line). No staining of untransfected CHO-K1 cells was observed using the Fluorescein conjugate of the anti-HA antibody (not shown). Data are presented as overlay histograms and are representative of at least five independent experiments.,'],\n",
       " 284: ['1471-2091-5-12-1-3',\n",
       "  'GFIG',\n",
       "  'Expression of HA-S1P 4 -G Î± i1  and HA-S1P 4 (E 122 Q)-G Î± i1 (C 351 I) in CHO-K1 cells.  Membranes from untransfected CHO-K1 cells (lane 2) and CHO-K1 cells stably expressing HA-S1P 4 -G Î± i1 (C 351 I) (A) or HA-S1P 4 (E 122 Q)-G Î± i1 (C 351 I) (B) were analysed by Western blotting using anti-HA (panel I) or anti-G Î± i1  (panel II) antibodies. Visualisation of immunoreactive proteins was achieved using chemiluminescence after incubation of the blot with appropriate HRP-conjugated secondary antibodies. The position of each HA-S1P 4  fusion protein is indicated by an arrow. Cell-surface expressed HA-S1P 4  receptor was detected by FACS analysis (panel III) using a Fluorescein conjugate of the anti-HA antibody (blue line). Cells were also stained with an isotype matched control antibody (red line). No staining of untransfected CHO-K1 cells was observed using the Fluorescein conjugate of the anti-HA antibody (not shown). Data are presented as overlay histograms and are representative of at least five independent experiments.,'],\n",
       " 285: ['1471-2091-5-12-1-4',\n",
       "  'GGEL',\n",
       "  'Expression of HA-S1P 4 -G Î± i1  and HA-S1P 4 (E 122 Q)-G Î± i1 (C 351 I) in CHO-K1 cells.  Membranes from untransfected CHO-K1 cells (lane 2) and CHO-K1 cells stably expressing HA-S1P 4 -G Î± i1 (C 351 I) (A) or HA-S1P 4 (E 122 Q)-G Î± i1 (C 351 I) (B) were analysed by Western blotting using anti-HA (panel I) or anti-G Î± i1  (panel II) antibodies. Visualisation of immunoreactive proteins was achieved using chemiluminescence after incubation of the blot with appropriate HRP-conjugated secondary antibodies. The position of each HA-S1P 4  fusion protein is indicated by an arrow. Cell-surface expressed HA-S1P 4  receptor was detected by FACS analysis (panel III) using a Fluorescein conjugate of the anti-HA antibody (blue line). Cells were also stained with an isotype matched control antibody (red line). No staining of untransfected CHO-K1 cells was observed using the Fluorescein conjugate of the anti-HA antibody (not shown). Data are presented as overlay histograms and are representative of at least five independent experiments.,'],\n",
       " 286: ['1471-2091-5-12-1-5',\n",
       "  'GGEL',\n",
       "  'Expression of HA-S1P 4 -G Î± i1  and HA-S1P 4 (E 122 Q)-G Î± i1 (C 351 I) in CHO-K1 cells.  Membranes from untransfected CHO-K1 cells (lane 2) and CHO-K1 cells stably expressing HA-S1P 4 -G Î± i1 (C 351 I) (A) or HA-S1P 4 (E 122 Q)-G Î± i1 (C 351 I) (B) were analysed by Western blotting using anti-HA (panel I) or anti-G Î± i1  (panel II) antibodies. Visualisation of immunoreactive proteins was achieved using chemiluminescence after incubation of the blot with appropriate HRP-conjugated secondary antibodies. The position of each HA-S1P 4  fusion protein is indicated by an arrow. Cell-surface expressed HA-S1P 4  receptor was detected by FACS analysis (panel III) using a Fluorescein conjugate of the anti-HA antibody (blue line). Cells were also stained with an isotype matched control antibody (red line). No staining of untransfected CHO-K1 cells was observed using the Fluorescein conjugate of the anti-HA antibody (not shown). Data are presented as overlay histograms and are representative of at least five independent experiments.,'],\n",
       " 287: ['1471-2091-5-12-1-6',\n",
       "  'GFIG',\n",
       "  'Expression of HA-S1P 4 -G Î± i1  and HA-S1P 4 (E 122 Q)-G Î± i1 (C 351 I) in CHO-K1 cells.  Membranes from untransfected CHO-K1 cells (lane 2) and CHO-K1 cells stably expressing HA-S1P 4 -G Î± i1 (C 351 I) (A) or HA-S1P 4 (E 122 Q)-G Î± i1 (C 351 I) (B) were analysed by Western blotting using anti-HA (panel I) or anti-G Î± i1  (panel II) antibodies. Visualisation of immunoreactive proteins was achieved using chemiluminescence after incubation of the blot with appropriate HRP-conjugated secondary antibodies. The position of each HA-S1P 4  fusion protein is indicated by an arrow. Cell-surface expressed HA-S1P 4  receptor was detected by FACS analysis (panel III) using a Fluorescein conjugate of the anti-HA antibody (blue line). Cells were also stained with an isotype matched control antibody (red line). No staining of untransfected CHO-K1 cells was observed using the Fluorescein conjugate of the anti-HA antibody (not shown). Data are presented as overlay histograms and are representative of at least five independent experiments.,'],\n",
       " 288: ['1471-2091-5-12-4-1',\n",
       "  'D3DR',\n",
       "  'Computational models of wild type S1P 4  and its E3.29(122)Q mutant with S1P and LPA species.  Computational models of the complexes between the wild type S1P 4  or its E 3.29(122) Q mutant with S1P or various LPA species generated by Autodock 3.0 and minimised using the MMFF94 forcefield in the MOE program. Complexes in each panel are shown from the same viewpoint with the extracellular end of the receptors oriented to the top of the figure. Standard element color codes are used with grey, white red, blue and magenta representing carbon, hydrogen, oxygen, nitrogen and phosphorous. Ribbons are shaded from red at the amino-terminus to blue at the carboxy-terminus. (A) Model of the complex between S1P (spacefilling) and the wild type S1P 4  receptor. Residues in the receptor involved in ion pairs with S1P are shown as stick models and labelled. (B) Superimposition of the wild type S1P 4  complex with S1P (orange) and the E 3.29(122) Q S1P 4  mutant complex with 14:0 LPA (green). For clarity, the only position at which the modelled amino acid position is shown for both receptor models is 3.29(122). Other residues had very similar optimised positions in the two model structures. (C) Superimposition of wild type S1P 4  complexes with 18:1 LPA (cyan), 16:0 LPA (yellow) and 14:0 LPA (green) on E 3.29(122) Q mutant complexes with 18:1 LPA (blue-green), 16:0 LPA (gold) and S1P (orange). For clarity, the only position at which modelled amino acid position is shown for both the wild type and mutant receptor models is 3.29(122). Other residues had very similar optimised positions in all model structures. (D) Space-filling models which represent the minimised extended conformation of each structure were constructed using SYBYL 6.9 software (Tripos Inc., St. Louis, MO., U.S.A.). The distance between phosphorus and terminal carbon atoms was predicted for each structure listed from top to bottom: 18:1 LPA, 27.0   ; 16:0 LPA, 26.7   ; 14:0 LPA, 24.2   ; S1P, 24.0   .,'],\n",
       " 289: ['1471-2091-5-12-4-2',\n",
       "  'D3DR',\n",
       "  'Computational models of wild type S1P 4  and its E3.29(122)Q mutant with S1P and LPA species.  Computational models of the complexes between the wild type S1P 4  or its E 3.29(122) Q mutant with S1P or various LPA species generated by Autodock 3.0 and minimised using the MMFF94 forcefield in the MOE program. Complexes in each panel are shown from the same viewpoint with the extracellular end of the receptors oriented to the top of the figure. Standard element color codes are used with grey, white red, blue and magenta representing carbon, hydrogen, oxygen, nitrogen and phosphorous. Ribbons are shaded from red at the amino-terminus to blue at the carboxy-terminus. (A) Model of the complex between S1P (spacefilling) and the wild type S1P 4  receptor. Residues in the receptor involved in ion pairs with S1P are shown as stick models and labelled. (B) Superimposition of the wild type S1P 4  complex with S1P (orange) and the E 3.29(122) Q S1P 4  mutant complex with 14:0 LPA (green). For clarity, the only position at which the modelled amino acid position is shown for both receptor models is 3.29(122). Other residues had very similar optimised positions in the two model structures. (C) Superimposition of wild type S1P 4  complexes with 18:1 LPA (cyan), 16:0 LPA (yellow) and 14:0 LPA (green) on E 3.29(122) Q mutant complexes with 18:1 LPA (blue-green), 16:0 LPA (gold) and S1P (orange). For clarity, the only position at which modelled amino acid position is shown for both the wild type and mutant receptor models is 3.29(122). Other residues had very similar optimised positions in all model structures. (D) Space-filling models which represent the minimised extended conformation of each structure were constructed using SYBYL 6.9 software (Tripos Inc., St. Louis, MO., U.S.A.). The distance between phosphorus and terminal carbon atoms was predicted for each structure listed from top to bottom: 18:1 LPA, 27.0   ; 16:0 LPA, 26.7   ; 14:0 LPA, 24.2   ; S1P, 24.0   .,'],\n",
       " 290: ['1471-2091-5-12-4-3',\n",
       "  'D3DR',\n",
       "  'Computational models of wild type S1P 4  and its E3.29(122)Q mutant with S1P and LPA species.  Computational models of the complexes between the wild type S1P 4  or its E 3.29(122) Q mutant with S1P or various LPA species generated by Autodock 3.0 and minimised using the MMFF94 forcefield in the MOE program. Complexes in each panel are shown from the same viewpoint with the extracellular end of the receptors oriented to the top of the figure. Standard element color codes are used with grey, white red, blue and magenta representing carbon, hydrogen, oxygen, nitrogen and phosphorous. Ribbons are shaded from red at the amino-terminus to blue at the carboxy-terminus. (A) Model of the complex between S1P (spacefilling) and the wild type S1P 4  receptor. Residues in the receptor involved in ion pairs with S1P are shown as stick models and labelled. (B) Superimposition of the wild type S1P 4  complex with S1P (orange) and the E 3.29(122) Q S1P 4  mutant complex with 14:0 LPA (green). For clarity, the only position at which the modelled amino acid position is shown for both receptor models is 3.29(122). Other residues had very similar optimised positions in the two model structures. (C) Superimposition of wild type S1P 4  complexes with 18:1 LPA (cyan), 16:0 LPA (yellow) and 14:0 LPA (green) on E 3.29(122) Q mutant complexes with 18:1 LPA (blue-green), 16:0 LPA (gold) and S1P (orange). For clarity, the only position at which modelled amino acid position is shown for both the wild type and mutant receptor models is 3.29(122). Other residues had very similar optimised positions in all model structures. (D) Space-filling models which represent the minimised extended conformation of each structure were constructed using SYBYL 6.9 software (Tripos Inc., St. Louis, MO., U.S.A.). The distance between phosphorus and terminal carbon atoms was predicted for each structure listed from top to bottom: 18:1 LPA, 27.0   ; 16:0 LPA, 26.7   ; 14:0 LPA, 24.2   ; S1P, 24.0   .,'],\n",
       " 291: ['1471-2091-5-12-4-4',\n",
       "  'D3DR',\n",
       "  'Computational models of wild type S1P 4  and its E3.29(122)Q mutant with S1P and LPA species.  Computational models of the complexes between the wild type S1P 4  or its E 3.29(122) Q mutant with S1P or various LPA species generated by Autodock 3.0 and minimised using the MMFF94 forcefield in the MOE program. Complexes in each panel are shown from the same viewpoint with the extracellular end of the receptors oriented to the top of the figure. Standard element color codes are used with grey, white red, blue and magenta representing carbon, hydrogen, oxygen, nitrogen and phosphorous. Ribbons are shaded from red at the amino-terminus to blue at the carboxy-terminus. (A) Model of the complex between S1P (spacefilling) and the wild type S1P 4  receptor. Residues in the receptor involved in ion pairs with S1P are shown as stick models and labelled. (B) Superimposition of the wild type S1P 4  complex with S1P (orange) and the E 3.29(122) Q S1P 4  mutant complex with 14:0 LPA (green). For clarity, the only position at which the modelled amino acid position is shown for both receptor models is 3.29(122). Other residues had very similar optimised positions in the two model structures. (C) Superimposition of wild type S1P 4  complexes with 18:1 LPA (cyan), 16:0 LPA (yellow) and 14:0 LPA (green) on E 3.29(122) Q mutant complexes with 18:1 LPA (blue-green), 16:0 LPA (gold) and S1P (orange). For clarity, the only position at which modelled amino acid position is shown for both the wild type and mutant receptor models is 3.29(122). Other residues had very similar optimised positions in all model structures. (D) Space-filling models which represent the minimised extended conformation of each structure were constructed using SYBYL 6.9 software (Tripos Inc., St. Louis, MO., U.S.A.). The distance between phosphorus and terminal carbon atoms was predicted for each structure listed from top to bottom: 18:1 LPA, 27.0   ; 16:0 LPA, 26.7   ; 14:0 LPA, 24.2   ; S1P, 24.0   .,'],\n",
       " 292: ['1471-2091-7-13-5-1',\n",
       "  'GGEL',\n",
       "  'hARD2 interacts with NATH and HIF-1 Î± . (A)HEK293 cells, transfected with plasmids encoding hARD2-V5 or lacZ-V5 as a negative control, were harvested and the lysates were immunoprecipitated (IP) with the anti-V5 antibody. The immunoprecipitates were analysed by SDS-PAGE and Western Blotting. The membrane was incubated with anti-NATH. The amount of lysate loaded on the gel represents approx. 10 % of the material used in the immunoprecipitation reaction. (B) MCF-7 cells were cotransfected with plasmids encoding HA-HIF-1 Î±  and X-press-hARD2 or X-press-lacZ as a negative control. After 48 hours the cells were collected and processed as (A). The membrane was incubated with anti-HA.,'],\n",
       " 293: ['1471-2091-7-13-5-2',\n",
       "  'GGEL',\n",
       "  'hARD2 interacts with NATH and HIF-1 Î± . (A)HEK293 cells, transfected with plasmids encoding hARD2-V5 or lacZ-V5 as a negative control, were harvested and the lysates were immunoprecipitated (IP) with the anti-V5 antibody. The immunoprecipitates were analysed by SDS-PAGE and Western Blotting. The membrane was incubated with anti-NATH. The amount of lysate loaded on the gel represents approx. 10 % of the material used in the immunoprecipitation reaction. (B) MCF-7 cells were cotransfected with plasmids encoding HA-HIF-1 Î±  and X-press-hARD2 or X-press-lacZ as a negative control. After 48 hours the cells were collected and processed as (A). The membrane was incubated with anti-HA.,'],\n",
       " 294: ['1471-2091-7-24-1-1',\n",
       "  'GGEL',\n",
       "  'Expression, purification and crystallization of BnDGAT1 (1 ï¿½ï¿½ 116) His 6 . Panel A: Induction of BnDGAT1 (1 ï¿½ï¿½ 116) His 6  expression in bacteria monitored by SDS-PAGE. From left to right: molecular mass markers (lane 1), clarified lysate from non-induced bacteria (lane 2), clarified lysate from two independent incubations of bacteria treated with IPTG (lanes 3 and 4), pellet obtained from clarified lysate of bacteria induced with IPTG (lane 5). Experiments to demonstrate induction of BnDGAT1 (1 ï¿½ï¿½ 116) His 6  synthesis were conducted using an incubation medium of 5 ml (instead of the 500 mL used for preparative work). Harvested  E. coli  were resuspended in 100 mM phosphate buffer (pH 7.4) containing 150 mM NaCl. The suspension was passed through a French pressure cell at 20,000 p.s.i. three times and the ruptured bacteria were centrifuged at 13,000  g  for 15 min to obtain a clarified lysate. The 13,000  g  pellet obtained from ruptured bacteria that had been induced with IPTG was resuspended in extraction buffer. Equal volumes of each sample were applied to the SDS gel. Panel B: Analysis by SDS-PAGE of BnDGAT1 (1 ï¿½ï¿½ 116) His 6  purified by immobilized nickel ion chromatography. From left to right: molecular mass markers (lane 1), pass-through fraction (lane 2), fraction eluted with equilibration buffer (lane 3), and fractions eluted with 4 ml portions of equilibration buffer containing 40, 60 and 80 mM imidazole, respectively (lanes 4 ï¿½ï¿½ 6). The protein load for lane 4 was about 10  Î¼ g. Ten microliter aliquots of each of the eluents were applied to the gel. Panel C: MALDI-TOF spectrum of BnDGAT1 (1 ï¿½ï¿½ 116) His 6 . The spectrum was acquired using a Voyager DE-Pro TOF MS (Applied Biosystems, Foster, CA, USA) using a 2-layer spotting technique. Matrix layer 1 was approximately 15 mg/mL  Î± -cyano-4-hydroxycinnamic acid dissolved in 1:2 (v/v) methanol:acetone. Matrix layer 2 was saturated  Î± -cyano-4-hydroxycinnamic acid in 1:2 (v/v) methanol/water. One microliter of layer 1 solution was spotted and allowed to dry. Protein sample (450  Î¼ g/mL) was mixed with layer 2 solution in a ratio between 1:1 and 1:4 (sample:matrix). One microliter of layer 2 solution was spotted on the dried layer 1 spot, and allowed to dry. The dried spot was washed 2 ï¿½ï¿½ 3 times with a drop of cold distilled water to remove salts prior to analysis by MALDI-TOF mass spectrometry. Panel D: Microcrystals of BnDGAT1 (1 ï¿½ï¿½ 116) His 6 . To begin preparation of microcrystals, a 12 mg/mL solution of BnDGAT1 (1 ï¿½ï¿½ 116) His 6  was dialyzed against 10 mM glycine-NaOH (pH 9.0) containing 20 mM NaCl and 1 mM EDTA. Initial screening for crystallization conditions using vapor diffusion and hanging drops resulted in several instances of microcrystals, which grew as plates, within 1 ï¿½ï¿½ 2 weeks. Microcrystals were obtained from ammonium sulfate and ammonium phosphate solutions over a wide pH range.'],\n",
       " 295: ['1471-2091-7-24-1-2',\n",
       "  'GGEL',\n",
       "  'Expression, purification and crystallization of BnDGAT1 (1 ï¿½ï¿½ 116) His 6 . Panel A: Induction of BnDGAT1 (1 ï¿½ï¿½ 116) His 6  expression in bacteria monitored by SDS-PAGE. From left to right: molecular mass markers (lane 1), clarified lysate from non-induced bacteria (lane 2), clarified lysate from two independent incubations of bacteria treated with IPTG (lanes 3 and 4), pellet obtained from clarified lysate of bacteria induced with IPTG (lane 5). Experiments to demonstrate induction of BnDGAT1 (1 ï¿½ï¿½ 116) His 6  synthesis were conducted using an incubation medium of 5 ml (instead of the 500 mL used for preparative work). Harvested  E. coli  were resuspended in 100 mM phosphate buffer (pH 7.4) containing 150 mM NaCl. The suspension was passed through a French pressure cell at 20,000 p.s.i. three times and the ruptured bacteria were centrifuged at 13,000  g  for 15 min to obtain a clarified lysate. The 13,000  g  pellet obtained from ruptured bacteria that had been induced with IPTG was resuspended in extraction buffer. Equal volumes of each sample were applied to the SDS gel. Panel B: Analysis by SDS-PAGE of BnDGAT1 (1 ï¿½ï¿½ 116) His 6  purified by immobilized nickel ion chromatography. From left to right: molecular mass markers (lane 1), pass-through fraction (lane 2), fraction eluted with equilibration buffer (lane 3), and fractions eluted with 4 ml portions of equilibration buffer containing 40, 60 and 80 mM imidazole, respectively (lanes 4 ï¿½ï¿½ 6). The protein load for lane 4 was about 10  Î¼ g. Ten microliter aliquots of each of the eluents were applied to the gel. Panel C: MALDI-TOF spectrum of BnDGAT1 (1 ï¿½ï¿½ 116) His 6 . The spectrum was acquired using a Voyager DE-Pro TOF MS (Applied Biosystems, Foster, CA, USA) using a 2-layer spotting technique. Matrix layer 1 was approximately 15 mg/mL  Î± -cyano-4-hydroxycinnamic acid dissolved in 1:2 (v/v) methanol:acetone. Matrix layer 2 was saturated  Î± -cyano-4-hydroxycinnamic acid in 1:2 (v/v) methanol/water. One microliter of layer 1 solution was spotted and allowed to dry. Protein sample (450  Î¼ g/mL) was mixed with layer 2 solution in a ratio between 1:1 and 1:4 (sample:matrix). One microliter of layer 2 solution was spotted on the dried layer 1 spot, and allowed to dry. The dried spot was washed 2 ï¿½ï¿½ 3 times with a drop of cold distilled water to remove salts prior to analysis by MALDI-TOF mass spectrometry. Panel D: Microcrystals of BnDGAT1 (1 ï¿½ï¿½ 116) His 6 . To begin preparation of microcrystals, a 12 mg/mL solution of BnDGAT1 (1 ï¿½ï¿½ 116) His 6  was dialyzed against 10 mM glycine-NaOH (pH 9.0) containing 20 mM NaCl and 1 mM EDTA. Initial screening for crystallization conditions using vapor diffusion and hanging drops resulted in several instances of microcrystals, which grew as plates, within 1 ï¿½ï¿½ 2 weeks. Microcrystals were obtained from ammonium sulfate and ammonium phosphate solutions over a wide pH range.'],\n",
       " 296: ['1471-2091-7-24-1-3',\n",
       "  'GFIG',\n",
       "  'Expression, purification and crystallization of BnDGAT1 (1 ï¿½ï¿½ 116) His 6 . Panel A: Induction of BnDGAT1 (1 ï¿½ï¿½ 116) His 6  expression in bacteria monitored by SDS-PAGE. From left to right: molecular mass markers (lane 1), clarified lysate from non-induced bacteria (lane 2), clarified lysate from two independent incubations of bacteria treated with IPTG (lanes 3 and 4), pellet obtained from clarified lysate of bacteria induced with IPTG (lane 5). Experiments to demonstrate induction of BnDGAT1 (1 ï¿½ï¿½ 116) His 6  synthesis were conducted using an incubation medium of 5 ml (instead of the 500 mL used for preparative work). Harvested  E. coli  were resuspended in 100 mM phosphate buffer (pH 7.4) containing 150 mM NaCl. The suspension was passed through a French pressure cell at 20,000 p.s.i. three times and the ruptured bacteria were centrifuged at 13,000  g  for 15 min to obtain a clarified lysate. The 13,000  g  pellet obtained from ruptured bacteria that had been induced with IPTG was resuspended in extraction buffer. Equal volumes of each sample were applied to the SDS gel. Panel B: Analysis by SDS-PAGE of BnDGAT1 (1 ï¿½ï¿½ 116) His 6  purified by immobilized nickel ion chromatography. From left to right: molecular mass markers (lane 1), pass-through fraction (lane 2), fraction eluted with equilibration buffer (lane 3), and fractions eluted with 4 ml portions of equilibration buffer containing 40, 60 and 80 mM imidazole, respectively (lanes 4 ï¿½ï¿½ 6). The protein load for lane 4 was about 10  Î¼ g. Ten microliter aliquots of each of the eluents were applied to the gel. Panel C: MALDI-TOF spectrum of BnDGAT1 (1 ï¿½ï¿½ 116) His 6 . The spectrum was acquired using a Voyager DE-Pro TOF MS (Applied Biosystems, Foster, CA, USA) using a 2-layer spotting technique. Matrix layer 1 was approximately 15 mg/mL  Î± -cyano-4-hydroxycinnamic acid dissolved in 1:2 (v/v) methanol:acetone. Matrix layer 2 was saturated  Î± -cyano-4-hydroxycinnamic acid in 1:2 (v/v) methanol/water. One microliter of layer 1 solution was spotted and allowed to dry. Protein sample (450  Î¼ g/mL) was mixed with layer 2 solution in a ratio between 1:1 and 1:4 (sample:matrix). One microliter of layer 2 solution was spotted on the dried layer 1 spot, and allowed to dry. The dried spot was washed 2 ï¿½ï¿½ 3 times with a drop of cold distilled water to remove salts prior to analysis by MALDI-TOF mass spectrometry. Panel D: Microcrystals of BnDGAT1 (1 ï¿½ï¿½ 116) His 6 . To begin preparation of microcrystals, a 12 mg/mL solution of BnDGAT1 (1 ï¿½ï¿½ 116) His 6  was dialyzed against 10 mM glycine-NaOH (pH 9.0) containing 20 mM NaCl and 1 mM EDTA. Initial screening for crystallization conditions using vapor diffusion and hanging drops resulted in several instances of microcrystals, which grew as plates, within 1 ï¿½ï¿½ 2 weeks. Microcrystals were obtained from ammonium sulfate and ammonium phosphate solutions over a wide pH range.'],\n",
       " 297: ['1471-2091-7-24-1-4',\n",
       "  'DMEL',\n",
       "  'Expression, purification and crystallization of BnDGAT1 (1 ï¿½ï¿½ 116) His 6 . Panel A: Induction of BnDGAT1 (1 ï¿½ï¿½ 116) His 6  expression in bacteria monitored by SDS-PAGE. From left to right: molecular mass markers (lane 1), clarified lysate from non-induced bacteria (lane 2), clarified lysate from two independent incubations of bacteria treated with IPTG (lanes 3 and 4), pellet obtained from clarified lysate of bacteria induced with IPTG (lane 5). Experiments to demonstrate induction of BnDGAT1 (1 ï¿½ï¿½ 116) His 6  synthesis were conducted using an incubation medium of 5 ml (instead of the 500 mL used for preparative work). Harvested  E. coli  were resuspended in 100 mM phosphate buffer (pH 7.4) containing 150 mM NaCl. The suspension was passed through a French pressure cell at 20,000 p.s.i. three times and the ruptured bacteria were centrifuged at 13,000  g  for 15 min to obtain a clarified lysate. The 13,000  g  pellet obtained from ruptured bacteria that had been induced with IPTG was resuspended in extraction buffer. Equal volumes of each sample were applied to the SDS gel. Panel B: Analysis by SDS-PAGE of BnDGAT1 (1 ï¿½ï¿½ 116) His 6  purified by immobilized nickel ion chromatography. From left to right: molecular mass markers (lane 1), pass-through fraction (lane 2), fraction eluted with equilibration buffer (lane 3), and fractions eluted with 4 ml portions of equilibration buffer containing 40, 60 and 80 mM imidazole, respectively (lanes 4 ï¿½ï¿½ 6). The protein load for lane 4 was about 10  Î¼ g. Ten microliter aliquots of each of the eluents were applied to the gel. Panel C: MALDI-TOF spectrum of BnDGAT1 (1 ï¿½ï¿½ 116) His 6 . The spectrum was acquired using a Voyager DE-Pro TOF MS (Applied Biosystems, Foster, CA, USA) using a 2-layer spotting technique. Matrix layer 1 was approximately 15 mg/mL  Î± -cyano-4-hydroxycinnamic acid dissolved in 1:2 (v/v) methanol:acetone. Matrix layer 2 was saturated  Î± -cyano-4-hydroxycinnamic acid in 1:2 (v/v) methanol/water. One microliter of layer 1 solution was spotted and allowed to dry. Protein sample (450  Î¼ g/mL) was mixed with layer 2 solution in a ratio between 1:1 and 1:4 (sample:matrix). One microliter of layer 2 solution was spotted on the dried layer 1 spot, and allowed to dry. The dried spot was washed 2 ï¿½ï¿½ 3 times with a drop of cold distilled water to remove salts prior to analysis by MALDI-TOF mass spectrometry. Panel D: Microcrystals of BnDGAT1 (1 ï¿½ï¿½ 116) His 6 . To begin preparation of microcrystals, a 12 mg/mL solution of BnDGAT1 (1 ï¿½ï¿½ 116) His 6  was dialyzed against 10 mM glycine-NaOH (pH 9.0) containing 20 mM NaCl and 1 mM EDTA. Initial screening for crystallization conditions using vapor diffusion and hanging drops resulted in several instances of microcrystals, which grew as plates, within 1 ï¿½ï¿½ 2 weeks. Microcrystals were obtained from ammonium sulfate and ammonium phosphate solutions over a wide pH range.'],\n",
       " 298: ['1471-2091-8-16-4-1',\n",
       "  'GGEL',\n",
       "  'Time course of the induction and decay of Spt7 forms .  A . CY1811 containing  YIplac211-GAL-Flag 3 -SPT7  was grown in minimal media containing 2% raffinose then diluted 1:5 into YP media containing 2% glucose (lane 2) or 2%galactose (lane 3) and grown for 30 minutes. Extracts were prepared by glass bead disruption and 100  Î¼ g separated by SDS-PAGE (5%). Flag 3 -Spt7 was detected by Western blotting with anti-Flag antibody. Lane one contains 100  Î¼ g of protein extract prepared after growth of BY4741 in galactose-containing media.  B . CY1811 containing  YIplac211-GAL  ï¿½ï¿½  Flag 3 -SPT7  was grown in raffinose. Expression of Spt7 was induced by the addition of galactose then inhibited by the addition of YP media containing 2% glucose. Cells were grown for 90 minutes which was empirically determined as time 0. Additional equal volume samples were taken at time 0,30,60, 120 and 240 minutes (lanes 2 ï¿½ï¿½ 7). Protein extracts were prepared and equal volumes separated by SDS-Page (5%). Flag 3 -Spt7 was detected by Western blotting (top panel) or stained with Coomassie Brilliant Blue (bottom panel).'],\n",
       " 299: ['1471-2091-8-16-4-2',\n",
       "  'GGEL',\n",
       "  'Time course of the induction and decay of Spt7 forms .  A . CY1811 containing  YIplac211-GAL-Flag 3 -SPT7  was grown in minimal media containing 2% raffinose then diluted 1:5 into YP media containing 2% glucose (lane 2) or 2%galactose (lane 3) and grown for 30 minutes. Extracts were prepared by glass bead disruption and 100  Î¼ g separated by SDS-PAGE (5%). Flag 3 -Spt7 was detected by Western blotting with anti-Flag antibody. Lane one contains 100  Î¼ g of protein extract prepared after growth of BY4741 in galactose-containing media.  B . CY1811 containing  YIplac211-GAL  ï¿½ï¿½  Flag 3 -SPT7  was grown in raffinose. Expression of Spt7 was induced by the addition of galactose then inhibited by the addition of YP media containing 2% glucose. Cells were grown for 90 minutes which was empirically determined as time 0. Additional equal volume samples were taken at time 0,30,60, 120 and 240 minutes (lanes 2 ï¿½ï¿½ 7). Protein extracts were prepared and equal volumes separated by SDS-Page (5%). Flag 3 -Spt7 was detected by Western blotting (top panel) or stained with Coomassie Brilliant Blue (bottom panel).'],\n",
       " 300: ['1471-2091-8-16-4-3',\n",
       "  'GGEL',\n",
       "  'Time course of the induction and decay of Spt7 forms .  A . CY1811 containing  YIplac211-GAL-Flag 3 -SPT7  was grown in minimal media containing 2% raffinose then diluted 1:5 into YP media containing 2% glucose (lane 2) or 2%galactose (lane 3) and grown for 30 minutes. Extracts were prepared by glass bead disruption and 100  Î¼ g separated by SDS-PAGE (5%). Flag 3 -Spt7 was detected by Western blotting with anti-Flag antibody. Lane one contains 100  Î¼ g of protein extract prepared after growth of BY4741 in galactose-containing media.  B . CY1811 containing  YIplac211-GAL  ï¿½ï¿½  Flag 3 -SPT7  was grown in raffinose. Expression of Spt7 was induced by the addition of galactose then inhibited by the addition of YP media containing 2% glucose. Cells were grown for 90 minutes which was empirically determined as time 0. Additional equal volume samples were taken at time 0,30,60, 120 and 240 minutes (lanes 2 ï¿½ï¿½ 7). Protein extracts were prepared and equal volumes separated by SDS-Page (5%). Flag 3 -Spt7 was detected by Western blotting (top panel) or stained with Coomassie Brilliant Blue (bottom panel).'],\n",
       " 301: ['1471-2091-9-17-5-1',\n",
       "  'GFIG',\n",
       "  \"Fluorescence titration of RNase(K41R) with PPL-L-alanine . Fluorescence was measured with 5  Î¼ M RNase(K41R) in 50 mM 4-methylmorpholine pH 7.5 at 25 Â° C. PPL-L-alanine concentrations were 10,12,15,20,25, 40 and 50  Î¼ M.  A , Fluorescence spectra;  B , nonlinear regression analysis gives a  K ' d  value of 23  Î¼ M. Bovine RNase A contains six tyrosine residues and no tryptophan residue [45].\"],\n",
       " 302: ['1471-2091-9-17-5-2',\n",
       "  'GFIG',\n",
       "  \"Fluorescence titration of RNase(K41R) with PPL-L-alanine . Fluorescence was measured with 5  Î¼ M RNase(K41R) in 50 mM 4-methylmorpholine pH 7.5 at 25 Â° C. PPL-L-alanine concentrations were 10,12,15,20,25, 40 and 50  Î¼ M.  A , Fluorescence spectra;  B , nonlinear regression analysis gives a  K ' d  value of 23  Î¼ M. Bovine RNase A contains six tyrosine residues and no tryptophan residue [45].\"],\n",
       " 303: ['1471-2091-9-18-5-1',\n",
       "  'DMFL',\n",
       "  'Visualization of compounds in living cells . Intracellular imaging of 8-[  -575]-cAMP (a-b) and of Pharos dye (c-d) in HEK293 (a,c) and in CHO (b,d) cells after 1 hour of treatment.,'],\n",
       " 304: ['1471-2091-9-18-5-2',\n",
       "  'DMFL',\n",
       "  'Visualization of compounds in living cells . Intracellular imaging of 8-[  -575]-cAMP (a-b) and of Pharos dye (c-d) in HEK293 (a,c) and in CHO (b,d) cells after 1 hour of treatment.,'],\n",
       " 305: ['1471-2091-9-18-5-3',\n",
       "  'DMFL',\n",
       "  'Visualization of compounds in living cells . Intracellular imaging of 8-[  -575]-cAMP (a-b) and of Pharos dye (c-d) in HEK293 (a,c) and in CHO (b,d) cells after 1 hour of treatment.,'],\n",
       " 306: ['1471-2091-9-18-5-4',\n",
       "  'DMFL',\n",
       "  'Visualization of compounds in living cells . Intracellular imaging of 8-[  -575]-cAMP (a-b) and of Pharos dye (c-d) in HEK293 (a,c) and in CHO (b,d) cells after 1 hour of treatment.,'],\n",
       " 307: ['1471-2091-9-4-3-10',\n",
       "  'GGEL',\n",
       "  'Rad23 UBL mutants show no detectable interaction with the proteasome  in vitro . All Rad23 constructs were bacterially expressed, purified and radiolabelled with [ Î³ - 32 P]ATP  in vitro  via an N-terminal HMK site. After quantification by scintillation counting and by Bradford assay, the radiolabelled proteins were normalized by counts and mixed with purified yeast proteasomes in a 1:10, 1:20 and 1:50 molar excess of proteasome over proteins. After incubation at 30 Â° C for 15 minutes,\" the mixtures were electrophoresed on a 3.5% native gel at 100 V for 5 hours at 4 Â° C. The arrows indicate free Rad23 proteins and the triangles indicate bands corresponding to proteasome-bound Rad23 proteins. \"\"I/P\"\" denotes the lane containing the input radiolabelled protein without any proteasome.\",'],\n",
       " 308: ['1471-2091-9-4-3-1',\n",
       "  'GMAT',\n",
       "  'Rad23 UBL mutants show no detectable interaction with the proteasome  in vitro . All Rad23 constructs were bacterially expressed, purified and radiolabelled with [ Î³ - 32 P]ATP  in vitro  via an N-terminal HMK site. After quantification by scintillation counting and by Bradford assay, the radiolabelled proteins were normalized by counts and mixed with purified yeast proteasomes in a 1:10, 1:20 and 1:50 molar excess of proteasome over proteins. After incubation at 30 Â° C for 15 minutes,\" the mixtures were electrophoresed on a 3.5% native gel at 100 V for 5 hours at 4 Â° C. The arrows indicate free Rad23 proteins and the triangles indicate bands corresponding to proteasome-bound Rad23 proteins. \"\"I/P\"\" denotes the lane containing the input radiolabelled protein without any proteasome.\",'],\n",
       " 309: ['1471-2091-9-4-3-2',\n",
       "  'GMAT',\n",
       "  'Rad23 UBL mutants show no detectable interaction with the proteasome  in vitro . All Rad23 constructs were bacterially expressed, purified and radiolabelled with [ Î³ - 32 P]ATP  in vitro  via an N-terminal HMK site. After quantification by scintillation counting and by Bradford assay, the radiolabelled proteins were normalized by counts and mixed with purified yeast proteasomes in a 1:10, 1:20 and 1:50 molar excess of proteasome over proteins. After incubation at 30 Â° C for 15 minutes,\" the mixtures were electrophoresed on a 3.5% native gel at 100 V for 5 hours at 4 Â° C. The arrows indicate free Rad23 proteins and the triangles indicate bands corresponding to proteasome-bound Rad23 proteins. \"\"I/P\"\" denotes the lane containing the input radiolabelled protein without any proteasome.\",'],\n",
       " 310: ['1471-2091-9-4-3-3',\n",
       "  'GMAT',\n",
       "  'Rad23 UBL mutants show no detectable interaction with the proteasome  in vitro . All Rad23 constructs were bacterially expressed, purified and radiolabelled with [ Î³ - 32 P]ATP  in vitro  via an N-terminal HMK site. After quantification by scintillation counting and by Bradford assay, the radiolabelled proteins were normalized by counts and mixed with purified yeast proteasomes in a 1:10, 1:20 and 1:50 molar excess of proteasome over proteins. After incubation at 30 Â° C for 15 minutes,\" the mixtures were electrophoresed on a 3.5% native gel at 100 V for 5 hours at 4 Â° C. The arrows indicate free Rad23 proteins and the triangles indicate bands corresponding to proteasome-bound Rad23 proteins. \"\"I/P\"\" denotes the lane containing the input radiolabelled protein without any proteasome.\",'],\n",
       " 311: ['1471-2091-9-4-3-4',\n",
       "  'GMAT',\n",
       "  'Rad23 UBL mutants show no detectable interaction with the proteasome  in vitro . All Rad23 constructs were bacterially expressed, purified and radiolabelled with [ Î³ - 32 P]ATP  in vitro  via an N-terminal HMK site. After quantification by scintillation counting and by Bradford assay, the radiolabelled proteins were normalized by counts and mixed with purified yeast proteasomes in a 1:10, 1:20 and 1:50 molar excess of proteasome over proteins. After incubation at 30 Â° C for 15 minutes,\" the mixtures were electrophoresed on a 3.5% native gel at 100 V for 5 hours at 4 Â° C. The arrows indicate free Rad23 proteins and the triangles indicate bands corresponding to proteasome-bound Rad23 proteins. \"\"I/P\"\" denotes the lane containing the input radiolabelled protein without any proteasome.\",'],\n",
       " 312: ['1471-2091-9-4-3-5',\n",
       "  'GGEL',\n",
       "  'Rad23 UBL mutants show no detectable interaction with the proteasome  in vitro . All Rad23 constructs were bacterially expressed, purified and radiolabelled with [ Î³ - 32 P]ATP  in vitro  via an N-terminal HMK site. After quantification by scintillation counting and by Bradford assay, the radiolabelled proteins were normalized by counts and mixed with purified yeast proteasomes in a 1:10, 1:20 and 1:50 molar excess of proteasome over proteins. After incubation at 30 Â° C for 15 minutes,\" the mixtures were electrophoresed on a 3.5% native gel at 100 V for 5 hours at 4 Â° C. The arrows indicate free Rad23 proteins and the triangles indicate bands corresponding to proteasome-bound Rad23 proteins. \"\"I/P\"\" denotes the lane containing the input radiolabelled protein without any proteasome.\",'],\n",
       " 313: ['1471-2091-9-4-3-6',\n",
       "  'GMAT',\n",
       "  'Rad23 UBL mutants show no detectable interaction with the proteasome  in vitro . All Rad23 constructs were bacterially expressed, purified and radiolabelled with [ Î³ - 32 P]ATP  in vitro  via an N-terminal HMK site. After quantification by scintillation counting and by Bradford assay, the radiolabelled proteins were normalized by counts and mixed with purified yeast proteasomes in a 1:10, 1:20 and 1:50 molar excess of proteasome over proteins. After incubation at 30 Â° C for 15 minutes,\" the mixtures were electrophoresed on a 3.5% native gel at 100 V for 5 hours at 4 Â° C. The arrows indicate free Rad23 proteins and the triangles indicate bands corresponding to proteasome-bound Rad23 proteins. \"\"I/P\"\" denotes the lane containing the input radiolabelled protein without any proteasome.\",'],\n",
       " 314: ['1471-2091-9-4-3-7',\n",
       "  'GMAT',\n",
       "  'Rad23 UBL mutants show no detectable interaction with the proteasome  in vitro . All Rad23 constructs were bacterially expressed, purified and radiolabelled with [ Î³ - 32 P]ATP  in vitro  via an N-terminal HMK site. After quantification by scintillation counting and by Bradford assay, the radiolabelled proteins were normalized by counts and mixed with purified yeast proteasomes in a 1:10, 1:20 and 1:50 molar excess of proteasome over proteins. After incubation at 30 Â° C for 15 minutes,\" the mixtures were electrophoresed on a 3.5% native gel at 100 V for 5 hours at 4 Â° C. The arrows indicate free Rad23 proteins and the triangles indicate bands corresponding to proteasome-bound Rad23 proteins. \"\"I/P\"\" denotes the lane containing the input radiolabelled protein without any proteasome.\",'],\n",
       " 315: ['1471-2091-9-4-3-8',\n",
       "  'GMAT',\n",
       "  'Rad23 UBL mutants show no detectable interaction with the proteasome  in vitro . All Rad23 constructs were bacterially expressed, purified and radiolabelled with [ Î³ - 32 P]ATP  in vitro  via an N-terminal HMK site. After quantification by scintillation counting and by Bradford assay, the radiolabelled proteins were normalized by counts and mixed with purified yeast proteasomes in a 1:10, 1:20 and 1:50 molar excess of proteasome over proteins. After incubation at 30 Â° C for 15 minutes,\" the mixtures were electrophoresed on a 3.5% native gel at 100 V for 5 hours at 4 Â° C. The arrows indicate free Rad23 proteins and the triangles indicate bands corresponding to proteasome-bound Rad23 proteins. \"\"I/P\"\" denotes the lane containing the input radiolabelled protein without any proteasome.\",'],\n",
       " 316: ['1471-2091-9-4-3-9',\n",
       "  'GMAT',\n",
       "  'Rad23 UBL mutants show no detectable interaction with the proteasome  in vitro . All Rad23 constructs were bacterially expressed, purified and radiolabelled with [ Î³ - 32 P]ATP  in vitro  via an N-terminal HMK site. After quantification by scintillation counting and by Bradford assay, the radiolabelled proteins were normalized by counts and mixed with purified yeast proteasomes in a 1:10, 1:20 and 1:50 molar excess of proteasome over proteins. After incubation at 30 Â° C for 15 minutes,\" the mixtures were electrophoresed on a 3.5% native gel at 100 V for 5 hours at 4 Â° C. The arrows indicate free Rad23 proteins and the triangles indicate bands corresponding to proteasome-bound Rad23 proteins. \"\"I/P\"\" denotes the lane containing the input radiolabelled protein without any proteasome.\",'],\n",
       " 317: ['1471-2091-9-5-6-1',\n",
       "  'GFIG',\n",
       "  'Circular dichroism spectroscopy of MCR1 as a function of temperature.  (A)  Thermal denaturation curves were recorded for MCR1 ( ï¿½ï¿½ ) and MBP (---). Denaturation curves are in units of molar ellipticity vs. temperature.  (B)  CD spectra of MCR1 samples after 60 min incubation periods at 20 Â° C, 30 Â° C, and 37 Â° C.  (C)  CD spectra of MCR1 recorded with increasing incubation times (from 0 to 120 min) at 20 Â° C, 30 Â° C and 37 Â° C. CD wavelength spectra in panels B and C are in units of molar ellipticity vs. wavelength.'],\n",
       " 318: ['1471-2091-9-5-6-2',\n",
       "  'GFIG',\n",
       "  'Circular dichroism spectroscopy of MCR1 as a function of temperature.  (A)  Thermal denaturation curves were recorded for MCR1 ( ï¿½ï¿½ ) and MBP (---). Denaturation curves are in units of molar ellipticity vs. temperature.  (B)  CD spectra of MCR1 samples after 60 min incubation periods at 20 Â° C, 30 Â° C, and 37 Â° C.  (C)  CD spectra of MCR1 recorded with increasing incubation times (from 0 to 120 min) at 20 Â° C, 30 Â° C and 37 Â° C. CD wavelength spectra in panels B and C are in units of molar ellipticity vs. wavelength.'],\n",
       " 319: ['1471-2091-9-5-6-3',\n",
       "  'GFIG',\n",
       "  'Circular dichroism spectroscopy of MCR1 as a function of temperature.  (A)  Thermal denaturation curves were recorded for MCR1 ( ï¿½ï¿½ ) and MBP (---). Denaturation curves are in units of molar ellipticity vs. temperature.  (B)  CD spectra of MCR1 samples after 60 min incubation periods at 20 Â° C, 30 Â° C, and 37 Â° C.  (C)  CD spectra of MCR1 recorded with increasing incubation times (from 0 to 120 min) at 20 Â° C, 30 Â° C and 37 Â° C. CD wavelength spectra in panels B and C are in units of molar ellipticity vs. wavelength.'],\n",
       " 320: ['1471-2091-9-5-6-4',\n",
       "  'GFIG',\n",
       "  'Circular dichroism spectroscopy of MCR1 as a function of temperature.  (A)  Thermal denaturation curves were recorded for MCR1 ( ï¿½ï¿½ ) and MBP (---). Denaturation curves are in units of molar ellipticity vs. temperature.  (B)  CD spectra of MCR1 samples after 60 min incubation periods at 20 Â° C, 30 Â° C, and 37 Â° C.  (C)  CD spectra of MCR1 recorded with increasing incubation times (from 0 to 120 min) at 20 Â° C, 30 Â° C and 37 Â° C. CD wavelength spectra in panels B and C are in units of molar ellipticity vs. wavelength.'],\n",
       " 321: ['1471-2091-9-5-6-5',\n",
       "  'GFIG',\n",
       "  'Circular dichroism spectroscopy of MCR1 as a function of temperature.  (A)  Thermal denaturation curves were recorded for MCR1 ( ï¿½ï¿½ ) and MBP (---). Denaturation curves are in units of molar ellipticity vs. temperature.  (B)  CD spectra of MCR1 samples after 60 min incubation periods at 20 Â° C, 30 Â° C, and 37 Â° C.  (C)  CD spectra of MCR1 recorded with increasing incubation times (from 0 to 120 min) at 20 Â° C, 30 Â° C and 37 Â° C. CD wavelength spectra in panels B and C are in units of molar ellipticity vs. wavelength.'],\n",
       " 322: ['1471-2105-10-113-6-1',\n",
       "  'GGEN',\n",
       "  'Sequence logos for some representative amino acid patterns . Specifically the following patterns are shown: (a) \"FE.P...F\",\" (b) \"\" [DLN]. [DLN] ... [KMR]. [ITV]  [CS]\"\"\",\" (c) \"\"V...EP...H\"\"\",\" (d) \"\"GPY.G\"\"\",\" (e) \"\"SP.NP.G\"\" and (f) \"\" [CS] ... [FHWY]. [FHWY] .N\"\".\",'],\n",
       " 323: ['1471-2105-10-113-6-2',\n",
       "  'GGEN',\n",
       "  'Sequence logos for some representative amino acid patterns . Specifically the following patterns are shown: (a) \"FE.P...F\",\" (b) \"\" [DLN]. [DLN] ... [KMR]. [ITV]  [CS]\"\"\",\" (c) \"\"V...EP...H\"\"\",\" (d) \"\"GPY.G\"\"\",\" (e) \"\"SP.NP.G\"\" and (f) \"\" [CS] ... [FHWY]. [FHWY] .N\"\".\",'],\n",
       " 324: ['1471-2105-10-113-6-3',\n",
       "  'GGEN',\n",
       "  'Sequence logos for some representative amino acid patterns . Specifically the following patterns are shown: (a) \"FE.P...F\",\" (b) \"\" [DLN]. [DLN] ... [KMR]. [ITV]  [CS]\"\"\",\" (c) \"\"V...EP...H\"\"\",\" (d) \"\"GPY.G\"\"\",\" (e) \"\"SP.NP.G\"\" and (f) \"\" [CS] ... [FHWY]. [FHWY] .N\"\".\",'],\n",
       " 325: ['1471-2105-10-113-6-4',\n",
       "  'GGEN',\n",
       "  'Sequence logos for some representative amino acid patterns . Specifically the following patterns are shown: (a) \"FE.P...F\",\" (b) \"\" [DLN]. [DLN] ... [KMR]. [ITV]  [CS]\"\"\",\" (c) \"\"V...EP...H\"\"\",\" (d) \"\"GPY.G\"\"\",\" (e) \"\"SP.NP.G\"\" and (f) \"\" [CS] ... [FHWY]. [FHWY] .N\"\".\",'],\n",
       " 326: ['1471-2105-10-113-6-5',\n",
       "  'GGEN',\n",
       "  'Sequence logos for some representative amino acid patterns . Specifically the following patterns are shown: (a) \"FE.P...F\",\" (b) \"\" [DLN]. [DLN] ... [KMR]. [ITV]  [CS]\"\"\",\" (c) \"\"V...EP...H\"\"\",\" (d) \"\"GPY.G\"\"\",\" (e) \"\"SP.NP.G\"\" and (f) \"\" [CS] ... [FHWY]. [FHWY] .N\"\".\",'],\n",
       " 327: ['1471-2105-10-113-6-6',\n",
       "  'GGEN',\n",
       "  'Sequence logos for some representative amino acid patterns . Specifically the following patterns are shown: (a) \"FE.P...F\",\" (b) \"\" [DLN]. [DLN] ... [KMR]. [ITV]  [CS]\"\"\",\" (c) \"\"V...EP...H\"\"\",\" (d) \"\"GPY.G\"\"\",\" (e) \"\"SP.NP.G\"\" and (f) \"\" [CS] ... [FHWY]. [FHWY] .N\"\".\",'],\n",
       " 328: ['1471-2105-10-113-7-1',\n",
       "  'GGEN',\n",
       "  'Sequence logos for the most representative property patterns . A sequence logo is provided for the highest scoring pattern of every property (Table 3). Different colors are used to discriminate between the amino acids that share or do not share the respective property. Specifically the highest scoring property patterns in the sequence logos are depicted in the following order: (1) hydrophobic, (2) aliphatic, (3) polar, (4) aromatic, (5) small, (6) positive, (7) proline, (8) negative.'],\n",
       " 329: ['1471-2105-10-113-7-2',\n",
       "  'GGEN',\n",
       "  'Sequence logos for the most representative property patterns . A sequence logo is provided for the highest scoring pattern of every property (Table 3). Different colors are used to discriminate between the amino acids that share or do not share the respective property. Specifically the highest scoring property patterns in the sequence logos are depicted in the following order: (1) hydrophobic, (2) aliphatic, (3) polar, (4) aromatic, (5) small, (6) positive, (7) proline, (8) negative.'],\n",
       " 330: ['1471-2105-10-113-7-3',\n",
       "  'GGEN',\n",
       "  'Sequence logos for the most representative property patterns . A sequence logo is provided for the highest scoring pattern of every property (Table 3). Different colors are used to discriminate between the amino acids that share or do not share the respective property. Specifically the highest scoring property patterns in the sequence logos are depicted in the following order: (1) hydrophobic, (2) aliphatic, (3) polar, (4) aromatic, (5) small, (6) positive, (7) proline, (8) negative.'],\n",
       " 331: ['1471-2105-10-113-7-4',\n",
       "  'GGEN',\n",
       "  'Sequence logos for the most representative property patterns . A sequence logo is provided for the highest scoring pattern of every property (Table 3). Different colors are used to discriminate between the amino acids that share or do not share the respective property. Specifically the highest scoring property patterns in the sequence logos are depicted in the following order: (1) hydrophobic, (2) aliphatic, (3) polar, (4) aromatic, (5) small, (6) positive, (7) proline, (8) negative.'],\n",
       " 332: ['1471-2105-10-113-7-5',\n",
       "  'GGEN',\n",
       "  'Sequence logos for the most representative property patterns . A sequence logo is provided for the highest scoring pattern of every property (Table 3). Different colors are used to discriminate between the amino acids that share or do not share the respective property. Specifically the highest scoring property patterns in the sequence logos are depicted in the following order: (1) hydrophobic, (2) aliphatic, (3) polar, (4) aromatic, (5) small, (6) positive, (7) proline, (8) negative.'],\n",
       " 333: ['1471-2105-10-113-7-6',\n",
       "  'GGEN',\n",
       "  'Sequence logos for the most representative property patterns . A sequence logo is provided for the highest scoring pattern of every property (Table 3). Different colors are used to discriminate between the amino acids that share or do not share the respective property. Specifically the highest scoring property patterns in the sequence logos are depicted in the following order: (1) hydrophobic, (2) aliphatic, (3) polar, (4) aromatic, (5) small, (6) positive, (7) proline, (8) negative.'],\n",
       " 334: ['1471-2105-10-113-7-7',\n",
       "  'GGEN',\n",
       "  'Sequence logos for the most representative property patterns . A sequence logo is provided for the highest scoring pattern of every property (Table 3). Different colors are used to discriminate between the amino acids that share or do not share the respective property. Specifically the highest scoring property patterns in the sequence logos are depicted in the following order: (1) hydrophobic, (2) aliphatic, (3) polar, (4) aromatic, (5) small, (6) positive, (7) proline, (8) negative.'],\n",
       " 335: ['1471-2105-10-113-7-8',\n",
       "  'GGEN',\n",
       "  'Sequence logos for the most representative property patterns . A sequence logo is provided for the highest scoring pattern of every property (Table 3). Different colors are used to discriminate between the amino acids that share or do not share the respective property. Specifically the highest scoring property patterns in the sequence logos are depicted in the following order: (1) hydrophobic, (2) aliphatic, (3) polar, (4) aromatic, (5) small, (6) positive, (7) proline, (8) negative.'],\n",
       " 336: ['1471-2105-10-156-2-1',\n",
       "  'GFIG',\n",
       "  'RMA decomposition of probe-level Affymetrix data . Panel A shows the background-adjusted and normalized probe-level data for  PRRX1 , from the Affymetrix mixture dataset (see Methods). The probes are displayed in the order which they map to the human genome (not to scale), and lines join all probe intensities of the same sample.  PRRX1  is expressed significantly higher in heart tissue compared to brain. Three replicates of pure heart tissue are shown as red lines; green lines represent pure brain tissue replicates and the blue lines represent a mixture of 75% brain tissue and 25% heart tissue. Panel B shows the estimated relative probe effects. Panel C shows the chip effects (i.e. summarized expression levels) and Panel D shows residuals, using the same colour scheme.'],\n",
       " 337: ['1471-2105-10-156-2-2',\n",
       "  'GFIG',\n",
       "  'RMA decomposition of probe-level Affymetrix data . Panel A shows the background-adjusted and normalized probe-level data for  PRRX1 , from the Affymetrix mixture dataset (see Methods). The probes are displayed in the order which they map to the human genome (not to scale), and lines join all probe intensities of the same sample.  PRRX1  is expressed significantly higher in heart tissue compared to brain. Three replicates of pure heart tissue are shown as red lines; green lines represent pure brain tissue replicates and the blue lines represent a mixture of 75% brain tissue and 25% heart tissue. Panel B shows the estimated relative probe effects. Panel C shows the chip effects (i.e. summarized expression levels) and Panel D shows residuals, using the same colour scheme.'],\n",
       " 338: ['1471-2105-10-156-2-3',\n",
       "  'GFIG',\n",
       "  'RMA decomposition of probe-level Affymetrix data . Panel A shows the background-adjusted and normalized probe-level data for  PRRX1 , from the Affymetrix mixture dataset (see Methods). The probes are displayed in the order which they map to the human genome (not to scale), and lines join all probe intensities of the same sample.  PRRX1  is expressed significantly higher in heart tissue compared to brain. Three replicates of pure heart tissue are shown as red lines; green lines represent pure brain tissue replicates and the blue lines represent a mixture of 75% brain tissue and 25% heart tissue. Panel B shows the estimated relative probe effects. Panel C shows the chip effects (i.e. summarized expression levels) and Panel D shows residuals, using the same colour scheme.'],\n",
       " 339: ['1471-2105-10-156-2-4',\n",
       "  'GFIG',\n",
       "  'RMA decomposition of probe-level Affymetrix data . Panel A shows the background-adjusted and normalized probe-level data for  PRRX1 , from the Affymetrix mixture dataset (see Methods). The probes are displayed in the order which they map to the human genome (not to scale), and lines join all probe intensities of the same sample.  PRRX1  is expressed significantly higher in heart tissue compared to brain. Three replicates of pure heart tissue are shown as red lines; green lines represent pure brain tissue replicates and the blue lines represent a mixture of 75% brain tissue and 25% heart tissue. Panel B shows the estimated relative probe effects. Panel C shows the chip effects (i.e. summarized expression levels) and Panel D shows residuals, using the same colour scheme.'],\n",
       " 340: ['1471-2105-10-157-2-1',\n",
       "  'D3DR',\n",
       "  'Our inner distance (ID) signature is compared, for instance, to Euclidean distance (ED) signature from  [ 7 ]. The first row shows the input four artificial proteins with the same main chain orientation but with different molecular shapes. The second row shows the ID and ED signatures. In each plot, the vertical axis represents distance distribution. Note that ID is not sensitive to shape deformation, so four signatures are almost consistent; in contrast, ED is strongly sensitive to deformation.'],\n",
       " 341: ['1471-2105-10-157-2-2',\n",
       "  'GHDR',\n",
       "  'Our inner distance (ID) signature is compared, for instance, to Euclidean distance (ED) signature from  [ 7 ]. The first row shows the input four artificial proteins with the same main chain orientation but with different molecular shapes. The second row shows the ID and ED signatures. In each plot, the vertical axis represents distance distribution. Note that ID is not sensitive to shape deformation, so four signatures are almost consistent; in contrast, ED is strongly sensitive to deformation.'],\n",
       " 342: ['1471-2105-10-157-2-3',\n",
       "  'D3DR',\n",
       "  'Our inner distance (ID) signature is compared, for instance, to Euclidean distance (ED) signature from  [ 7 ]. The first row shows the input four artificial proteins with the same main chain orientation but with different molecular shapes. The second row shows the ID and ED signatures. In each plot, the vertical axis represents distance distribution. Note that ID is not sensitive to shape deformation, so four signatures are almost consistent; in contrast, ED is strongly sensitive to deformation.'],\n",
       " 343: ['1471-2105-10-157-2-4',\n",
       "  'D3DR',\n",
       "  'Our inner distance (ID) signature is compared, for instance, to Euclidean distance (ED) signature from  [ 7 ]. The first row shows the input four artificial proteins with the same main chain orientation but with different molecular shapes. The second row shows the ID and ED signatures. In each plot, the vertical axis represents distance distribution. Note that ID is not sensitive to shape deformation, so four signatures are almost consistent; in contrast, ED is strongly sensitive to deformation.'],\n",
       " 344: ['1471-2105-10-157-2-5',\n",
       "  'GFIG',\n",
       "  'Our inner distance (ID) signature is compared, for instance, to Euclidean distance (ED) signature from  [ 7 ]. The first row shows the input four artificial proteins with the same main chain orientation but with different molecular shapes. The second row shows the ID and ED signatures. In each plot, the vertical axis represents distance distribution. Note that ID is not sensitive to shape deformation, so four signatures are almost consistent; in contrast, ED is strongly sensitive to deformation.'],\n",
       " 345: ['1471-2105-10-157-2-6',\n",
       "  'GFIG',\n",
       "  'Our inner distance (ID) signature is compared, for instance, to Euclidean distance (ED) signature from  [ 7 ]. The first row shows the input four artificial proteins with the same main chain orientation but with different molecular shapes. The second row shows the ID and ED signatures. In each plot, the vertical axis represents distance distribution. Note that ID is not sensitive to shape deformation, so four signatures are almost consistent; in contrast, ED is strongly sensitive to deformation.'],\n",
       " 346: ['1471-2105-10-157-7-1',\n",
       "  'D3DR',\n",
       "  'The ID signature compared, for instance, to EDsignature . The first row shows the input two conformations  1j5nA  (left) and  1lwmA  (right) of the same protein. The second row shows the ID and ED signatures. Note that ID is not sensitive to shape deformation, so two signatures are very close; in contrast, ED is strongly sensitive to deformation.,'],\n",
       " 347: ['1471-2105-10-157-7-2',\n",
       "  'D3DR',\n",
       "  'The ID signature compared, for instance, to EDsignature . The first row shows the input two conformations  1j5nA  (left) and  1lwmA  (right) of the same protein. The second row shows the ID and ED signatures. Note that ID is not sensitive to shape deformation, so two signatures are very close; in contrast, ED is strongly sensitive to deformation.,'],\n",
       " 348: ['1471-2105-10-157-7-3',\n",
       "  'GFIG',\n",
       "  'The ID signature compared, for instance, to EDsignature . The first row shows the input two conformations  1j5nA  (left) and  1lwmA  (right) of the same protein. The second row shows the ID and ED signatures. Note that ID is not sensitive to shape deformation, so two signatures are very close; in contrast, ED is strongly sensitive to deformation.,'],\n",
       " 349: ['1471-2105-10-157-7-4',\n",
       "  'GFIG',\n",
       "  'The ID signature compared, for instance, to EDsignature . The first row shows the input two conformations  1j5nA  (left) and  1lwmA  (right) of the same protein. The second row shows the ID and ED signatures. Note that ID is not sensitive to shape deformation, so two signatures are very close; in contrast, ED is strongly sensitive to deformation.,'],\n",
       " 350: ['1471-2105-10-214-2-1',\n",
       "  'GFIG',\n",
       "  \"Efficiency of detection by Pearson's correlation coefficient in three classes of probe set pairs .  A, B, C : We defined, in network HG-U133-NR10, three classes of probe set pairs that had positive and negative correlation scores of, respectively,  >  10 and  >  -10 (mainly positively correlated and labelled CORR in  A ),  >  10 and  <  -10 (positively and negatively correlated and labelled CORR  &  ANTI in  B ) and  <  10 and  <  -10 (mainly negatively correlated and labelled ANTI in  C ). For each class we selected 30,000 random probe set pairs; the Figure indicates the percentage of the pairs that had a Pearson's correlation coefficient of greater than 0.5 or less than -0.5 (correlation coefficients were calculated on the ranks, but similar results were obtained with the log2 of signals). A probe set pair has its positive (red) and negative (blue) score plotted on the x-axis, and its Pearson's correlation coefficient plotted on the y-axis (a probe set pair is therefore represented by two points located on a line parallel to the x-axis).  D, E, F : For the three subsets of probe set pairs whose Pearson's correlation coefficients were greater than 0.60 ( D ), 0.50 ( E ), or 0.40 ( F ), we have displayed the sorted values of the corresponding positive and negative scores ((a probe set pair is therefore represented by two points). The percentage of pairs that also existed in the network is indicated on the left of the curve, and the right side of the curve shows the ratio between the total number of pairs found to be positively correlated in the network and the number of pairs that would have been selected if the process of selection by Pearson's correlation coefficient, here applied to 30,000 probe set pairs, had been extended to the entire set of probe set pairs. Figures for negatively correlated probe sets were similar (not shown).\"],\n",
       " 351: ['1471-2105-10-214-2-2',\n",
       "  'GFIG',\n",
       "  \"Efficiency of detection by Pearson's correlation coefficient in three classes of probe set pairs .  A, B, C : We defined, in network HG-U133-NR10, three classes of probe set pairs that had positive and negative correlation scores of, respectively,  >  10 and  >  -10 (mainly positively correlated and labelled CORR in  A ),  >  10 and  <  -10 (positively and negatively correlated and labelled CORR  &  ANTI in  B ) and  <  10 and  <  -10 (mainly negatively correlated and labelled ANTI in  C ). For each class we selected 30,000 random probe set pairs; the Figure indicates the percentage of the pairs that had a Pearson's correlation coefficient of greater than 0.5 or less than -0.5 (correlation coefficients were calculated on the ranks, but similar results were obtained with the log2 of signals). A probe set pair has its positive (red) and negative (blue) score plotted on the x-axis, and its Pearson's correlation coefficient plotted on the y-axis (a probe set pair is therefore represented by two points located on a line parallel to the x-axis).  D, E, F : For the three subsets of probe set pairs whose Pearson's correlation coefficients were greater than 0.60 ( D ), 0.50 ( E ), or 0.40 ( F ), we have displayed the sorted values of the corresponding positive and negative scores ((a probe set pair is therefore represented by two points). The percentage of pairs that also existed in the network is indicated on the left of the curve, and the right side of the curve shows the ratio between the total number of pairs found to be positively correlated in the network and the number of pairs that would have been selected if the process of selection by Pearson's correlation coefficient, here applied to 30,000 probe set pairs, had been extended to the entire set of probe set pairs. Figures for negatively correlated probe sets were similar (not shown).\"],\n",
       " 352: ['1471-2105-10-214-2-3',\n",
       "  'GFIG',\n",
       "  \"Efficiency of detection by Pearson's correlation coefficient in three classes of probe set pairs .  A, B, C : We defined, in network HG-U133-NR10, three classes of probe set pairs that had positive and negative correlation scores of, respectively,  >  10 and  >  -10 (mainly positively correlated and labelled CORR in  A ),  >  10 and  <  -10 (positively and negatively correlated and labelled CORR  &  ANTI in  B ) and  <  10 and  <  -10 (mainly negatively correlated and labelled ANTI in  C ). For each class we selected 30,000 random probe set pairs; the Figure indicates the percentage of the pairs that had a Pearson's correlation coefficient of greater than 0.5 or less than -0.5 (correlation coefficients were calculated on the ranks, but similar results were obtained with the log2 of signals). A probe set pair has its positive (red) and negative (blue) score plotted on the x-axis, and its Pearson's correlation coefficient plotted on the y-axis (a probe set pair is therefore represented by two points located on a line parallel to the x-axis).  D, E, F : For the three subsets of probe set pairs whose Pearson's correlation coefficients were greater than 0.60 ( D ), 0.50 ( E ), or 0.40 ( F ), we have displayed the sorted values of the corresponding positive and negative scores ((a probe set pair is therefore represented by two points). The percentage of pairs that also existed in the network is indicated on the left of the curve, and the right side of the curve shows the ratio between the total number of pairs found to be positively correlated in the network and the number of pairs that would have been selected if the process of selection by Pearson's correlation coefficient, here applied to 30,000 probe set pairs, had been extended to the entire set of probe set pairs. Figures for negatively correlated probe sets were similar (not shown).\"],\n",
       " 353: ['1471-2105-10-214-2-4',\n",
       "  'GFIG',\n",
       "  \"Efficiency of detection by Pearson's correlation coefficient in three classes of probe set pairs .  A, B, C : We defined, in network HG-U133-NR10, three classes of probe set pairs that had positive and negative correlation scores of, respectively,  >  10 and  >  -10 (mainly positively correlated and labelled CORR in  A ),  >  10 and  <  -10 (positively and negatively correlated and labelled CORR  &  ANTI in  B ) and  <  10 and  <  -10 (mainly negatively correlated and labelled ANTI in  C ). For each class we selected 30,000 random probe set pairs; the Figure indicates the percentage of the pairs that had a Pearson's correlation coefficient of greater than 0.5 or less than -0.5 (correlation coefficients were calculated on the ranks, but similar results were obtained with the log2 of signals). A probe set pair has its positive (red) and negative (blue) score plotted on the x-axis, and its Pearson's correlation coefficient plotted on the y-axis (a probe set pair is therefore represented by two points located on a line parallel to the x-axis).  D, E, F : For the three subsets of probe set pairs whose Pearson's correlation coefficients were greater than 0.60 ( D ), 0.50 ( E ), or 0.40 ( F ), we have displayed the sorted values of the corresponding positive and negative scores ((a probe set pair is therefore represented by two points). The percentage of pairs that also existed in the network is indicated on the left of the curve, and the right side of the curve shows the ratio between the total number of pairs found to be positively correlated in the network and the number of pairs that would have been selected if the process of selection by Pearson's correlation coefficient, here applied to 30,000 probe set pairs, had been extended to the entire set of probe set pairs. Figures for negatively correlated probe sets were similar (not shown).\"],\n",
       " 354: ['1471-2105-10-214-2-5',\n",
       "  'GFIG',\n",
       "  \"Efficiency of detection by Pearson's correlation coefficient in three classes of probe set pairs .  A, B, C : We defined, in network HG-U133-NR10, three classes of probe set pairs that had positive and negative correlation scores of, respectively,  >  10 and  >  -10 (mainly positively correlated and labelled CORR in  A ),  >  10 and  <  -10 (positively and negatively correlated and labelled CORR  &  ANTI in  B ) and  <  10 and  <  -10 (mainly negatively correlated and labelled ANTI in  C ). For each class we selected 30,000 random probe set pairs; the Figure indicates the percentage of the pairs that had a Pearson's correlation coefficient of greater than 0.5 or less than -0.5 (correlation coefficients were calculated on the ranks, but similar results were obtained with the log2 of signals). A probe set pair has its positive (red) and negative (blue) score plotted on the x-axis, and its Pearson's correlation coefficient plotted on the y-axis (a probe set pair is therefore represented by two points located on a line parallel to the x-axis).  D, E, F : For the three subsets of probe set pairs whose Pearson's correlation coefficients were greater than 0.60 ( D ), 0.50 ( E ), or 0.40 ( F ), we have displayed the sorted values of the corresponding positive and negative scores ((a probe set pair is therefore represented by two points). The percentage of pairs that also existed in the network is indicated on the left of the curve, and the right side of the curve shows the ratio between the total number of pairs found to be positively correlated in the network and the number of pairs that would have been selected if the process of selection by Pearson's correlation coefficient, here applied to 30,000 probe set pairs, had been extended to the entire set of probe set pairs. Figures for negatively correlated probe sets were similar (not shown).\"],\n",
       " 355: ['1471-2105-10-214-2-6',\n",
       "  'GFIG',\n",
       "  \"Efficiency of detection by Pearson's correlation coefficient in three classes of probe set pairs .  A, B, C : We defined, in network HG-U133-NR10, three classes of probe set pairs that had positive and negative correlation scores of, respectively,  >  10 and  >  -10 (mainly positively correlated and labelled CORR in  A ),  >  10 and  <  -10 (positively and negatively correlated and labelled CORR  &  ANTI in  B ) and  <  10 and  <  -10 (mainly negatively correlated and labelled ANTI in  C ). For each class we selected 30,000 random probe set pairs; the Figure indicates the percentage of the pairs that had a Pearson's correlation coefficient of greater than 0.5 or less than -0.5 (correlation coefficients were calculated on the ranks, but similar results were obtained with the log2 of signals). A probe set pair has its positive (red) and negative (blue) score plotted on the x-axis, and its Pearson's correlation coefficient plotted on the y-axis (a probe set pair is therefore represented by two points located on a line parallel to the x-axis).  D, E, F : For the three subsets of probe set pairs whose Pearson's correlation coefficients were greater than 0.60 ( D ), 0.50 ( E ), or 0.40 ( F ), we have displayed the sorted values of the corresponding positive and negative scores ((a probe set pair is therefore represented by two points). The percentage of pairs that also existed in the network is indicated on the left of the curve, and the right side of the curve shows the ratio between the total number of pairs found to be positively correlated in the network and the number of pairs that would have been selected if the process of selection by Pearson's correlation coefficient, here applied to 30,000 probe set pairs, had been extended to the entire set of probe set pairs. Figures for negatively correlated probe sets were similar (not shown).\"],\n",
       " 356: ['1471-2105-10-214-8-1',\n",
       "  'GFIG',\n",
       "  'Geometrical and topological network similarity . Two Keiko maps are compared, one constructed on MG-U74v2-NR1 and another on RG-U34-NR1. Only unique and multiple probe set pairs are taken into account in the display and statistics.  A  ï¿½ï¿½  Six clusters found by Gene DIVER in the RG-U34-NR1 map are displayed in the upper right corner. The corresponding points on the MG-U74v2-NR1 map are displayed in the lower left corner. All the points belonging to a given cluster in the RG-U34-NR1 map are linked by a coloured line,\" and the same is true for the corresponding points in the MG-U74v2-NR1 map.  B  ï¿½ï¿½  Two points that correspond to a pair of probe sets are linked by a line coloured according to the cluster they belong to.  C  ï¿½ï¿½  In blue is plotted the probability density of the distance in the RG-U34-NR1 map between one point and the closest point in the MG-U74v2-NR1 map. In red is plotted the probability density of the mean distance in a series of random permutations of RG-U34-NR1. The black vertical line indicates the median value of the first distribution. The magenta vertical line marks the mean of the 100 median values calculated on permuted maps. The ratio between these two values is used to score the \"\"distance similarity\"\" between the two maps.  D  ï¿½ï¿½  In blue is plotted the probability density of the ranks in the RG-U34-NR1 map between one point and the closest point in the MG-U74v2-NR1 map. In red is plotted the probability density of the mean rank in a series of random permutations of RG-U34-NR1. The ratio between the two values marked by the two vertical lines is used to calculate a ranking score that scores the \"\"rank similarity\"\" between the two maps.\",'],\n",
       " 357: ['1471-2105-10-214-8-2',\n",
       "  'GFIG',\n",
       "  'Geometrical and topological network similarity . Two Keiko maps are compared, one constructed on MG-U74v2-NR1 and another on RG-U34-NR1. Only unique and multiple probe set pairs are taken into account in the display and statistics.  A  ï¿½ï¿½  Six clusters found by Gene DIVER in the RG-U34-NR1 map are displayed in the upper right corner. The corresponding points on the MG-U74v2-NR1 map are displayed in the lower left corner. All the points belonging to a given cluster in the RG-U34-NR1 map are linked by a coloured line,\" and the same is true for the corresponding points in the MG-U74v2-NR1 map.  B  ï¿½ï¿½  Two points that correspond to a pair of probe sets are linked by a line coloured according to the cluster they belong to.  C  ï¿½ï¿½  In blue is plotted the probability density of the distance in the RG-U34-NR1 map between one point and the closest point in the MG-U74v2-NR1 map. In red is plotted the probability density of the mean distance in a series of random permutations of RG-U34-NR1. The black vertical line indicates the median value of the first distribution. The magenta vertical line marks the mean of the 100 median values calculated on permuted maps. The ratio between these two values is used to score the \"\"distance similarity\"\" between the two maps.  D  ï¿½ï¿½  In blue is plotted the probability density of the ranks in the RG-U34-NR1 map between one point and the closest point in the MG-U74v2-NR1 map. In red is plotted the probability density of the mean rank in a series of random permutations of RG-U34-NR1. The ratio between the two values marked by the two vertical lines is used to calculate a ranking score that scores the \"\"rank similarity\"\" between the two maps.\",'],\n",
       " 358: ['1471-2105-10-214-8-3',\n",
       "  'GFIG',\n",
       "  'Geometrical and topological network similarity . Two Keiko maps are compared, one constructed on MG-U74v2-NR1 and another on RG-U34-NR1. Only unique and multiple probe set pairs are taken into account in the display and statistics.  A  ï¿½ï¿½  Six clusters found by Gene DIVER in the RG-U34-NR1 map are displayed in the upper right corner. The corresponding points on the MG-U74v2-NR1 map are displayed in the lower left corner. All the points belonging to a given cluster in the RG-U34-NR1 map are linked by a coloured line,\" and the same is true for the corresponding points in the MG-U74v2-NR1 map.  B  ï¿½ï¿½  Two points that correspond to a pair of probe sets are linked by a line coloured according to the cluster they belong to.  C  ï¿½ï¿½  In blue is plotted the probability density of the distance in the RG-U34-NR1 map between one point and the closest point in the MG-U74v2-NR1 map. In red is plotted the probability density of the mean distance in a series of random permutations of RG-U34-NR1. The black vertical line indicates the median value of the first distribution. The magenta vertical line marks the mean of the 100 median values calculated on permuted maps. The ratio between these two values is used to score the \"\"distance similarity\"\" between the two maps.  D  ï¿½ï¿½  In blue is plotted the probability density of the ranks in the RG-U34-NR1 map between one point and the closest point in the MG-U74v2-NR1 map. In red is plotted the probability density of the mean rank in a series of random permutations of RG-U34-NR1. The ratio between the two values marked by the two vertical lines is used to calculate a ranking score that scores the \"\"rank similarity\"\" between the two maps.\",'],\n",
       " 359: ['1471-2105-10-214-8-4',\n",
       "  'GFIG',\n",
       "  'Geometrical and topological network similarity . Two Keiko maps are compared, one constructed on MG-U74v2-NR1 and another on RG-U34-NR1. Only unique and multiple probe set pairs are taken into account in the display and statistics.  A  ï¿½ï¿½  Six clusters found by Gene DIVER in the RG-U34-NR1 map are displayed in the upper right corner. The corresponding points on the MG-U74v2-NR1 map are displayed in the lower left corner. All the points belonging to a given cluster in the RG-U34-NR1 map are linked by a coloured line,\" and the same is true for the corresponding points in the MG-U74v2-NR1 map.  B  ï¿½ï¿½  Two points that correspond to a pair of probe sets are linked by a line coloured according to the cluster they belong to.  C  ï¿½ï¿½  In blue is plotted the probability density of the distance in the RG-U34-NR1 map between one point and the closest point in the MG-U74v2-NR1 map. In red is plotted the probability density of the mean distance in a series of random permutations of RG-U34-NR1. The black vertical line indicates the median value of the first distribution. The magenta vertical line marks the mean of the 100 median values calculated on permuted maps. The ratio between these two values is used to score the \"\"distance similarity\"\" between the two maps.  D  ï¿½ï¿½  In blue is plotted the probability density of the ranks in the RG-U34-NR1 map between one point and the closest point in the MG-U74v2-NR1 map. In red is plotted the probability density of the mean rank in a series of random permutations of RG-U34-NR1. The ratio between the two values marked by the two vertical lines is used to calculate a ranking score that scores the \"\"rank similarity\"\" between the two maps.\",'],\n",
       " 360: ['1471-2105-10-254-5-1',\n",
       "  'GFLO',\n",
       "  'Representing hierarchical information in Ultra-Structure . Panel A: conceptual representation of hierarchical information. Briefly, a Dataset consists of multiple Runs, each of which has a number of Precursor Ions, which in turn can produce multiple Product Ions. Panel B: one possible schema for this hierarchy using traditional ER modeling. Each level of the hierarchy is represented by a separate table. Various attributes of entities are represented using a number of additional columns. The structure of the hierarchy (i.e., the inter-level links) are represented using foreign key relationships, shown here with arrows extending from the referring data to their referents. For instance,\" each Run is from a single Dataset (indicated by its \"\"Dataset\"\" column)\", while a single Dataset may have several Runs that refer to it. Panel C: reformulation of the same information using Ultra-Structure ruleforms. Entities at all levels of the hierarchy are declared in the BioEntity existential ruleform. The various attributes of each entity are represented by individual BioEntity Attribute rules. Finally, the structure of the hierarchy is declared using BioEntity Network rules. The specific nature of the links between entities is declared explicitly, using the appropriate Relationship. These same tables could be used to represent any other hierarchy of BioEntities, regardless of depth, whereas the traditional ER model would require as many tables as there are hierarchical levels. As mentioned earlier, the actual BioEntity Attribute and BioEntity Network ruleforms have an additional Resource factor (which in this case would be used to denote the particular data file the data come from) as well as additional considerations that are omitted here for simplicity and clarity.,'],\n",
       " 361: ['1471-2105-10-254-5-2',\n",
       "  'GFLO',\n",
       "  'Representing hierarchical information in Ultra-Structure . Panel A: conceptual representation of hierarchical information. Briefly, a Dataset consists of multiple Runs, each of which has a number of Precursor Ions, which in turn can produce multiple Product Ions. Panel B: one possible schema for this hierarchy using traditional ER modeling. Each level of the hierarchy is represented by a separate table. Various attributes of entities are represented using a number of additional columns. The structure of the hierarchy (i.e., the inter-level links) are represented using foreign key relationships, shown here with arrows extending from the referring data to their referents. For instance,\" each Run is from a single Dataset (indicated by its \"\"Dataset\"\" column)\", while a single Dataset may have several Runs that refer to it. Panel C: reformulation of the same information using Ultra-Structure ruleforms. Entities at all levels of the hierarchy are declared in the BioEntity existential ruleform. The various attributes of each entity are represented by individual BioEntity Attribute rules. Finally, the structure of the hierarchy is declared using BioEntity Network rules. The specific nature of the links between entities is declared explicitly, using the appropriate Relationship. These same tables could be used to represent any other hierarchy of BioEntities, regardless of depth, whereas the traditional ER model would require as many tables as there are hierarchical levels. As mentioned earlier, the actual BioEntity Attribute and BioEntity Network ruleforms have an additional Resource factor (which in this case would be used to denote the particular data file the data come from) as well as additional considerations that are omitted here for simplicity and clarity.,'],\n",
       " 362: ['1471-2105-10-254-5-3',\n",
       "  'GTAB',\n",
       "  'Representing hierarchical information in Ultra-Structure . Panel A: conceptual representation of hierarchical information. Briefly, a Dataset consists of multiple Runs, each of which has a number of Precursor Ions, which in turn can produce multiple Product Ions. Panel B: one possible schema for this hierarchy using traditional ER modeling. Each level of the hierarchy is represented by a separate table. Various attributes of entities are represented using a number of additional columns. The structure of the hierarchy (i.e., the inter-level links) are represented using foreign key relationships, shown here with arrows extending from the referring data to their referents. For instance,\" each Run is from a single Dataset (indicated by its \"\"Dataset\"\" column)\", while a single Dataset may have several Runs that refer to it. Panel C: reformulation of the same information using Ultra-Structure ruleforms. Entities at all levels of the hierarchy are declared in the BioEntity existential ruleform. The various attributes of each entity are represented by individual BioEntity Attribute rules. Finally, the structure of the hierarchy is declared using BioEntity Network rules. The specific nature of the links between entities is declared explicitly, using the appropriate Relationship. These same tables could be used to represent any other hierarchy of BioEntities, regardless of depth, whereas the traditional ER model would require as many tables as there are hierarchical levels. As mentioned earlier, the actual BioEntity Attribute and BioEntity Network ruleforms have an additional Resource factor (which in this case would be used to denote the particular data file the data come from) as well as additional considerations that are omitted here for simplicity and clarity.,'],\n",
       " 363: ['1471-2105-10-254-5-4',\n",
       "  'GTAB',\n",
       "  'Representing hierarchical information in Ultra-Structure . Panel A: conceptual representation of hierarchical information. Briefly, a Dataset consists of multiple Runs, each of which has a number of Precursor Ions, which in turn can produce multiple Product Ions. Panel B: one possible schema for this hierarchy using traditional ER modeling. Each level of the hierarchy is represented by a separate table. Various attributes of entities are represented using a number of additional columns. The structure of the hierarchy (i.e., the inter-level links) are represented using foreign key relationships, shown here with arrows extending from the referring data to their referents. For instance,\" each Run is from a single Dataset (indicated by its \"\"Dataset\"\" column)\", while a single Dataset may have several Runs that refer to it. Panel C: reformulation of the same information using Ultra-Structure ruleforms. Entities at all levels of the hierarchy are declared in the BioEntity existential ruleform. The various attributes of each entity are represented by individual BioEntity Attribute rules. Finally, the structure of the hierarchy is declared using BioEntity Network rules. The specific nature of the links between entities is declared explicitly, using the appropriate Relationship. These same tables could be used to represent any other hierarchy of BioEntities, regardless of depth, whereas the traditional ER model would require as many tables as there are hierarchical levels. As mentioned earlier, the actual BioEntity Attribute and BioEntity Network ruleforms have an additional Resource factor (which in this case would be used to denote the particular data file the data come from) as well as additional considerations that are omitted here for simplicity and clarity.,'],\n",
       " 364: ['1471-2105-10-254-5-5',\n",
       "  'GTAB',\n",
       "  'Representing hierarchical information in Ultra-Structure . Panel A: conceptual representation of hierarchical information. Briefly, a Dataset consists of multiple Runs, each of which has a number of Precursor Ions, which in turn can produce multiple Product Ions. Panel B: one possible schema for this hierarchy using traditional ER modeling. Each level of the hierarchy is represented by a separate table. Various attributes of entities are represented using a number of additional columns. The structure of the hierarchy (i.e., the inter-level links) are represented using foreign key relationships, shown here with arrows extending from the referring data to their referents. For instance,\" each Run is from a single Dataset (indicated by its \"\"Dataset\"\" column)\", while a single Dataset may have several Runs that refer to it. Panel C: reformulation of the same information using Ultra-Structure ruleforms. Entities at all levels of the hierarchy are declared in the BioEntity existential ruleform. The various attributes of each entity are represented by individual BioEntity Attribute rules. Finally, the structure of the hierarchy is declared using BioEntity Network rules. The specific nature of the links between entities is declared explicitly, using the appropriate Relationship. These same tables could be used to represent any other hierarchy of BioEntities, regardless of depth, whereas the traditional ER model would require as many tables as there are hierarchical levels. As mentioned earlier, the actual BioEntity Attribute and BioEntity Network ruleforms have an additional Resource factor (which in this case would be used to denote the particular data file the data come from) as well as additional considerations that are omitted here for simplicity and clarity.,'],\n",
       " 365: ['1471-2105-10-254-8-1',\n",
       "  'GGEN',\n",
       "  'Example ruleform for querying Locations by Relationship . Panel A: schematic of two locations from the 18th draft of the human genome declared in the system to have some overlap, with Genomic Location 1 overlapping the 5\\' end of Genomic Location 2. This relationship can be specified in terms of relationships between corresponding pairs of Attributes of those Locations. Panel B: We make the types of relationships we wish to query explicit, and can now allow us to perform queries based on Locations. For example,\" we define here a set of 5 rules that specify a relationship of \"\"Location 1 overlaps the 5\\'-end of Location 2\"\" that holds between Locations in the 18th draft of the human genome (specified by the \"\"Context\"\" and \"\"Target Context\"\" columns). Other ruleforms (not shown) contain meta-rules that ensure the appropriate rules are chosen for a given scenario. In this case\",\" the rules for determining a 5\\' overlap depend on what nucleic acid strand (forward or reverse) a given genomic region lies; here we show the rules that apply for the forward strand only. These rules specify what relationships must hold between attributes of our query Location (\"\"Location 1 Attribute\"\" consideration) and those of any other Location (\"\"Location 2 Attribute\"\" consideration) whose 5\\' region is overlapped by our query Location. For instance\", the first rule states that both Locations must be on the same sequence (i.e., chromosome), while the second states that they must be on the same strand. The remaining rules constrain relationships between the start and stop positions of the Locations such that the desired relationship holds. Any kind of relationship that can be expressed in terms of relationships between Location Attributes can be represented in this way.'],\n",
       " 366: ['1471-2105-10-254-8-2',\n",
       "  'GFIG',\n",
       "  'Example ruleform for querying Locations by Relationship . Panel A: schematic of two locations from the 18th draft of the human genome declared in the system to have some overlap, with Genomic Location 1 overlapping the 5\\' end of Genomic Location 2. This relationship can be specified in terms of relationships between corresponding pairs of Attributes of those Locations. Panel B: We make the types of relationships we wish to query explicit, and can now allow us to perform queries based on Locations. For example,\" we define here a set of 5 rules that specify a relationship of \"\"Location 1 overlaps the 5\\'-end of Location 2\"\" that holds between Locations in the 18th draft of the human genome (specified by the \"\"Context\"\" and \"\"Target Context\"\" columns). Other ruleforms (not shown) contain meta-rules that ensure the appropriate rules are chosen for a given scenario. In this case\",\" the rules for determining a 5\\' overlap depend on what nucleic acid strand (forward or reverse) a given genomic region lies; here we show the rules that apply for the forward strand only. These rules specify what relationships must hold between attributes of our query Location (\"\"Location 1 Attribute\"\" consideration) and those of any other Location (\"\"Location 2 Attribute\"\" consideration) whose 5\\' region is overlapped by our query Location. For instance\", the first rule states that both Locations must be on the same sequence (i.e., chromosome), while the second states that they must be on the same strand. The remaining rules constrain relationships between the start and stop positions of the Locations such that the desired relationship holds. Any kind of relationship that can be expressed in terms of relationships between Location Attributes can be represented in this way.'],\n",
       " 367: ['1471-2105-10-254-8-3',\n",
       "  'GTAB',\n",
       "  'Example ruleform for querying Locations by Relationship . Panel A: schematic of two locations from the 18th draft of the human genome declared in the system to have some overlap, with Genomic Location 1 overlapping the 5\\' end of Genomic Location 2. This relationship can be specified in terms of relationships between corresponding pairs of Attributes of those Locations. Panel B: We make the types of relationships we wish to query explicit, and can now allow us to perform queries based on Locations. For example,\" we define here a set of 5 rules that specify a relationship of \"\"Location 1 overlaps the 5\\'-end of Location 2\"\" that holds between Locations in the 18th draft of the human genome (specified by the \"\"Context\"\" and \"\"Target Context\"\" columns). Other ruleforms (not shown) contain meta-rules that ensure the appropriate rules are chosen for a given scenario. In this case\",\" the rules for determining a 5\\' overlap depend on what nucleic acid strand (forward or reverse) a given genomic region lies; here we show the rules that apply for the forward strand only. These rules specify what relationships must hold between attributes of our query Location (\"\"Location 1 Attribute\"\" consideration) and those of any other Location (\"\"Location 2 Attribute\"\" consideration) whose 5\\' region is overlapped by our query Location. For instance\", the first rule states that both Locations must be on the same sequence (i.e., chromosome), while the second states that they must be on the same strand. The remaining rules constrain relationships between the start and stop positions of the Locations such that the desired relationship holds. Any kind of relationship that can be expressed in terms of relationships between Location Attributes can be represented in this way.'],\n",
       " 368: ['1471-2105-10-273-3-1',\n",
       "  'GGEN',\n",
       "  'Homology network of PLP-dependent enzymes . Nodes represent Hidden Markov models (HMMs) of PLP-dependent families. Edges represent homology connections (E  <  10 -5 ) between families established by HMM-HMM comparisons [ 18 ]. Black edges connect protein families with the most significant similarities (E  <  10 -50 ). The network is visualized with the \"Degree sorted circle layout\" of Cytoscape [ 19 ]. Colors were mapped into nodes using the structural group of the protein family as a node property.,'],\n",
       " 369: ['1471-2105-10-273-3-2',\n",
       "  'GHDR',\n",
       "  'Homology network of PLP-dependent enzymes . Nodes represent Hidden Markov models (HMMs) of PLP-dependent families. Edges represent homology connections (E  <  10 -5 ) between families established by HMM-HMM comparisons [ 18 ]. Black edges connect protein families with the most significant similarities (E  <  10 -50 ). The network is visualized with the \"Degree sorted circle layout\" of Cytoscape [ 19 ]. Colors were mapped into nodes using the structural group of the protein family as a node property.,'],\n",
       " 370: ['1471-2105-10-273-3-3',\n",
       "  'GSYS',\n",
       "  'Homology network of PLP-dependent enzymes . Nodes represent Hidden Markov models (HMMs) of PLP-dependent families. Edges represent homology connections (E  <  10 -5 ) between families established by HMM-HMM comparisons [ 18 ]. Black edges connect protein families with the most significant similarities (E  <  10 -50 ). The network is visualized with the \"Degree sorted circle layout\" of Cytoscape [ 19 ]. Colors were mapped into nodes using the structural group of the protein family as a node property.,'],\n",
       " 371: ['1471-2105-10-298-1-1',\n",
       "  'GGEN',\n",
       "  'Wired phylogenetic trees of V_set and C2_set domains (definitions from Pfam database  [ 21 ] ), respectively zoomed on nodes  41  and  43  (yellow areas) . All protein sequences are from the Ensembl database [ 22 ]. The multiple alignment was built and annotated using the MACSIMS program [ 23 ] and the phylogeny was produced and visualized using the Figenix platform [ 4 ].  [Number]  represents the identifier of the closest node.  Number1_Number2_Number3  represents bootstrap values for the closest internal node (from three construction algorithms [ 4 ]). Red circles on nodes represent duplication events. Purple circles represent the detected homolog domains shuffling events. Labeled trees for V_set and C2_set domains are provided [see Additional file  2 ], [see Additional file  3 ].,'],\n",
       " 372: ['1471-2105-10-298-1-2',\n",
       "  'GGEN',\n",
       "  'Wired phylogenetic trees of V_set and C2_set domains (definitions from Pfam database  [ 21 ] ), respectively zoomed on nodes  41  and  43  (yellow areas) . All protein sequences are from the Ensembl database [ 22 ]. The multiple alignment was built and annotated using the MACSIMS program [ 23 ] and the phylogeny was produced and visualized using the Figenix platform [ 4 ].  [Number]  represents the identifier of the closest node.  Number1_Number2_Number3  represents bootstrap values for the closest internal node (from three construction algorithms [ 4 ]). Red circles on nodes represent duplication events. Purple circles represent the detected homolog domains shuffling events. Labeled trees for V_set and C2_set domains are provided [see Additional file  2 ], [see Additional file  3 ].,'],\n",
       " 373: ['1471-2105-10-298-1-3',\n",
       "  'GFLO',\n",
       "  'Wired phylogenetic trees of V_set and C2_set domains (definitions from Pfam database  [ 21 ] ), respectively zoomed on nodes  41  and  43  (yellow areas) . All protein sequences are from the Ensembl database [ 22 ]. The multiple alignment was built and annotated using the MACSIMS program [ 23 ] and the phylogeny was produced and visualized using the Figenix platform [ 4 ].  [Number]  represents the identifier of the closest node.  Number1_Number2_Number3  represents bootstrap values for the closest internal node (from three construction algorithms [ 4 ]). Red circles on nodes represent duplication events. Purple circles represent the detected homolog domains shuffling events. Labeled trees for V_set and C2_set domains are provided [see Additional file  2 ], [see Additional file  3 ].,'],\n",
       " 374: ['1471-2105-10-298-1-4',\n",
       "  'GGEN',\n",
       "  'Wired phylogenetic trees of V_set and C2_set domains (definitions from Pfam database  [ 21 ] ), respectively zoomed on nodes  41  and  43  (yellow areas) . All protein sequences are from the Ensembl database [ 22 ]. The multiple alignment was built and annotated using the MACSIMS program [ 23 ] and the phylogeny was produced and visualized using the Figenix platform [ 4 ].  [Number]  represents the identifier of the closest node.  Number1_Number2_Number3  represents bootstrap values for the closest internal node (from three construction algorithms [ 4 ]). Red circles on nodes represent duplication events. Purple circles represent the detected homolog domains shuffling events. Labeled trees for V_set and C2_set domains are provided [see Additional file  2 ], [see Additional file  3 ].,'],\n",
       " 375: ['1471-2105-10-298-1-5',\n",
       "  'GGEN',\n",
       "  'Wired phylogenetic trees of V_set and C2_set domains (definitions from Pfam database  [ 21 ] ), respectively zoomed on nodes  41  and  43  (yellow areas) . All protein sequences are from the Ensembl database [ 22 ]. The multiple alignment was built and annotated using the MACSIMS program [ 23 ] and the phylogeny was produced and visualized using the Figenix platform [ 4 ].  [Number]  represents the identifier of the closest node.  Number1_Number2_Number3  represents bootstrap values for the closest internal node (from three construction algorithms [ 4 ]). Red circles on nodes represent duplication events. Purple circles represent the detected homolog domains shuffling events. Labeled trees for V_set and C2_set domains are provided [see Additional file  2 ], [see Additional file  3 ].,'],\n",
       " 376: ['1471-2105-10-298-1-6',\n",
       "  'GSYS',\n",
       "  'Wired phylogenetic trees of V_set and C2_set domains (definitions from Pfam database  [ 21 ] ), respectively zoomed on nodes  41  and  43  (yellow areas) . All protein sequences are from the Ensembl database [ 22 ]. The multiple alignment was built and annotated using the MACSIMS program [ 23 ] and the phylogeny was produced and visualized using the Figenix platform [ 4 ].  [Number]  represents the identifier of the closest node.  Number1_Number2_Number3  represents bootstrap values for the closest internal node (from three construction algorithms [ 4 ]). Red circles on nodes represent duplication events. Purple circles represent the detected homolog domains shuffling events. Labeled trees for V_set and C2_set domains are provided [see Additional file  2 ], [see Additional file  3 ].,'],\n",
       " 377: ['1471-2105-10-351-2-1',\n",
       "  'D3DR',\n",
       "  'Examples of linear motifs with functional residues protruding outwards from the structural domain surface . a) A very exposed instance (in white) of LIG_RGD in a loop of SCOP domain d1 mfn_2; b) An instance (in violet) of LIG_RGD in a region outside a domain (SCOP d1ssua_); c) An instance (in pink) of MOD_SUMO in an exposed loop of the d1kpsd_ SCOP domain; d) The MOD_CMANNOS C-Mannosylation site (in magenta) in the SCOP domain d1k2aa_; e) The two MOD_N-GLC_1 N-glycosylation sites (in yellow) in the SCOP domain d1qm3a_; f) The N-glycosylation site (in red) in the SCOP domain d1fl7b_; g), h) The N-glycosylation site (in green) in the SCOP domain d1o7ae2 and (in red) in the SCOP domain d1n26a1.'],\n",
       " 378: ['1471-2105-10-351-2-2',\n",
       "  'D3DR',\n",
       "  'Examples of linear motifs with functional residues protruding outwards from the structural domain surface . a) A very exposed instance (in white) of LIG_RGD in a loop of SCOP domain d1 mfn_2; b) An instance (in violet) of LIG_RGD in a region outside a domain (SCOP d1ssua_); c) An instance (in pink) of MOD_SUMO in an exposed loop of the d1kpsd_ SCOP domain; d) The MOD_CMANNOS C-Mannosylation site (in magenta) in the SCOP domain d1k2aa_; e) The two MOD_N-GLC_1 N-glycosylation sites (in yellow) in the SCOP domain d1qm3a_; f) The N-glycosylation site (in red) in the SCOP domain d1fl7b_; g), h) The N-glycosylation site (in green) in the SCOP domain d1o7ae2 and (in red) in the SCOP domain d1n26a1.'],\n",
       " 379: ['1471-2105-10-351-2-3',\n",
       "  'D3DR',\n",
       "  'Examples of linear motifs with functional residues protruding outwards from the structural domain surface . a) A very exposed instance (in white) of LIG_RGD in a loop of SCOP domain d1 mfn_2; b) An instance (in violet) of LIG_RGD in a region outside a domain (SCOP d1ssua_); c) An instance (in pink) of MOD_SUMO in an exposed loop of the d1kpsd_ SCOP domain; d) The MOD_CMANNOS C-Mannosylation site (in magenta) in the SCOP domain d1k2aa_; e) The two MOD_N-GLC_1 N-glycosylation sites (in yellow) in the SCOP domain d1qm3a_; f) The N-glycosylation site (in red) in the SCOP domain d1fl7b_; g), h) The N-glycosylation site (in green) in the SCOP domain d1o7ae2 and (in red) in the SCOP domain d1n26a1.'],\n",
       " 380: ['1471-2105-10-351-2-4',\n",
       "  'D3DR',\n",
       "  'Examples of linear motifs with functional residues protruding outwards from the structural domain surface . a) A very exposed instance (in white) of LIG_RGD in a loop of SCOP domain d1 mfn_2; b) An instance (in violet) of LIG_RGD in a region outside a domain (SCOP d1ssua_); c) An instance (in pink) of MOD_SUMO in an exposed loop of the d1kpsd_ SCOP domain; d) The MOD_CMANNOS C-Mannosylation site (in magenta) in the SCOP domain d1k2aa_; e) The two MOD_N-GLC_1 N-glycosylation sites (in yellow) in the SCOP domain d1qm3a_; f) The N-glycosylation site (in red) in the SCOP domain d1fl7b_; g), h) The N-glycosylation site (in green) in the SCOP domain d1o7ae2 and (in red) in the SCOP domain d1n26a1.'],\n",
       " 381: ['1471-2105-10-351-2-5',\n",
       "  'D3DR',\n",
       "  'Examples of linear motifs with functional residues protruding outwards from the structural domain surface . a) A very exposed instance (in white) of LIG_RGD in a loop of SCOP domain d1 mfn_2; b) An instance (in violet) of LIG_RGD in a region outside a domain (SCOP d1ssua_); c) An instance (in pink) of MOD_SUMO in an exposed loop of the d1kpsd_ SCOP domain; d) The MOD_CMANNOS C-Mannosylation site (in magenta) in the SCOP domain d1k2aa_; e) The two MOD_N-GLC_1 N-glycosylation sites (in yellow) in the SCOP domain d1qm3a_; f) The N-glycosylation site (in red) in the SCOP domain d1fl7b_; g), h) The N-glycosylation site (in green) in the SCOP domain d1o7ae2 and (in red) in the SCOP domain d1n26a1.'],\n",
       " 382: ['1471-2105-10-351-2-6',\n",
       "  'D3DR',\n",
       "  'Examples of linear motifs with functional residues protruding outwards from the structural domain surface . a) A very exposed instance (in white) of LIG_RGD in a loop of SCOP domain d1 mfn_2; b) An instance (in violet) of LIG_RGD in a region outside a domain (SCOP d1ssua_); c) An instance (in pink) of MOD_SUMO in an exposed loop of the d1kpsd_ SCOP domain; d) The MOD_CMANNOS C-Mannosylation site (in magenta) in the SCOP domain d1k2aa_; e) The two MOD_N-GLC_1 N-glycosylation sites (in yellow) in the SCOP domain d1qm3a_; f) The N-glycosylation site (in red) in the SCOP domain d1fl7b_; g), h) The N-glycosylation site (in green) in the SCOP domain d1o7ae2 and (in red) in the SCOP domain d1n26a1.'],\n",
       " 383: ['1471-2105-10-351-2-7',\n",
       "  'D3DR',\n",
       "  'Examples of linear motifs with functional residues protruding outwards from the structural domain surface . a) A very exposed instance (in white) of LIG_RGD in a loop of SCOP domain d1 mfn_2; b) An instance (in violet) of LIG_RGD in a region outside a domain (SCOP d1ssua_); c) An instance (in pink) of MOD_SUMO in an exposed loop of the d1kpsd_ SCOP domain; d) The MOD_CMANNOS C-Mannosylation site (in magenta) in the SCOP domain d1k2aa_; e) The two MOD_N-GLC_1 N-glycosylation sites (in yellow) in the SCOP domain d1qm3a_; f) The N-glycosylation site (in red) in the SCOP domain d1fl7b_; g), h) The N-glycosylation site (in green) in the SCOP domain d1o7ae2 and (in red) in the SCOP domain d1n26a1.'],\n",
       " 384: ['1471-2105-10-351-2-8',\n",
       "  'D3DR',\n",
       "  'Examples of linear motifs with functional residues protruding outwards from the structural domain surface . a) A very exposed instance (in white) of LIG_RGD in a loop of SCOP domain d1 mfn_2; b) An instance (in violet) of LIG_RGD in a region outside a domain (SCOP d1ssua_); c) An instance (in pink) of MOD_SUMO in an exposed loop of the d1kpsd_ SCOP domain; d) The MOD_CMANNOS C-Mannosylation site (in magenta) in the SCOP domain d1k2aa_; e) The two MOD_N-GLC_1 N-glycosylation sites (in yellow) in the SCOP domain d1qm3a_; f) The N-glycosylation site (in red) in the SCOP domain d1fl7b_; g), h) The N-glycosylation site (in green) in the SCOP domain d1o7ae2 and (in red) in the SCOP domain d1n26a1.'],\n",
       " 385: ['1471-2105-10-435-2-1',\n",
       "  'GFIG',\n",
       "  'Performance of Correlation and CRE Prediction Methods across microarray data:  Regulated  genes only . The overall pattern for the PPV (A), Sensitivity (B) and F-Measure (C), Specificity (D), and NPV (E) for the analysis of regulated genes using correlation and/or CRE detection prediction methods for determining TF targets. As shown here the PPV shows a generally consistent pattern across all datasets, while the sensitivity and F measure are shown to be higher for the Spearman,    1 CRE and Spearman +    1 CRE prediction methods (A-C). Also shown is the consistent pattern for specificity and NPV (D and E) across all prediction methods except the    1 CRE which shows a high degree of variability. PPV (positive predictive value), NPV (negative predictive value), CRE over.: CRE Over-representation,   1 CRE: detection of 1 or more CREs.,'],\n",
       " 386: ['1471-2105-10-435-2-2',\n",
       "  'GFIG',\n",
       "  'Performance of Correlation and CRE Prediction Methods across microarray data:  Regulated  genes only . The overall pattern for the PPV (A), Sensitivity (B) and F-Measure (C), Specificity (D), and NPV (E) for the analysis of regulated genes using correlation and/or CRE detection prediction methods for determining TF targets. As shown here the PPV shows a generally consistent pattern across all datasets, while the sensitivity and F measure are shown to be higher for the Spearman,    1 CRE and Spearman +    1 CRE prediction methods (A-C). Also shown is the consistent pattern for specificity and NPV (D and E) across all prediction methods except the    1 CRE which shows a high degree of variability. PPV (positive predictive value), NPV (negative predictive value), CRE over.: CRE Over-representation,   1 CRE: detection of 1 or more CREs.,'],\n",
       " 387: ['1471-2105-10-435-2-3',\n",
       "  'GFIG',\n",
       "  'Performance of Correlation and CRE Prediction Methods across microarray data:  Regulated  genes only . The overall pattern for the PPV (A), Sensitivity (B) and F-Measure (C), Specificity (D), and NPV (E) for the analysis of regulated genes using correlation and/or CRE detection prediction methods for determining TF targets. As shown here the PPV shows a generally consistent pattern across all datasets, while the sensitivity and F measure are shown to be higher for the Spearman,    1 CRE and Spearman +    1 CRE prediction methods (A-C). Also shown is the consistent pattern for specificity and NPV (D and E) across all prediction methods except the    1 CRE which shows a high degree of variability. PPV (positive predictive value), NPV (negative predictive value), CRE over.: CRE Over-representation,   1 CRE: detection of 1 or more CREs.,'],\n",
       " 388: ['1471-2105-10-435-2-4',\n",
       "  'GFIG',\n",
       "  'Performance of Correlation and CRE Prediction Methods across microarray data:  Regulated  genes only . The overall pattern for the PPV (A), Sensitivity (B) and F-Measure (C), Specificity (D), and NPV (E) for the analysis of regulated genes using correlation and/or CRE detection prediction methods for determining TF targets. As shown here the PPV shows a generally consistent pattern across all datasets, while the sensitivity and F measure are shown to be higher for the Spearman,    1 CRE and Spearman +    1 CRE prediction methods (A-C). Also shown is the consistent pattern for specificity and NPV (D and E) across all prediction methods except the    1 CRE which shows a high degree of variability. PPV (positive predictive value), NPV (negative predictive value), CRE over.: CRE Over-representation,   1 CRE: detection of 1 or more CREs.,'],\n",
       " 389: ['1471-2105-10-435-2-5',\n",
       "  'GFIG',\n",
       "  'Performance of Correlation and CRE Prediction Methods across microarray data:  Regulated  genes only . The overall pattern for the PPV (A), Sensitivity (B) and F-Measure (C), Specificity (D), and NPV (E) for the analysis of regulated genes using correlation and/or CRE detection prediction methods for determining TF targets. As shown here the PPV shows a generally consistent pattern across all datasets, while the sensitivity and F measure are shown to be higher for the Spearman,    1 CRE and Spearman +    1 CRE prediction methods (A-C). Also shown is the consistent pattern for specificity and NPV (D and E) across all prediction methods except the    1 CRE which shows a high degree of variability. PPV (positive predictive value), NPV (negative predictive value), CRE over.: CRE Over-representation,   1 CRE: detection of 1 or more CREs.,'],\n",
       " 390: ['1471-2105-10-435-2-6',\n",
       "  'GFIG',\n",
       "  'Performance of Correlation and CRE Prediction Methods across microarray data:  Regulated  genes only . The overall pattern for the PPV (A), Sensitivity (B) and F-Measure (C), Specificity (D), and NPV (E) for the analysis of regulated genes using correlation and/or CRE detection prediction methods for determining TF targets. As shown here the PPV shows a generally consistent pattern across all datasets, while the sensitivity and F measure are shown to be higher for the Spearman,    1 CRE and Spearman +    1 CRE prediction methods (A-C). Also shown is the consistent pattern for specificity and NPV (D and E) across all prediction methods except the    1 CRE which shows a high degree of variability. PPV (positive predictive value), NPV (negative predictive value), CRE over.: CRE Over-representation,   1 CRE: detection of 1 or more CREs.,'],\n",
       " 391: ['1471-2105-10-75-2-1',\n",
       "  'DMTR',\n",
       "  'Edge extraction on a microscopy image of a vesicle . An electron microscopy image of a vesicle (a), to which the edge extraction scheme is applied. The magnitude of the directional field extracted from the curvelet coefficients is shown in (b), with an extract showing the direction of the field in a small part of the image shown in (c). The last image (d) shows the final result of the edge extraction overlaid on the original image, after the edges have been extracted using the non-maximal suppression, and extended along the directional field to connect the different edge segments. Image courtesy of Prof. Urs Greber, University of Z Ã¼ rich.'],\n",
       " 392: ['1471-2105-10-75-2-2',\n",
       "  'DMTR',\n",
       "  'Edge extraction on a microscopy image of a vesicle . An electron microscopy image of a vesicle (a), to which the edge extraction scheme is applied. The magnitude of the directional field extracted from the curvelet coefficients is shown in (b), with an extract showing the direction of the field in a small part of the image shown in (c). The last image (d) shows the final result of the edge extraction overlaid on the original image, after the edges have been extracted using the non-maximal suppression, and extended along the directional field to connect the different edge segments. Image courtesy of Prof. Urs Greber, University of Z Ã¼ rich.'],\n",
       " 393: ['1471-2105-10-75-2-3',\n",
       "  'GFIG',\n",
       "  'Edge extraction on a microscopy image of a vesicle . An electron microscopy image of a vesicle (a), to which the edge extraction scheme is applied. The magnitude of the directional field extracted from the curvelet coefficients is shown in (b), with an extract showing the direction of the field in a small part of the image shown in (c). The last image (d) shows the final result of the edge extraction overlaid on the original image, after the edges have been extracted using the non-maximal suppression, and extended along the directional field to connect the different edge segments. Image courtesy of Prof. Urs Greber, University of Z Ã¼ rich.'],\n",
       " 394: ['1471-2105-10-75-2-4',\n",
       "  'DMEL',\n",
       "  'Edge extraction on a microscopy image of a vesicle . An electron microscopy image of a vesicle (a), to which the edge extraction scheme is applied. The magnitude of the directional field extracted from the curvelet coefficients is shown in (b), with an extract showing the direction of the field in a small part of the image shown in (c). The last image (d) shows the final result of the edge extraction overlaid on the original image, after the edges have been extracted using the non-maximal suppression, and extended along the directional field to connect the different edge segments. Image courtesy of Prof. Urs Greber, University of Z Ã¼ rich.'],\n",
       " 395: ['1471-2105-10-S12-S12-6-1',\n",
       "  'GFIG',\n",
       "  'Evaluation of linear dispersion coefficient (R) . R coefficients calculated for three cell populations in wound-healing assay (a). Deviation from the expected, i.e. towards the center of the wound, direction and R values are reported for each time point for NIH3T3 (b) and NIHRas (c) populations.,'],\n",
       " 396: ['1471-2105-10-S12-S12-6-2',\n",
       "  'GFIG',\n",
       "  'Evaluation of linear dispersion coefficient (R) . R coefficients calculated for three cell populations in wound-healing assay (a). Deviation from the expected, i.e. towards the center of the wound, direction and R values are reported for each time point for NIH3T3 (b) and NIHRas (c) populations.,'],\n",
       " 397: ['1471-2105-10-S12-S12-6-3',\n",
       "  'GFIG',\n",
       "  'Evaluation of linear dispersion coefficient (R) . R coefficients calculated for three cell populations in wound-healing assay (a). Deviation from the expected, i.e. towards the center of the wound, direction and R values are reported for each time point for NIH3T3 (b) and NIHRas (c) populations.,'],\n",
       " 398: ['1471-2105-10-S12-S12-7-1',\n",
       "  'GFIG',\n",
       "  'Relationship between deviation angle and linearity . a) Deviation from the expected direction, plotted as a function of linearity, in a population subjected to the wound stimulus. b) External, middle and internal sub-populations, identified according to their distance from the wound edge. c) R coefficient for the three sub-populations, compared to the threshold level for P = 0.01.'],\n",
       " 399: ['1471-2105-10-S12-S12-7-2',\n",
       "  'DMEL',\n",
       "  'Relationship between deviation angle and linearity . a) Deviation from the expected direction, plotted as a function of linearity, in a population subjected to the wound stimulus. b) External, middle and internal sub-populations, identified according to their distance from the wound edge. c) R coefficient for the three sub-populations, compared to the threshold level for P = 0.01.'],\n",
       " 400: ['1471-2105-10-S12-S12-7-3',\n",
       "  'GFIG',\n",
       "  'Relationship between deviation angle and linearity . a) Deviation from the expected direction, plotted as a function of linearity, in a population subjected to the wound stimulus. b) External, middle and internal sub-populations, identified according to their distance from the wound edge. c) R coefficient for the three sub-populations, compared to the threshold level for P = 0.01.'],\n",
       " 401: ['1471-2105-10-S12-S12-8-1',\n",
       "  'GSCR',\n",
       "  'Time plots for the three subpopulations described in figure 7 . Deviation angle and R coefficient, for the external, middle and internal cell subpopulation. In all plots, expected angle (black) and threshold level for P = 0.01 (red) are reported as a reference.'],\n",
       " 402: ['1471-2105-10-S12-S12-8-2',\n",
       "  'GSCR',\n",
       "  'Time plots for the three subpopulations described in figure 7 . Deviation angle and R coefficient, for the external, middle and internal cell subpopulation. In all plots, expected angle (black) and threshold level for P = 0.01 (red) are reported as a reference.'],\n",
       " 403: ['1471-2105-10-S12-S12-8-3',\n",
       "  'GSCR',\n",
       "  'Time plots for the three subpopulations described in figure 7 . Deviation angle and R coefficient, for the external, middle and internal cell subpopulation. In all plots, expected angle (black) and threshold level for P = 0.01 (red) are reported as a reference.'],\n",
       " 404: ['1471-2105-10-S1-S39-9-1',\n",
       "  'GSYS',\n",
       "  'Pseudoknot-loops . The lighter base pairs and lighter unpaired bases represent the particular loop. In the lower structure the arc (3, 24) is lighter, since it is minimal crossing with respect to the arc (9, 30),  not  contained in any pseudoknot-loop.,'],\n",
       " 405: ['1471-2105-10-S1-S39-9-2',\n",
       "  'GFIG',\n",
       "  'Pseudoknot-loops . The lighter base pairs and lighter unpaired bases represent the particular loop. In the lower structure the arc (3, 24) is lighter, since it is minimal crossing with respect to the arc (9, 30),  not  contained in any pseudoknot-loop.,'],\n",
       " 406: ['1471-2105-10-S1-S39-9-3',\n",
       "  'GSYS',\n",
       "  'Pseudoknot-loops . The lighter base pairs and lighter unpaired bases represent the particular loop. In the lower structure the arc (3, 24) is lighter, since it is minimal crossing with respect to the arc (9, 30),  not  contained in any pseudoknot-loop.,'],\n",
       " 407: ['1471-2105-10-S1-S39-9-4',\n",
       "  'GFIG',\n",
       "  'Pseudoknot-loops . The lighter base pairs and lighter unpaired bases represent the particular loop. In the lower structure the arc (3, 24) is lighter, since it is minimal crossing with respect to the arc (9, 30),  not  contained in any pseudoknot-loop.,'],\n",
       " 408: ['1471-2105-11-141-6-1',\n",
       "  'GFIG',\n",
       "  'Determining loss of viability due to heat shock using YODA . BY4742 cells show a loss in viability following 55 Â° C heat shock. A) Average growth curves from three technical replicates are shown for each heat shock treatment. B) Survival curves from two independent Bioscreen runs as well as CFUs. All experiments assume the 0 minute treatment to have 100% cell viability.,'],\n",
       " 409: ['1471-2105-11-141-6-2',\n",
       "  'GFIG',\n",
       "  'Determining loss of viability due to heat shock using YODA . BY4742 cells show a loss in viability following 55 Â° C heat shock. A) Average growth curves from three technical replicates are shown for each heat shock treatment. B) Survival curves from two independent Bioscreen runs as well as CFUs. All experiments assume the 0 minute treatment to have 100% cell viability.,'],\n",
       " 410: ['1471-2105-11-141-7-1',\n",
       "  'GFIG',\n",
       "  'Determining rapamycin sensitivity using YODA . Cells from A) BY4742, B)  fpr1  ,\" and C)  tor1    overnight cultures were inoculated into YPD or YPD containing 10 ng/mL rapamycin. Average growth curves from three technical replicates are shown along with D) doubling times generated by YODA. The \"\"interval\"\" method was used to calculate doubling times.\",'],\n",
       " 411: ['1471-2105-11-141-7-2',\n",
       "  'GFIG',\n",
       "  'Determining rapamycin sensitivity using YODA . Cells from A) BY4742, B)  fpr1  ,\" and C)  tor1    overnight cultures were inoculated into YPD or YPD containing 10 ng/mL rapamycin. Average growth curves from three technical replicates are shown along with D) doubling times generated by YODA. The \"\"interval\"\" method was used to calculate doubling times.\",'],\n",
       " 412: ['1471-2105-11-141-7-3',\n",
       "  'GFIG',\n",
       "  'Determining rapamycin sensitivity using YODA . Cells from A) BY4742, B)  fpr1  ,\" and C)  tor1    overnight cultures were inoculated into YPD or YPD containing 10 ng/mL rapamycin. Average growth curves from three technical replicates are shown along with D) doubling times generated by YODA. The \"\"interval\"\" method was used to calculate doubling times.\",'],\n",
       " 413: ['1471-2105-11-141-7-4',\n",
       "  'GFIG',\n",
       "  'Determining rapamycin sensitivity using YODA . Cells from A) BY4742, B)  fpr1  ,\" and C)  tor1    overnight cultures were inoculated into YPD or YPD containing 10 ng/mL rapamycin. Average growth curves from three technical replicates are shown along with D) doubling times generated by YODA. The \"\"interval\"\" method was used to calculate doubling times.\",'],\n",
       " 414: ['1471-2105-11-183-6-1',\n",
       "  'GFIG',\n",
       "  \"Results of applying the Weibull-normal mixture model to the neuroblastoma data sets - A and B show the favorable and unfavorable group, respectively .  (A1, B1) . Comparison between the fitted Weibull-normal distribution (green line) with two components ('On' and 'Off') and the empirical distribution (brown line). The pink line shows the component of expression intensities of 'On' genes, the blue line that of 'Off' genes.  (A2, B2)  Density function corresponding to the fitted Weibull-normal distribution function (green line) and its components (pink and blue lines).,\"],\n",
       " 415: ['1471-2105-11-183-6-2',\n",
       "  'GFIG',\n",
       "  \"Results of applying the Weibull-normal mixture model to the neuroblastoma data sets - A and B show the favorable and unfavorable group, respectively .  (A1, B1) . Comparison between the fitted Weibull-normal distribution (green line) with two components ('On' and 'Off') and the empirical distribution (brown line). The pink line shows the component of expression intensities of 'On' genes, the blue line that of 'Off' genes.  (A2, B2)  Density function corresponding to the fitted Weibull-normal distribution function (green line) and its components (pink and blue lines).,\"],\n",
       " 416: ['1471-2105-11-183-6-3',\n",
       "  'GFIG',\n",
       "  \"Results of applying the Weibull-normal mixture model to the neuroblastoma data sets - A and B show the favorable and unfavorable group, respectively .  (A1, B1) . Comparison between the fitted Weibull-normal distribution (green line) with two components ('On' and 'Off') and the empirical distribution (brown line). The pink line shows the component of expression intensities of 'On' genes, the blue line that of 'Off' genes.  (A2, B2)  Density function corresponding to the fitted Weibull-normal distribution function (green line) and its components (pink and blue lines).,\"],\n",
       " 417: ['1471-2105-11-183-6-4',\n",
       "  'GFIG',\n",
       "  \"Results of applying the Weibull-normal mixture model to the neuroblastoma data sets - A and B show the favorable and unfavorable group, respectively .  (A1, B1) . Comparison between the fitted Weibull-normal distribution (green line) with two components ('On' and 'Off') and the empirical distribution (brown line). The pink line shows the component of expression intensities of 'On' genes, the blue line that of 'Off' genes.  (A2, B2)  Density function corresponding to the fitted Weibull-normal distribution function (green line) and its components (pink and blue lines).,\"],\n",
       " 418: ['1471-2105-11-191-10-1',\n",
       "  'GFIG',\n",
       "  'Analysis of the variable sets separately using PCA . Representation of the samples from the validation set by the projection onto the first two principal components from the copy number data (left panel) and gene expression data (right panel), extracted from the tuning set. Samples with different leukemia subtypes are shown with different colors. The E2A/PBX1 subgroup, well separated from the rest using the correlation-maximizing methods (Figure 5), is not emerging as a deviating group when the data sets are analyzed individually with PCA. The T-ALL subgroup (light green) has a specific gene expression pattern, and is deviating from the rest using PCA on only the gene expression data (right panel). However, since these subjects do not have a specific copy number pattern, they are less discernible using CCA.,'],\n",
       " 419: ['1471-2105-11-191-10-2',\n",
       "  'GFIG',\n",
       "  'Analysis of the variable sets separately using PCA . Representation of the samples from the validation set by the projection onto the first two principal components from the copy number data (left panel) and gene expression data (right panel), extracted from the tuning set. Samples with different leukemia subtypes are shown with different colors. The E2A/PBX1 subgroup, well separated from the rest using the correlation-maximizing methods (Figure 5), is not emerging as a deviating group when the data sets are analyzed individually with PCA. The T-ALL subgroup (light green) has a specific gene expression pattern, and is deviating from the rest using PCA on only the gene expression data (right panel). However, since these subjects do not have a specific copy number pattern, they are less discernible using CCA.,'],\n",
       " 420: ['1471-2105-11-193-5-1',\n",
       "  'GCHE',\n",
       "  'Chemical structures of the five ligand molecules in the HIV protease complexes listed in Table 3 .,'],\n",
       " 421: ['1471-2105-11-193-5-2',\n",
       "  'GCHE',\n",
       "  'Chemical structures of the five ligand molecules in the HIV protease complexes listed in Table 3 .,'],\n",
       " 422: ['1471-2105-11-193-5-3',\n",
       "  'GCHE',\n",
       "  'Chemical structures of the five ligand molecules in the HIV protease complexes listed in Table 3 .,'],\n",
       " 423: ['1471-2105-11-193-5-4',\n",
       "  'GCHE',\n",
       "  'Chemical structures of the five ligand molecules in the HIV protease complexes listed in Table 3 .,'],\n",
       " 424: ['1471-2105-11-193-5-5',\n",
       "  'GCHE',\n",
       "  'Chemical structures of the five ligand molecules in the HIV protease complexes listed in Table 3 .,'],\n",
       " 425: ['1471-2105-11-193-8-1',\n",
       "  'GCHE',\n",
       "  'Chemical structures of the six ligand molecules in the trypsin complexes listed in Table 4 .,'],\n",
       " 426: ['1471-2105-11-193-8-2',\n",
       "  'GCHE',\n",
       "  'Chemical structures of the six ligand molecules in the trypsin complexes listed in Table 4 .,'],\n",
       " 427: ['1471-2105-11-193-8-3',\n",
       "  'GCHE',\n",
       "  'Chemical structures of the six ligand molecules in the trypsin complexes listed in Table 4 .,'],\n",
       " 428: ['1471-2105-11-193-8-4',\n",
       "  'GCHE',\n",
       "  'Chemical structures of the six ligand molecules in the trypsin complexes listed in Table 4 .,'],\n",
       " 429: ['1471-2105-11-193-8-5',\n",
       "  'GCHE',\n",
       "  'Chemical structures of the six ligand molecules in the trypsin complexes listed in Table 4 .,'],\n",
       " 430: ['1471-2105-11-193-8-6',\n",
       "  'GCHE',\n",
       "  'Chemical structures of the six ligand molecules in the trypsin complexes listed in Table 4 .,'],\n",
       " 431: ['1471-2105-11-203-2-1',\n",
       "  'GGEN',\n",
       "  'Schematic diagrams of the 4 forms of BS reads, mapping and post processing . 2 A . BS reads may be in one of the 4 forms: +FW, +RC, -FW, -RC. 2 B . Bowtie aligns C/T converted reads to the C/T converted strands. During the post processing, the number of mismatches is counted except those between read Ts and genomic Cs. Low-quality mappings with many mismatches are removed.'],\n",
       " 432: ['1471-2105-11-203-2-2',\n",
       "  'GGEN',\n",
       "  'Schematic diagrams of the 4 forms of BS reads, mapping and post processing . 2 A . BS reads may be in one of the 4 forms: +FW, +RC, -FW, -RC. 2 B . Bowtie aligns C/T converted reads to the C/T converted strands. During the post processing, the number of mismatches is counted except those between read Ts and genomic Cs. Low-quality mappings with many mismatches are removed.'],\n",
       " 433: ['1471-2105-11-203-2-3',\n",
       "  'GSYS',\n",
       "  'Schematic diagrams of the 4 forms of BS reads, mapping and post processing . 2 A . BS reads may be in one of the 4 forms: +FW, +RC, -FW, -RC. 2 B . Bowtie aligns C/T converted reads to the C/T converted strands. During the post processing, the number of mismatches is counted except those between read Ts and genomic Cs. Low-quality mappings with many mismatches are removed.'],\n",
       " 434: ['1471-2105-11-291-15-1',\n",
       "  'GFIG',\n",
       "  \"Systematic bias of 'Langmuir'-expression estimates (Eq. (12)) with respect to the estimates which consider washing (Eq. (11)) : Part a and b: Correlation plot between both estimates and their logged difference,  . The graphs were calculated assuming K P,h  = const for two non-specific background levels (red and black curves) and for PM and MM probes (dotted and solid curves) assuming the survival fractions w PM,S (t) = 0.95, w MM,S (t) = 0.50 and w P,N (t) = 0.1. Neglecting washing underestimates the expression degree especially at small and large expression values. Part c: The survival fraction of bound probes depends on the intensity (or, equivalently, probe occupancy) before (t = 0) and after washing (t  >  0). The graph for t = 0 was re-plotted from Figure 5a using Eq. (17) with w max  = 0.9, w min  = 0.06,  Î³  = 1.6 and a' = 0.1. The graph for t = 6 refers to the standard number of washing cycles. It is obtained from the t = 0 graph by making the substitution logI(t) = logI(0)+log(w(t)) in the argument (see Eq. (8)). Part d shows the bias of the Langmuir approximation assuming a constant transcript concentration and variable K P,S  and thus a variable survival fraction w(  ) which has been taken from part c of the figure for t = 6. The dashed curves labelled with '(+)' and '(-)' in panels c and d refer to 50%-deviations of the washing function, log w(  ) +/-  = log w(  ) Â· (1.5) (+/-)1 , to estimate the effect of the scattering of the probe level data from the mean (compare with Figure 5a). The bias of the Langmuir-approximation strongly resembles that shown in part b. Note that the bias applies to PM and MM probes as well in this case.\"],\n",
       " 435: ['1471-2105-11-291-15-2',\n",
       "  'GFIG',\n",
       "  \"Systematic bias of 'Langmuir'-expression estimates (Eq. (12)) with respect to the estimates which consider washing (Eq. (11)) : Part a and b: Correlation plot between both estimates and their logged difference,  . The graphs were calculated assuming K P,h  = const for two non-specific background levels (red and black curves) and for PM and MM probes (dotted and solid curves) assuming the survival fractions w PM,S (t) = 0.95, w MM,S (t) = 0.50 and w P,N (t) = 0.1. Neglecting washing underestimates the expression degree especially at small and large expression values. Part c: The survival fraction of bound probes depends on the intensity (or, equivalently, probe occupancy) before (t = 0) and after washing (t  >  0). The graph for t = 0 was re-plotted from Figure 5a using Eq. (17) with w max  = 0.9, w min  = 0.06,  Î³  = 1.6 and a' = 0.1. The graph for t = 6 refers to the standard number of washing cycles. It is obtained from the t = 0 graph by making the substitution logI(t) = logI(0)+log(w(t)) in the argument (see Eq. (8)). Part d shows the bias of the Langmuir approximation assuming a constant transcript concentration and variable K P,S  and thus a variable survival fraction w(  ) which has been taken from part c of the figure for t = 6. The dashed curves labelled with '(+)' and '(-)' in panels c and d refer to 50%-deviations of the washing function, log w(  ) +/-  = log w(  ) Â· (1.5) (+/-)1 , to estimate the effect of the scattering of the probe level data from the mean (compare with Figure 5a). The bias of the Langmuir-approximation strongly resembles that shown in part b. Note that the bias applies to PM and MM probes as well in this case.\"],\n",
       " 436: ['1471-2105-11-291-15-3',\n",
       "  'GFIG',\n",
       "  \"Systematic bias of 'Langmuir'-expression estimates (Eq. (12)) with respect to the estimates which consider washing (Eq. (11)) : Part a and b: Correlation plot between both estimates and their logged difference,  . The graphs were calculated assuming K P,h  = const for two non-specific background levels (red and black curves) and for PM and MM probes (dotted and solid curves) assuming the survival fractions w PM,S (t) = 0.95, w MM,S (t) = 0.50 and w P,N (t) = 0.1. Neglecting washing underestimates the expression degree especially at small and large expression values. Part c: The survival fraction of bound probes depends on the intensity (or, equivalently, probe occupancy) before (t = 0) and after washing (t  >  0). The graph for t = 0 was re-plotted from Figure 5a using Eq. (17) with w max  = 0.9, w min  = 0.06,  Î³  = 1.6 and a' = 0.1. The graph for t = 6 refers to the standard number of washing cycles. It is obtained from the t = 0 graph by making the substitution logI(t) = logI(0)+log(w(t)) in the argument (see Eq. (8)). Part d shows the bias of the Langmuir approximation assuming a constant transcript concentration and variable K P,S  and thus a variable survival fraction w(  ) which has been taken from part c of the figure for t = 6. The dashed curves labelled with '(+)' and '(-)' in panels c and d refer to 50%-deviations of the washing function, log w(  ) +/-  = log w(  ) Â· (1.5) (+/-)1 , to estimate the effect of the scattering of the probe level data from the mean (compare with Figure 5a). The bias of the Langmuir-approximation strongly resembles that shown in part b. Note that the bias applies to PM and MM probes as well in this case.\"],\n",
       " 437: ['1471-2105-11-291-15-4',\n",
       "  'GFIG',\n",
       "  \"Systematic bias of 'Langmuir'-expression estimates (Eq. (12)) with respect to the estimates which consider washing (Eq. (11)) : Part a and b: Correlation plot between both estimates and their logged difference,  . The graphs were calculated assuming K P,h  = const for two non-specific background levels (red and black curves) and for PM and MM probes (dotted and solid curves) assuming the survival fractions w PM,S (t) = 0.95, w MM,S (t) = 0.50 and w P,N (t) = 0.1. Neglecting washing underestimates the expression degree especially at small and large expression values. Part c: The survival fraction of bound probes depends on the intensity (or, equivalently, probe occupancy) before (t = 0) and after washing (t  >  0). The graph for t = 0 was re-plotted from Figure 5a using Eq. (17) with w max  = 0.9, w min  = 0.06,  Î³  = 1.6 and a' = 0.1. The graph for t = 6 refers to the standard number of washing cycles. It is obtained from the t = 0 graph by making the substitution logI(t) = logI(0)+log(w(t)) in the argument (see Eq. (8)). Part d shows the bias of the Langmuir approximation assuming a constant transcript concentration and variable K P,S  and thus a variable survival fraction w(  ) which has been taken from part c of the figure for t = 6. The dashed curves labelled with '(+)' and '(-)' in panels c and d refer to 50%-deviations of the washing function, log w(  ) +/-  = log w(  ) Â· (1.5) (+/-)1 , to estimate the effect of the scattering of the probe level data from the mean (compare with Figure 5a). The bias of the Langmuir-approximation strongly resembles that shown in part b. Note that the bias applies to PM and MM probes as well in this case.\"],\n",
       " 438: ['1471-2105-11-314-10-10',\n",
       "  'GFIG',\n",
       "  'Score plots of gene expressions in ovarian cancer data . Score plots of the selected 11 genes by pAUCBoost based on ovarian cancer data. The rug plot at the bottoms of each score plot shows the observations from normal controls; the rug plot for ovarian cancer cases is described at the top of each score plot.,'],\n",
       " 439: ['1471-2105-11-314-10-11',\n",
       "  'GFIG',\n",
       "  'Score plots of gene expressions in ovarian cancer data . Score plots of the selected 11 genes by pAUCBoost based on ovarian cancer data. The rug plot at the bottoms of each score plot shows the observations from normal controls; the rug plot for ovarian cancer cases is described at the top of each score plot.,'],\n",
       " 440: ['1471-2105-11-314-10-1',\n",
       "  'GFIG',\n",
       "  'Score plots of gene expressions in ovarian cancer data . Score plots of the selected 11 genes by pAUCBoost based on ovarian cancer data. The rug plot at the bottoms of each score plot shows the observations from normal controls; the rug plot for ovarian cancer cases is described at the top of each score plot.,'],\n",
       " 441: ['1471-2105-11-314-10-2',\n",
       "  'GFIG',\n",
       "  'Score plots of gene expressions in ovarian cancer data . Score plots of the selected 11 genes by pAUCBoost based on ovarian cancer data. The rug plot at the bottoms of each score plot shows the observations from normal controls; the rug plot for ovarian cancer cases is described at the top of each score plot.,'],\n",
       " 442: ['1471-2105-11-314-10-3',\n",
       "  'GFIG',\n",
       "  'Score plots of gene expressions in ovarian cancer data . Score plots of the selected 11 genes by pAUCBoost based on ovarian cancer data. The rug plot at the bottoms of each score plot shows the observations from normal controls; the rug plot for ovarian cancer cases is described at the top of each score plot.,'],\n",
       " 443: ['1471-2105-11-314-10-4',\n",
       "  'GFIG',\n",
       "  'Score plots of gene expressions in ovarian cancer data . Score plots of the selected 11 genes by pAUCBoost based on ovarian cancer data. The rug plot at the bottoms of each score plot shows the observations from normal controls; the rug plot for ovarian cancer cases is described at the top of each score plot.,'],\n",
       " 444: ['1471-2105-11-314-10-5',\n",
       "  'GFIG',\n",
       "  'Score plots of gene expressions in ovarian cancer data . Score plots of the selected 11 genes by pAUCBoost based on ovarian cancer data. The rug plot at the bottoms of each score plot shows the observations from normal controls; the rug plot for ovarian cancer cases is described at the top of each score plot.,'],\n",
       " 445: ['1471-2105-11-314-10-6',\n",
       "  'GFIG',\n",
       "  'Score plots of gene expressions in ovarian cancer data . Score plots of the selected 11 genes by pAUCBoost based on ovarian cancer data. The rug plot at the bottoms of each score plot shows the observations from normal controls; the rug plot for ovarian cancer cases is described at the top of each score plot.,'],\n",
       " 446: ['1471-2105-11-314-10-7',\n",
       "  'GFIG',\n",
       "  'Score plots of gene expressions in ovarian cancer data . Score plots of the selected 11 genes by pAUCBoost based on ovarian cancer data. The rug plot at the bottoms of each score plot shows the observations from normal controls; the rug plot for ovarian cancer cases is described at the top of each score plot.,'],\n",
       " 447: ['1471-2105-11-314-10-8',\n",
       "  'GFIG',\n",
       "  'Score plots of gene expressions in ovarian cancer data . Score plots of the selected 11 genes by pAUCBoost based on ovarian cancer data. The rug plot at the bottoms of each score plot shows the observations from normal controls; the rug plot for ovarian cancer cases is described at the top of each score plot.,'],\n",
       " 448: ['1471-2105-11-314-10-9',\n",
       "  'GFIG',\n",
       "  'Score plots of gene expressions in ovarian cancer data . Score plots of the selected 11 genes by pAUCBoost based on ovarian cancer data. The rug plot at the bottoms of each score plot shows the observations from normal controls; the rug plot for ovarian cancer cases is described at the top of each score plot.,'],\n",
       " 449: ['1471-2105-11-314-11-1',\n",
       "  'GFIG',\n",
       "  'Score plots of gene expressions in leukemia data . Score plots of the selected 5 genes by pAUCBoost based on leukemia data. The rug plot at the bottoms of each score plot shows the observations from ALL; the rug plot for AML is described at the top of each score plot.,'],\n",
       " 450: ['1471-2105-11-314-11-2',\n",
       "  'GFIG',\n",
       "  'Score plots of gene expressions in leukemia data . Score plots of the selected 5 genes by pAUCBoost based on leukemia data. The rug plot at the bottoms of each score plot shows the observations from ALL; the rug plot for AML is described at the top of each score plot.,'],\n",
       " 451: ['1471-2105-11-314-11-3',\n",
       "  'GFIG',\n",
       "  'Score plots of gene expressions in leukemia data . Score plots of the selected 5 genes by pAUCBoost based on leukemia data. The rug plot at the bottoms of each score plot shows the observations from ALL; the rug plot for AML is described at the top of each score plot.,'],\n",
       " 452: ['1471-2105-11-314-11-4',\n",
       "  'GFIG',\n",
       "  'Score plots of gene expressions in leukemia data . Score plots of the selected 5 genes by pAUCBoost based on leukemia data. The rug plot at the bottoms of each score plot shows the observations from ALL; the rug plot for AML is described at the top of each score plot.,'],\n",
       " 453: ['1471-2105-11-314-11-5',\n",
       "  'GFIG',\n",
       "  'Score plots of gene expressions in leukemia data . Score plots of the selected 5 genes by pAUCBoost based on leukemia data. The rug plot at the bottoms of each score plot shows the observations from ALL; the rug plot for AML is described at the top of each score plot.,'],\n",
       " 454: ['1471-2105-11-320-3-1',\n",
       "  'GGEN',\n",
       "  'Caspase cleavage products identified based on MS/MS-data . Protein identifications were done with Mascot against a database of predicted caspase cleavage products created with Pripper (Vote-classifier: SVM-6-4 and RF-12-12). Complete protein sequences are shown with all the identified peptides (red), caspase cleavage motifs (yellow) and peptides identified at the cleavage site (black box). Peptide spectrum matches for the cleavage site peptides are also shown. A) Cytokeratin 18 identified from the 2-DE sample B) Potential new caspase target leukosialin identified from the iTRAQ-sample.'],\n",
       " 455: ['1471-2105-11-320-3-2',\n",
       "  'GFIG',\n",
       "  'Caspase cleavage products identified based on MS/MS-data . Protein identifications were done with Mascot against a database of predicted caspase cleavage products created with Pripper (Vote-classifier: SVM-6-4 and RF-12-12). Complete protein sequences are shown with all the identified peptides (red), caspase cleavage motifs (yellow) and peptides identified at the cleavage site (black box). Peptide spectrum matches for the cleavage site peptides are also shown. A) Cytokeratin 18 identified from the 2-DE sample B) Potential new caspase target leukosialin identified from the iTRAQ-sample.'],\n",
       " 456: ['1471-2105-11-320-3-3',\n",
       "  'GGEN',\n",
       "  'Caspase cleavage products identified based on MS/MS-data . Protein identifications were done with Mascot against a database of predicted caspase cleavage products created with Pripper (Vote-classifier: SVM-6-4 and RF-12-12). Complete protein sequences are shown with all the identified peptides (red), caspase cleavage motifs (yellow) and peptides identified at the cleavage site (black box). Peptide spectrum matches for the cleavage site peptides are also shown. A) Cytokeratin 18 identified from the 2-DE sample B) Potential new caspase target leukosialin identified from the iTRAQ-sample.'],\n",
       " 457: ['1471-2105-11-320-3-4',\n",
       "  'GFIG',\n",
       "  'Caspase cleavage products identified based on MS/MS-data . Protein identifications were done with Mascot against a database of predicted caspase cleavage products created with Pripper (Vote-classifier: SVM-6-4 and RF-12-12). Complete protein sequences are shown with all the identified peptides (red), caspase cleavage motifs (yellow) and peptides identified at the cleavage site (black box). Peptide spectrum matches for the cleavage site peptides are also shown. A) Cytokeratin 18 identified from the 2-DE sample B) Potential new caspase target leukosialin identified from the iTRAQ-sample.'],\n",
       " 458: ['1471-2105-11-333-1-1',\n",
       "  'GFIG',\n",
       "  'Amino acid distribution and solvent accessibility . The incidence of three different classes of amino acids as a function of distance from the membrane center. A) The subset of residues that have accessibility lower or equal to 25% (buried). B) The subset of residues that have accessibility higher than 25% (exposed). The coloring differentiates between polar, intermediate and hydrophobic residues. The two lower figures show the log of the difference in substitution rates between the globular and membrane regions plotted against the biological hydrophobicity value for each amino acid type. In C) the rate for buried sites is shown while in D) the rate for exposed sites is shown.'],\n",
       " 459: ['1471-2105-11-333-1-2',\n",
       "  'GFIG',\n",
       "  'Amino acid distribution and solvent accessibility . The incidence of three different classes of amino acids as a function of distance from the membrane center. A) The subset of residues that have accessibility lower or equal to 25% (buried). B) The subset of residues that have accessibility higher than 25% (exposed). The coloring differentiates between polar, intermediate and hydrophobic residues. The two lower figures show the log of the difference in substitution rates between the globular and membrane regions plotted against the biological hydrophobicity value for each amino acid type. In C) the rate for buried sites is shown while in D) the rate for exposed sites is shown.'],\n",
       " 460: ['1471-2105-11-333-1-3',\n",
       "  'GFIG',\n",
       "  'Amino acid distribution and solvent accessibility . The incidence of three different classes of amino acids as a function of distance from the membrane center. A) The subset of residues that have accessibility lower or equal to 25% (buried). B) The subset of residues that have accessibility higher than 25% (exposed). The coloring differentiates between polar, intermediate and hydrophobic residues. The two lower figures show the log of the difference in substitution rates between the globular and membrane regions plotted against the biological hydrophobicity value for each amino acid type. In C) the rate for buried sites is shown while in D) the rate for exposed sites is shown.'],\n",
       " 461: ['1471-2105-11-333-1-4',\n",
       "  'GFIG',\n",
       "  'Amino acid distribution and solvent accessibility . The incidence of three different classes of amino acids as a function of distance from the membrane center. A) The subset of residues that have accessibility lower or equal to 25% (buried). B) The subset of residues that have accessibility higher than 25% (exposed). The coloring differentiates between polar, intermediate and hydrophobic residues. The two lower figures show the log of the difference in substitution rates between the globular and membrane regions plotted against the biological hydrophobicity value for each amino acid type. In C) the rate for buried sites is shown while in D) the rate for exposed sites is shown.'],\n",
       " 462: ['1471-2105-11-333-2-1',\n",
       "  'GFIG',\n",
       "  'Relative substitution rate and solvent accessibility . A) The relative substitution rate as a function of Z-coordinate. Evolutionarily conserved sites have low values and variable sites have high independently on their distance from the membrane center. Sites that are solvent accessible and inaccessible are colored differently. B) The relative substitution rate as a function of relative accessibility for all residues show a linear relationship.,'],\n",
       " 463: ['1471-2105-11-333-2-2',\n",
       "  'GFIG',\n",
       "  'Relative substitution rate and solvent accessibility . A) The relative substitution rate as a function of Z-coordinate. Evolutionarily conserved sites have low values and variable sites have high independently on their distance from the membrane center. Sites that are solvent accessible and inaccessible are colored differently. B) The relative substitution rate as a function of relative accessibility for all residues show a linear relationship.,'],\n",
       " 464: ['1471-2105-11-345-6-1',\n",
       "  'GFIG',\n",
       "  'N50 vs. combined mismatch and no-match error rate for  de novo  assembly of real data . See main text and the caption for Table 1 for explanation of the error rates.,'],\n",
       " 465: ['1471-2105-11-345-6-2',\n",
       "  'GFIG',\n",
       "  'N50 vs. combined mismatch and no-match error rate for  de novo  assembly of real data . See main text and the caption for Table 1 for explanation of the error rates.,'],\n",
       " 466: ['1471-2105-11-345-7-1',\n",
       "  'GFIG',\n",
       "  'N50 vs. combined rearrangement error rate for  de novo  assembly of real data . See main text and the caption for Table 1 for explanation of the error rates,'],\n",
       " 467: ['1471-2105-11-345-7-2',\n",
       "  'GFIG',\n",
       "  'N50 vs. combined rearrangement error rate for  de novo  assembly of real data . See main text and the caption for Table 1 for explanation of the error rates,'],\n",
       " 468: ['1471-2105-11-374-1-1',\n",
       "  'D3DR',\n",
       "  'Four-distance definition of inter-residue interactions for the Asn-Gln pair . (a) To define the interaction, one pair of side chain atom is chosen in the first residue (Asn) and another pair is chosen in the second residue (Gln). The four distances (OE1-OD1, OE1-ND2, NE2-OD1, NE2-ND2) define mutual positions of chosen side chain atoms. (b) A 2-dimensional projection (OD1-NE2 versus ND2-OE1 distance) of the 4-distance distributions for the Asn-Gln pair. Peaks in the histogram indicate preferred distance combinations. The peak indicated by the arrow corresponds to the arrangement of side chain atoms in the panel (a). The histogram was built with a bin size of 0.25   , without smoothing. The histogram is represented as a contour plot generated with MATLAB. All 4-distance distributions can be viewed in the supporting web site  http://bioinfo.weizmann.ac.il/hunter/ .'],\n",
       " 469: ['1471-2105-11-374-1-2',\n",
       "  'GFIG',\n",
       "  'Four-distance definition of inter-residue interactions for the Asn-Gln pair . (a) To define the interaction, one pair of side chain atom is chosen in the first residue (Asn) and another pair is chosen in the second residue (Gln). The four distances (OE1-OD1, OE1-ND2, NE2-OD1, NE2-ND2) define mutual positions of chosen side chain atoms. (b) A 2-dimensional projection (OD1-NE2 versus ND2-OE1 distance) of the 4-distance distributions for the Asn-Gln pair. Peaks in the histogram indicate preferred distance combinations. The peak indicated by the arrow corresponds to the arrangement of side chain atoms in the panel (a). The histogram was built with a bin size of 0.25   , without smoothing. The histogram is represented as a contour plot generated with MATLAB. All 4-distance distributions can be viewed in the supporting web site  http://bioinfo.weizmann.ac.il/hunter/ .'],\n",
       " 470: ['1471-2105-11-402-6-1',\n",
       "  'D3DR',\n",
       "  'Visualization of the overall orientation and prediction results on CCD-IBD complex PDB: 2b4j . (a) The overall orientation of CCD-IBD complex; (b) Protein-protein interaction predictions of CCD-IBD complex. The orientation of the complex is illustrated by a smooth spline between consecutive alpha carbon positions. Left graph denotes the natural orientation, while the right one illustrates the protein-protein interaction prediction of the complex. In the right graph, blue sphere stands for TP residue, bluetint one stands for FP residue, and gold sphere demonstrates FN residue. All other residues (not shown as colored spheres) are true negatives (TN). Note that the orientation of the complex in the right graph is varied a little to clearly show the predictions of protein interface residues. Additionally each sphere represents an alpha-carbon atom of each residue. We used RasTop  http://www.geneinfinity.org/rastop/  software to display the structure of this complex.,'],\n",
       " 471: ['1471-2105-11-402-6-2',\n",
       "  'D3DR',\n",
       "  'Visualization of the overall orientation and prediction results on CCD-IBD complex PDB: 2b4j . (a) The overall orientation of CCD-IBD complex; (b) Protein-protein interaction predictions of CCD-IBD complex. The orientation of the complex is illustrated by a smooth spline between consecutive alpha carbon positions. Left graph denotes the natural orientation, while the right one illustrates the protein-protein interaction prediction of the complex. In the right graph, blue sphere stands for TP residue, bluetint one stands for FP residue, and gold sphere demonstrates FN residue. All other residues (not shown as colored spheres) are true negatives (TN). Note that the orientation of the complex in the right graph is varied a little to clearly show the predictions of protein interface residues. Additionally each sphere represents an alpha-carbon atom of each residue. We used RasTop  http://www.geneinfinity.org/rastop/  software to display the structure of this complex.,'],\n",
       " 472: ['1471-2105-11-434-6-1',\n",
       "  'GFIG',\n",
       "  'The density distributions of the outlierness value,  O , and the corresponding false discovery rates from the benchmark results for HIV-1 and HCV nucleotide sequences . For all the HIV-1 (a) and HCV (b) sequences used in the benchmark tests (Table 1), the  O  values were surveyed and plotted as the histograms that were separately normalized for the cases concordant with (bar) and discordant to (line) LANL genotypes/subtypes. After filtering out the cases having  O  >  cutoff, the discordant ones were counted as false positives. The false positive rates and the proportion of the sequences retained (coverage) were plotted against the  O  cutoff for HIV-1 (c) and HCV (d) sequences. The suggested cutoff is shown by a dashed line. The revised plots after removing the HCV sequences of suspicious genotypes are available in Supplementary Figure 5 in Additional File  3 .,'],\n",
       " 473: ['1471-2105-11-434-6-2',\n",
       "  'GFIG',\n",
       "  'The density distributions of the outlierness value,  O , and the corresponding false discovery rates from the benchmark results for HIV-1 and HCV nucleotide sequences . For all the HIV-1 (a) and HCV (b) sequences used in the benchmark tests (Table 1), the  O  values were surveyed and plotted as the histograms that were separately normalized for the cases concordant with (bar) and discordant to (line) LANL genotypes/subtypes. After filtering out the cases having  O  >  cutoff, the discordant ones were counted as false positives. The false positive rates and the proportion of the sequences retained (coverage) were plotted against the  O  cutoff for HIV-1 (c) and HCV (d) sequences. The suggested cutoff is shown by a dashed line. The revised plots after removing the HCV sequences of suspicious genotypes are available in Supplementary Figure 5 in Additional File  3 .,'],\n",
       " 474: ['1471-2105-11-434-6-3',\n",
       "  'GFIG',\n",
       "  'The density distributions of the outlierness value,  O , and the corresponding false discovery rates from the benchmark results for HIV-1 and HCV nucleotide sequences . For all the HIV-1 (a) and HCV (b) sequences used in the benchmark tests (Table 1), the  O  values were surveyed and plotted as the histograms that were separately normalized for the cases concordant with (bar) and discordant to (line) LANL genotypes/subtypes. After filtering out the cases having  O  >  cutoff, the discordant ones were counted as false positives. The false positive rates and the proportion of the sequences retained (coverage) were plotted against the  O  cutoff for HIV-1 (c) and HCV (d) sequences. The suggested cutoff is shown by a dashed line. The revised plots after removing the HCV sequences of suspicious genotypes are available in Supplementary Figure 5 in Additional File  3 .,'],\n",
       " 475: ['1471-2105-11-434-6-4',\n",
       "  'GFIG',\n",
       "  'The density distributions of the outlierness value,  O , and the corresponding false discovery rates from the benchmark results for HIV-1 and HCV nucleotide sequences . For all the HIV-1 (a) and HCV (b) sequences used in the benchmark tests (Table 1), the  O  values were surveyed and plotted as the histograms that were separately normalized for the cases concordant with (bar) and discordant to (line) LANL genotypes/subtypes. After filtering out the cases having  O  >  cutoff, the discordant ones were counted as false positives. The false positive rates and the proportion of the sequences retained (coverage) were plotted against the  O  cutoff for HIV-1 (c) and HCV (d) sequences. The suggested cutoff is shown by a dashed line. The revised plots after removing the HCV sequences of suspicious genotypes are available in Supplementary Figure 5 in Additional File  3 .,'],\n",
       " 476: ['1471-2105-11-458-4-1',\n",
       "  'GFLO',\n",
       "  \"The UMLStermFinder . (A) The underlying algorithms. A GO term is associated with a Unique Concept Identifier (CUI) using the UMLS classes file (MRSAT). The UMLS relations file (MRREL) is then used to identify CUI terms that are parental to the input GO terms, recursively traversing the DAG up to 3 levels up. All the aliases to any of these terms are also extracted. Each CUI found in the resulting sub-graph is associated with their corresponding strings, which are searched for any of the following: 'immune', 'neuro', 'inflam', 'brain', 'lymph' or 'nerve'. The outputs of the algorithm are the terms that contain these strings. Arrows of different types denote different sources from which relationships were asserted. (B) An example of how UMLStermFinder works. Using the GO term 'muscle hypertrophy' (GO:0014896) to search the UMLS ontologies, the corresponding concept identifier is found and used to find the concept's parents and synonyms (framed CUIs) of the initial concept and its parents. After all concepts are collected, they are searched for neural and/or immune-related string content (grey boxes). The UMLStermFinder finds concepts from several ontologies; by each arrow in the figure appears the name of the source ontology from which the relationship was asserted. A Perl script implementing the UMLStermFinder algorithm is freely available as Additional file  7 .,\"],\n",
       " 477: ['1471-2105-11-458-4-2',\n",
       "  'GSYS',\n",
       "  \"The UMLStermFinder . (A) The underlying algorithms. A GO term is associated with a Unique Concept Identifier (CUI) using the UMLS classes file (MRSAT). The UMLS relations file (MRREL) is then used to identify CUI terms that are parental to the input GO terms, recursively traversing the DAG up to 3 levels up. All the aliases to any of these terms are also extracted. Each CUI found in the resulting sub-graph is associated with their corresponding strings, which are searched for any of the following: 'immune', 'neuro', 'inflam', 'brain', 'lymph' or 'nerve'. The outputs of the algorithm are the terms that contain these strings. Arrows of different types denote different sources from which relationships were asserted. (B) An example of how UMLStermFinder works. Using the GO term 'muscle hypertrophy' (GO:0014896) to search the UMLS ontologies, the corresponding concept identifier is found and used to find the concept's parents and synonyms (framed CUIs) of the initial concept and its parents. After all concepts are collected, they are searched for neural and/or immune-related string content (grey boxes). The UMLStermFinder finds concepts from several ontologies; by each arrow in the figure appears the name of the source ontology from which the relationship was asserted. A Perl script implementing the UMLStermFinder algorithm is freely available as Additional file  7 .,\"],\n",
       " 478: ['1471-2105-11-469-8-1',\n",
       "  'GFIG',\n",
       "  'Differentially expressed genes mapped on the pathway ontology . Color indicates the ratio of up/down regulated genes; height shows the log value of total number of differentially expressed genes. Most of the upper side of ontology categories is down-regulated, while some categories on the lower-left side are up-regulated. Zoomed in view (center) shows the detail of amino acids biosynthesis. (bottom) shows that two superpathways (histidine, purine, and pyrimidine biosynthesis, and chorismate) have many more genes differentially expressed downward than other superpathways.,'],\n",
       " 479: ['1471-2105-11-469-8-2',\n",
       "  'GFIG',\n",
       "  'Differentially expressed genes mapped on the pathway ontology . Color indicates the ratio of up/down regulated genes; height shows the log value of total number of differentially expressed genes. Most of the upper side of ontology categories is down-regulated, while some categories on the lower-left side are up-regulated. Zoomed in view (center) shows the detail of amino acids biosynthesis. (bottom) shows that two superpathways (histidine, purine, and pyrimidine biosynthesis, and chorismate) have many more genes differentially expressed downward than other superpathways.,'],\n",
       " 480: ['1471-2105-11-469-8-3',\n",
       "  'GFIG',\n",
       "  'Differentially expressed genes mapped on the pathway ontology . Color indicates the ratio of up/down regulated genes; height shows the log value of total number of differentially expressed genes. Most of the upper side of ontology categories is down-regulated, while some categories on the lower-left side are up-regulated. Zoomed in view (center) shows the detail of amino acids biosynthesis. (bottom) shows that two superpathways (histidine, purine, and pyrimidine biosynthesis, and chorismate) have many more genes differentially expressed downward than other superpathways.,'],\n",
       " 481: ['1471-2105-11-470-7-1',\n",
       "  'GFIG',\n",
       "  'Similarity of topologies for different tree algorithms . Average level of similarity for topology comparisons (when rewiring using  network shuffle ) made for a fixed number of shared species. The average similarity for each method is different, and the trends seen in PAML show marked difference to those methods taken from Phylip. The variance of similarities for each tree construction method increases as more species are compared. This may potentially bias the score results for the topological comparisons, thus suggesting the need to be vigilant when comparing graph ensemble results with empirical data. The  network shuffle  results exhibit a distribution of comparisons over the species similar enough for this to not affect the overall conclusions (see Additional File  1 ).,'],\n",
       " 482: ['1471-2105-11-470-7-2',\n",
       "  'GFIG',\n",
       "  'Similarity of topologies for different tree algorithms . Average level of similarity for topology comparisons (when rewiring using  network shuffle ) made for a fixed number of shared species. The average similarity for each method is different, and the trends seen in PAML show marked difference to those methods taken from Phylip. The variance of similarities for each tree construction method increases as more species are compared. This may potentially bias the score results for the topological comparisons, thus suggesting the need to be vigilant when comparing graph ensemble results with empirical data. The  network shuffle  results exhibit a distribution of comparisons over the species similar enough for this to not affect the overall conclusions (see Additional File  1 ).,'],\n",
       " 483: ['1471-2105-11-470-7-3',\n",
       "  'GFIG',\n",
       "  'Similarity of topologies for different tree algorithms . Average level of similarity for topology comparisons (when rewiring using  network shuffle ) made for a fixed number of shared species. The average similarity for each method is different, and the trends seen in PAML show marked difference to those methods taken from Phylip. The variance of similarities for each tree construction method increases as more species are compared. This may potentially bias the score results for the topological comparisons, thus suggesting the need to be vigilant when comparing graph ensemble results with empirical data. The  network shuffle  results exhibit a distribution of comparisons over the species similar enough for this to not affect the overall conclusions (see Additional File  1 ).,'],\n",
       " 484: ['1471-2105-11-480-3-1',\n",
       "  'D3DR',\n",
       "  'Illustration of the inner distance . The red dashed line denotes the inner distance, which is the shortest path within the shape boundary surface that connect two landmark points  x  and  y . The right molecule is one deformation to the left one, and the relative change of the inner distances between the corresponding pair of points (e.g.  x  and  y ) during shape deformation are small. In contrast, the black bold line denotes the Euclidean distance defined as the length of the line segment between two landmark points  x  and  y . Note that the Euclidean distance does not have the property of deformation invariant in contrast to the inner distance. This is because, the Euclidean distance does not consider whether the line segment crosses shape boundaries.,'],\n",
       " 485: ['1471-2105-11-480-3-2',\n",
       "  'D3DR',\n",
       "  'Illustration of the inner distance . The red dashed line denotes the inner distance, which is the shortest path within the shape boundary surface that connect two landmark points  x  and  y . The right molecule is one deformation to the left one, and the relative change of the inner distances between the corresponding pair of points (e.g.  x  and  y ) during shape deformation are small. In contrast, the black bold line denotes the Euclidean distance defined as the length of the line segment between two landmark points  x  and  y . Note that the Euclidean distance does not have the property of deformation invariant in contrast to the inner distance. This is because, the Euclidean distance does not consider whether the line segment crosses shape boundaries.,'],\n",
       " 486: ['1471-2105-11-480-5-1',\n",
       "  'D3DR',\n",
       "  'The DD descriptor is compared to ID for the morph deformations between two conformations of GroEL: 1AON and 1KP8 . The first row shows the input four molecules with the same main chain orientation but with different surface shapes, where the arrows mark topological changes. The second row shows the DD and ID descriptors. Note that DD is not sensitive to shape deformation with topological changes, so four histograms are almost consistent; in contrast, ID is sensitive to topological changes.'],\n",
       " 487: ['1471-2105-11-480-5-2',\n",
       "  'D3DR',\n",
       "  'The DD descriptor is compared to ID for the morph deformations between two conformations of GroEL: 1AON and 1KP8 . The first row shows the input four molecules with the same main chain orientation but with different surface shapes, where the arrows mark topological changes. The second row shows the DD and ID descriptors. Note that DD is not sensitive to shape deformation with topological changes, so four histograms are almost consistent; in contrast, ID is sensitive to topological changes.'],\n",
       " 488: ['1471-2105-11-480-5-3',\n",
       "  'D3DR',\n",
       "  'The DD descriptor is compared to ID for the morph deformations between two conformations of GroEL: 1AON and 1KP8 . The first row shows the input four molecules with the same main chain orientation but with different surface shapes, where the arrows mark topological changes. The second row shows the DD and ID descriptors. Note that DD is not sensitive to shape deformation with topological changes, so four histograms are almost consistent; in contrast, ID is sensitive to topological changes.'],\n",
       " 489: ['1471-2105-11-480-5-4',\n",
       "  'D3DR',\n",
       "  'The DD descriptor is compared to ID for the morph deformations between two conformations of GroEL: 1AON and 1KP8 . The first row shows the input four molecules with the same main chain orientation but with different surface shapes, where the arrows mark topological changes. The second row shows the DD and ID descriptors. Note that DD is not sensitive to shape deformation with topological changes, so four histograms are almost consistent; in contrast, ID is sensitive to topological changes.'],\n",
       " 490: ['1471-2105-11-480-5-5',\n",
       "  'GFIG',\n",
       "  'The DD descriptor is compared to ID for the morph deformations between two conformations of GroEL: 1AON and 1KP8 . The first row shows the input four molecules with the same main chain orientation but with different surface shapes, where the arrows mark topological changes. The second row shows the DD and ID descriptors. Note that DD is not sensitive to shape deformation with topological changes, so four histograms are almost consistent; in contrast, ID is sensitive to topological changes.'],\n",
       " 491: ['1471-2105-11-480-5-6',\n",
       "  'GFIG',\n",
       "  'The DD descriptor is compared to ID for the morph deformations between two conformations of GroEL: 1AON and 1KP8 . The first row shows the input four molecules with the same main chain orientation but with different surface shapes, where the arrows mark topological changes. The second row shows the DD and ID descriptors. Note that DD is not sensitive to shape deformation with topological changes, so four histograms are almost consistent; in contrast, ID is sensitive to topological changes.'],\n",
       " 492: ['1471-2105-11-529-3-1',\n",
       "  'GFIG',\n",
       "  'Graphical illustration of how a posterior probability is calculated for a SNP . Graphs of the mixture prior  p ( g j ), conditional likelihood  h ( g j | G j ,   2 ) and conditional posterior distribution  p ( g j  | g -j, y ) as given in equations (A1) and (A2) of Additional file 1 for  Î³ = 0.05 , ï¿½ï¿½ Î» = 10 , ï¿½ï¿½  e 2 = 1  and  n = 500 . The Dirac Delta function is replaced by a DE with  Î» s  =  1000  i.e. a Spike at 0. The posterior probability  Î³ j  of SNP  j  being in LD with QTL is calculated from equation (A3) by numerical integration. Figures A and B show the distributions when  G j  is 0.19 and 0.11 respectively, where  G j  is the conditional maximum likelihood estimate of  g j .,'],\n",
       " 493: ['1471-2105-11-529-3-2',\n",
       "  'GFIG',\n",
       "  'Graphical illustration of how a posterior probability is calculated for a SNP . Graphs of the mixture prior  p ( g j ), conditional likelihood  h ( g j | G j ,   2 ) and conditional posterior distribution  p ( g j  | g -j, y ) as given in equations (A1) and (A2) of Additional file 1 for  Î³ = 0.05 , ï¿½ï¿½ Î» = 10 , ï¿½ï¿½  e 2 = 1  and  n = 500 . The Dirac Delta function is replaced by a DE with  Î» s  =  1000  i.e. a Spike at 0. The posterior probability  Î³ j  of SNP  j  being in LD with QTL is calculated from equation (A3) by numerical integration. Figures A and B show the distributions when  G j  is 0.19 and 0.11 respectively, where  G j  is the conditional maximum likelihood estimate of  g j .,'],\n",
       " 494: ['1471-2105-11-562-4-1',\n",
       "  'GFIG',\n",
       "  'Correlation with gene expression and CESSM dataset . (a) Pearson correlation between gene expression similarity and semantic similarity on a  S .  cerevisiae  dataset containing 5,000 randomly selected protein pairs are shown. (b - d) Correlation between semantic similarity and sequence, enzyme commission (EC), protein family (Pfam) similarity using the online CESSM tool. The evaluation was performed for cellular component (CC), biological process (BP), and molecular function (MF) ontologies of GO using maximum (MAX) approach for combining multiple GO annotations.'],\n",
       " 495: ['1471-2105-11-562-4-2',\n",
       "  'GFIG',\n",
       "  'Correlation with gene expression and CESSM dataset . (a) Pearson correlation between gene expression similarity and semantic similarity on a  S .  cerevisiae  dataset containing 5,000 randomly selected protein pairs are shown. (b - d) Correlation between semantic similarity and sequence, enzyme commission (EC), protein family (Pfam) similarity using the online CESSM tool. The evaluation was performed for cellular component (CC), biological process (BP), and molecular function (MF) ontologies of GO using maximum (MAX) approach for combining multiple GO annotations.'],\n",
       " 496: ['1471-2105-11-562-4-3',\n",
       "  'GFIG',\n",
       "  'Correlation with gene expression and CESSM dataset . (a) Pearson correlation between gene expression similarity and semantic similarity on a  S .  cerevisiae  dataset containing 5,000 randomly selected protein pairs are shown. (b - d) Correlation between semantic similarity and sequence, enzyme commission (EC), protein family (Pfam) similarity using the online CESSM tool. The evaluation was performed for cellular component (CC), biological process (BP), and molecular function (MF) ontologies of GO using maximum (MAX) approach for combining multiple GO annotations.'],\n",
       " 497: ['1471-2105-11-562-4-4',\n",
       "  'GFIG',\n",
       "  'Correlation with gene expression and CESSM dataset . (a) Pearson correlation between gene expression similarity and semantic similarity on a  S .  cerevisiae  dataset containing 5,000 randomly selected protein pairs are shown. (b - d) Correlation between semantic similarity and sequence, enzyme commission (EC), protein family (Pfam) similarity using the online CESSM tool. The evaluation was performed for cellular component (CC), biological process (BP), and molecular function (MF) ontologies of GO using maximum (MAX) approach for combining multiple GO annotations.'],\n",
       " 498: ['1471-2105-11-91-2-1',\n",
       "  'GSCR',\n",
       "  'BisoGenet Client input options . Screenshots of Bisogenet input options. The text box on the left enables introducing a list of identifiers. a) Identifiers tab: a combobox allows choosing the organism to which the query genes/proteins belong to and the type of biological entities to be identified: genes only, proteins only or both. b) Data Settings: a tree component enables choosing data sources and the type of experimental methods to be considered. c) Method tab: it gives two alternatives for network building, first to build a network considering as nodes only those gene/proteins identified from the input list and second to build a network that includes in addition to those identified from the input list, neighbors located up to a distances of N edges (N is defined by the user). d) Output tab: It allows the user to choose network representation in terms of genes, proteins or both with coding relations.,'],\n",
       " 499: ['1471-2105-11-91-2-2',\n",
       "  'GSCR',\n",
       "  'BisoGenet Client input options . Screenshots of Bisogenet input options. The text box on the left enables introducing a list of identifiers. a) Identifiers tab: a combobox allows choosing the organism to which the query genes/proteins belong to and the type of biological entities to be identified: genes only, proteins only or both. b) Data Settings: a tree component enables choosing data sources and the type of experimental methods to be considered. c) Method tab: it gives two alternatives for network building, first to build a network considering as nodes only those gene/proteins identified from the input list and second to build a network that includes in addition to those identified from the input list, neighbors located up to a distances of N edges (N is defined by the user). d) Output tab: It allows the user to choose network representation in terms of genes, proteins or both with coding relations.,'],\n",
       " 500: ['1471-2105-11-91-2-3',\n",
       "  'GSCR',\n",
       "  'BisoGenet Client input options . Screenshots of Bisogenet input options. The text box on the left enables introducing a list of identifiers. a) Identifiers tab: a combobox allows choosing the organism to which the query genes/proteins belong to and the type of biological entities to be identified: genes only, proteins only or both. b) Data Settings: a tree component enables choosing data sources and the type of experimental methods to be considered. c) Method tab: it gives two alternatives for network building, first to build a network considering as nodes only those gene/proteins identified from the input list and second to build a network that includes in addition to those identified from the input list, neighbors located up to a distances of N edges (N is defined by the user). d) Output tab: It allows the user to choose network representation in terms of genes, proteins or both with coding relations.,'],\n",
       " 501: ['1471-2105-11-91-2-4',\n",
       "  'GSCR',\n",
       "  'BisoGenet Client input options . Screenshots of Bisogenet input options. The text box on the left enables introducing a list of identifiers. a) Identifiers tab: a combobox allows choosing the organism to which the query genes/proteins belong to and the type of biological entities to be identified: genes only, proteins only or both. b) Data Settings: a tree component enables choosing data sources and the type of experimental methods to be considered. c) Method tab: it gives two alternatives for network building, first to build a network considering as nodes only those gene/proteins identified from the input list and second to build a network that includes in addition to those identified from the input list, neighbors located up to a distances of N edges (N is defined by the user). d) Output tab: It allows the user to choose network representation in terms of genes, proteins or both with coding relations.,'],\n",
       " 502: ['1471-2105-11-S11-S2-1-1',\n",
       "  'D3DR',\n",
       "  'Examples of four different representations of the protein (1hdmA).  A is surface representation using all-atoms (AASurf). B is a backbone representation using all heavy atoms in the main-chain (CACNO). C and D are simplified backbone representations which are composed of the atoms C Î± , C, and N (CACN) and only the C Î±  atoms, respectively. E shows the 3DZD invariant values of the four representations.'],\n",
       " 503: ['1471-2105-11-S11-S2-1-2',\n",
       "  'D3DR',\n",
       "  'Examples of four different representations of the protein (1hdmA).  A is surface representation using all-atoms (AASurf). B is a backbone representation using all heavy atoms in the main-chain (CACNO). C and D are simplified backbone representations which are composed of the atoms C Î± , C, and N (CACN) and only the C Î±  atoms, respectively. E shows the 3DZD invariant values of the four representations.'],\n",
       " 504: ['1471-2105-11-S11-S2-1-3',\n",
       "  'D3DR',\n",
       "  'Examples of four different representations of the protein (1hdmA).  A is surface representation using all-atoms (AASurf). B is a backbone representation using all heavy atoms in the main-chain (CACNO). C and D are simplified backbone representations which are composed of the atoms C Î± , C, and N (CACN) and only the C Î±  atoms, respectively. E shows the 3DZD invariant values of the four representations.'],\n",
       " 505: ['1471-2105-11-S11-S2-1-4',\n",
       "  'D3DR',\n",
       "  'Examples of four different representations of the protein (1hdmA).  A is surface representation using all-atoms (AASurf). B is a backbone representation using all heavy atoms in the main-chain (CACNO). C and D are simplified backbone representations which are composed of the atoms C Î± , C, and N (CACN) and only the C Î±  atoms, respectively. E shows the 3DZD invariant values of the four representations.'],\n",
       " 506: ['1471-2105-11-S11-S2-1-5',\n",
       "  'GFIG',\n",
       "  'Examples of four different representations of the protein (1hdmA).  A is surface representation using all-atoms (AASurf). B is a backbone representation using all heavy atoms in the main-chain (CACNO). C and D are simplified backbone representations which are composed of the atoms C Î± , C, and N (CACN) and only the C Î±  atoms, respectively. E shows the 3DZD invariant values of the four representations.'],\n",
       " 507: ['1471-2105-11-S11-S2-3-1',\n",
       "  'GFIG',\n",
       "  'Effect of the sphericity and the tail-like structure on the retrieval performance.  The effect of the sphericity and the tail-like structure to the precision-recall AUC of AASurf and CACNO representations is shown. A shows the effect of the sphericity, and B shows the effect of the number of the residues which reside further than two times of the radius of gyration (tail-like structures).'],\n",
       " 508: ['1471-2105-11-S11-S2-3-2',\n",
       "  'GFIG',\n",
       "  'Effect of the sphericity and the tail-like structure on the retrieval performance.  The effect of the sphericity and the tail-like structure to the precision-recall AUC of AASurf and CACNO representations is shown. A shows the effect of the sphericity, and B shows the effect of the number of the residues which reside further than two times of the radius of gyration (tail-like structures).'],\n",
       " 509: ['1471-2105-11-S11-S4-1-1',\n",
       "  'GMAT',\n",
       "  'Illustration of the minimum CFG for a string  ï¿½ï¿½ abcabcab ï¿½ï¿½  In the minimum CFG, the substring  ï¿½ï¿½ abc ï¿½ï¿½  is generated from a nonterminal symbol  A abc . Then,  z abc  = 1,  x 1,3  = 1, and  x 4,6  = 1. To generate  ï¿½ï¿½ abcabc ï¿½ï¿½ , that is,  x 1,6  = 1, at least one variable of  y 1, k ,6  ( k  = 1,  ,5) must be 1, that is, the substring  s 1,6  must be divided into  s 1, k  and  s k + 1,6 . Here,  y 1,3,6  = 1 because  x 1,3  =  x 4,6  = 1. Then, the production rule  A abcabc     A abc A abc  appears.'],\n",
       " 510: ['1471-2105-11-S11-S4-1-2',\n",
       "  'GTAB',\n",
       "  'Illustration of the minimum CFG for a string  ï¿½ï¿½ abcabcab ï¿½ï¿½  In the minimum CFG, the substring  ï¿½ï¿½ abc ï¿½ï¿½  is generated from a nonterminal symbol  A abc . Then,  z abc  = 1,  x 1,3  = 1, and  x 4,6  = 1. To generate  ï¿½ï¿½ abcabc ï¿½ï¿½ , that is,  x 1,6  = 1, at least one variable of  y 1, k ,6  ( k  = 1,  ,5) must be 1, that is, the substring  s 1,6  must be divided into  s 1, k  and  s k + 1,6 . Here,  y 1,3,6  = 1 because  x 1,3  =  x 4,6  = 1. Then, the production rule  A abcabc     A abc A abc  appears.'],\n",
       " 511: ['1471-2105-11-S11-S4-1-3',\n",
       "  'GMAT',\n",
       "  'Illustration of the minimum CFG for a string  ï¿½ï¿½ abcabcab ï¿½ï¿½  In the minimum CFG, the substring  ï¿½ï¿½ abc ï¿½ï¿½  is generated from a nonterminal symbol  A abc . Then,  z abc  = 1,  x 1,3  = 1, and  x 4,6  = 1. To generate  ï¿½ï¿½ abcabc ï¿½ï¿½ , that is,  x 1,6  = 1, at least one variable of  y 1, k ,6  ( k  = 1,  ,5) must be 1, that is, the substring  s 1,6  must be divided into  s 1, k  and  s k + 1,6 . Here,  y 1,3,6  = 1 because  x 1,3  =  x 4,6  = 1. Then, the production rule  A abcabc     A abc A abc  appears.'],\n",
       " 512: ['1471-2105-11-S11-S4-1-4',\n",
       "  'GMAT',\n",
       "  'Illustration of the minimum CFG for a string  ï¿½ï¿½ abcabcab ï¿½ï¿½  In the minimum CFG, the substring  ï¿½ï¿½ abc ï¿½ï¿½  is generated from a nonterminal symbol  A abc . Then,  z abc  = 1,  x 1,3  = 1, and  x 4,6  = 1. To generate  ï¿½ï¿½ abcabc ï¿½ï¿½ , that is,  x 1,6  = 1, at least one variable of  y 1, k ,6  ( k  = 1,  ,5) must be 1, that is, the substring  s 1,6  must be divided into  s 1, k  and  s k + 1,6 . Here,  y 1,3,6  = 1 because  x 1,3  =  x 4,6  = 1. Then, the production rule  A abcabc     A abc A abc  appears.'],\n",
       " 513: ['1471-2105-11-S11-S4-1-5',\n",
       "  'GMAT',\n",
       "  'Illustration of the minimum CFG for a string  ï¿½ï¿½ abcabcab ï¿½ï¿½  In the minimum CFG, the substring  ï¿½ï¿½ abc ï¿½ï¿½  is generated from a nonterminal symbol  A abc . Then,  z abc  = 1,  x 1,3  = 1, and  x 4,6  = 1. To generate  ï¿½ï¿½ abcabc ï¿½ï¿½ , that is,  x 1,6  = 1, at least one variable of  y 1, k ,6  ( k  = 1,  ,5) must be 1, that is, the substring  s 1,6  must be divided into  s 1, k  and  s k + 1,6 . Here,  y 1,3,6  = 1 because  x 1,3  =  x 4,6  = 1. Then, the production rule  A abcabc     A abc A abc  appears.'],\n",
       " 514: ['1471-2105-11-S11-S4-1-6',\n",
       "  'GMAT',\n",
       "  'Illustration of the minimum CFG for a string  ï¿½ï¿½ abcabcab ï¿½ï¿½  In the minimum CFG, the substring  ï¿½ï¿½ abc ï¿½ï¿½  is generated from a nonterminal symbol  A abc . Then,  z abc  = 1,  x 1,3  = 1, and  x 4,6  = 1. To generate  ï¿½ï¿½ abcabc ï¿½ï¿½ , that is,  x 1,6  = 1, at least one variable of  y 1, k ,6  ( k  = 1,  ,5) must be 1, that is, the substring  s 1,6  must be divided into  s 1, k  and  s k + 1,6 . Here,  y 1,3,6  = 1 because  x 1,3  =  x 4,6  = 1. Then, the production rule  A abcabc     A abc A abc  appears.'],\n",
       " 515: ['1471-2105-11-S1-S3-2-1',\n",
       "  'D3DR',\n",
       "  'Example of the surface predicted by the present method . This example employs the two subunits of RNA ploymerase II (PDB ID:  2HZM ), Med18 (chain B) and Med20 (chain A), to show the predicted surface relative to the interface residues. The protein chain in  spacefill  mode is the target subunit used in surface identification; the protein chain that is displayed in  stick  mode is treated as the interacting partner of the target subunit. The predicted surface that overlaps the interface residues is shown in yellow, and the non-overlapping region is shown in red. Med18 is the target subunit in (a), and Med20 is the target subunit in (b).,'],\n",
       " 516: ['1471-2105-11-S1-S3-2-2',\n",
       "  'D3DR',\n",
       "  'Example of the surface predicted by the present method . This example employs the two subunits of RNA ploymerase II (PDB ID:  2HZM ), Med18 (chain B) and Med20 (chain A), to show the predicted surface relative to the interface residues. The protein chain in  spacefill  mode is the target subunit used in surface identification; the protein chain that is displayed in  stick  mode is treated as the interacting partner of the target subunit. The predicted surface that overlaps the interface residues is shown in yellow, and the non-overlapping region is shown in red. Med18 is the target subunit in (a), and Med20 is the target subunit in (b).,'],\n",
       " 517: ['1471-2105-11-S1-S8-2-1',\n",
       "  'GFIG',\n",
       "  '2-fold cross-validation performances of prediction methods .  (A)  ROC curves of the regulatory predictions using different methods with a knowledge base (KBR cutoff and KBR SVM) and without a knowledge base(NR cutoff).  (B)  ROC curves of the CLR algorithms with and without prior knowledge, and the expression classifier of the SEREND algorithm.'],\n",
       " 518: ['1471-2105-11-S1-S8-2-2',\n",
       "  'GFIG',\n",
       "  '2-fold cross-validation performances of prediction methods .  (A)  ROC curves of the regulatory predictions using different methods with a knowledge base (KBR cutoff and KBR SVM) and without a knowledge base(NR cutoff).  (B)  ROC curves of the CLR algorithms with and without prior knowledge, and the expression classifier of the SEREND algorithm.'],\n",
       " 519: ['1471-2105-11-S6-S11-10-1',\n",
       "  'GNCP',\n",
       "  'Left side view of the binocular display and right side hinges of the arm, and Binocular VICON Model (VST file).'],\n",
       " 520: ['1471-2105-11-S6-S11-10-2',\n",
       "  'GNCP',\n",
       "  'Left side view of the binocular display and right side hinges of the arm, and Binocular VICON Model (VST file).'],\n",
       " 521: ['1471-2105-11-S6-S11-10-3',\n",
       "  'D3DR',\n",
       "  'Left side view of the binocular display and right side hinges of the arm, and Binocular VICON Model (VST file).'],\n",
       " 522: ['1471-2105-11-S6-S23-4-1',\n",
       "  'DVDM',\n",
       "  'The mask of Figure  3   is applied to a raw dermoscopy image (a).  The process removed the noise in upper-right corner as shown in (b). The mask of Figure 3  is employed as denoising filter on current image. Also, most of the bubbles are disappeared on the left of the original image (a).'],\n",
       " 523: ['1471-2105-11-S6-S23-4-2',\n",
       "  'DMLI',\n",
       "  'The mask of Figure  3   is applied to a raw dermoscopy image (a).  The process removed the noise in upper-right corner as shown in (b). The mask of Figure 3  is employed as denoising filter on current image. Also, most of the bubbles are disappeared on the left of the original image (a).'],\n",
       " 524: ['1471-2105-12-14-1-1',\n",
       "  'GFIG',\n",
       "  'The frequency distributions of a) degree; b) closeness; c) betweenness; and d) clustering coefficient for disease-associated and polymorphism SAPs . In Figure 1c, the  x -axis indicates the  betweenness  values scaled by sequence length. Here, the intervals for the frequency calculation were set to 1,0.03, 1 and 0.1.,'],\n",
       " 525: ['1471-2105-12-14-1-2',\n",
       "  'GFIG',\n",
       "  'The frequency distributions of a) degree; b) closeness; c) betweenness; and d) clustering coefficient for disease-associated and polymorphism SAPs . In Figure 1c, the  x -axis indicates the  betweenness  values scaled by sequence length. Here, the intervals for the frequency calculation were set to 1,0.03, 1 and 0.1.,'],\n",
       " 526: ['1471-2105-12-14-1-3',\n",
       "  'GFIG',\n",
       "  'The frequency distributions of a) degree; b) closeness; c) betweenness; and d) clustering coefficient for disease-associated and polymorphism SAPs . In Figure 1c, the  x -axis indicates the  betweenness  values scaled by sequence length. Here, the intervals for the frequency calculation were set to 1,0.03, 1 and 0.1.,'],\n",
       " 527: ['1471-2105-12-14-1-4',\n",
       "  'GFIG',\n",
       "  'The frequency distributions of a) degree; b) closeness; c) betweenness; and d) clustering coefficient for disease-associated and polymorphism SAPs . In Figure 1c, the  x -axis indicates the  betweenness  values scaled by sequence length. Here, the intervals for the frequency calculation were set to 1,0.03, 1 and 0.1.,'],\n",
       " 528: ['1471-2105-12-247-9-1',\n",
       "  'D3DR',\n",
       "  'Illustration of the functional role of three words . A:  DODQ  corresponds to calcium-binding sites. B:  YUOD  contains residues involved in nucleotide-binding sites. C:  RUDO  contains residues involved in SAH/SAM-binding sites. Structural words are highlighted in red and ligands in blue.,'],\n",
       " 529: ['1471-2105-12-247-9-2',\n",
       "  'D3DR',\n",
       "  'Illustration of the functional role of three words . A:  DODQ  corresponds to calcium-binding sites. B:  YUOD  contains residues involved in nucleotide-binding sites. C:  RUDO  contains residues involved in SAH/SAM-binding sites. Structural words are highlighted in red and ligands in blue.,'],\n",
       " 530: ['1471-2105-12-247-9-3',\n",
       "  'D3DR',\n",
       "  'Illustration of the functional role of three words . A:  DODQ  corresponds to calcium-binding sites. B:  YUOD  contains residues involved in nucleotide-binding sites. C:  RUDO  contains residues involved in SAH/SAM-binding sites. Structural words are highlighted in red and ligands in blue.,'],\n",
       " 531: ['1471-2105-12-253-1-1',\n",
       "  'GFIG',\n",
       "  'Choosing the number of dimensions in sPLS-DA . Estimated classification error rates for Brain and SNP (10 cross-validation averaged 10 times) with respect to each sPLS-DA dimension. The different lines represent the number of variables selected on each dimension (going from 5 to  p ).,'],\n",
       " 532: ['1471-2105-12-253-1-2',\n",
       "  'GFIG',\n",
       "  'Choosing the number of dimensions in sPLS-DA . Estimated classification error rates for Brain and SNP (10 cross-validation averaged 10 times) with respect to each sPLS-DA dimension. The different lines represent the number of variables selected on each dimension (going from 5 to  p ).,'],\n",
       " 533: ['1471-2105-12-253-2-1',\n",
       "  'GFIG',\n",
       "  'Comparisons of the classification performance with other variable selection approaches . Estimated classification error rates for Leukemia, SRBCT, Brain, GCM and the SNP data set (10 cross-validation averaged 10 times) with respect to the number of selected genes (from 5 to  p ) for the wrapper approaches and the sparse exploratory approaches.'],\n",
       " 534: ['1471-2105-12-253-2-2',\n",
       "  'GFIG',\n",
       "  'Comparisons of the classification performance with other variable selection approaches . Estimated classification error rates for Leukemia, SRBCT, Brain, GCM and the SNP data set (10 cross-validation averaged 10 times) with respect to the number of selected genes (from 5 to  p ) for the wrapper approaches and the sparse exploratory approaches.'],\n",
       " 535: ['1471-2105-12-253-2-3',\n",
       "  'GFIG',\n",
       "  'Comparisons of the classification performance with other variable selection approaches . Estimated classification error rates for Leukemia, SRBCT, Brain, GCM and the SNP data set (10 cross-validation averaged 10 times) with respect to the number of selected genes (from 5 to  p ) for the wrapper approaches and the sparse exploratory approaches.'],\n",
       " 536: ['1471-2105-12-253-2-4',\n",
       "  'GFIG',\n",
       "  'Comparisons of the classification performance with other variable selection approaches . Estimated classification error rates for Leukemia, SRBCT, Brain, GCM and the SNP data set (10 cross-validation averaged 10 times) with respect to the number of selected genes (from 5 to  p ) for the wrapper approaches and the sparse exploratory approaches.'],\n",
       " 537: ['1471-2105-12-253-2-5',\n",
       "  'GFIG',\n",
       "  'Comparisons of the classification performance with other variable selection approaches . Estimated classification error rates for Leukemia, SRBCT, Brain, GCM and the SNP data set (10 cross-validation averaged 10 times) with respect to the number of selected genes (from 5 to  p ) for the wrapper approaches and the sparse exploratory approaches.'],\n",
       " 538: ['1471-2105-12-38-5-1',\n",
       "  'GFIG',\n",
       "  'OTU construction accuracy for the Titanium data set as a function of percent sequence difference for the different noise removal algorithms . Results are given for AmpliconNoise with   s  = 0.01 (A) and   s  = 0.04 (B), single-linkage preclustering at 2% (C), and the DeNoiser algorithm (D). Reads classified as chimeric by comparison with the references were removed. The solid black portion gives the number of OTUs comprised of reference sequences and denoised pyrosequenced reads. These are good OTUs. The grey area OTUs formed only from reference sequences. These correspond to true OTUs that are missed. The diagonal shaded area those OTUs containing only pyrosequenced reads and hence are noise.,'],\n",
       " 539: ['1471-2105-12-38-5-2',\n",
       "  'GFIG',\n",
       "  'OTU construction accuracy for the Titanium data set as a function of percent sequence difference for the different noise removal algorithms . Results are given for AmpliconNoise with   s  = 0.01 (A) and   s  = 0.04 (B), single-linkage preclustering at 2% (C), and the DeNoiser algorithm (D). Reads classified as chimeric by comparison with the references were removed. The solid black portion gives the number of OTUs comprised of reference sequences and denoised pyrosequenced reads. These are good OTUs. The grey area OTUs formed only from reference sequences. These correspond to true OTUs that are missed. The diagonal shaded area those OTUs containing only pyrosequenced reads and hence are noise.,'],\n",
       " 540: ['1471-2105-12-38-5-3',\n",
       "  'GFIG',\n",
       "  'OTU construction accuracy for the Titanium data set as a function of percent sequence difference for the different noise removal algorithms . Results are given for AmpliconNoise with   s  = 0.01 (A) and   s  = 0.04 (B), single-linkage preclustering at 2% (C), and the DeNoiser algorithm (D). Reads classified as chimeric by comparison with the references were removed. The solid black portion gives the number of OTUs comprised of reference sequences and denoised pyrosequenced reads. These are good OTUs. The grey area OTUs formed only from reference sequences. These correspond to true OTUs that are missed. The diagonal shaded area those OTUs containing only pyrosequenced reads and hence are noise.,'],\n",
       " 541: ['1471-2105-12-38-5-4',\n",
       "  'GFIG',\n",
       "  'OTU construction accuracy for the Titanium data set as a function of percent sequence difference for the different noise removal algorithms . Results are given for AmpliconNoise with   s  = 0.01 (A) and   s  = 0.04 (B), single-linkage preclustering at 2% (C), and the DeNoiser algorithm (D). Reads classified as chimeric by comparison with the references were removed. The solid black portion gives the number of OTUs comprised of reference sequences and denoised pyrosequenced reads. These are good OTUs. The grey area OTUs formed only from reference sequences. These correspond to true OTUs that are missed. The diagonal shaded area those OTUs containing only pyrosequenced reads and hence are noise.,'],\n",
       " 542: ['1471-2105-12-39-8-1',\n",
       "  'GFIG',\n",
       "  'Number of binding sites identified in treatment and control sample using different sets of parameters . (A - C) Box-plots show the ratios of predicted binding site counts in treatment and control achieved with different parameter choices. Lower values suggest higher specificity. (D) The number of predicted nucleosomes in the control sample is plotted versus the number of nucleosomes identified from the mononucleosome sample. Different colours correspond to different background rate cut-offs while different symbols indicate different cut-off levels for support region rates.,'],\n",
       " 543: ['1471-2105-12-39-8-2',\n",
       "  'GFIG',\n",
       "  'Number of binding sites identified in treatment and control sample using different sets of parameters . (A - C) Box-plots show the ratios of predicted binding site counts in treatment and control achieved with different parameter choices. Lower values suggest higher specificity. (D) The number of predicted nucleosomes in the control sample is plotted versus the number of nucleosomes identified from the mononucleosome sample. Different colours correspond to different background rate cut-offs while different symbols indicate different cut-off levels for support region rates.,'],\n",
       " 544: ['1471-2105-12-39-8-3',\n",
       "  'GFIG',\n",
       "  'Number of binding sites identified in treatment and control sample using different sets of parameters . (A - C) Box-plots show the ratios of predicted binding site counts in treatment and control achieved with different parameter choices. Lower values suggest higher specificity. (D) The number of predicted nucleosomes in the control sample is plotted versus the number of nucleosomes identified from the mononucleosome sample. Different colours correspond to different background rate cut-offs while different symbols indicate different cut-off levels for support region rates.,'],\n",
       " 545: ['1471-2105-12-39-8-4',\n",
       "  'GFIG',\n",
       "  'Number of binding sites identified in treatment and control sample using different sets of parameters . (A - C) Box-plots show the ratios of predicted binding site counts in treatment and control achieved with different parameter choices. Lower values suggest higher specificity. (D) The number of predicted nucleosomes in the control sample is plotted versus the number of nucleosomes identified from the mononucleosome sample. Different colours correspond to different background rate cut-offs while different symbols indicate different cut-off levels for support region rates.,'],\n",
       " 546: ['1471-2105-12-40-4-1',\n",
       "  'GFIG',\n",
       "  'Transcription feature plots of predicted ncRNA transcripts . Transcription feature plots of predicted ncRNA transcripts (blue arrows) covering annotated ribosomal RNAs (red arrows). For a detailed legend see figure 2. In the first example the RNAz prediction is shorter than the annotated ncRNA (left), while it is much longer in the second example (right). In both cases the prediction of the transcript boundaries were improved by  NOCO RNAc.'],\n",
       " 547: ['1471-2105-12-40-4-2',\n",
       "  'GFIG',\n",
       "  'Transcription feature plots of predicted ncRNA transcripts . Transcription feature plots of predicted ncRNA transcripts (blue arrows) covering annotated ribosomal RNAs (red arrows). For a detailed legend see figure 2. In the first example the RNAz prediction is shorter than the annotated ncRNA (left), while it is much longer in the second example (right). In both cases the prediction of the transcript boundaries were improved by  NOCO RNAc.'],\n",
       " 548: ['1471-2105-12-5-3-1',\n",
       "  'GFIG',\n",
       "  'Performance of the reconstruction algorithm (simulated data) . Simulation results (average and standard error) for quasispecies reconstruction algorithm runs ( 10 ) for parameter set of  n = 1,100, w+1 = 7, k = 200, q = 50, x = 10 , sample size of  10,0, varying  m  and guide distribution selection policy (continuous line for maximum likelihood, dashed for random choice). Panel (a) depicts proportion of correct reconstructions, while panel (b) proportion of total reconstructions.,'],\n",
       " 549: ['1471-2105-12-5-3-2',\n",
       "  'GFIG',\n",
       "  'Performance of the reconstruction algorithm (simulated data) . Simulation results (average and standard error) for quasispecies reconstruction algorithm runs ( 10 ) for parameter set of  n = 1,100, w+1 = 7, k = 200, q = 50, x = 10 , sample size of  10,0, varying  m  and guide distribution selection policy (continuous line for maximum likelihood, dashed for random choice). Panel (a) depicts proportion of correct reconstructions, while panel (b) proportion of total reconstructions.,'],\n",
       " 550: ['1471-2105-5-207-7-1',\n",
       "  'GFIG',\n",
       "  '(A) We plot the results of the hierarchical model for  C T ( k ) for different configurations. 3 initial nodes and up to 7 iterations (circles), 4 initial nodes and up to 5 iterations (triangles), 5 initial nodes and up to 4 iterations (squares). Prom top to bottom (3 initial nodes (black), 4 initial nodes (red), 5 initial nodes (green)), we show with lines the results of  C T ( k ) obtained by means of Eq. (8). (B) The lines have the same meaning as before and the diamonds correspond to the experimental data for reactions from the KEGG database [14]. Experimental data involves 163 organisms. Circles (red): Experimental data binned into seven intervals according to degree (1  < k    8  < , ..., 128  < k    256, 256  < k    512). Figures in log-linear scale.,'],\n",
       " 551: ['1471-2105-5-207-7-2',\n",
       "  'GFIG',\n",
       "  '(A) We plot the results of the hierarchical model for  C T ( k ) for different configurations. 3 initial nodes and up to 7 iterations (circles), 4 initial nodes and up to 5 iterations (triangles), 5 initial nodes and up to 4 iterations (squares). Prom top to bottom (3 initial nodes (black), 4 initial nodes (red), 5 initial nodes (green)), we show with lines the results of  C T ( k ) obtained by means of Eq. (8). (B) The lines have the same meaning as before and the diamonds correspond to the experimental data for reactions from the KEGG database [14]. Experimental data involves 163 organisms. Circles (red): Experimental data binned into seven intervals according to degree (1  < k    8  < , ..., 128  < k    256, 256  < k    512). Figures in log-linear scale.,'],\n",
       " 552: ['1471-2105-5-62-1-1',\n",
       "  'GSYS',\n",
       "  'The maximal coverage alignment problem : For a query sequence to which multiple local alignments (through standard alignment methods or database search) have been generated,\" we wish to pick the combination of local alignments that covers the maximum proportion of the query sequence.  A)  Representing a series of alignments as a graph. Each lettered alignment is shown as a node. Letters indicate the order of these sequences in the reference sequence (i.e. the fourth alignment is letter \"\"a\"\" because it occurs first in the reference sequence). Edges join nodes with permissible overlap. The path through this graph corresponding to the OAC is shown with darkened arrows. At right is a representation of some of the data stored by each node and edge. The five most important pieces of information stored in the nodes are the starting position and length of the alignment ( start  and  len )\", the alignment score (if that is used as an objective function),\" the sequence alignment for the node in question (needed when overlap is permitted and alignment scores are the objective function) and the list of \"\"in nodes:\"\" those nodes with a directed edge leading to the current node. Edges store the extent of overlap between the nodes they connect (which is also easily calculated from the two nodes\\' values of  start  and  len .  B)  Pseudo-code implementation of the last step in our algorithm\", the depth-first search (see text). The dot (.) operator represents access to data structure members.,'],\n",
       " 553: ['1471-2105-5-62-1-2',\n",
       "  'GPLI',\n",
       "  'The maximal coverage alignment problem : For a query sequence to which multiple local alignments (through standard alignment methods or database search) have been generated,\" we wish to pick the combination of local alignments that covers the maximum proportion of the query sequence.  A)  Representing a series of alignments as a graph. Each lettered alignment is shown as a node. Letters indicate the order of these sequences in the reference sequence (i.e. the fourth alignment is letter \"\"a\"\" because it occurs first in the reference sequence). Edges join nodes with permissible overlap. The path through this graph corresponding to the OAC is shown with darkened arrows. At right is a representation of some of the data stored by each node and edge. The five most important pieces of information stored in the nodes are the starting position and length of the alignment ( start  and  len )\", the alignment score (if that is used as an objective function),\" the sequence alignment for the node in question (needed when overlap is permitted and alignment scores are the objective function) and the list of \"\"in nodes:\"\" those nodes with a directed edge leading to the current node. Edges store the extent of overlap between the nodes they connect (which is also easily calculated from the two nodes\\' values of  start  and  len .  B)  Pseudo-code implementation of the last step in our algorithm\", the depth-first search (see text). The dot (.) operator represents access to data structure members.,'],\n",
       " 554: ['1471-2105-6-131-1-1',\n",
       "  'GFIG',\n",
       "  'Venn diagram of the prediction sets.  Venn diagram obtained from the comparison of the three prediction sets: Ensembl (E), SGP2 (S) and TWINSCAN (T). (A) Description of each subset in the Venn diagram. (B) Total number of intron assemblies (IAs) populating each subset. (C) Percentage of experimentally verified IAs for each subset (top) and number of assayed IAs (bottom). (D) Percentage of correctly predicted splice junctions (top) from the experimentally verified IAs (bottom).'],\n",
       " 555: ['1471-2105-6-131-1-2',\n",
       "  'GFIG',\n",
       "  'Venn diagram of the prediction sets.  Venn diagram obtained from the comparison of the three prediction sets: Ensembl (E), SGP2 (S) and TWINSCAN (T). (A) Description of each subset in the Venn diagram. (B) Total number of intron assemblies (IAs) populating each subset. (C) Percentage of experimentally verified IAs for each subset (top) and number of assayed IAs (bottom). (D) Percentage of correctly predicted splice junctions (top) from the experimentally verified IAs (bottom).'],\n",
       " 556: ['1471-2105-6-131-1-3',\n",
       "  'GFIG',\n",
       "  'Venn diagram of the prediction sets.  Venn diagram obtained from the comparison of the three prediction sets: Ensembl (E), SGP2 (S) and TWINSCAN (T). (A) Description of each subset in the Venn diagram. (B) Total number of intron assemblies (IAs) populating each subset. (C) Percentage of experimentally verified IAs for each subset (top) and number of assayed IAs (bottom). (D) Percentage of correctly predicted splice junctions (top) from the experimentally verified IAs (bottom).'],\n",
       " 557: ['1471-2105-6-131-1-4',\n",
       "  'GFIG',\n",
       "  'Venn diagram of the prediction sets.  Venn diagram obtained from the comparison of the three prediction sets: Ensembl (E), SGP2 (S) and TWINSCAN (T). (A) Description of each subset in the Venn diagram. (B) Total number of intron assemblies (IAs) populating each subset. (C) Percentage of experimentally verified IAs for each subset (top) and number of assayed IAs (bottom). (D) Percentage of correctly predicted splice junctions (top) from the experimentally verified IAs (bottom).'],\n",
       " 558: ['1471-2105-6-131-2-1',\n",
       "  'GFIG',\n",
       "  \"Comparison of two predictions.  From the comparison of two predictions (a) we obtain three differentiated sets of intron assemblies (IAs): the set of IAs that are identical in both transcripts ('A and B'), and two set of the IAs that are in one prediction but not in the other ('A not B' and 'B not A'). When two sets have the same intron with different outside boundaries for the flanking exons these boundaries are taken from the intersection of the exons. Ensembl predictions (b) have in general more than one transcript per gene (two top yellow tracks). The intersecting intron assemblies (IAs) are therefore defined as the longest non-redundant IAs common between the transcripts from either prediction. For the novel IAs we take the longest non-redundant IAs in one that are not present in the other set.\"],\n",
       " 559: ['1471-2105-6-131-2-2',\n",
       "  'GFIG',\n",
       "  \"Comparison of two predictions.  From the comparison of two predictions (a) we obtain three differentiated sets of intron assemblies (IAs): the set of IAs that are identical in both transcripts ('A and B'), and two set of the IAs that are in one prediction but not in the other ('A not B' and 'B not A'). When two sets have the same intron with different outside boundaries for the flanking exons these boundaries are taken from the intersection of the exons. Ensembl predictions (b) have in general more than one transcript per gene (two top yellow tracks). The intersecting intron assemblies (IAs) are therefore defined as the longest non-redundant IAs common between the transcripts from either prediction. For the novel IAs we take the longest non-redundant IAs in one that are not present in the other set.\"],\n",
       " 560: ['1471-2105-6-131-2-3',\n",
       "  'GGEN',\n",
       "  \"Comparison of two predictions.  From the comparison of two predictions (a) we obtain three differentiated sets of intron assemblies (IAs): the set of IAs that are identical in both transcripts ('A and B'), and two set of the IAs that are in one prediction but not in the other ('A not B' and 'B not A'). When two sets have the same intron with different outside boundaries for the flanking exons these boundaries are taken from the intersection of the exons. Ensembl predictions (b) have in general more than one transcript per gene (two top yellow tracks). The intersecting intron assemblies (IAs) are therefore defined as the longest non-redundant IAs common between the transcripts from either prediction. For the novel IAs we take the longest non-redundant IAs in one that are not present in the other set.\"],\n",
       " 561: ['1471-2105-6-131-2-4',\n",
       "  'GFIG',\n",
       "  \"Comparison of two predictions.  From the comparison of two predictions (a) we obtain three differentiated sets of intron assemblies (IAs): the set of IAs that are identical in both transcripts ('A and B'), and two set of the IAs that are in one prediction but not in the other ('A not B' and 'B not A'). When two sets have the same intron with different outside boundaries for the flanking exons these boundaries are taken from the intersection of the exons. Ensembl predictions (b) have in general more than one transcript per gene (two top yellow tracks). The intersecting intron assemblies (IAs) are therefore defined as the longest non-redundant IAs common between the transcripts from either prediction. For the novel IAs we take the longest non-redundant IAs in one that are not present in the other set.\"],\n",
       " 562: ['1471-2105-6-131-3-1',\n",
       "  'GFIG',\n",
       "  'Ensembl extensions . Exon extensions to Ensembl predictions can be obtained from exons predicted by TWINSCAN and/or SGP2. These exons can either (a) be part of a transcript with exons in common with the Ensembl transcript (linked) or (b) be part of a close but non-overlapping transcript (unlinked).,'],\n",
       " 563: ['1471-2105-6-131-3-2',\n",
       "  'GSCR',\n",
       "  'Ensembl extensions . Exon extensions to Ensembl predictions can be obtained from exons predicted by TWINSCAN and/or SGP2. These exons can either (a) be part of a transcript with exons in common with the Ensembl transcript (linked) or (b) be part of a close but non-overlapping transcript (unlinked).,'],\n",
       " 564: ['1471-2105-6-131-3-3',\n",
       "  'GFIG',\n",
       "  'Ensembl extensions . Exon extensions to Ensembl predictions can be obtained from exons predicted by TWINSCAN and/or SGP2. These exons can either (a) be part of a transcript with exons in common with the Ensembl transcript (linked) or (b) be part of a close but non-overlapping transcript (unlinked).,'],\n",
       " 565: ['1471-2105-6-155-3-10',\n",
       "  'GFIG',\n",
       "  'Prediction profiles of the 19 protein concentrations for the test experiment for the caspase system. Solid line: real system; dashed line: prediction with estimated parameters after first iteration (suboptimal experiment with suboptimal measurements); dash-dotted line: prediction with estimated parameters after second iteration (suboptimal experiment with optimal measurements); dotted line: prediction with estimated parameters after second iteration (optimal experiment with suboptimal measurements),'],\n",
       " 566: ['1471-2105-6-155-3-11',\n",
       "  'GFIG',\n",
       "  'Prediction profiles of the 19 protein concentrations for the test experiment for the caspase system. Solid line: real system; dashed line: prediction with estimated parameters after first iteration (suboptimal experiment with suboptimal measurements); dash-dotted line: prediction with estimated parameters after second iteration (suboptimal experiment with optimal measurements); dotted line: prediction with estimated parameters after second iteration (optimal experiment with suboptimal measurements),'],\n",
       " 567: ['1471-2105-6-155-3-12',\n",
       "  'GFIG',\n",
       "  'Prediction profiles of the 19 protein concentrations for the test experiment for the caspase system. Solid line: real system; dashed line: prediction with estimated parameters after first iteration (suboptimal experiment with suboptimal measurements); dash-dotted line: prediction with estimated parameters after second iteration (suboptimal experiment with optimal measurements); dotted line: prediction with estimated parameters after second iteration (optimal experiment with suboptimal measurements),'],\n",
       " 568: ['1471-2105-6-155-3-13',\n",
       "  'GFIG',\n",
       "  'Prediction profiles of the 19 protein concentrations for the test experiment for the caspase system. Solid line: real system; dashed line: prediction with estimated parameters after first iteration (suboptimal experiment with suboptimal measurements); dash-dotted line: prediction with estimated parameters after second iteration (suboptimal experiment with optimal measurements); dotted line: prediction with estimated parameters after second iteration (optimal experiment with suboptimal measurements),'],\n",
       " 569: ['1471-2105-6-155-3-14',\n",
       "  'GFIG',\n",
       "  'Prediction profiles of the 19 protein concentrations for the test experiment for the caspase system. Solid line: real system; dashed line: prediction with estimated parameters after first iteration (suboptimal experiment with suboptimal measurements); dash-dotted line: prediction with estimated parameters after second iteration (suboptimal experiment with optimal measurements); dotted line: prediction with estimated parameters after second iteration (optimal experiment with suboptimal measurements),'],\n",
       " 570: ['1471-2105-6-155-3-15',\n",
       "  'GFIG',\n",
       "  'Prediction profiles of the 19 protein concentrations for the test experiment for the caspase system. Solid line: real system; dashed line: prediction with estimated parameters after first iteration (suboptimal experiment with suboptimal measurements); dash-dotted line: prediction with estimated parameters after second iteration (suboptimal experiment with optimal measurements); dotted line: prediction with estimated parameters after second iteration (optimal experiment with suboptimal measurements),'],\n",
       " 571: ['1471-2105-6-155-3-16',\n",
       "  'GFIG',\n",
       "  'Prediction profiles of the 19 protein concentrations for the test experiment for the caspase system. Solid line: real system; dashed line: prediction with estimated parameters after first iteration (suboptimal experiment with suboptimal measurements); dash-dotted line: prediction with estimated parameters after second iteration (suboptimal experiment with optimal measurements); dotted line: prediction with estimated parameters after second iteration (optimal experiment with suboptimal measurements),'],\n",
       " 572: ['1471-2105-6-155-3-17',\n",
       "  'GFIG',\n",
       "  'Prediction profiles of the 19 protein concentrations for the test experiment for the caspase system. Solid line: real system; dashed line: prediction with estimated parameters after first iteration (suboptimal experiment with suboptimal measurements); dash-dotted line: prediction with estimated parameters after second iteration (suboptimal experiment with optimal measurements); dotted line: prediction with estimated parameters after second iteration (optimal experiment with suboptimal measurements),'],\n",
       " 573: ['1471-2105-6-155-3-18',\n",
       "  'GFIG',\n",
       "  'Prediction profiles of the 19 protein concentrations for the test experiment for the caspase system. Solid line: real system; dashed line: prediction with estimated parameters after first iteration (suboptimal experiment with suboptimal measurements); dash-dotted line: prediction with estimated parameters after second iteration (suboptimal experiment with optimal measurements); dotted line: prediction with estimated parameters after second iteration (optimal experiment with suboptimal measurements),'],\n",
       " 574: ['1471-2105-6-155-3-19',\n",
       "  'GFIG',\n",
       "  'Prediction profiles of the 19 protein concentrations for the test experiment for the caspase system. Solid line: real system; dashed line: prediction with estimated parameters after first iteration (suboptimal experiment with suboptimal measurements); dash-dotted line: prediction with estimated parameters after second iteration (suboptimal experiment with optimal measurements); dotted line: prediction with estimated parameters after second iteration (optimal experiment with suboptimal measurements),'],\n",
       " 575: ['1471-2105-6-155-3-1',\n",
       "  'GFIG',\n",
       "  'Prediction profiles of the 19 protein concentrations for the test experiment for the caspase system. Solid line: real system; dashed line: prediction with estimated parameters after first iteration (suboptimal experiment with suboptimal measurements); dash-dotted line: prediction with estimated parameters after second iteration (suboptimal experiment with optimal measurements); dotted line: prediction with estimated parameters after second iteration (optimal experiment with suboptimal measurements),'],\n",
       " 576: ['1471-2105-6-155-3-2',\n",
       "  'GFIG',\n",
       "  'Prediction profiles of the 19 protein concentrations for the test experiment for the caspase system. Solid line: real system; dashed line: prediction with estimated parameters after first iteration (suboptimal experiment with suboptimal measurements); dash-dotted line: prediction with estimated parameters after second iteration (suboptimal experiment with optimal measurements); dotted line: prediction with estimated parameters after second iteration (optimal experiment with suboptimal measurements),'],\n",
       " 577: ['1471-2105-6-155-3-3',\n",
       "  'GFIG',\n",
       "  'Prediction profiles of the 19 protein concentrations for the test experiment for the caspase system. Solid line: real system; dashed line: prediction with estimated parameters after first iteration (suboptimal experiment with suboptimal measurements); dash-dotted line: prediction with estimated parameters after second iteration (suboptimal experiment with optimal measurements); dotted line: prediction with estimated parameters after second iteration (optimal experiment with suboptimal measurements),'],\n",
       " 578: ['1471-2105-6-155-3-4',\n",
       "  'GFIG',\n",
       "  'Prediction profiles of the 19 protein concentrations for the test experiment for the caspase system. Solid line: real system; dashed line: prediction with estimated parameters after first iteration (suboptimal experiment with suboptimal measurements); dash-dotted line: prediction with estimated parameters after second iteration (suboptimal experiment with optimal measurements); dotted line: prediction with estimated parameters after second iteration (optimal experiment with suboptimal measurements),'],\n",
       " 579: ['1471-2105-6-155-3-5',\n",
       "  'GFIG',\n",
       "  'Prediction profiles of the 19 protein concentrations for the test experiment for the caspase system. Solid line: real system; dashed line: prediction with estimated parameters after first iteration (suboptimal experiment with suboptimal measurements); dash-dotted line: prediction with estimated parameters after second iteration (suboptimal experiment with optimal measurements); dotted line: prediction with estimated parameters after second iteration (optimal experiment with suboptimal measurements),'],\n",
       " 580: ['1471-2105-6-155-3-6',\n",
       "  'GFIG',\n",
       "  'Prediction profiles of the 19 protein concentrations for the test experiment for the caspase system. Solid line: real system; dashed line: prediction with estimated parameters after first iteration (suboptimal experiment with suboptimal measurements); dash-dotted line: prediction with estimated parameters after second iteration (suboptimal experiment with optimal measurements); dotted line: prediction with estimated parameters after second iteration (optimal experiment with suboptimal measurements),'],\n",
       " 581: ['1471-2105-6-155-3-7',\n",
       "  'GFIG',\n",
       "  'Prediction profiles of the 19 protein concentrations for the test experiment for the caspase system. Solid line: real system; dashed line: prediction with estimated parameters after first iteration (suboptimal experiment with suboptimal measurements); dash-dotted line: prediction with estimated parameters after second iteration (suboptimal experiment with optimal measurements); dotted line: prediction with estimated parameters after second iteration (optimal experiment with suboptimal measurements),'],\n",
       " 582: ['1471-2105-6-155-3-8',\n",
       "  'GFIG',\n",
       "  'Prediction profiles of the 19 protein concentrations for the test experiment for the caspase system. Solid line: real system; dashed line: prediction with estimated parameters after first iteration (suboptimal experiment with suboptimal measurements); dash-dotted line: prediction with estimated parameters after second iteration (suboptimal experiment with optimal measurements); dotted line: prediction with estimated parameters after second iteration (optimal experiment with suboptimal measurements),'],\n",
       " 583: ['1471-2105-6-155-3-9',\n",
       "  'GFIG',\n",
       "  'Prediction profiles of the 19 protein concentrations for the test experiment for the caspase system. Solid line: real system; dashed line: prediction with estimated parameters after first iteration (suboptimal experiment with suboptimal measurements); dash-dotted line: prediction with estimated parameters after second iteration (suboptimal experiment with optimal measurements); dotted line: prediction with estimated parameters after second iteration (optimal experiment with suboptimal measurements),'],\n",
       " 584: ['1471-2105-6-164-5-1',\n",
       "  'GFLO',\n",
       "  'Workflow of the web service PROMI. In step one the user specifies the motif and selects (may be user submitted) protein files. For sane results step two can be used to refine the selection derived by step one (by disabling false positive matches) and to limit the matches for a balanced ratio within the classes. Step three is the presentation of the complete and pairwise calculations in plots (compare Figure 2) as well as tables breaking down the multiple alignments into ratios of amino acids per position.,'],\n",
       " 585: ['1471-2105-6-164-5-2',\n",
       "  'GSCR',\n",
       "  'Workflow of the web service PROMI. In step one the user specifies the motif and selects (may be user submitted) protein files. For sane results step two can be used to refine the selection derived by step one (by disabling false positive matches) and to limit the matches for a balanced ratio within the classes. Step three is the presentation of the complete and pairwise calculations in plots (compare Figure 2) as well as tables breaking down the multiple alignments into ratios of amino acids per position.,'],\n",
       " 586: ['1471-2105-6-27-5-1',\n",
       "  'GFIG',\n",
       "  \"Comparison of segmentation results based on Breast Cancer Cell lines using the adaptive criterion and Jong criterion.  Results of the segmentation procedure for Breast cancer cell lines Bt474, chromosomes 9 and 10. Fluoresence log 2 -ratios are plotted according to their location on the genome in megabases. Left profiles are segmented using the adaptive criterion and right profiles using Jong's criterion. The adaptive method detects a break-point at 1.58 MB on chromosome 9 that seems to be an outlier, and detects a putative deleted region on chromosome 10 at 1.76 MB.,\"],\n",
       " 587: ['1471-2105-6-27-5-2',\n",
       "  'GFIG',\n",
       "  \"Comparison of segmentation results based on Breast Cancer Cell lines using the adaptive criterion and Jong criterion.  Results of the segmentation procedure for Breast cancer cell lines Bt474, chromosomes 9 and 10. Fluoresence log 2 -ratios are plotted according to their location on the genome in megabases. Left profiles are segmented using the adaptive criterion and right profiles using Jong's criterion. The adaptive method detects a break-point at 1.58 MB on chromosome 9 that seems to be an outlier, and detects a putative deleted region on chromosome 10 at 1.76 MB.,\"],\n",
       " 588: ['1471-2105-6-27-5-3',\n",
       "  'GFIG',\n",
       "  \"Comparison of segmentation results based on Breast Cancer Cell lines using the adaptive criterion and Jong criterion.  Results of the segmentation procedure for Breast cancer cell lines Bt474, chromosomes 9 and 10. Fluoresence log 2 -ratios are plotted according to their location on the genome in megabases. Left profiles are segmented using the adaptive criterion and right profiles using Jong's criterion. The adaptive method detects a break-point at 1.58 MB on chromosome 9 that seems to be an outlier, and detects a putative deleted region on chromosome 10 at 1.76 MB.,\"],\n",
       " 589: ['1471-2105-6-27-5-4',\n",
       "  'GFIG',\n",
       "  \"Comparison of segmentation results based on Breast Cancer Cell lines using the adaptive criterion and Jong criterion.  Results of the segmentation procedure for Breast cancer cell lines Bt474, chromosomes 9 and 10. Fluoresence log 2 -ratios are plotted according to their location on the genome in megabases. Left profiles are segmented using the adaptive criterion and right profiles using Jong's criterion. The adaptive method detects a break-point at 1.58 MB on chromosome 9 that seems to be an outlier, and detects a putative deleted region on chromosome 10 at 1.76 MB.,\"],\n",
       " 590: ['1471-2105-6-30-15-1',\n",
       "  'GFIG',\n",
       "  'Impact of parameter settings on ACO-HPPFP-3 performance in 2D and 3D: relative weights of pheromone and heuristic information . Effect of the relative weights of pheromone information,  Î± , and heuristic information,  Î² , on the average CPU time required for obtaining minimal energy conformations of Sequence S1-8 in 2D (length 64, left side) and Sequence S2-5 in 3D (length 48, right side).,'],\n",
       " 591: ['1471-2105-6-30-15-2',\n",
       "  'GFIG',\n",
       "  'Impact of parameter settings on ACO-HPPFP-3 performance in 2D and 3D: relative weights of pheromone and heuristic information . Effect of the relative weights of pheromone information,  Î± , and heuristic information,  Î² , on the average CPU time required for obtaining minimal energy conformations of Sequence S1-8 in 2D (length 64, left side) and Sequence S2-5 in 3D (length 48, right side).,'],\n",
       " 592: ['1471-2105-6-52-1-1',\n",
       "  'GSCR',\n",
       "  'WEBnm@ architecture.  a.  The main page of WEBnm@ is a form where users can input a structure file in the PDB format.  b.  The server consists of two parts, the graphical web interface and the core of the program, written in Python, which performs the actual computation. The two parts communicate via a web application server, BIAZ.,'],\n",
       " 593: ['1471-2105-6-52-1-2',\n",
       "  'GFLO',\n",
       "  'WEBnm@ architecture.  a.  The main page of WEBnm@ is a form where users can input a structure file in the PDB format.  b.  The server consists of two parts, the graphical web interface and the core of the program, written in Python, which performs the actual computation. The two parts communicate via a web application server, BIAZ.,'],\n",
       " 594: ['1471-2105-6-6-1-1',\n",
       "  'GGEN',\n",
       "  'Analysis of plant CNB domains . (A) Arabidopsis CNB domains (CNTE1, KAT1 and CNGC2) were aligned against several well studied CNB domains including regulatory subunits of PKA (RI Î±  and RII Î² ), Epac1, Epac2, and cyclic GMP dependent kinase 2 (CGK2) from humans, HCN2 from mouse and  E. coli  CAP. Highlighted on the alignment are glycine residues involved in loop structures (dark grey arrows), residues forming the hydrophobic pocket for cNMP binding (green arrows) and residues proposed to contact the phosphate of the cNMP (blue arrows). The highly conserved helix capping acidic residue is shown in red. Secondary structure is denoted by arrows above the alignment, with light blue for alpha helices and pink for beta sheets and is based on the secondary structure of HCN2. (B) A homology model of atCNTE1 was generated from the known structures of CNB domains. Key residues are shown as stick representations and are colored and labeled according to the color scheme described in (A). The cGMP ligand is shown in magenta and is based on the structure of cGMP bound to HCN2 [pdb: 1Q3E] superimposed over our model. Figure was generated with Molscript [83]  and Raster3D [84].'],\n",
       " 595: ['1471-2105-6-6-1-2',\n",
       "  'GGEN',\n",
       "  'Analysis of plant CNB domains . (A) Arabidopsis CNB domains (CNTE1, KAT1 and CNGC2) were aligned against several well studied CNB domains including regulatory subunits of PKA (RI Î±  and RII Î² ), Epac1, Epac2, and cyclic GMP dependent kinase 2 (CGK2) from humans, HCN2 from mouse and  E. coli  CAP. Highlighted on the alignment are glycine residues involved in loop structures (dark grey arrows), residues forming the hydrophobic pocket for cNMP binding (green arrows) and residues proposed to contact the phosphate of the cNMP (blue arrows). The highly conserved helix capping acidic residue is shown in red. Secondary structure is denoted by arrows above the alignment, with light blue for alpha helices and pink for beta sheets and is based on the secondary structure of HCN2. (B) A homology model of atCNTE1 was generated from the known structures of CNB domains. Key residues are shown as stick representations and are colored and labeled according to the color scheme described in (A). The cGMP ligand is shown in magenta and is based on the structure of cGMP bound to HCN2 [pdb: 1Q3E] superimposed over our model. Figure was generated with Molscript [83]  and Raster3D [84].'],\n",
       " 596: ['1471-2105-6-6-1-3',\n",
       "  'D3DR',\n",
       "  'Analysis of plant CNB domains . (A) Arabidopsis CNB domains (CNTE1, KAT1 and CNGC2) were aligned against several well studied CNB domains including regulatory subunits of PKA (RI Î±  and RII Î² ), Epac1, Epac2, and cyclic GMP dependent kinase 2 (CGK2) from humans, HCN2 from mouse and  E. coli  CAP. Highlighted on the alignment are glycine residues involved in loop structures (dark grey arrows), residues forming the hydrophobic pocket for cNMP binding (green arrows) and residues proposed to contact the phosphate of the cNMP (blue arrows). The highly conserved helix capping acidic residue is shown in red. Secondary structure is denoted by arrows above the alignment, with light blue for alpha helices and pink for beta sheets and is based on the secondary structure of HCN2. (B) A homology model of atCNTE1 was generated from the known structures of CNB domains. Key residues are shown as stick representations and are colored and labeled according to the color scheme described in (A). The cGMP ligand is shown in magenta and is based on the structure of cGMP bound to HCN2 [pdb: 1Q3E] superimposed over our model. Figure was generated with Molscript [83]  and Raster3D [84].'],\n",
       " 597: ['1471-2105-6-62-10-1',\n",
       "  'GFIG',\n",
       "  'Effect of amplitude normalization on plateau scattering in 96   replica . Axes: vertical  ï¿½ï¿½  Fluorescence, horizontal  ï¿½ï¿½  Cycle. Data for plate 3 from Table 2.'],\n",
       " 598: ['1471-2105-6-62-10-2',\n",
       "  'GFIG',\n",
       "  'Effect of amplitude normalization on plateau scattering in 96   replica . Axes: vertical  ï¿½ï¿½  Fluorescence, horizontal  ï¿½ï¿½  Cycle. Data for plate 3 from Table 2.'],\n",
       " 599: ['1471-2105-6-91-3-1',\n",
       "  'D3DR',\n",
       "  'Comparison of the model structure of HPr from E. coli with the corresponding X-ray and NMR structures . A comparison of the modeled HPr homology structure with the structures experimentally determined by NMR spectroscopy (1HDN) and X-ray crystallography (1POH). The structures are shown in the same orientation as in Fig. 1 with the radius of the backbone splines indicating the RMSD of the C Î±   atom positions in the respective structures. (A) Overall good agreement between the model structure (yellow) and the X-ray structure (blue) is obtained. Deviations are mainly seen in loop regions and in the orientation of helices a and b. RMSD values for the C Î±   atom positions of the X-ray structure 1POH have been derived from the crystallographic B-factors,  f B , using the Debye-Waller equation   where isotropic displacement from the mean atom positions was assumed. (B) Comparison of the model (yellow) and the NMR structure (red). Deviations are seen in the same regions as before. (C) X-ray (blue) and NMR (red) structures superimpose well. Interestingly, deviations between them are mainly observed in regions where the two structures also diverge from the homology model.'],\n",
       " 600: ['1471-2105-6-91-3-2',\n",
       "  'D3DR',\n",
       "  'Comparison of the model structure of HPr from E. coli with the corresponding X-ray and NMR structures . A comparison of the modeled HPr homology structure with the structures experimentally determined by NMR spectroscopy (1HDN) and X-ray crystallography (1POH). The structures are shown in the same orientation as in Fig. 1 with the radius of the backbone splines indicating the RMSD of the C Î±   atom positions in the respective structures. (A) Overall good agreement between the model structure (yellow) and the X-ray structure (blue) is obtained. Deviations are mainly seen in loop regions and in the orientation of helices a and b. RMSD values for the C Î±   atom positions of the X-ray structure 1POH have been derived from the crystallographic B-factors,  f B , using the Debye-Waller equation   where isotropic displacement from the mean atom positions was assumed. (B) Comparison of the model (yellow) and the NMR structure (red). Deviations are seen in the same regions as before. (C) X-ray (blue) and NMR (red) structures superimpose well. Interestingly, deviations between them are mainly observed in regions where the two structures also diverge from the homology model.'],\n",
       " 601: ['1471-2105-6-91-3-3',\n",
       "  'D3DR',\n",
       "  'Comparison of the model structure of HPr from E. coli with the corresponding X-ray and NMR structures . A comparison of the modeled HPr homology structure with the structures experimentally determined by NMR spectroscopy (1HDN) and X-ray crystallography (1POH). The structures are shown in the same orientation as in Fig. 1 with the radius of the backbone splines indicating the RMSD of the C Î±   atom positions in the respective structures. (A) Overall good agreement between the model structure (yellow) and the X-ray structure (blue) is obtained. Deviations are mainly seen in loop regions and in the orientation of helices a and b. RMSD values for the C Î±   atom positions of the X-ray structure 1POH have been derived from the crystallographic B-factors,  f B , using the Debye-Waller equation   where isotropic displacement from the mean atom positions was assumed. (B) Comparison of the model (yellow) and the NMR structure (red). Deviations are seen in the same regions as before. (C) X-ray (blue) and NMR (red) structures superimpose well. Interestingly, deviations between them are mainly observed in regions where the two structures also diverge from the homology model.'],\n",
       " 602: ['1471-2105-6-91-4-1',\n",
       "  'D3DR',\n",
       "  'Importance of torsion angle restraints exemplified on HPr from Streptococcus faecalis . On the left hand side the model structure calculated with PERMOL using 427 torsion angle restraints and 41 hydrogen bonds is displayed, while on the right hand side the target X-ray structure 1PTF is shown. The RMSD value for the heavy atoms of the two structures is 0.328 nm. Restraints for torsion angles and hydrogen bonds were directly generated from the X-ray structure 1PTF.'],\n",
       " 603: ['1471-2105-6-91-4-2',\n",
       "  'D3DR',\n",
       "  'Importance of torsion angle restraints exemplified on HPr from Streptococcus faecalis . On the left hand side the model structure calculated with PERMOL using 427 torsion angle restraints and 41 hydrogen bonds is displayed, while on the right hand side the target X-ray structure 1PTF is shown. The RMSD value for the heavy atoms of the two structures is 0.328 nm. Restraints for torsion angles and hydrogen bonds were directly generated from the X-ray structure 1PTF.'],\n",
       " 604: ['1471-2105-6-S2-S11-9-1',\n",
       "  'GFIG',\n",
       "  'Intensity-and concentration-based ratio calculation.  Intensity-based ratio calculation is accurate only when the calibration curve (I~C) is a linear and pass the origin (the dashed blue line). When the calibration curve is nonlinear (red curve), intensity-based ratio calculation underestimate the true concentration differences. Such an underestimation of concentration difference is universal for both two-color and one-color platforms. For one-color platform, the calibration functions  f 1  and  f 2  are the same, whereas for two-color platform,  f 1  and  f 2  reflect the differences of the two dyes as discussed in this study.,'],\n",
       " 605: ['1471-2105-6-S2-S11-9-2',\n",
       "  'GMAT',\n",
       "  'Intensity-and concentration-based ratio calculation.  Intensity-based ratio calculation is accurate only when the calibration curve (I~C) is a linear and pass the origin (the dashed blue line). When the calibration curve is nonlinear (red curve), intensity-based ratio calculation underestimate the true concentration differences. Such an underestimation of concentration difference is universal for both two-color and one-color platforms. For one-color platform, the calibration functions  f 1  and  f 2  are the same, whereas for two-color platform,  f 1  and  f 2  reflect the differences of the two dyes as discussed in this study.,'],\n",
       " 606: ['1471-2105-7-175-3-1',\n",
       "  'GFIG',\n",
       "  'ROC Analysis of Yeast Cell Cycle and Rosetta Data Sets . ROC analysis for NMF and LS-NMF for the yeast cell cycle data is summarized in sections a and b, while section c shows the results for the Rosetta Compendium analysis, where variance estimates are available. In a, the total area under the ROC curves is shown for LS-NMF (solid line) and NMF (dashed line). In b, LS-NMF and NMF are compared to hierarchical clustering at a single dimensionality (the LS-NMF and NMF curves are superimposed as there is no difference). In c, the areas under the ROC curve for analysis of the Rosetta data are shown for LS-NMF (solid line), NMF (dash-dot line), K-means clustering (dashed line), and NMF on scaled data (C-NMF, dotted line). Here the variance information allows far better results to be obtained, either by scaling or by use of LS-NMF. The advantages of the approach used in LS-NMF are discussed in the text.,'],\n",
       " 607: ['1471-2105-7-175-3-2',\n",
       "  'GFIG',\n",
       "  'ROC Analysis of Yeast Cell Cycle and Rosetta Data Sets . ROC analysis for NMF and LS-NMF for the yeast cell cycle data is summarized in sections a and b, while section c shows the results for the Rosetta Compendium analysis, where variance estimates are available. In a, the total area under the ROC curves is shown for LS-NMF (solid line) and NMF (dashed line). In b, LS-NMF and NMF are compared to hierarchical clustering at a single dimensionality (the LS-NMF and NMF curves are superimposed as there is no difference). In c, the areas under the ROC curve for analysis of the Rosetta data are shown for LS-NMF (solid line), NMF (dash-dot line), K-means clustering (dashed line), and NMF on scaled data (C-NMF, dotted line). Here the variance information allows far better results to be obtained, either by scaling or by use of LS-NMF. The advantages of the approach used in LS-NMF are discussed in the text.,'],\n",
       " 608: ['1471-2105-7-175-3-3',\n",
       "  'GFIG',\n",
       "  'ROC Analysis of Yeast Cell Cycle and Rosetta Data Sets . ROC analysis for NMF and LS-NMF for the yeast cell cycle data is summarized in sections a and b, while section c shows the results for the Rosetta Compendium analysis, where variance estimates are available. In a, the total area under the ROC curves is shown for LS-NMF (solid line) and NMF (dashed line). In b, LS-NMF and NMF are compared to hierarchical clustering at a single dimensionality (the LS-NMF and NMF curves are superimposed as there is no difference). In c, the areas under the ROC curve for analysis of the Rosetta data are shown for LS-NMF (solid line), NMF (dash-dot line), K-means clustering (dashed line), and NMF on scaled data (C-NMF, dotted line). Here the variance information allows far better results to be obtained, either by scaling or by use of LS-NMF. The advantages of the approach used in LS-NMF are discussed in the text.,'],\n",
       " 609: ['1471-2105-7-215-2-1',\n",
       "  'GFIG',\n",
       "  'Comparison of GP and GOT models . Comparison of estimated log 2 (fold change) and the corresponding  p -value estimates for the GP and GOT model results of the WS ecotype values in Table 4 (a) This is a scatter plot of the estimated log 2 (fold change) values from the GP and GOT models; these values are essentially identical, (b) This is a scatter plot of the - log 10  (p-value), again for the GP and GOT models. The dotted lines correspond to - log 10 (0.05), as our significance cutoff is 0.05. For the GP model, it is the points to the right of the dotted line that are significant. For the GOT model, it is the points above the dotted line that are significant.'],\n",
       " 610: ['1471-2105-7-215-2-2',\n",
       "  'GFIG',\n",
       "  'Comparison of GP and GOT models . Comparison of estimated log 2 (fold change) and the corresponding  p -value estimates for the GP and GOT model results of the WS ecotype values in Table 4 (a) This is a scatter plot of the estimated log 2 (fold change) values from the GP and GOT models; these values are essentially identical, (b) This is a scatter plot of the - log 10  (p-value), again for the GP and GOT models. The dotted lines correspond to - log 10 (0.05), as our significance cutoff is 0.05. For the GP model, it is the points to the right of the dotted line that are significant. For the GOT model, it is the points above the dotted line that are significant.'],\n",
       " 611: ['1471-2105-7-227-6-1',\n",
       "  'GTAB',\n",
       "  \"LD between a complex microsatellite and SNPs. (a) Previous work [22] indicated SNP alleles in LD with two size ranges of the  CSH1 .01 microsatellite. The lower size range has dinucleotide spacing, the upper has tetranucleotide spacing. This suggested two major lineages. (b) Plotting interallelic LD between a SNP (GH1V004) and the  CSH1 .01 microsatellite demonstrates clear LD with the two lineages. The common SNP alleles associate with the lower size range and the rare SNP alleles associate with the upper size range. Results are boxed in red where the haplotype frequency is significantly higher than expected (D'    0) and blue where it is significantly lower (D' < 0). (c) SNP haplotypes (four SNPs, including GH1V004) confirm these findings and demonstrate the ability of MIDAS to handle haplotype data as a multi-allelic marker.,\"],\n",
       " 612: ['1471-2105-7-227-6-2',\n",
       "  'GFIG',\n",
       "  \"LD between a complex microsatellite and SNPs. (a) Previous work [22] indicated SNP alleles in LD with two size ranges of the  CSH1 .01 microsatellite. The lower size range has dinucleotide spacing, the upper has tetranucleotide spacing. This suggested two major lineages. (b) Plotting interallelic LD between a SNP (GH1V004) and the  CSH1 .01 microsatellite demonstrates clear LD with the two lineages. The common SNP alleles associate with the lower size range and the rare SNP alleles associate with the upper size range. Results are boxed in red where the haplotype frequency is significantly higher than expected (D'    0) and blue where it is significantly lower (D' < 0). (c) SNP haplotypes (four SNPs, including GH1V004) confirm these findings and demonstrate the ability of MIDAS to handle haplotype data as a multi-allelic marker.,\"],\n",
       " 613: ['1471-2105-7-227-6-3',\n",
       "  'GGEN',\n",
       "  \"LD between a complex microsatellite and SNPs. (a) Previous work [22] indicated SNP alleles in LD with two size ranges of the  CSH1 .01 microsatellite. The lower size range has dinucleotide spacing, the upper has tetranucleotide spacing. This suggested two major lineages. (b) Plotting interallelic LD between a SNP (GH1V004) and the  CSH1 .01 microsatellite demonstrates clear LD with the two lineages. The common SNP alleles associate with the lower size range and the rare SNP alleles associate with the upper size range. Results are boxed in red where the haplotype frequency is significantly higher than expected (D'    0) and blue where it is significantly lower (D' < 0). (c) SNP haplotypes (four SNPs, including GH1V004) confirm these findings and demonstrate the ability of MIDAS to handle haplotype data as a multi-allelic marker.,\"],\n",
       " 614: ['1471-2105-7-227-6-4',\n",
       "  'GSYS',\n",
       "  \"LD between a complex microsatellite and SNPs. (a) Previous work [22] indicated SNP alleles in LD with two size ranges of the  CSH1 .01 microsatellite. The lower size range has dinucleotide spacing, the upper has tetranucleotide spacing. This suggested two major lineages. (b) Plotting interallelic LD between a SNP (GH1V004) and the  CSH1 .01 microsatellite demonstrates clear LD with the two lineages. The common SNP alleles associate with the lower size range and the rare SNP alleles associate with the upper size range. Results are boxed in red where the haplotype frequency is significantly higher than expected (D'    0) and blue where it is significantly lower (D' < 0). (c) SNP haplotypes (four SNPs, including GH1V004) confirm these findings and demonstrate the ability of MIDAS to handle haplotype data as a multi-allelic marker.,\"],\n",
       " 615: ['1471-2105-7-227-6-5',\n",
       "  'GFIG',\n",
       "  \"LD between a complex microsatellite and SNPs. (a) Previous work [22] indicated SNP alleles in LD with two size ranges of the  CSH1 .01 microsatellite. The lower size range has dinucleotide spacing, the upper has tetranucleotide spacing. This suggested two major lineages. (b) Plotting interallelic LD between a SNP (GH1V004) and the  CSH1 .01 microsatellite demonstrates clear LD with the two lineages. The common SNP alleles associate with the lower size range and the rare SNP alleles associate with the upper size range. Results are boxed in red where the haplotype frequency is significantly higher than expected (D'    0) and blue where it is significantly lower (D' < 0). (c) SNP haplotypes (four SNPs, including GH1V004) confirm these findings and demonstrate the ability of MIDAS to handle haplotype data as a multi-allelic marker.,\"],\n",
       " 616: ['1471-2105-7-250-2-1',\n",
       "  'GSYS',\n",
       "  'Graphical model representation of the LDA model (left) and the variational distribution used to approximate the posterior in LDA (right) [22]. LDA defines a distribution on a collection of documents in much the same manner that a profile hidden Markov model yields a distribution on a set of (biological) sequences [31] . The corpus depicted contains  M  documents and each is a sequence of  N  words. Open circles are parameters ( Î± ,  Î² ,  Î³ ,   ) or latent variables ( Î¸ ,  z ). The shaded circle is the observed word variable ( w ) and boxes (plates) represent replicates. The Dirichlet parameter,  Î± , and topic-word matrix,  Î² , are corpus-level parameters sampled once in the process of generating a corpus. The topic proportions,  Î¸ , is a document-level variable sampled from  Î± once per document. The topic,  z , is a word-level variable sampled from  Î¸   once for each word in a document. Formally, a  K -topic LDA specifies a two-level probabilistic process that generates a document as follows, (i) a  K -dimensional vector,  Î¸ , is chosen from the distribution  p ( Î¸ | Î± ), and (ii) words are sampled repeatedly from the document-specific mixture distribution,  p ( w| Î¸ ). Exact inference and parameter estimation involve calculating the posterior distribution on a document  p ( Î¸ ,  z | w ,  Î± ,  Î² ). This is intractable because the latent variables are coupled via the edge between  Î¸   and  z . The posterior can be approximated by computing the variational Dirichlet parameter  Î³   and the variational multinomial parameter     for each word in the document. The subscripts  m ,  n , and  k  on a parameter ( Î² ,  Î³ ,   ) or variable ( Î¸ ,  z ,  w)  donate the  m  th document,  nth  word and kth topic respectively. Note that the Dirichlet variable  Î±   is a distinct component of the probability model and not merely an expression of uncertainty about a parameter. This differs from profile hidden Markov models where a mixture of Dirichlet distributions is used as a prior for amino acid/nucleotide probability distributions.,'],\n",
       " 617: ['1471-2105-7-250-2-2',\n",
       "  'GFIG',\n",
       "  'Graphical model representation of the LDA model (left) and the variational distribution used to approximate the posterior in LDA (right) [22]. LDA defines a distribution on a collection of documents in much the same manner that a profile hidden Markov model yields a distribution on a set of (biological) sequences [31] . The corpus depicted contains  M  documents and each is a sequence of  N  words. Open circles are parameters ( Î± ,  Î² ,  Î³ ,   ) or latent variables ( Î¸ ,  z ). The shaded circle is the observed word variable ( w ) and boxes (plates) represent replicates. The Dirichlet parameter,  Î± , and topic-word matrix,  Î² , are corpus-level parameters sampled once in the process of generating a corpus. The topic proportions,  Î¸ , is a document-level variable sampled from  Î± once per document. The topic,  z , is a word-level variable sampled from  Î¸   once for each word in a document. Formally, a  K -topic LDA specifies a two-level probabilistic process that generates a document as follows, (i) a  K -dimensional vector,  Î¸ , is chosen from the distribution  p ( Î¸ | Î± ), and (ii) words are sampled repeatedly from the document-specific mixture distribution,  p ( w| Î¸ ). Exact inference and parameter estimation involve calculating the posterior distribution on a document  p ( Î¸ ,  z | w ,  Î± ,  Î² ). This is intractable because the latent variables are coupled via the edge between  Î¸   and  z . The posterior can be approximated by computing the variational Dirichlet parameter  Î³   and the variational multinomial parameter     for each word in the document. The subscripts  m ,  n , and  k  on a parameter ( Î² ,  Î³ ,   ) or variable ( Î¸ ,  z ,  w)  donate the  m  th document,  nth  word and kth topic respectively. Note that the Dirichlet variable  Î±   is a distinct component of the probability model and not merely an expression of uncertainty about a parameter. This differs from profile hidden Markov models where a mixture of Dirichlet distributions is used as a prior for amino acid/nucleotide probability distributions.,'],\n",
       " 618: ['1471-2105-7-251-6-1',\n",
       "  'GFIG',\n",
       "  'Effects of rescaling, quantile normalization and a different summary procedure on subsequent analysis . Intermediate time points were interpolated using a simple linear procedure using neighbouring time points (for example, the 8 hours time point was interpolated using the 6 and 10 hours time points). A relative interpolation error was then computed from each gene using the formula ( X T  - X P )/ X T  where  X T  is the true value and  X P  the predicted one. In each plot there are three curves which, from top to bottom correspond to the third quartile, median and first quartile of the relative interpolation error. This procedure was repeated for each replicate separately before (A), after (B) rescaling, after quantile normalization applied on MAS5 summarized values (C) and on RMA summarized data (D). In (B) scaling imbalances have been corrected whilst the performance of quantile normalization and RMA summarized data are inferior in that respect (C, D). Only those genes whose neighbouring time points have a detection p-value less than or equal to 0.001 were included, which corresponds to 2, 500  ï¿½ï¿½  5, 000 genes/time point.,'],\n",
       " 619: ['1471-2105-7-251-6-2',\n",
       "  'GFIG',\n",
       "  'Effects of rescaling, quantile normalization and a different summary procedure on subsequent analysis . Intermediate time points were interpolated using a simple linear procedure using neighbouring time points (for example, the 8 hours time point was interpolated using the 6 and 10 hours time points). A relative interpolation error was then computed from each gene using the formula ( X T  - X P )/ X T  where  X T  is the true value and  X P  the predicted one. In each plot there are three curves which, from top to bottom correspond to the third quartile, median and first quartile of the relative interpolation error. This procedure was repeated for each replicate separately before (A), after (B) rescaling, after quantile normalization applied on MAS5 summarized values (C) and on RMA summarized data (D). In (B) scaling imbalances have been corrected whilst the performance of quantile normalization and RMA summarized data are inferior in that respect (C, D). Only those genes whose neighbouring time points have a detection p-value less than or equal to 0.001 were included, which corresponds to 2, 500  ï¿½ï¿½  5, 000 genes/time point.,'],\n",
       " 620: ['1471-2105-7-251-6-3',\n",
       "  'GFIG',\n",
       "  'Effects of rescaling, quantile normalization and a different summary procedure on subsequent analysis . Intermediate time points were interpolated using a simple linear procedure using neighbouring time points (for example, the 8 hours time point was interpolated using the 6 and 10 hours time points). A relative interpolation error was then computed from each gene using the formula ( X T  - X P )/ X T  where  X T  is the true value and  X P  the predicted one. In each plot there are three curves which, from top to bottom correspond to the third quartile, median and first quartile of the relative interpolation error. This procedure was repeated for each replicate separately before (A), after (B) rescaling, after quantile normalization applied on MAS5 summarized values (C) and on RMA summarized data (D). In (B) scaling imbalances have been corrected whilst the performance of quantile normalization and RMA summarized data are inferior in that respect (C, D). Only those genes whose neighbouring time points have a detection p-value less than or equal to 0.001 were included, which corresponds to 2, 500  ï¿½ï¿½  5, 000 genes/time point.,'],\n",
       " 621: ['1471-2105-7-251-6-4',\n",
       "  'GFIG',\n",
       "  'Effects of rescaling, quantile normalization and a different summary procedure on subsequent analysis . Intermediate time points were interpolated using a simple linear procedure using neighbouring time points (for example, the 8 hours time point was interpolated using the 6 and 10 hours time points). A relative interpolation error was then computed from each gene using the formula ( X T  - X P )/ X T  where  X T  is the true value and  X P  the predicted one. In each plot there are three curves which, from top to bottom correspond to the third quartile, median and first quartile of the relative interpolation error. This procedure was repeated for each replicate separately before (A), after (B) rescaling, after quantile normalization applied on MAS5 summarized values (C) and on RMA summarized data (D). In (B) scaling imbalances have been corrected whilst the performance of quantile normalization and RMA summarized data are inferior in that respect (C, D). Only those genes whose neighbouring time points have a detection p-value less than or equal to 0.001 were included, which corresponds to 2, 500  ï¿½ï¿½  5, 000 genes/time point.,'],\n",
       " 622: ['1471-2105-7-251-6-5',\n",
       "  'GFIG',\n",
       "  'Effects of rescaling, quantile normalization and a different summary procedure on subsequent analysis . Intermediate time points were interpolated using a simple linear procedure using neighbouring time points (for example, the 8 hours time point was interpolated using the 6 and 10 hours time points). A relative interpolation error was then computed from each gene using the formula ( X T  - X P )/ X T  where  X T  is the true value and  X P  the predicted one. In each plot there are three curves which, from top to bottom correspond to the third quartile, median and first quartile of the relative interpolation error. This procedure was repeated for each replicate separately before (A), after (B) rescaling, after quantile normalization applied on MAS5 summarized values (C) and on RMA summarized data (D). In (B) scaling imbalances have been corrected whilst the performance of quantile normalization and RMA summarized data are inferior in that respect (C, D). Only those genes whose neighbouring time points have a detection p-value less than or equal to 0.001 were included, which corresponds to 2, 500  ï¿½ï¿½  5, 000 genes/time point.,'],\n",
       " 623: ['1471-2105-7-251-6-6',\n",
       "  'GFIG',\n",
       "  'Effects of rescaling, quantile normalization and a different summary procedure on subsequent analysis . Intermediate time points were interpolated using a simple linear procedure using neighbouring time points (for example, the 8 hours time point was interpolated using the 6 and 10 hours time points). A relative interpolation error was then computed from each gene using the formula ( X T  - X P )/ X T  where  X T  is the true value and  X P  the predicted one. In each plot there are three curves which, from top to bottom correspond to the third quartile, median and first quartile of the relative interpolation error. This procedure was repeated for each replicate separately before (A), after (B) rescaling, after quantile normalization applied on MAS5 summarized values (C) and on RMA summarized data (D). In (B) scaling imbalances have been corrected whilst the performance of quantile normalization and RMA summarized data are inferior in that respect (C, D). Only those genes whose neighbouring time points have a detection p-value less than or equal to 0.001 were included, which corresponds to 2, 500  ï¿½ï¿½  5, 000 genes/time point.,'],\n",
       " 624: ['1471-2105-7-251-6-7',\n",
       "  'GFIG',\n",
       "  'Effects of rescaling, quantile normalization and a different summary procedure on subsequent analysis . Intermediate time points were interpolated using a simple linear procedure using neighbouring time points (for example, the 8 hours time point was interpolated using the 6 and 10 hours time points). A relative interpolation error was then computed from each gene using the formula ( X T  - X P )/ X T  where  X T  is the true value and  X P  the predicted one. In each plot there are three curves which, from top to bottom correspond to the third quartile, median and first quartile of the relative interpolation error. This procedure was repeated for each replicate separately before (A), after (B) rescaling, after quantile normalization applied on MAS5 summarized values (C) and on RMA summarized data (D). In (B) scaling imbalances have been corrected whilst the performance of quantile normalization and RMA summarized data are inferior in that respect (C, D). Only those genes whose neighbouring time points have a detection p-value less than or equal to 0.001 were included, which corresponds to 2, 500  ï¿½ï¿½  5, 000 genes/time point.,'],\n",
       " 625: ['1471-2105-7-251-6-8',\n",
       "  'GFIG',\n",
       "  'Effects of rescaling, quantile normalization and a different summary procedure on subsequent analysis . Intermediate time points were interpolated using a simple linear procedure using neighbouring time points (for example, the 8 hours time point was interpolated using the 6 and 10 hours time points). A relative interpolation error was then computed from each gene using the formula ( X T  - X P )/ X T  where  X T  is the true value and  X P  the predicted one. In each plot there are three curves which, from top to bottom correspond to the third quartile, median and first quartile of the relative interpolation error. This procedure was repeated for each replicate separately before (A), after (B) rescaling, after quantile normalization applied on MAS5 summarized values (C) and on RMA summarized data (D). In (B) scaling imbalances have been corrected whilst the performance of quantile normalization and RMA summarized data are inferior in that respect (C, D). Only those genes whose neighbouring time points have a detection p-value less than or equal to 0.001 were included, which corresponds to 2, 500  ï¿½ï¿½  5, 000 genes/time point.,'],\n",
       " 626: ['1471-2105-7-270-5-1',\n",
       "  'GFIG',\n",
       "  'Ortholuge R1    R2 plots (Ratio1 versus Ratio2) for selected eukaryotic data, where each point represents one putative ortholog group. (A) Putative orthologous groups identified using RBH for mouse-rat-human (Figure 4 shows the corresponding histogram). (B) Putative orthologs groups for mouse-rat-human from a higher quality (more precise) dataset (see Methods). It is expected that this more precise data set comprises primarily true orthologs. (C) A lower quality data set of RBH-predicted orthologous groups for cattle-human-mouse, where cattle genes have been identified from an incomplete genome sequence. (D), (E), (F) are zoomed-in versions of (A), (B), (C), respectively, with axes shown from 0 to 2 instead of 0 to 30. Note that most orthologous groups exhibit low Ratio1 and Ratio2 values, in all three data sets. For example, in panels A and D, about 86% of orthologs have Ratio1 and Ratio2 values less than 1. However, the higher quality data set (panels B and E) contains fewer points at higher Ratio values versus the RBH-predicted data set. The lower quality data set contains more points with very high Ratio2 values (i.e. only 73% of points have Ratio1 and Ratio2 values less than 1), potentially reflecting the increased occurrence of probable cattle paralogs (i.e. paralogs being misidentified as orthologs by an RBH-analysis with an incomplete cattle genome).'],\n",
       " 627: ['1471-2105-7-270-5-2',\n",
       "  'GFIG',\n",
       "  'Ortholuge R1    R2 plots (Ratio1 versus Ratio2) for selected eukaryotic data, where each point represents one putative ortholog group. (A) Putative orthologous groups identified using RBH for mouse-rat-human (Figure 4 shows the corresponding histogram). (B) Putative orthologs groups for mouse-rat-human from a higher quality (more precise) dataset (see Methods). It is expected that this more precise data set comprises primarily true orthologs. (C) A lower quality data set of RBH-predicted orthologous groups for cattle-human-mouse, where cattle genes have been identified from an incomplete genome sequence. (D), (E), (F) are zoomed-in versions of (A), (B), (C), respectively, with axes shown from 0 to 2 instead of 0 to 30. Note that most orthologous groups exhibit low Ratio1 and Ratio2 values, in all three data sets. For example, in panels A and D, about 86% of orthologs have Ratio1 and Ratio2 values less than 1. However, the higher quality data set (panels B and E) contains fewer points at higher Ratio values versus the RBH-predicted data set. The lower quality data set contains more points with very high Ratio2 values (i.e. only 73% of points have Ratio1 and Ratio2 values less than 1), potentially reflecting the increased occurrence of probable cattle paralogs (i.e. paralogs being misidentified as orthologs by an RBH-analysis with an incomplete cattle genome).'],\n",
       " 628: ['1471-2105-7-270-5-3',\n",
       "  'GFIG',\n",
       "  'Ortholuge R1    R2 plots (Ratio1 versus Ratio2) for selected eukaryotic data, where each point represents one putative ortholog group. (A) Putative orthologous groups identified using RBH for mouse-rat-human (Figure 4 shows the corresponding histogram). (B) Putative orthologs groups for mouse-rat-human from a higher quality (more precise) dataset (see Methods). It is expected that this more precise data set comprises primarily true orthologs. (C) A lower quality data set of RBH-predicted orthologous groups for cattle-human-mouse, where cattle genes have been identified from an incomplete genome sequence. (D), (E), (F) are zoomed-in versions of (A), (B), (C), respectively, with axes shown from 0 to 2 instead of 0 to 30. Note that most orthologous groups exhibit low Ratio1 and Ratio2 values, in all three data sets. For example, in panels A and D, about 86% of orthologs have Ratio1 and Ratio2 values less than 1. However, the higher quality data set (panels B and E) contains fewer points at higher Ratio values versus the RBH-predicted data set. The lower quality data set contains more points with very high Ratio2 values (i.e. only 73% of points have Ratio1 and Ratio2 values less than 1), potentially reflecting the increased occurrence of probable cattle paralogs (i.e. paralogs being misidentified as orthologs by an RBH-analysis with an incomplete cattle genome).'],\n",
       " 629: ['1471-2105-7-270-5-4',\n",
       "  'GFIG',\n",
       "  'Ortholuge R1    R2 plots (Ratio1 versus Ratio2) for selected eukaryotic data, where each point represents one putative ortholog group. (A) Putative orthologous groups identified using RBH for mouse-rat-human (Figure 4 shows the corresponding histogram). (B) Putative orthologs groups for mouse-rat-human from a higher quality (more precise) dataset (see Methods). It is expected that this more precise data set comprises primarily true orthologs. (C) A lower quality data set of RBH-predicted orthologous groups for cattle-human-mouse, where cattle genes have been identified from an incomplete genome sequence. (D), (E), (F) are zoomed-in versions of (A), (B), (C), respectively, with axes shown from 0 to 2 instead of 0 to 30. Note that most orthologous groups exhibit low Ratio1 and Ratio2 values, in all three data sets. For example, in panels A and D, about 86% of orthologs have Ratio1 and Ratio2 values less than 1. However, the higher quality data set (panels B and E) contains fewer points at higher Ratio values versus the RBH-predicted data set. The lower quality data set contains more points with very high Ratio2 values (i.e. only 73% of points have Ratio1 and Ratio2 values less than 1), potentially reflecting the increased occurrence of probable cattle paralogs (i.e. paralogs being misidentified as orthologs by an RBH-analysis with an incomplete cattle genome).'],\n",
       " 630: ['1471-2105-7-270-5-5',\n",
       "  'GFIG',\n",
       "  'Ortholuge R1    R2 plots (Ratio1 versus Ratio2) for selected eukaryotic data, where each point represents one putative ortholog group. (A) Putative orthologous groups identified using RBH for mouse-rat-human (Figure 4 shows the corresponding histogram). (B) Putative orthologs groups for mouse-rat-human from a higher quality (more precise) dataset (see Methods). It is expected that this more precise data set comprises primarily true orthologs. (C) A lower quality data set of RBH-predicted orthologous groups for cattle-human-mouse, where cattle genes have been identified from an incomplete genome sequence. (D), (E), (F) are zoomed-in versions of (A), (B), (C), respectively, with axes shown from 0 to 2 instead of 0 to 30. Note that most orthologous groups exhibit low Ratio1 and Ratio2 values, in all three data sets. For example, in panels A and D, about 86% of orthologs have Ratio1 and Ratio2 values less than 1. However, the higher quality data set (panels B and E) contains fewer points at higher Ratio values versus the RBH-predicted data set. The lower quality data set contains more points with very high Ratio2 values (i.e. only 73% of points have Ratio1 and Ratio2 values less than 1), potentially reflecting the increased occurrence of probable cattle paralogs (i.e. paralogs being misidentified as orthologs by an RBH-analysis with an incomplete cattle genome).'],\n",
       " 631: ['1471-2105-7-270-5-6',\n",
       "  'GFIG',\n",
       "  'Ortholuge R1    R2 plots (Ratio1 versus Ratio2) for selected eukaryotic data, where each point represents one putative ortholog group. (A) Putative orthologous groups identified using RBH for mouse-rat-human (Figure 4 shows the corresponding histogram). (B) Putative orthologs groups for mouse-rat-human from a higher quality (more precise) dataset (see Methods). It is expected that this more precise data set comprises primarily true orthologs. (C) A lower quality data set of RBH-predicted orthologous groups for cattle-human-mouse, where cattle genes have been identified from an incomplete genome sequence. (D), (E), (F) are zoomed-in versions of (A), (B), (C), respectively, with axes shown from 0 to 2 instead of 0 to 30. Note that most orthologous groups exhibit low Ratio1 and Ratio2 values, in all three data sets. For example, in panels A and D, about 86% of orthologs have Ratio1 and Ratio2 values less than 1. However, the higher quality data set (panels B and E) contains fewer points at higher Ratio values versus the RBH-predicted data set. The lower quality data set contains more points with very high Ratio2 values (i.e. only 73% of points have Ratio1 and Ratio2 values less than 1), potentially reflecting the increased occurrence of probable cattle paralogs (i.e. paralogs being misidentified as orthologs by an RBH-analysis with an incomplete cattle genome).'],\n",
       " 632: ['1471-2105-7-270-6-1',\n",
       "  'GFIG',\n",
       "  'Ortholuge R1    R2 plots for the prokaryotic data,\" illustrating two ortholog data sets and a true-negative data set. (A) Putative orthologous groups from an RBH-predicted data set. (B) Probable true orthologs from a higher quality (more precise) data set. (C) True-negative orthologs (i.e. true paralogs) from the \"\"gene-loss simulation\"\" data set. Darker dots represent putative orthologous groups which have had an ingroup1 true-negative (paralog) introduced into the group. Lighter dots represent putative orthologous groups which have had an ingroup2 true-negative (paralog) introduced into the group. (D)\", (E), (F) are zoomed-in versions of (A), (B), (C), respectively, with axes shown from 0 to 2 instead of 0 to 10. Most putative ortholog groups (particularly for the high quality data set) exhibit low Ratio1 and Ratio2 values (for example, all values are less than 1 for the points in the high quality data set plot), whereas most true-negative groups exhibit higher Ratio1 and Ratio2 values (i.e. only 9% of ingroup1 true negative introductions, and 6% of ingroup2 true negative introductions, have points with Ratio1 and Ratio2 values less than 1).'],\n",
       " 633: ['1471-2105-7-270-6-2',\n",
       "  'GFIG',\n",
       "  'Ortholuge R1    R2 plots for the prokaryotic data,\" illustrating two ortholog data sets and a true-negative data set. (A) Putative orthologous groups from an RBH-predicted data set. (B) Probable true orthologs from a higher quality (more precise) data set. (C) True-negative orthologs (i.e. true paralogs) from the \"\"gene-loss simulation\"\" data set. Darker dots represent putative orthologous groups which have had an ingroup1 true-negative (paralog) introduced into the group. Lighter dots represent putative orthologous groups which have had an ingroup2 true-negative (paralog) introduced into the group. (D)\", (E), (F) are zoomed-in versions of (A), (B), (C), respectively, with axes shown from 0 to 2 instead of 0 to 10. Most putative ortholog groups (particularly for the high quality data set) exhibit low Ratio1 and Ratio2 values (for example, all values are less than 1 for the points in the high quality data set plot), whereas most true-negative groups exhibit higher Ratio1 and Ratio2 values (i.e. only 9% of ingroup1 true negative introductions, and 6% of ingroup2 true negative introductions, have points with Ratio1 and Ratio2 values less than 1).'],\n",
       " 634: ['1471-2105-7-270-6-3',\n",
       "  'GFIG',\n",
       "  'Ortholuge R1    R2 plots for the prokaryotic data,\" illustrating two ortholog data sets and a true-negative data set. (A) Putative orthologous groups from an RBH-predicted data set. (B) Probable true orthologs from a higher quality (more precise) data set. (C) True-negative orthologs (i.e. true paralogs) from the \"\"gene-loss simulation\"\" data set. Darker dots represent putative orthologous groups which have had an ingroup1 true-negative (paralog) introduced into the group. Lighter dots represent putative orthologous groups which have had an ingroup2 true-negative (paralog) introduced into the group. (D)\", (E), (F) are zoomed-in versions of (A), (B), (C), respectively, with axes shown from 0 to 2 instead of 0 to 10. Most putative ortholog groups (particularly for the high quality data set) exhibit low Ratio1 and Ratio2 values (for example, all values are less than 1 for the points in the high quality data set plot), whereas most true-negative groups exhibit higher Ratio1 and Ratio2 values (i.e. only 9% of ingroup1 true negative introductions, and 6% of ingroup2 true negative introductions, have points with Ratio1 and Ratio2 values less than 1).'],\n",
       " 635: ['1471-2105-7-270-6-4',\n",
       "  'GFIG',\n",
       "  'Ortholuge R1    R2 plots for the prokaryotic data,\" illustrating two ortholog data sets and a true-negative data set. (A) Putative orthologous groups from an RBH-predicted data set. (B) Probable true orthologs from a higher quality (more precise) data set. (C) True-negative orthologs (i.e. true paralogs) from the \"\"gene-loss simulation\"\" data set. Darker dots represent putative orthologous groups which have had an ingroup1 true-negative (paralog) introduced into the group. Lighter dots represent putative orthologous groups which have had an ingroup2 true-negative (paralog) introduced into the group. (D)\", (E), (F) are zoomed-in versions of (A), (B), (C), respectively, with axes shown from 0 to 2 instead of 0 to 10. Most putative ortholog groups (particularly for the high quality data set) exhibit low Ratio1 and Ratio2 values (for example, all values are less than 1 for the points in the high quality data set plot), whereas most true-negative groups exhibit higher Ratio1 and Ratio2 values (i.e. only 9% of ingroup1 true negative introductions, and 6% of ingroup2 true negative introductions, have points with Ratio1 and Ratio2 values less than 1).'],\n",
       " 636: ['1471-2105-7-270-6-5',\n",
       "  'GFIG',\n",
       "  'Ortholuge R1    R2 plots for the prokaryotic data,\" illustrating two ortholog data sets and a true-negative data set. (A) Putative orthologous groups from an RBH-predicted data set. (B) Probable true orthologs from a higher quality (more precise) data set. (C) True-negative orthologs (i.e. true paralogs) from the \"\"gene-loss simulation\"\" data set. Darker dots represent putative orthologous groups which have had an ingroup1 true-negative (paralog) introduced into the group. Lighter dots represent putative orthologous groups which have had an ingroup2 true-negative (paralog) introduced into the group. (D)\", (E), (F) are zoomed-in versions of (A), (B), (C), respectively, with axes shown from 0 to 2 instead of 0 to 10. Most putative ortholog groups (particularly for the high quality data set) exhibit low Ratio1 and Ratio2 values (for example, all values are less than 1 for the points in the high quality data set plot), whereas most true-negative groups exhibit higher Ratio1 and Ratio2 values (i.e. only 9% of ingroup1 true negative introductions, and 6% of ingroup2 true negative introductions, have points with Ratio1 and Ratio2 values less than 1).'],\n",
       " 637: ['1471-2105-7-270-6-6',\n",
       "  'GFIG',\n",
       "  'Ortholuge R1    R2 plots for the prokaryotic data,\" illustrating two ortholog data sets and a true-negative data set. (A) Putative orthologous groups from an RBH-predicted data set. (B) Probable true orthologs from a higher quality (more precise) data set. (C) True-negative orthologs (i.e. true paralogs) from the \"\"gene-loss simulation\"\" data set. Darker dots represent putative orthologous groups which have had an ingroup1 true-negative (paralog) introduced into the group. Lighter dots represent putative orthologous groups which have had an ingroup2 true-negative (paralog) introduced into the group. (D)\", (E), (F) are zoomed-in versions of (A), (B), (C), respectively, with axes shown from 0 to 2 instead of 0 to 10. Most putative ortholog groups (particularly for the high quality data set) exhibit low Ratio1 and Ratio2 values (for example, all values are less than 1 for the points in the high quality data set plot), whereas most true-negative groups exhibit higher Ratio1 and Ratio2 values (i.e. only 9% of ingroup1 true negative introductions, and 6% of ingroup2 true negative introductions, have points with Ratio1 and Ratio2 values less than 1).'],\n",
       " 638: ['1471-2105-7-270-8-1',\n",
       "  'GFIG',\n",
       "  'Example of the generation of cut-offs for classification of ssd-orthologs and probable paralogs, based on an iterative-true-negative analysis (i.e. based on an introduction of random sets of true-negatives). The particular analysis illustrated here is a Ratio1 analysis for the mouse, rat, human RefSeq RBH dataset, with true-negatives introduced into the mouse (ingroup1) set. In panel A, the number of putative orthologous groups in each ratio range for the true-negative-transformed data set is shown for the whole data set (light shaded bars) and for just the introduced true-negatives only (dark shaded bars). Note how the distribution of the data set differs from that of the true negatives (i.e. introduced paralogs). In panel B, the proportion of randomly introduced true-negatives at 0.5 ratio range intervals is used to formulate cut-offs (denoted by dashed lines) for classifying ssd-orthologs and probable paralogs for the analysis. For the ssd-orthologs cut-off (left-most dashed line),\" no more than 10% true negatives in a given ratio range are permitted for the ssd-orthologs range. For the probable paralogs cut-off (right-most dashed line) the proportion of true negatives is at or above 50 percent. The resulting middle region bounded by these two cut-off points establishes the \"\"uncertain\"\" orthology class ratio range. Dashed-lines denoting these particular cut-offs are also illustrated on the figure in Panel A for reference. This approach for a true-negative analysis and cut-off generation is also performed for Ratio2 [ Additional file 1 ] and the combination of cut-offs for Ratio1 and Ratio2 are used to classify putative orthologous groups from another data set (such as an RBH-predicted data set) into the three classification levels of \"\"probable ssd-ortholog\"\"\",\" \"\"uncertain\"\" and \"\"probable paralogs\"\". Panel C schematically shows the areas of an R1    R2 that would be classified in this way\", with the cut-off numbers in this particular example matching the RefSeq RBH-based mouse-rat-human analysis (see Table 2 for how these ranges are numerically determined).'],\n",
       " 639: ['1471-2105-7-270-8-2',\n",
       "  'GFIG',\n",
       "  'Example of the generation of cut-offs for classification of ssd-orthologs and probable paralogs, based on an iterative-true-negative analysis (i.e. based on an introduction of random sets of true-negatives). The particular analysis illustrated here is a Ratio1 analysis for the mouse, rat, human RefSeq RBH dataset, with true-negatives introduced into the mouse (ingroup1) set. In panel A, the number of putative orthologous groups in each ratio range for the true-negative-transformed data set is shown for the whole data set (light shaded bars) and for just the introduced true-negatives only (dark shaded bars). Note how the distribution of the data set differs from that of the true negatives (i.e. introduced paralogs). In panel B, the proportion of randomly introduced true-negatives at 0.5 ratio range intervals is used to formulate cut-offs (denoted by dashed lines) for classifying ssd-orthologs and probable paralogs for the analysis. For the ssd-orthologs cut-off (left-most dashed line),\" no more than 10% true negatives in a given ratio range are permitted for the ssd-orthologs range. For the probable paralogs cut-off (right-most dashed line) the proportion of true negatives is at or above 50 percent. The resulting middle region bounded by these two cut-off points establishes the \"\"uncertain\"\" orthology class ratio range. Dashed-lines denoting these particular cut-offs are also illustrated on the figure in Panel A for reference. This approach for a true-negative analysis and cut-off generation is also performed for Ratio2 [ Additional file 1 ] and the combination of cut-offs for Ratio1 and Ratio2 are used to classify putative orthologous groups from another data set (such as an RBH-predicted data set) into the three classification levels of \"\"probable ssd-ortholog\"\"\",\" \"\"uncertain\"\" and \"\"probable paralogs\"\". Panel C schematically shows the areas of an R1    R2 that would be classified in this way\", with the cut-off numbers in this particular example matching the RefSeq RBH-based mouse-rat-human analysis (see Table 2 for how these ranges are numerically determined).'],\n",
       " 640: ['1471-2105-7-270-8-3',\n",
       "  'GFIG',\n",
       "  'Example of the generation of cut-offs for classification of ssd-orthologs and probable paralogs, based on an iterative-true-negative analysis (i.e. based on an introduction of random sets of true-negatives). The particular analysis illustrated here is a Ratio1 analysis for the mouse, rat, human RefSeq RBH dataset, with true-negatives introduced into the mouse (ingroup1) set. In panel A, the number of putative orthologous groups in each ratio range for the true-negative-transformed data set is shown for the whole data set (light shaded bars) and for just the introduced true-negatives only (dark shaded bars). Note how the distribution of the data set differs from that of the true negatives (i.e. introduced paralogs). In panel B, the proportion of randomly introduced true-negatives at 0.5 ratio range intervals is used to formulate cut-offs (denoted by dashed lines) for classifying ssd-orthologs and probable paralogs for the analysis. For the ssd-orthologs cut-off (left-most dashed line),\" no more than 10% true negatives in a given ratio range are permitted for the ssd-orthologs range. For the probable paralogs cut-off (right-most dashed line) the proportion of true negatives is at or above 50 percent. The resulting middle region bounded by these two cut-off points establishes the \"\"uncertain\"\" orthology class ratio range. Dashed-lines denoting these particular cut-offs are also illustrated on the figure in Panel A for reference. This approach for a true-negative analysis and cut-off generation is also performed for Ratio2 [ Additional file 1 ] and the combination of cut-offs for Ratio1 and Ratio2 are used to classify putative orthologous groups from another data set (such as an RBH-predicted data set) into the three classification levels of \"\"probable ssd-ortholog\"\"\",\" \"\"uncertain\"\" and \"\"probable paralogs\"\". Panel C schematically shows the areas of an R1    R2 that would be classified in this way\", with the cut-off numbers in this particular example matching the RefSeq RBH-based mouse-rat-human analysis (see Table 2 for how these ranges are numerically determined).'],\n",
       " 641: ['1471-2105-7-332-1-1',\n",
       "  'GFIG',\n",
       "  'The modeling concept and framework . (a) A feedback link from metabolite 1 to regulator 1 is established (black dash arc). Two perturbation responses are explained by the model. The up regulation of gene 2 by supplying metabolite 1 is explained by the path marked by red arrows (metabolite 1    regulator 1    gene 2). The decrease of flux 3 by deleting enzyme (gene) 1 is explained by the path marked by green arrows (enzyme 1    flux 1    metabolite 2    flux 2    metabolite 3    flux 3). Edges (metabolite 1, regulator 1) and (regulator 1, gene2) must be both positive or both negative in order to explain the first response. (b) The framework of the joint model. A factor graph model is constructed from metabolic and regulatory networks and perturbation data. A learning algorithm is applied to find the identities and functions of feedback links and active paths. These information are used to predict each perturbation response. By comparing with the observed data, each response is explained or contradicted with the prediction.'],\n",
       " 642: ['1471-2105-7-332-1-2',\n",
       "  'GFLO',\n",
       "  'The modeling concept and framework . (a) A feedback link from metabolite 1 to regulator 1 is established (black dash arc). Two perturbation responses are explained by the model. The up regulation of gene 2 by supplying metabolite 1 is explained by the path marked by red arrows (metabolite 1    regulator 1    gene 2). The decrease of flux 3 by deleting enzyme (gene) 1 is explained by the path marked by green arrows (enzyme 1    flux 1    metabolite 2    flux 2    metabolite 3    flux 3). Edges (metabolite 1, regulator 1) and (regulator 1, gene2) must be both positive or both negative in order to explain the first response. (b) The framework of the joint model. A factor graph model is constructed from metabolic and regulatory networks and perturbation data. A learning algorithm is applied to find the identities and functions of feedback links and active paths. These information are used to predict each perturbation response. By comparing with the observed data, each response is explained or contradicted with the prediction.'],\n",
       " 643: ['1471-2105-7-376-2-1',\n",
       "  'GFIG',\n",
       "  'Multiple Alignment Accuracy . A: Alignment accuracy varies across tools and divergences. Mean four species alignment accuracy for each tool was measured as a function of total divergence distance. B: Alignment accuracy improves with the presence of transcription factor binding sites. Mean improved alignment accuracy of enhancers over background sequences for four species alignments was measured as a function of total divergence distance. C: Dividing a fixed total divergence up with more species improves alignment accuracy. Mean Mlagan alignment accuracy for two, three and four species trees was measured as a function of total divergence distance. D: Adding in-group species to a pair of species has no effect on the alignment accuracy of the pair. Mean improved alignment accuracy of three species alignments over two species alignments, where the divergence distance between Seq1 and Seq3 in the three species alignment was the same as the divergence distances between Seq1 and Seq2 in the two species alignment, was measured as a function of divergence distance. E  &  F: Alignment accuracy varies across branches in a tree and is best for leaf-to-leaf alignments and worst for node-to-node alignments, with the exception of highly diverged enhancers. Mean Clustalw alignment accuracy along branches in three and four species trees subtracted from mean two species alignment accuracy, where divergence along each branch is the same as the two species divergence, was measured in background sequences (E) and enhancers (F) as a function of divergence distance.,'],\n",
       " 644: ['1471-2105-7-376-2-2',\n",
       "  'GFIG',\n",
       "  'Multiple Alignment Accuracy . A: Alignment accuracy varies across tools and divergences. Mean four species alignment accuracy for each tool was measured as a function of total divergence distance. B: Alignment accuracy improves with the presence of transcription factor binding sites. Mean improved alignment accuracy of enhancers over background sequences for four species alignments was measured as a function of total divergence distance. C: Dividing a fixed total divergence up with more species improves alignment accuracy. Mean Mlagan alignment accuracy for two, three and four species trees was measured as a function of total divergence distance. D: Adding in-group species to a pair of species has no effect on the alignment accuracy of the pair. Mean improved alignment accuracy of three species alignments over two species alignments, where the divergence distance between Seq1 and Seq3 in the three species alignment was the same as the divergence distances between Seq1 and Seq2 in the two species alignment, was measured as a function of divergence distance. E  &  F: Alignment accuracy varies across branches in a tree and is best for leaf-to-leaf alignments and worst for node-to-node alignments, with the exception of highly diverged enhancers. Mean Clustalw alignment accuracy along branches in three and four species trees subtracted from mean two species alignment accuracy, where divergence along each branch is the same as the two species divergence, was measured in background sequences (E) and enhancers (F) as a function of divergence distance.,'],\n",
       " 645: ['1471-2105-7-376-2-3',\n",
       "  'GFIG',\n",
       "  'Multiple Alignment Accuracy . A: Alignment accuracy varies across tools and divergences. Mean four species alignment accuracy for each tool was measured as a function of total divergence distance. B: Alignment accuracy improves with the presence of transcription factor binding sites. Mean improved alignment accuracy of enhancers over background sequences for four species alignments was measured as a function of total divergence distance. C: Dividing a fixed total divergence up with more species improves alignment accuracy. Mean Mlagan alignment accuracy for two, three and four species trees was measured as a function of total divergence distance. D: Adding in-group species to a pair of species has no effect on the alignment accuracy of the pair. Mean improved alignment accuracy of three species alignments over two species alignments, where the divergence distance between Seq1 and Seq3 in the three species alignment was the same as the divergence distances between Seq1 and Seq2 in the two species alignment, was measured as a function of divergence distance. E  &  F: Alignment accuracy varies across branches in a tree and is best for leaf-to-leaf alignments and worst for node-to-node alignments, with the exception of highly diverged enhancers. Mean Clustalw alignment accuracy along branches in three and four species trees subtracted from mean two species alignment accuracy, where divergence along each branch is the same as the two species divergence, was measured in background sequences (E) and enhancers (F) as a function of divergence distance.,'],\n",
       " 646: ['1471-2105-7-376-2-4',\n",
       "  'GFIG',\n",
       "  'Multiple Alignment Accuracy . A: Alignment accuracy varies across tools and divergences. Mean four species alignment accuracy for each tool was measured as a function of total divergence distance. B: Alignment accuracy improves with the presence of transcription factor binding sites. Mean improved alignment accuracy of enhancers over background sequences for four species alignments was measured as a function of total divergence distance. C: Dividing a fixed total divergence up with more species improves alignment accuracy. Mean Mlagan alignment accuracy for two, three and four species trees was measured as a function of total divergence distance. D: Adding in-group species to a pair of species has no effect on the alignment accuracy of the pair. Mean improved alignment accuracy of three species alignments over two species alignments, where the divergence distance between Seq1 and Seq3 in the three species alignment was the same as the divergence distances between Seq1 and Seq2 in the two species alignment, was measured as a function of divergence distance. E  &  F: Alignment accuracy varies across branches in a tree and is best for leaf-to-leaf alignments and worst for node-to-node alignments, with the exception of highly diverged enhancers. Mean Clustalw alignment accuracy along branches in three and four species trees subtracted from mean two species alignment accuracy, where divergence along each branch is the same as the two species divergence, was measured in background sequences (E) and enhancers (F) as a function of divergence distance.,'],\n",
       " 647: ['1471-2105-7-376-2-5',\n",
       "  'GFIG',\n",
       "  'Multiple Alignment Accuracy . A: Alignment accuracy varies across tools and divergences. Mean four species alignment accuracy for each tool was measured as a function of total divergence distance. B: Alignment accuracy improves with the presence of transcription factor binding sites. Mean improved alignment accuracy of enhancers over background sequences for four species alignments was measured as a function of total divergence distance. C: Dividing a fixed total divergence up with more species improves alignment accuracy. Mean Mlagan alignment accuracy for two, three and four species trees was measured as a function of total divergence distance. D: Adding in-group species to a pair of species has no effect on the alignment accuracy of the pair. Mean improved alignment accuracy of three species alignments over two species alignments, where the divergence distance between Seq1 and Seq3 in the three species alignment was the same as the divergence distances between Seq1 and Seq2 in the two species alignment, was measured as a function of divergence distance. E  &  F: Alignment accuracy varies across branches in a tree and is best for leaf-to-leaf alignments and worst for node-to-node alignments, with the exception of highly diverged enhancers. Mean Clustalw alignment accuracy along branches in three and four species trees subtracted from mean two species alignment accuracy, where divergence along each branch is the same as the two species divergence, was measured in background sequences (E) and enhancers (F) as a function of divergence distance.,'],\n",
       " 648: ['1471-2105-7-376-2-6',\n",
       "  'GFIG',\n",
       "  'Multiple Alignment Accuracy . A: Alignment accuracy varies across tools and divergences. Mean four species alignment accuracy for each tool was measured as a function of total divergence distance. B: Alignment accuracy improves with the presence of transcription factor binding sites. Mean improved alignment accuracy of enhancers over background sequences for four species alignments was measured as a function of total divergence distance. C: Dividing a fixed total divergence up with more species improves alignment accuracy. Mean Mlagan alignment accuracy for two, three and four species trees was measured as a function of total divergence distance. D: Adding in-group species to a pair of species has no effect on the alignment accuracy of the pair. Mean improved alignment accuracy of three species alignments over two species alignments, where the divergence distance between Seq1 and Seq3 in the three species alignment was the same as the divergence distances between Seq1 and Seq2 in the two species alignment, was measured as a function of divergence distance. E  &  F: Alignment accuracy varies across branches in a tree and is best for leaf-to-leaf alignments and worst for node-to-node alignments, with the exception of highly diverged enhancers. Mean Clustalw alignment accuracy along branches in three and four species trees subtracted from mean two species alignment accuracy, where divergence along each branch is the same as the two species divergence, was measured in background sequences (E) and enhancers (F) as a function of divergence distance.,'],\n",
       " 649: ['1471-2105-7-388-5-1',\n",
       "  'GFIG',\n",
       "  'Classification accuracy using different normalization methods . Cross validation results of predictive performance of the same diagnostic signature used with different normalization strategies for diagnostic microarrays. The left plot shows classification accuracies for distinguishing TEL-AML1 from other groups in leukemia ( p s  =  p n  = 5). The right plot shows classification accuracies for distinguishing normal from adenocarcinomas in lung ( p s  =  p n  = 3). The boxplots are sorted by increasing median accuracy. When they have the same median the mean was used for sorting.,'],\n",
       " 650: ['1471-2105-7-388-5-2',\n",
       "  'GFIG',\n",
       "  'Classification accuracy using different normalization methods . Cross validation results of predictive performance of the same diagnostic signature used with different normalization strategies for diagnostic microarrays. The left plot shows classification accuracies for distinguishing TEL-AML1 from other groups in leukemia ( p s  =  p n  = 5). The right plot shows classification accuracies for distinguishing normal from adenocarcinomas in lung ( p s  =  p n  = 3). The boxplots are sorted by increasing median accuracy. When they have the same median the mean was used for sorting.,'],\n",
       " 651: ['1471-2105-7-389-5-1',\n",
       "  'GTAB',\n",
       "  'Minimum size enhanced suffix arrays for worst case analysis . Enhanced suffix arrays for text  S  = cagataaccgtcttggc, consisting of all strings of length  m  = 2 over an alphabet of size 4, and  T  = ccaaacaccc, consisting of all strings of length  m  = 3 over an alphabet of size 2.'],\n",
       " 652: ['1471-2105-7-389-5-2',\n",
       "  'GTAB',\n",
       "  'Minimum size enhanced suffix arrays for worst case analysis . Enhanced suffix arrays for text  S  = cagataaccgtcttggc, consisting of all strings of length  m  = 2 over an alphabet of size 4, and  T  = ccaaacaccc, consisting of all strings of length  m  = 3 over an alphabet of size 2.'],\n",
       " 653: ['1471-2105-7-445-3-1',\n",
       "  'GFIG',\n",
       "  'Histogram of feasible range of flux values for the product formation rates for the metabolic network of  C. glutamicum  through individual elementary mode. The figure depicts the ranges of absolute values for formation of biomass, lysine and trehalose. The details are the same as given in the legend of Figure 2.'],\n",
       " 654: ['1471-2105-7-445-3-2',\n",
       "  'GFIG',\n",
       "  'Histogram of feasible range of flux values for the product formation rates for the metabolic network of  C. glutamicum  through individual elementary mode. The figure depicts the ranges of absolute values for formation of biomass, lysine and trehalose. The details are the same as given in the legend of Figure 2.'],\n",
       " 655: ['1471-2105-7-445-3-3',\n",
       "  'GFIG',\n",
       "  'Histogram of feasible range of flux values for the product formation rates for the metabolic network of  C. glutamicum  through individual elementary mode. The figure depicts the ranges of absolute values for formation of biomass, lysine and trehalose. The details are the same as given in the legend of Figure 2.'],\n",
       " 656: ['1471-2105-7-445-3-4',\n",
       "  'GFIG',\n",
       "  'Histogram of feasible range of flux values for the product formation rates for the metabolic network of  C. glutamicum  through individual elementary mode. The figure depicts the ranges of absolute values for formation of biomass, lysine and trehalose. The details are the same as given in the legend of Figure 2.'],\n",
       " 657: ['1471-2105-7-445-3-5',\n",
       "  'GFIG',\n",
       "  'Histogram of feasible range of flux values for the product formation rates for the metabolic network of  C. glutamicum  through individual elementary mode. The figure depicts the ranges of absolute values for formation of biomass, lysine and trehalose. The details are the same as given in the legend of Figure 2.'],\n",
       " 658: ['1471-2105-7-445-3-6',\n",
       "  'GFIG',\n",
       "  'Histogram of feasible range of flux values for the product formation rates for the metabolic network of  C. glutamicum  through individual elementary mode. The figure depicts the ranges of absolute values for formation of biomass, lysine and trehalose. The details are the same as given in the legend of Figure 2.'],\n",
       " 659: ['1471-2105-7-445-3-7',\n",
       "  'GFIG',\n",
       "  'Histogram of feasible range of flux values for the product formation rates for the metabolic network of  C. glutamicum  through individual elementary mode. The figure depicts the ranges of absolute values for formation of biomass, lysine and trehalose. The details are the same as given in the legend of Figure 2.'],\n",
       " 660: ['1471-2105-7-445-3-8',\n",
       "  'GFIG',\n",
       "  'Histogram of feasible range of flux values for the product formation rates for the metabolic network of  C. glutamicum  through individual elementary mode. The figure depicts the ranges of absolute values for formation of biomass, lysine and trehalose. The details are the same as given in the legend of Figure 2.'],\n",
       " 661: ['1471-2105-7-445-3-9',\n",
       "  'GFIG',\n",
       "  'Histogram of feasible range of flux values for the product formation rates for the metabolic network of  C. glutamicum  through individual elementary mode. The figure depicts the ranges of absolute values for formation of biomass, lysine and trehalose. The details are the same as given in the legend of Figure 2.'],\n",
       " 662: ['1471-2105-7-460-1-1',\n",
       "  'D3DR',\n",
       "  'Backbone smoothing and turning angle series of the structure with SCOP id d1b6ra2 . (A) Stereo images of overlapping backbone and smoothed backbone with smoothing radius d = 3; (B) turning angle series along the smoothed backbone with different angle defining distances, with X-axis labeled by DSSP [38] secondary structure annotation. Data series: A1, A2,..., A5: angle series with angle defining distance d = 1,2,...,5, respectively. All structural diagrams are prepared by using PyMOL [39] .,'],\n",
       " 663: ['1471-2105-7-460-1-2',\n",
       "  'D3DR',\n",
       "  'Backbone smoothing and turning angle series of the structure with SCOP id d1b6ra2 . (A) Stereo images of overlapping backbone and smoothed backbone with smoothing radius d = 3; (B) turning angle series along the smoothed backbone with different angle defining distances, with X-axis labeled by DSSP [38] secondary structure annotation. Data series: A1, A2,..., A5: angle series with angle defining distance d = 1,2,...,5, respectively. All structural diagrams are prepared by using PyMOL [39] .,'],\n",
       " 664: ['1471-2105-7-460-1-3',\n",
       "  'GFIG',\n",
       "  'Backbone smoothing and turning angle series of the structure with SCOP id d1b6ra2 . (A) Stereo images of overlapping backbone and smoothed backbone with smoothing radius d = 3; (B) turning angle series along the smoothed backbone with different angle defining distances, with X-axis labeled by DSSP [38] secondary structure annotation. Data series: A1, A2,..., A5: angle series with angle defining distance d = 1,2,...,5, respectively. All structural diagrams are prepared by using PyMOL [39] .,'],\n",
       " 665: ['1471-2105-7-471-7-1',\n",
       "  'GFIG',\n",
       "  'Comparison of the alignment accuracy values of trees with the same maximum evolutionary distance and different topologies . The accuracy curves for both topologies are very similar, independently of the topology employed. The input trees differed in where evolution had occurred, at the tips or internal branches.,'],\n",
       " 666: ['1471-2105-7-471-7-2',\n",
       "  'GFIG',\n",
       "  'Comparison of the alignment accuracy values of trees with the same maximum evolutionary distance and different topologies . The accuracy curves for both topologies are very similar, independently of the topology employed. The input trees differed in where evolution had occurred, at the tips or internal branches.,'],\n",
       " 667: ['1471-2105-7-477-10-1',\n",
       "  'GFIG',\n",
       "  'Number of observed eMRs and their occurrences in  Sulfolobus solfataricus, Bacillus subtilis  and  Escherichia coli . On each plot, red and green curves refer to eMRs located in the original genome sequence, while blue and purple curves refer to eMRs located in shuffled sequences computed using shuffleseq from the EMBOSS package [42]. For each curve,  NbMR  refers to the total number of eMRs observed for a given size and above (cumulative values), and  NbOcc  refers to the grand total of eMR occurrences observed for a given size and above (cumulative values). [a] displays   N b O c c  N b M R   MathType@MTEF@5@5@+=feaafiart1ev1aaatCvAUfKttLearuWrP9MDH5MBPbIqV92AaeXatLxBI9gBaebbnrfifHhDYfgasaacH8akY=wiFfYdH8Gipec8Eeeu0xXdbba9frFj0=OqFfea0dXdd9vqai=hGuQ8kuc9pgc9s8qqaq=dirpe0xb9q8qiLsFr0=vr0=vr0dc8meaabaqaciaacaGaaeqabaqabeGadaaakeaadaWcaaqaamaaqaeabaGaemOta4KaemOyaiMaem4ta8Kaem4yamMaem4yamgaleqabeqdcqGHris5aaGcbaWaaabqaeaacqWGobGtcqWGIbGycqWGnbqtcqWGsbGuaSqabeqaniabggHiLdaaaaaa@3BE7@ , [b] displays 100.   N b O c c  N b M R   MathType@MTEF@5@5@+=feaafiart1ev1aaatCvAUfKttLearuWrP9MDH5MBPbIqV92AaeXatLxBI9gBaebbnrfifHhDYfgasaacH8akY=wiFfYdH8Gipec8Eeeu0xXdbba9frFj0=OqFfea0dXdd9vqai=hGuQ8kuc9pgc9s8qqaq=dirpe0xb9q8qiLsFr0=vr0=vr0dc8meaabaqaciaacaGaaeqabaqabeGadaaakeaadaWcaaqaamaakaaabaWaaabqaeaacqWGobGtcqWGIbGycqWGpbWtcqWGJbWycqWGJbWyaSqabeqaniabggHiLdaaleqaaaGcbaWaaabqaeaacqWGobGtcqWGIbGycqWGnbqtcqWGsbGuaSqabeqaniabggHiLdaaaaaa@3C02@ , and [c] displays  2  N b M R   MathType@MTEF@5@5@+=feaafiart1ev1aaatCvAUfKttLearuWrP9MDH5MBPbIqV92AaeXatLxBI9gBaebbnrfifHhDYfgasaacH8akY=wiFfYdH8Gipec8Eeeu0xXdbba9frFj0=OqFfea0dXdd9vqai=hGuQ8kuc9pgc9s8qqaq=dirpe0xb9q8qiLsFr0=vr0=vr0dc8meaabaqaciaacaGaaeqabaqabeGadaaakeaadaWcaaqaamaakaaabaGaeGOmaidaleqaaaGcbaWaaabqaeaacqWGobGtcqWGIbGycqWGnbqtcqWGsbGuaSqabeqaniabggHiLdaaaaaa@34A9@ .'],\n",
       " 668: ['1471-2105-7-477-10-2',\n",
       "  'GFIG',\n",
       "  'Number of observed eMRs and their occurrences in  Sulfolobus solfataricus, Bacillus subtilis  and  Escherichia coli . On each plot, red and green curves refer to eMRs located in the original genome sequence, while blue and purple curves refer to eMRs located in shuffled sequences computed using shuffleseq from the EMBOSS package [42]. For each curve,  NbMR  refers to the total number of eMRs observed for a given size and above (cumulative values), and  NbOcc  refers to the grand total of eMR occurrences observed for a given size and above (cumulative values). [a] displays   N b O c c  N b M R   MathType@MTEF@5@5@+=feaafiart1ev1aaatCvAUfKttLearuWrP9MDH5MBPbIqV92AaeXatLxBI9gBaebbnrfifHhDYfgasaacH8akY=wiFfYdH8Gipec8Eeeu0xXdbba9frFj0=OqFfea0dXdd9vqai=hGuQ8kuc9pgc9s8qqaq=dirpe0xb9q8qiLsFr0=vr0=vr0dc8meaabaqaciaacaGaaeqabaqabeGadaaakeaadaWcaaqaamaaqaeabaGaemOta4KaemOyaiMaem4ta8Kaem4yamMaem4yamgaleqabeqdcqGHris5aaGcbaWaaabqaeaacqWGobGtcqWGIbGycqWGnbqtcqWGsbGuaSqabeqaniabggHiLdaaaaaa@3BE7@ , [b] displays 100.   N b O c c  N b M R   MathType@MTEF@5@5@+=feaafiart1ev1aaatCvAUfKttLearuWrP9MDH5MBPbIqV92AaeXatLxBI9gBaebbnrfifHhDYfgasaacH8akY=wiFfYdH8Gipec8Eeeu0xXdbba9frFj0=OqFfea0dXdd9vqai=hGuQ8kuc9pgc9s8qqaq=dirpe0xb9q8qiLsFr0=vr0=vr0dc8meaabaqaciaacaGaaeqabaqabeGadaaakeaadaWcaaqaamaakaaabaWaaabqaeaacqWGobGtcqWGIbGycqWGpbWtcqWGJbWycqWGJbWyaSqabeqaniabggHiLdaaleqaaaGcbaWaaabqaeaacqWGobGtcqWGIbGycqWGnbqtcqWGsbGuaSqabeqaniabggHiLdaaaaaa@3C02@ , and [c] displays  2  N b M R   MathType@MTEF@5@5@+=feaafiart1ev1aaatCvAUfKttLearuWrP9MDH5MBPbIqV92AaeXatLxBI9gBaebbnrfifHhDYfgasaacH8akY=wiFfYdH8Gipec8Eeeu0xXdbba9frFj0=OqFfea0dXdd9vqai=hGuQ8kuc9pgc9s8qqaq=dirpe0xb9q8qiLsFr0=vr0=vr0dc8meaabaqaciaacaGaaeqabaqabeGadaaakeaadaWcaaqaamaakaaabaGaeGOmaidaleqaaaGcbaWaaabqaeaacqWGobGtcqWGIbGycqWGnbqtcqWGsbGuaSqabeqaniabggHiLdaaaaaa@34A9@ .'],\n",
       " 669: ['1471-2105-7-477-10-3',\n",
       "  'GFIG',\n",
       "  'Number of observed eMRs and their occurrences in  Sulfolobus solfataricus, Bacillus subtilis  and  Escherichia coli . On each plot, red and green curves refer to eMRs located in the original genome sequence, while blue and purple curves refer to eMRs located in shuffled sequences computed using shuffleseq from the EMBOSS package [42]. For each curve,  NbMR  refers to the total number of eMRs observed for a given size and above (cumulative values), and  NbOcc  refers to the grand total of eMR occurrences observed for a given size and above (cumulative values). [a] displays   N b O c c  N b M R   MathType@MTEF@5@5@+=feaafiart1ev1aaatCvAUfKttLearuWrP9MDH5MBPbIqV92AaeXatLxBI9gBaebbnrfifHhDYfgasaacH8akY=wiFfYdH8Gipec8Eeeu0xXdbba9frFj0=OqFfea0dXdd9vqai=hGuQ8kuc9pgc9s8qqaq=dirpe0xb9q8qiLsFr0=vr0=vr0dc8meaabaqaciaacaGaaeqabaqabeGadaaakeaadaWcaaqaamaaqaeabaGaemOta4KaemOyaiMaem4ta8Kaem4yamMaem4yamgaleqabeqdcqGHris5aaGcbaWaaabqaeaacqWGobGtcqWGIbGycqWGnbqtcqWGsbGuaSqabeqaniabggHiLdaaaaaa@3BE7@ , [b] displays 100.   N b O c c  N b M R   MathType@MTEF@5@5@+=feaafiart1ev1aaatCvAUfKttLearuWrP9MDH5MBPbIqV92AaeXatLxBI9gBaebbnrfifHhDYfgasaacH8akY=wiFfYdH8Gipec8Eeeu0xXdbba9frFj0=OqFfea0dXdd9vqai=hGuQ8kuc9pgc9s8qqaq=dirpe0xb9q8qiLsFr0=vr0=vr0dc8meaabaqaciaacaGaaeqabaqabeGadaaakeaadaWcaaqaamaakaaabaWaaabqaeaacqWGobGtcqWGIbGycqWGpbWtcqWGJbWycqWGJbWyaSqabeqaniabggHiLdaaleqaaaGcbaWaaabqaeaacqWGobGtcqWGIbGycqWGnbqtcqWGsbGuaSqabeqaniabggHiLdaaaaaa@3C02@ , and [c] displays  2  N b M R   MathType@MTEF@5@5@+=feaafiart1ev1aaatCvAUfKttLearuWrP9MDH5MBPbIqV92AaeXatLxBI9gBaebbnrfifHhDYfgasaacH8akY=wiFfYdH8Gipec8Eeeu0xXdbba9frFj0=OqFfea0dXdd9vqai=hGuQ8kuc9pgc9s8qqaq=dirpe0xb9q8qiLsFr0=vr0=vr0dc8meaabaqaciaacaGaaeqabaqabeGadaaakeaadaWcaaqaamaakaaabaGaeGOmaidaleqaaaGcbaWaaabqaeaacqWGobGtcqWGIbGycqWGnbqtcqWGsbGuaSqabeqaniabggHiLdaaaaaa@34A9@ .'],\n",
       " 670: ['1471-2105-7-477-10-4',\n",
       "  'GFIG',\n",
       "  'Number of observed eMRs and their occurrences in  Sulfolobus solfataricus, Bacillus subtilis  and  Escherichia coli . On each plot, red and green curves refer to eMRs located in the original genome sequence, while blue and purple curves refer to eMRs located in shuffled sequences computed using shuffleseq from the EMBOSS package [42]. For each curve,  NbMR  refers to the total number of eMRs observed for a given size and above (cumulative values), and  NbOcc  refers to the grand total of eMR occurrences observed for a given size and above (cumulative values). [a] displays   N b O c c  N b M R   MathType@MTEF@5@5@+=feaafiart1ev1aaatCvAUfKttLearuWrP9MDH5MBPbIqV92AaeXatLxBI9gBaebbnrfifHhDYfgasaacH8akY=wiFfYdH8Gipec8Eeeu0xXdbba9frFj0=OqFfea0dXdd9vqai=hGuQ8kuc9pgc9s8qqaq=dirpe0xb9q8qiLsFr0=vr0=vr0dc8meaabaqaciaacaGaaeqabaqabeGadaaakeaadaWcaaqaamaaqaeabaGaemOta4KaemOyaiMaem4ta8Kaem4yamMaem4yamgaleqabeqdcqGHris5aaGcbaWaaabqaeaacqWGobGtcqWGIbGycqWGnbqtcqWGsbGuaSqabeqaniabggHiLdaaaaaa@3BE7@ , [b] displays 100.   N b O c c  N b M R   MathType@MTEF@5@5@+=feaafiart1ev1aaatCvAUfKttLearuWrP9MDH5MBPbIqV92AaeXatLxBI9gBaebbnrfifHhDYfgasaacH8akY=wiFfYdH8Gipec8Eeeu0xXdbba9frFj0=OqFfea0dXdd9vqai=hGuQ8kuc9pgc9s8qqaq=dirpe0xb9q8qiLsFr0=vr0=vr0dc8meaabaqaciaacaGaaeqabaqabeGadaaakeaadaWcaaqaamaakaaabaWaaabqaeaacqWGobGtcqWGIbGycqWGpbWtcqWGJbWycqWGJbWyaSqabeqaniabggHiLdaaleqaaaGcbaWaaabqaeaacqWGobGtcqWGIbGycqWGnbqtcqWGsbGuaSqabeqaniabggHiLdaaaaaa@3C02@ , and [c] displays  2  N b M R   MathType@MTEF@5@5@+=feaafiart1ev1aaatCvAUfKttLearuWrP9MDH5MBPbIqV92AaeXatLxBI9gBaebbnrfifHhDYfgasaacH8akY=wiFfYdH8Gipec8Eeeu0xXdbba9frFj0=OqFfea0dXdd9vqai=hGuQ8kuc9pgc9s8qqaq=dirpe0xb9q8qiLsFr0=vr0=vr0dc8meaabaqaciaacaGaaeqabaqabeGadaaakeaadaWcaaqaamaakaaabaGaeGOmaidaleqaaaGcbaWaaabqaeaacqWGobGtcqWGIbGycqWGnbqtcqWGsbGuaSqabeqaniabggHiLdaaaaaa@34A9@ .'],\n",
       " 671: ['1471-2105-7-477-10-5',\n",
       "  'GFIG',\n",
       "  'Number of observed eMRs and their occurrences in  Sulfolobus solfataricus, Bacillus subtilis  and  Escherichia coli . On each plot, red and green curves refer to eMRs located in the original genome sequence, while blue and purple curves refer to eMRs located in shuffled sequences computed using shuffleseq from the EMBOSS package [42]. For each curve,  NbMR  refers to the total number of eMRs observed for a given size and above (cumulative values), and  NbOcc  refers to the grand total of eMR occurrences observed for a given size and above (cumulative values). [a] displays   N b O c c  N b M R   MathType@MTEF@5@5@+=feaafiart1ev1aaatCvAUfKttLearuWrP9MDH5MBPbIqV92AaeXatLxBI9gBaebbnrfifHhDYfgasaacH8akY=wiFfYdH8Gipec8Eeeu0xXdbba9frFj0=OqFfea0dXdd9vqai=hGuQ8kuc9pgc9s8qqaq=dirpe0xb9q8qiLsFr0=vr0=vr0dc8meaabaqaciaacaGaaeqabaqabeGadaaakeaadaWcaaqaamaaqaeabaGaemOta4KaemOyaiMaem4ta8Kaem4yamMaem4yamgaleqabeqdcqGHris5aaGcbaWaaabqaeaacqWGobGtcqWGIbGycqWGnbqtcqWGsbGuaSqabeqaniabggHiLdaaaaaa@3BE7@ , [b] displays 100.   N b O c c  N b M R   MathType@MTEF@5@5@+=feaafiart1ev1aaatCvAUfKttLearuWrP9MDH5MBPbIqV92AaeXatLxBI9gBaebbnrfifHhDYfgasaacH8akY=wiFfYdH8Gipec8Eeeu0xXdbba9frFj0=OqFfea0dXdd9vqai=hGuQ8kuc9pgc9s8qqaq=dirpe0xb9q8qiLsFr0=vr0=vr0dc8meaabaqaciaacaGaaeqabaqabeGadaaakeaadaWcaaqaamaakaaabaWaaabqaeaacqWGobGtcqWGIbGycqWGpbWtcqWGJbWycqWGJbWyaSqabeqaniabggHiLdaaleqaaaGcbaWaaabqaeaacqWGobGtcqWGIbGycqWGnbqtcqWGsbGuaSqabeqaniabggHiLdaaaaaa@3C02@ , and [c] displays  2  N b M R   MathType@MTEF@5@5@+=feaafiart1ev1aaatCvAUfKttLearuWrP9MDH5MBPbIqV92AaeXatLxBI9gBaebbnrfifHhDYfgasaacH8akY=wiFfYdH8Gipec8Eeeu0xXdbba9frFj0=OqFfea0dXdd9vqai=hGuQ8kuc9pgc9s8qqaq=dirpe0xb9q8qiLsFr0=vr0=vr0dc8meaabaqaciaacaGaaeqabaqabeGadaaakeaadaWcaaqaamaakaaabaGaeGOmaidaleqaaaGcbaWaaabqaeaacqWGobGtcqWGIbGycqWGnbqtcqWGsbGuaSqabeqaniabggHiLdaaaaaa@34A9@ .'],\n",
       " 672: ['1471-2105-7-477-10-6',\n",
       "  'GFIG',\n",
       "  'Number of observed eMRs and their occurrences in  Sulfolobus solfataricus, Bacillus subtilis  and  Escherichia coli . On each plot, red and green curves refer to eMRs located in the original genome sequence, while blue and purple curves refer to eMRs located in shuffled sequences computed using shuffleseq from the EMBOSS package [42]. For each curve,  NbMR  refers to the total number of eMRs observed for a given size and above (cumulative values), and  NbOcc  refers to the grand total of eMR occurrences observed for a given size and above (cumulative values). [a] displays   N b O c c  N b M R   MathType@MTEF@5@5@+=feaafiart1ev1aaatCvAUfKttLearuWrP9MDH5MBPbIqV92AaeXatLxBI9gBaebbnrfifHhDYfgasaacH8akY=wiFfYdH8Gipec8Eeeu0xXdbba9frFj0=OqFfea0dXdd9vqai=hGuQ8kuc9pgc9s8qqaq=dirpe0xb9q8qiLsFr0=vr0=vr0dc8meaabaqaciaacaGaaeqabaqabeGadaaakeaadaWcaaqaamaaqaeabaGaemOta4KaemOyaiMaem4ta8Kaem4yamMaem4yamgaleqabeqdcqGHris5aaGcbaWaaabqaeaacqWGobGtcqWGIbGycqWGnbqtcqWGsbGuaSqabeqaniabggHiLdaaaaaa@3BE7@ , [b] displays 100.   N b O c c  N b M R   MathType@MTEF@5@5@+=feaafiart1ev1aaatCvAUfKttLearuWrP9MDH5MBPbIqV92AaeXatLxBI9gBaebbnrfifHhDYfgasaacH8akY=wiFfYdH8Gipec8Eeeu0xXdbba9frFj0=OqFfea0dXdd9vqai=hGuQ8kuc9pgc9s8qqaq=dirpe0xb9q8qiLsFr0=vr0=vr0dc8meaabaqaciaacaGaaeqabaqabeGadaaakeaadaWcaaqaamaakaaabaWaaabqaeaacqWGobGtcqWGIbGycqWGpbWtcqWGJbWycqWGJbWyaSqabeqaniabggHiLdaaleqaaaGcbaWaaabqaeaacqWGobGtcqWGIbGycqWGnbqtcqWGsbGuaSqabeqaniabggHiLdaaaaaa@3C02@ , and [c] displays  2  N b M R   MathType@MTEF@5@5@+=feaafiart1ev1aaatCvAUfKttLearuWrP9MDH5MBPbIqV92AaeXatLxBI9gBaebbnrfifHhDYfgasaacH8akY=wiFfYdH8Gipec8Eeeu0xXdbba9frFj0=OqFfea0dXdd9vqai=hGuQ8kuc9pgc9s8qqaq=dirpe0xb9q8qiLsFr0=vr0=vr0dc8meaabaqaciaacaGaaeqabaqabeGadaaakeaadaWcaaqaamaakaaabaGaeGOmaidaleqaaaGcbaWaaabqaeaacqWGobGtcqWGIbGycqWGnbqtcqWGsbGuaSqabeqaniabggHiLdaaaaaa@34A9@ .'],\n",
       " 673: ['1471-2105-7-516-1-1',\n",
       "  'GFIG',\n",
       "  'The distribution of the number of siRNA experiments per study is highly skewed in the  siRecords  collection.  A.  Studies were categorized based on the number of siRNA experiments reported. Only 6 out of the 1,417 studies (0.4%) reported  >  30 siRNA experiments per study.  B.  The distribution of the total number of records in each category. Six hundred and fifty-seven records (representing 17.5% of the entire dataset) originated from the 6 studies with  >  30 records per study.'],\n",
       " 674: ['1471-2105-7-516-1-2',\n",
       "  'GFIG',\n",
       "  'The distribution of the number of siRNA experiments per study is highly skewed in the  siRecords  collection.  A.  Studies were categorized based on the number of siRNA experiments reported. Only 6 out of the 1,417 studies (0.4%) reported  >  30 siRNA experiments per study.  B.  The distribution of the total number of records in each category. Six hundred and fifty-seven records (representing 17.5% of the entire dataset) originated from the 6 studies with  >  30 records per study.'],\n",
       " 675: ['1471-2105-7-526-5-1',\n",
       "  'GFIG',\n",
       "  'Denoising Step-by-Step . Demonstration of the contribution of each step of the analysis for pattern extraction on Chr.1 for the glioma data-set.  A)  Average profile of expression for Chr 1 for all the 27 tumour cases.  B)  Main pattern of expression (81% of total variability) detected by SVD.  C)  Effect of de-noising on the SVD pattern.  D)  Additional noise removal by inter-probe distance penalization.,'],\n",
       " 676: ['1471-2105-7-526-5-2',\n",
       "  'GFIG',\n",
       "  'Denoising Step-by-Step . Demonstration of the contribution of each step of the analysis for pattern extraction on Chr.1 for the glioma data-set.  A)  Average profile of expression for Chr 1 for all the 27 tumour cases.  B)  Main pattern of expression (81% of total variability) detected by SVD.  C)  Effect of de-noising on the SVD pattern.  D)  Additional noise removal by inter-probe distance penalization.,'],\n",
       " 677: ['1471-2105-7-526-5-3',\n",
       "  'GFIG',\n",
       "  'Denoising Step-by-Step . Demonstration of the contribution of each step of the analysis for pattern extraction on Chr.1 for the glioma data-set.  A)  Average profile of expression for Chr 1 for all the 27 tumour cases.  B)  Main pattern of expression (81% of total variability) detected by SVD.  C)  Effect of de-noising on the SVD pattern.  D)  Additional noise removal by inter-probe distance penalization.,'],\n",
       " 678: ['1471-2105-7-526-5-4',\n",
       "  'GFIG',\n",
       "  'Denoising Step-by-Step . Demonstration of the contribution of each step of the analysis for pattern extraction on Chr.1 for the glioma data-set.  A)  Average profile of expression for Chr 1 for all the 27 tumour cases.  B)  Main pattern of expression (81% of total variability) detected by SVD.  C)  Effect of de-noising on the SVD pattern.  D)  Additional noise removal by inter-probe distance penalization.,'],\n",
       " 679: ['1471-2105-7-527-7-1',\n",
       "  'GFIG',\n",
       "  'Maximum accuracies plotted by divergence .  S ,  S A , and  S L  are the maximum alignment identity produced for each sequence pair by log-affine, affine, and logarithmic gap costs respectively. The subfigures are the same as in Figure 5.,'],\n",
       " 680: ['1471-2105-7-527-7-2',\n",
       "  'GFIG',\n",
       "  'Maximum accuracies plotted by divergence .  S ,  S A , and  S L  are the maximum alignment identity produced for each sequence pair by log-affine, affine, and logarithmic gap costs respectively. The subfigures are the same as in Figure 5.,'],\n",
       " 681: ['1471-2105-7-527-7-3',\n",
       "  'GFIG',\n",
       "  'Maximum accuracies plotted by divergence .  S ,  S A , and  S L  are the maximum alignment identity produced for each sequence pair by log-affine, affine, and logarithmic gap costs respectively. The subfigures are the same as in Figure 5.,'],\n",
       " 682: ['1471-2105-7-527-7-4',\n",
       "  'GFIG',\n",
       "  'Maximum accuracies plotted by divergence .  S ,  S A , and  S L  are the maximum alignment identity produced for each sequence pair by log-affine, affine, and logarithmic gap costs respectively. The subfigures are the same as in Figure 5.,'],\n",
       " 683: ['1471-2105-7-527-7-5',\n",
       "  'GFIG',\n",
       "  'Maximum accuracies plotted by divergence .  S ,  S A , and  S L  are the maximum alignment identity produced for each sequence pair by log-affine, affine, and logarithmic gap costs respectively. The subfigures are the same as in Figure 5.,'],\n",
       " 684: ['1471-2105-7-527-7-6',\n",
       "  'GFIG',\n",
       "  'Maximum accuracies plotted by divergence .  S ,  S A , and  S L  are the maximum alignment identity produced for each sequence pair by log-affine, affine, and logarithmic gap costs respectively. The subfigures are the same as in Figure 5.,'],\n",
       " 685: ['1471-2105-7-S4-S19-4-1',\n",
       "  'GFIG',\n",
       "  'Clustering results of SOM . Each rectangle corresponds to a node of the map. The black thick line in the rectangle corresponds to the profile of the node, and the grey lines correspond to the profiles of the genes in that cluster. The black bars on the left of the profiles are proportional to the number of genes in the clusters.'],\n",
       " 686: ['1471-2105-7-S4-S19-4-2',\n",
       "  'GFIG',\n",
       "  'Clustering results of SOM . Each rectangle corresponds to a node of the map. The black thick line in the rectangle corresponds to the profile of the node, and the grey lines correspond to the profiles of the genes in that cluster. The black bars on the left of the profiles are proportional to the number of genes in the clusters.'],\n",
       " 687: ['1471-2105-7-S4-S19-4-3',\n",
       "  'GFIG',\n",
       "  'Clustering results of SOM . Each rectangle corresponds to a node of the map. The black thick line in the rectangle corresponds to the profile of the node, and the grey lines correspond to the profiles of the genes in that cluster. The black bars on the left of the profiles are proportional to the number of genes in the clusters.'],\n",
       " 688: ['1471-2105-7-S4-S19-4-4',\n",
       "  'GFIG',\n",
       "  'Clustering results of SOM . Each rectangle corresponds to a node of the map. The black thick line in the rectangle corresponds to the profile of the node, and the grey lines correspond to the profiles of the genes in that cluster. The black bars on the left of the profiles are proportional to the number of genes in the clusters.'],\n",
       " 689: ['1471-2105-7-S4-S19-4-5',\n",
       "  'GFIG',\n",
       "  'Clustering results of SOM . Each rectangle corresponds to a node of the map. The black thick line in the rectangle corresponds to the profile of the node, and the grey lines correspond to the profiles of the genes in that cluster. The black bars on the left of the profiles are proportional to the number of genes in the clusters.'],\n",
       " 690: ['1471-2105-7-S4-S19-4-6',\n",
       "  'GFIG',\n",
       "  'Clustering results of SOM . Each rectangle corresponds to a node of the map. The black thick line in the rectangle corresponds to the profile of the node, and the grey lines correspond to the profiles of the genes in that cluster. The black bars on the left of the profiles are proportional to the number of genes in the clusters.'],\n",
       " 691: ['1471-2105-8-107-1-1',\n",
       "  'GFIG',\n",
       "  'Comparison of real-time PCR and simulated melting profiles for different  Naegleria  species . Plots (solid lines) of the first derivatives (for either fluorescence, theta or absorbance) versus temperature for  Naegleria fowleri  (A, B, C),  Naegleria lovaniensis  (D, E, F) and  Naegleria australiensis  (G, H, I) for either DNA amplified and melted in the presence of SYTO9 (A, D, G) or melting simulations conducted using the amplicon DNA sequences and either MELTSIM (B, E, H) or POLAND (C, F, I) respectively. The melt map for each sequence (base position versus predicted  T m ) determined by MELTSIM is indicated by magenta circles. The equivalent profiles calculated by POLAND are indicated by blue plus signs or red squares for first order and second order reactions respectively.,'],\n",
       " 692: ['1471-2105-8-107-1-2',\n",
       "  'GFIG',\n",
       "  'Comparison of real-time PCR and simulated melting profiles for different  Naegleria  species . Plots (solid lines) of the first derivatives (for either fluorescence, theta or absorbance) versus temperature for  Naegleria fowleri  (A, B, C),  Naegleria lovaniensis  (D, E, F) and  Naegleria australiensis  (G, H, I) for either DNA amplified and melted in the presence of SYTO9 (A, D, G) or melting simulations conducted using the amplicon DNA sequences and either MELTSIM (B, E, H) or POLAND (C, F, I) respectively. The melt map for each sequence (base position versus predicted  T m ) determined by MELTSIM is indicated by magenta circles. The equivalent profiles calculated by POLAND are indicated by blue plus signs or red squares for first order and second order reactions respectively.,'],\n",
       " 693: ['1471-2105-8-107-1-3',\n",
       "  'GFIG',\n",
       "  'Comparison of real-time PCR and simulated melting profiles for different  Naegleria  species . Plots (solid lines) of the first derivatives (for either fluorescence, theta or absorbance) versus temperature for  Naegleria fowleri  (A, B, C),  Naegleria lovaniensis  (D, E, F) and  Naegleria australiensis  (G, H, I) for either DNA amplified and melted in the presence of SYTO9 (A, D, G) or melting simulations conducted using the amplicon DNA sequences and either MELTSIM (B, E, H) or POLAND (C, F, I) respectively. The melt map for each sequence (base position versus predicted  T m ) determined by MELTSIM is indicated by magenta circles. The equivalent profiles calculated by POLAND are indicated by blue plus signs or red squares for first order and second order reactions respectively.,'],\n",
       " 694: ['1471-2105-8-107-1-4',\n",
       "  'GFIG',\n",
       "  'Comparison of real-time PCR and simulated melting profiles for different  Naegleria  species . Plots (solid lines) of the first derivatives (for either fluorescence, theta or absorbance) versus temperature for  Naegleria fowleri  (A, B, C),  Naegleria lovaniensis  (D, E, F) and  Naegleria australiensis  (G, H, I) for either DNA amplified and melted in the presence of SYTO9 (A, D, G) or melting simulations conducted using the amplicon DNA sequences and either MELTSIM (B, E, H) or POLAND (C, F, I) respectively. The melt map for each sequence (base position versus predicted  T m ) determined by MELTSIM is indicated by magenta circles. The equivalent profiles calculated by POLAND are indicated by blue plus signs or red squares for first order and second order reactions respectively.,'],\n",
       " 695: ['1471-2105-8-107-1-5',\n",
       "  'GFIG',\n",
       "  'Comparison of real-time PCR and simulated melting profiles for different  Naegleria  species . Plots (solid lines) of the first derivatives (for either fluorescence, theta or absorbance) versus temperature for  Naegleria fowleri  (A, B, C),  Naegleria lovaniensis  (D, E, F) and  Naegleria australiensis  (G, H, I) for either DNA amplified and melted in the presence of SYTO9 (A, D, G) or melting simulations conducted using the amplicon DNA sequences and either MELTSIM (B, E, H) or POLAND (C, F, I) respectively. The melt map for each sequence (base position versus predicted  T m ) determined by MELTSIM is indicated by magenta circles. The equivalent profiles calculated by POLAND are indicated by blue plus signs or red squares for first order and second order reactions respectively.,'],\n",
       " 696: ['1471-2105-8-107-1-6',\n",
       "  'GFIG',\n",
       "  'Comparison of real-time PCR and simulated melting profiles for different  Naegleria  species . Plots (solid lines) of the first derivatives (for either fluorescence, theta or absorbance) versus temperature for  Naegleria fowleri  (A, B, C),  Naegleria lovaniensis  (D, E, F) and  Naegleria australiensis  (G, H, I) for either DNA amplified and melted in the presence of SYTO9 (A, D, G) or melting simulations conducted using the amplicon DNA sequences and either MELTSIM (B, E, H) or POLAND (C, F, I) respectively. The melt map for each sequence (base position versus predicted  T m ) determined by MELTSIM is indicated by magenta circles. The equivalent profiles calculated by POLAND are indicated by blue plus signs or red squares for first order and second order reactions respectively.,'],\n",
       " 697: ['1471-2105-8-107-1-7',\n",
       "  'GFIG',\n",
       "  'Comparison of real-time PCR and simulated melting profiles for different  Naegleria  species . Plots (solid lines) of the first derivatives (for either fluorescence, theta or absorbance) versus temperature for  Naegleria fowleri  (A, B, C),  Naegleria lovaniensis  (D, E, F) and  Naegleria australiensis  (G, H, I) for either DNA amplified and melted in the presence of SYTO9 (A, D, G) or melting simulations conducted using the amplicon DNA sequences and either MELTSIM (B, E, H) or POLAND (C, F, I) respectively. The melt map for each sequence (base position versus predicted  T m ) determined by MELTSIM is indicated by magenta circles. The equivalent profiles calculated by POLAND are indicated by blue plus signs or red squares for first order and second order reactions respectively.,'],\n",
       " 698: ['1471-2105-8-107-1-8',\n",
       "  'GFIG',\n",
       "  'Comparison of real-time PCR and simulated melting profiles for different  Naegleria  species . Plots (solid lines) of the first derivatives (for either fluorescence, theta or absorbance) versus temperature for  Naegleria fowleri  (A, B, C),  Naegleria lovaniensis  (D, E, F) and  Naegleria australiensis  (G, H, I) for either DNA amplified and melted in the presence of SYTO9 (A, D, G) or melting simulations conducted using the amplicon DNA sequences and either MELTSIM (B, E, H) or POLAND (C, F, I) respectively. The melt map for each sequence (base position versus predicted  T m ) determined by MELTSIM is indicated by magenta circles. The equivalent profiles calculated by POLAND are indicated by blue plus signs or red squares for first order and second order reactions respectively.,'],\n",
       " 699: ['1471-2105-8-107-1-9',\n",
       "  'GFIG',\n",
       "  'Comparison of real-time PCR and simulated melting profiles for different  Naegleria  species . Plots (solid lines) of the first derivatives (for either fluorescence, theta or absorbance) versus temperature for  Naegleria fowleri  (A, B, C),  Naegleria lovaniensis  (D, E, F) and  Naegleria australiensis  (G, H, I) for either DNA amplified and melted in the presence of SYTO9 (A, D, G) or melting simulations conducted using the amplicon DNA sequences and either MELTSIM (B, E, H) or POLAND (C, F, I) respectively. The melt map for each sequence (base position versus predicted  T m ) determined by MELTSIM is indicated by magenta circles. The equivalent profiles calculated by POLAND are indicated by blue plus signs or red squares for first order and second order reactions respectively.,'],\n",
       " 700: ['1471-2105-8-107-2-1',\n",
       "  'GFIG',\n",
       "  'Comparison of real-time PCR and simulated melting profiles for different  Cryptosporidium  species . Plots (solid lines) of the first derivatives (for either fluorescence, theta or absorbance) versus temperature for  Cryptosporidium parvum  (A, B, C) and  Cryptosporidium muris  (D, E, F) for either DNA amplified and melted in the presence of SYTO9 (A, D) or melting simulations conducted using the amplicon DNA sequences and either MELTSIM (B, E) or POLAND (C, F) respectively. The melt map for each sequence (base position versus predicted  T m ) determined by MELTSIM is indicated by magenta circles. The equivalent profiles calculated by POLAND are indicated by blue plus signs or red squares for first order and second order reactions respectively.,'],\n",
       " 701: ['1471-2105-8-107-2-2',\n",
       "  'GFIG',\n",
       "  'Comparison of real-time PCR and simulated melting profiles for different  Cryptosporidium  species . Plots (solid lines) of the first derivatives (for either fluorescence, theta or absorbance) versus temperature for  Cryptosporidium parvum  (A, B, C) and  Cryptosporidium muris  (D, E, F) for either DNA amplified and melted in the presence of SYTO9 (A, D) or melting simulations conducted using the amplicon DNA sequences and either MELTSIM (B, E) or POLAND (C, F) respectively. The melt map for each sequence (base position versus predicted  T m ) determined by MELTSIM is indicated by magenta circles. The equivalent profiles calculated by POLAND are indicated by blue plus signs or red squares for first order and second order reactions respectively.,'],\n",
       " 702: ['1471-2105-8-107-2-3',\n",
       "  'GFIG',\n",
       "  'Comparison of real-time PCR and simulated melting profiles for different  Cryptosporidium  species . Plots (solid lines) of the first derivatives (for either fluorescence, theta or absorbance) versus temperature for  Cryptosporidium parvum  (A, B, C) and  Cryptosporidium muris  (D, E, F) for either DNA amplified and melted in the presence of SYTO9 (A, D) or melting simulations conducted using the amplicon DNA sequences and either MELTSIM (B, E) or POLAND (C, F) respectively. The melt map for each sequence (base position versus predicted  T m ) determined by MELTSIM is indicated by magenta circles. The equivalent profiles calculated by POLAND are indicated by blue plus signs or red squares for first order and second order reactions respectively.,'],\n",
       " 703: ['1471-2105-8-107-2-4',\n",
       "  'GFIG',\n",
       "  'Comparison of real-time PCR and simulated melting profiles for different  Cryptosporidium  species . Plots (solid lines) of the first derivatives (for either fluorescence, theta or absorbance) versus temperature for  Cryptosporidium parvum  (A, B, C) and  Cryptosporidium muris  (D, E, F) for either DNA amplified and melted in the presence of SYTO9 (A, D) or melting simulations conducted using the amplicon DNA sequences and either MELTSIM (B, E) or POLAND (C, F) respectively. The melt map for each sequence (base position versus predicted  T m ) determined by MELTSIM is indicated by magenta circles. The equivalent profiles calculated by POLAND are indicated by blue plus signs or red squares for first order and second order reactions respectively.,'],\n",
       " 704: ['1471-2105-8-107-2-5',\n",
       "  'GFIG',\n",
       "  'Comparison of real-time PCR and simulated melting profiles for different  Cryptosporidium  species . Plots (solid lines) of the first derivatives (for either fluorescence, theta or absorbance) versus temperature for  Cryptosporidium parvum  (A, B, C) and  Cryptosporidium muris  (D, E, F) for either DNA amplified and melted in the presence of SYTO9 (A, D) or melting simulations conducted using the amplicon DNA sequences and either MELTSIM (B, E) or POLAND (C, F) respectively. The melt map for each sequence (base position versus predicted  T m ) determined by MELTSIM is indicated by magenta circles. The equivalent profiles calculated by POLAND are indicated by blue plus signs or red squares for first order and second order reactions respectively.,'],\n",
       " 705: ['1471-2105-8-107-2-6',\n",
       "  'GFIG',\n",
       "  'Comparison of real-time PCR and simulated melting profiles for different  Cryptosporidium  species . Plots (solid lines) of the first derivatives (for either fluorescence, theta or absorbance) versus temperature for  Cryptosporidium parvum  (A, B, C) and  Cryptosporidium muris  (D, E, F) for either DNA amplified and melted in the presence of SYTO9 (A, D) or melting simulations conducted using the amplicon DNA sequences and either MELTSIM (B, E) or POLAND (C, F) respectively. The melt map for each sequence (base position versus predicted  T m ) determined by MELTSIM is indicated by magenta circles. The equivalent profiles calculated by POLAND are indicated by blue plus signs or red squares for first order and second order reactions respectively.,'],\n",
       " 706: ['1471-2105-8-232-3-1',\n",
       "  'GFIG',\n",
       "  'Cumulative overlap for the helices of MscL . Plots for the rotation (a) and translation (b) displacements of each helix around and along its own axis, respectively. Overlap (c) and cumulative overlap (d) of the rotation of the TM bundle around its axis.'],\n",
       " 707: ['1471-2105-8-232-3-2',\n",
       "  'GFIG',\n",
       "  'Cumulative overlap for the helices of MscL . Plots for the rotation (a) and translation (b) displacements of each helix around and along its own axis, respectively. Overlap (c) and cumulative overlap (d) of the rotation of the TM bundle around its axis.'],\n",
       " 708: ['1471-2105-8-232-3-3',\n",
       "  'GFIG',\n",
       "  'Cumulative overlap for the helices of MscL . Plots for the rotation (a) and translation (b) displacements of each helix around and along its own axis, respectively. Overlap (c) and cumulative overlap (d) of the rotation of the TM bundle around its axis.'],\n",
       " 709: ['1471-2105-8-232-3-4',\n",
       "  'GFIG',\n",
       "  'Cumulative overlap for the helices of MscL . Plots for the rotation (a) and translation (b) displacements of each helix around and along its own axis, respectively. Overlap (c) and cumulative overlap (d) of the rotation of the TM bundle around its axis.'],\n",
       " 710: ['1471-2105-8-265-2-1',\n",
       "  'GFIG',\n",
       "  'Functional and localizational co-occurrence of interacting proteins with respect to their reliability . The proportion of the interacting pairs that co-occur in the same functional and localizational categories, with respect to their reliability measured by semantic similarity, semantic interactivity and co-expression. We used (a) the functional categories on the third level in a hierarchy from MIPS, which includes 170 different functions, and (b) 50 distinct localizational categories from MIPS. The semantic similarity and semantic interactivity provide positive correlations with the co-occurrence in functions and localization.,'],\n",
       " 711: ['1471-2105-8-265-2-2',\n",
       "  'GFIG',\n",
       "  'Functional and localizational co-occurrence of interacting proteins with respect to their reliability . The proportion of the interacting pairs that co-occur in the same functional and localizational categories, with respect to their reliability measured by semantic similarity, semantic interactivity and co-expression. We used (a) the functional categories on the third level in a hierarchy from MIPS, which includes 170 different functions, and (b) 50 distinct localizational categories from MIPS. The semantic similarity and semantic interactivity provide positive correlations with the co-occurrence in functions and localization.,'],\n",
       " 712: ['1471-2105-8-28-1-1',\n",
       "  'GFIG',\n",
       "  'S2d and S1d for the BRCA and Lymphoma data sets. (a) A scatter plot of the BRCA data, with log  S ^   MathType@MTEF@5@5@+=feaafiart1ev1aaatCvAUfKttLearuWrP9MDH5MBPbIqV92AaeXatLxBI9gBaebbnrfifHhDYfgasaacH8akY=wiFfYdH8Gipec8Eeeu0xXdbba9frFj0=OqFfea0dXdd9vqai=hGuQ8kuc9pgc9s8qqaq=dirpe0xb9q8qiLsFr0=vr0=vr0dc8meaabaqaciaacaGaaeqabaqabeGadaaakeaacuWGtbWugaqcaaaa@2DEB@  on the horizontal axis and log  se  on the vertical axis. Each symbol corresponds to a gene. The isolines shown are the local fdr based on the S2d method. (b) The same as (a) for the Lymphoma data. (c) The local fdr for the BRCA data, shown as a function of log  S ^   MathType@MTEF@5@5@+=feaafiart1ev1aaatCvAUfKttLearuWrP9MDH5MBPbIqV92AaeXatLxBI9gBaebbnrfifHhDYfgasaacH8akY=wiFfYdH8Gipec8Eeeu0xXdbba9frFj0=OqFfea0dXdd9vqai=hGuQ8kuc9pgc9s8qqaq=dirpe0xb9q8qiLsFr0=vr0=vr0dc8meaabaqaciaacaGaaeqabaqabeGadaaakeaacuWGtbWugaqcaaaa@2DEB@  on the horizontal axis. The black line shows the local fdr computed via S1d, the red line shows the fdr based on S2d, averaged across the log standard errors as shown in (a) above. (d) The same as (c) for the Lymphoma data.,'],\n",
       " 713: ['1471-2105-8-28-1-2',\n",
       "  'GFIG',\n",
       "  'S2d and S1d for the BRCA and Lymphoma data sets. (a) A scatter plot of the BRCA data, with log  S ^   MathType@MTEF@5@5@+=feaafiart1ev1aaatCvAUfKttLearuWrP9MDH5MBPbIqV92AaeXatLxBI9gBaebbnrfifHhDYfgasaacH8akY=wiFfYdH8Gipec8Eeeu0xXdbba9frFj0=OqFfea0dXdd9vqai=hGuQ8kuc9pgc9s8qqaq=dirpe0xb9q8qiLsFr0=vr0=vr0dc8meaabaqaciaacaGaaeqabaqabeGadaaakeaacuWGtbWugaqcaaaa@2DEB@  on the horizontal axis and log  se  on the vertical axis. Each symbol corresponds to a gene. The isolines shown are the local fdr based on the S2d method. (b) The same as (a) for the Lymphoma data. (c) The local fdr for the BRCA data, shown as a function of log  S ^   MathType@MTEF@5@5@+=feaafiart1ev1aaatCvAUfKttLearuWrP9MDH5MBPbIqV92AaeXatLxBI9gBaebbnrfifHhDYfgasaacH8akY=wiFfYdH8Gipec8Eeeu0xXdbba9frFj0=OqFfea0dXdd9vqai=hGuQ8kuc9pgc9s8qqaq=dirpe0xb9q8qiLsFr0=vr0=vr0dc8meaabaqaciaacaGaaeqabaqabeGadaaakeaacuWGtbWugaqcaaaa@2DEB@  on the horizontal axis. The black line shows the local fdr computed via S1d, the red line shows the fdr based on S2d, averaged across the log standard errors as shown in (a) above. (d) The same as (c) for the Lymphoma data.,'],\n",
       " 714: ['1471-2105-8-28-1-3',\n",
       "  'GFIG',\n",
       "  'S2d and S1d for the BRCA and Lymphoma data sets. (a) A scatter plot of the BRCA data, with log  S ^   MathType@MTEF@5@5@+=feaafiart1ev1aaatCvAUfKttLearuWrP9MDH5MBPbIqV92AaeXatLxBI9gBaebbnrfifHhDYfgasaacH8akY=wiFfYdH8Gipec8Eeeu0xXdbba9frFj0=OqFfea0dXdd9vqai=hGuQ8kuc9pgc9s8qqaq=dirpe0xb9q8qiLsFr0=vr0=vr0dc8meaabaqaciaacaGaaeqabaqabeGadaaakeaacuWGtbWugaqcaaaa@2DEB@  on the horizontal axis and log  se  on the vertical axis. Each symbol corresponds to a gene. The isolines shown are the local fdr based on the S2d method. (b) The same as (a) for the Lymphoma data. (c) The local fdr for the BRCA data, shown as a function of log  S ^   MathType@MTEF@5@5@+=feaafiart1ev1aaatCvAUfKttLearuWrP9MDH5MBPbIqV92AaeXatLxBI9gBaebbnrfifHhDYfgasaacH8akY=wiFfYdH8Gipec8Eeeu0xXdbba9frFj0=OqFfea0dXdd9vqai=hGuQ8kuc9pgc9s8qqaq=dirpe0xb9q8qiLsFr0=vr0=vr0dc8meaabaqaciaacaGaaeqabaqabeGadaaakeaacuWGtbWugaqcaaaa@2DEB@  on the horizontal axis. The black line shows the local fdr computed via S1d, the red line shows the fdr based on S2d, averaged across the log standard errors as shown in (a) above. (d) The same as (c) for the Lymphoma data.,'],\n",
       " 715: ['1471-2105-8-28-1-4',\n",
       "  'GFIG',\n",
       "  'S2d and S1d for the BRCA and Lymphoma data sets. (a) A scatter plot of the BRCA data, with log  S ^   MathType@MTEF@5@5@+=feaafiart1ev1aaatCvAUfKttLearuWrP9MDH5MBPbIqV92AaeXatLxBI9gBaebbnrfifHhDYfgasaacH8akY=wiFfYdH8Gipec8Eeeu0xXdbba9frFj0=OqFfea0dXdd9vqai=hGuQ8kuc9pgc9s8qqaq=dirpe0xb9q8qiLsFr0=vr0=vr0dc8meaabaqaciaacaGaaeqabaqabeGadaaakeaacuWGtbWugaqcaaaa@2DEB@  on the horizontal axis and log  se  on the vertical axis. Each symbol corresponds to a gene. The isolines shown are the local fdr based on the S2d method. (b) The same as (a) for the Lymphoma data. (c) The local fdr for the BRCA data, shown as a function of log  S ^   MathType@MTEF@5@5@+=feaafiart1ev1aaatCvAUfKttLearuWrP9MDH5MBPbIqV92AaeXatLxBI9gBaebbnrfifHhDYfgasaacH8akY=wiFfYdH8Gipec8Eeeu0xXdbba9frFj0=OqFfea0dXdd9vqai=hGuQ8kuc9pgc9s8qqaq=dirpe0xb9q8qiLsFr0=vr0=vr0dc8meaabaqaciaacaGaaeqabaqabeGadaaakeaacuWGtbWugaqcaaaa@2DEB@  on the horizontal axis. The black line shows the local fdr computed via S1d, the red line shows the fdr based on S2d, averaged across the log standard errors as shown in (a) above. (d) The same as (c) for the Lymphoma data.,'],\n",
       " 716: ['1471-2105-8-299-6-1',\n",
       "  'GFIG',\n",
       "  'Analysis of RMT approach for robustness to noise . Increasing levels of Gaussian noise are added to the yeast cell cycling microarray expression profiles. The mean of noise is zero and its standard deviation (  Noise ) is set to 1,2,5,10,20,30,40, and 50% of the average of absolute expression value of whole dataset. (A) Percentage of preserved links over total links in the modules perturbed by noise (red), and over total links in original modules (green) at different levels of noise. (B) Percentage of preserved Genes over total genes in the modules perturbed by noise (red), and over total genes in original modules (green) at different levels of noise. (C) Increased noise decreases the cutoff to separate modules.'],\n",
       " 717: ['1471-2105-8-299-6-2',\n",
       "  'GFIG',\n",
       "  'Analysis of RMT approach for robustness to noise . Increasing levels of Gaussian noise are added to the yeast cell cycling microarray expression profiles. The mean of noise is zero and its standard deviation (  Noise ) is set to 1,2,5,10,20,30,40, and 50% of the average of absolute expression value of whole dataset. (A) Percentage of preserved links over total links in the modules perturbed by noise (red), and over total links in original modules (green) at different levels of noise. (B) Percentage of preserved Genes over total genes in the modules perturbed by noise (red), and over total genes in original modules (green) at different levels of noise. (C) Increased noise decreases the cutoff to separate modules.'],\n",
       " 718: ['1471-2105-8-299-6-3',\n",
       "  'GTAB',\n",
       "  'Analysis of RMT approach for robustness to noise . Increasing levels of Gaussian noise are added to the yeast cell cycling microarray expression profiles. The mean of noise is zero and its standard deviation (  Noise ) is set to 1,2,5,10,20,30,40, and 50% of the average of absolute expression value of whole dataset. (A) Percentage of preserved links over total links in the modules perturbed by noise (red), and over total links in original modules (green) at different levels of noise. (B) Percentage of preserved Genes over total genes in the modules perturbed by noise (red), and over total genes in original modules (green) at different levels of noise. (C) Increased noise decreases the cutoff to separate modules.'],\n",
       " 719: ['1471-2105-8-359-3-1',\n",
       "  'GFIG',\n",
       "  '(A) Distribution of the FDR versus d-statistic versus log fold change as shown in the representative 0 ï¿½ï¿½ 2 hour HisH4 data. (B) 0 ï¿½ï¿½ 2 hour HisH4 data, corroborates that no length based bias is introduced the estimation of the d-statistic and/or FDR.'],\n",
       " 720: ['1471-2105-8-359-3-2',\n",
       "  'GFIG',\n",
       "  '(A) Distribution of the FDR versus d-statistic versus log fold change as shown in the representative 0 ï¿½ï¿½ 2 hour HisH4 data. (B) 0 ï¿½ï¿½ 2 hour HisH4 data, corroborates that no length based bias is introduced the estimation of the d-statistic and/or FDR.'],\n",
       " 721: ['1471-2105-8-359-3-3',\n",
       "  'GFIG',\n",
       "  '(A) Distribution of the FDR versus d-statistic versus log fold change as shown in the representative 0 ï¿½ï¿½ 2 hour HisH4 data. (B) 0 ï¿½ï¿½ 2 hour HisH4 data, corroborates that no length based bias is introduced the estimation of the d-statistic and/or FDR.'],\n",
       " 722: ['1471-2105-8-373-1-1',\n",
       "  'GFIG',\n",
       "  '(a) Correlation between the differences in experimental inhibitor ln(k on ) and differences in electrostatic potentials of AchE and (b). leave-two-out cross-validation for prediction of k on  values for the inhibitor TFK+ and AChE . Each point in part (a) represents a pair of AChE variants for which the natural log (ln) of the difference in association rate constant, k on , for the inhibitor TFK+ is plotted on the x-axis and the average electrostatic potential difference in the comparison region is plotted on the y-axis. The straight line corresponds to the best linear fit and is given by y = -1.39*x. The data are for wild-type AChE and 10 mutants (see Table 1, no k on value for TFK+ is available for mutant 04). For leave-two-out cross-validation predictions presented in (b), 2 cases were omitted when deriving the correlation factor  Î±   in formula (1) and these 2 cases were predicted using formula (1) with the derived factor  Î± . Predictions are shown as vertical lines connecting minimum and maximum values from all (55) different predictions.,'],\n",
       " 723: ['1471-2105-8-373-1-2',\n",
       "  'GFIG',\n",
       "  '(a) Correlation between the differences in experimental inhibitor ln(k on ) and differences in electrostatic potentials of AchE and (b). leave-two-out cross-validation for prediction of k on  values for the inhibitor TFK+ and AChE . Each point in part (a) represents a pair of AChE variants for which the natural log (ln) of the difference in association rate constant, k on , for the inhibitor TFK+ is plotted on the x-axis and the average electrostatic potential difference in the comparison region is plotted on the y-axis. The straight line corresponds to the best linear fit and is given by y = -1.39*x. The data are for wild-type AChE and 10 mutants (see Table 1, no k on value for TFK+ is available for mutant 04). For leave-two-out cross-validation predictions presented in (b), 2 cases were omitted when deriving the correlation factor  Î±   in formula (1) and these 2 cases were predicted using formula (1) with the derived factor  Î± . Predictions are shown as vertical lines connecting minimum and maximum values from all (55) different predictions.,'],\n",
       " 724: ['1471-2105-8-373-4-1',\n",
       "  'GFIG',\n",
       "  '(a) Correlation between experimental ln(k cat  /K m ) for superoxide and electrostatic potential differences for SOD from different organisms or under different conditions, and (b) experimental and calculated pH dependence of the rate constant for association of superoxide with bovine SOD . (a) The reference point (0,0) marked by a cross is for bovine SOD at pH 7.7 and 20 mM ionic strength with corresponding experimental rate constant of 3.8 10 9 M -1 s -1  [26]. The other 2 crosses are for human and  Photobacterium leiognathi  SODs with experimental rate constants of 2.5 and 8.5 10 9 M -1 s -1 , respectively, at pH7 and 20 mM ionic strength [55]. Open circles : bovine SOD at pH 7.7 under ionic strength values of 20,40,90, 160 and 250 mM [26] . Connected filled circles: bovine SOD at 20 mM ionic strength and pH values ranging from 7.7 to 12.3 [26]. All points can be approximated by a linear relation y = 0.45*x (R-coefficient 0.97). The ionic strength dependence alone can be fit with y = 0.3*x (R-coefficient 0.99). On panel (b), experimental rates [26]  are shown as filled circles. The pH dependence of the rates is calculated by assigning 2 different values of the dielectric constant of the protein interior ( Îµ ), 4 and 78, when computing residue pK a  values (see Methods section). The value of 78 was used for pK a  calculations for all titratable residues and was expected to give better agreement with experiment [28].,'],\n",
       " 725: ['1471-2105-8-373-4-2',\n",
       "  'GFIG',\n",
       "  '(a) Correlation between experimental ln(k cat  /K m ) for superoxide and electrostatic potential differences for SOD from different organisms or under different conditions, and (b) experimental and calculated pH dependence of the rate constant for association of superoxide with bovine SOD . (a) The reference point (0,0) marked by a cross is for bovine SOD at pH 7.7 and 20 mM ionic strength with corresponding experimental rate constant of 3.8 10 9 M -1 s -1  [26]. The other 2 crosses are for human and  Photobacterium leiognathi  SODs with experimental rate constants of 2.5 and 8.5 10 9 M -1 s -1 , respectively, at pH7 and 20 mM ionic strength [55]. Open circles : bovine SOD at pH 7.7 under ionic strength values of 20,40,90, 160 and 250 mM [26] . Connected filled circles: bovine SOD at 20 mM ionic strength and pH values ranging from 7.7 to 12.3 [26]. All points can be approximated by a linear relation y = 0.45*x (R-coefficient 0.97). The ionic strength dependence alone can be fit with y = 0.3*x (R-coefficient 0.99). On panel (b), experimental rates [26]  are shown as filled circles. The pH dependence of the rates is calculated by assigning 2 different values of the dielectric constant of the protein interior ( Îµ ), 4 and 78, when computing residue pK a  values (see Methods section). The value of 78 was used for pK a  calculations for all titratable residues and was expected to give better agreement with experiment [28].,'],\n",
       " 726: ['1471-2105-8-408-4-1',\n",
       "  'GFIG',\n",
       "  'Scatter plots of statistical metrics for the derived and artificial functional modules. Each data point represents statistical value for a certain functional module (x-axis) and its artificially created corresponding module (y-axis). The red dashed line corresponds to the line y = x. When a data point is below the line then the artificial module has a lower statistical value than the derived one, while the opposite stands for the case a data point is above the line. When the data point is on the line it means that the derived and its corresponding artificial module have the same value.  (A)  The metric used in this plot is connectivity density, which is a measure of how densely connected is a specific module.  (B)  Representation of R measuring the coverage in protein complexes of a detected functional module. It is evident from both diagrams that in all cases the derived from DMSP functional modules have better statistical values than the artificial ones.,'],\n",
       " 727: ['1471-2105-8-408-4-2',\n",
       "  'GFIG',\n",
       "  'Scatter plots of statistical metrics for the derived and artificial functional modules. Each data point represents statistical value for a certain functional module (x-axis) and its artificially created corresponding module (y-axis). The red dashed line corresponds to the line y = x. When a data point is below the line then the artificial module has a lower statistical value than the derived one, while the opposite stands for the case a data point is above the line. When the data point is on the line it means that the derived and its corresponding artificial module have the same value.  (A)  The metric used in this plot is connectivity density, which is a measure of how densely connected is a specific module.  (B)  Representation of R measuring the coverage in protein complexes of a detected functional module. It is evident from both diagrams that in all cases the derived from DMSP functional modules have better statistical values than the artificial ones.,'],\n",
       " 728: ['1471-2105-8-421-6-1',\n",
       "  'GFIG',\n",
       "  'Extracellular fluxes and growth rate calculated from the measured concentrations .  Î¼   is the biomass growth rate,  v g , the flux of glucose,  v Q  the flux of glutamine,  v L  the flux of lactate,  v A  the flux of alanine and  v NH 4  the flux of ammonia. Fluxes are calculated with the middle point Euler approximation (black solid line) and the backward Euler approximation (green dotted line). In addition, fluxes calculated with the backward Euler approximation and filtered with a standard moving average of order 2 are also depicted (blue solid line).,'],\n",
       " 729: ['1471-2105-8-421-6-2',\n",
       "  'GFIG',\n",
       "  'Extracellular fluxes and growth rate calculated from the measured concentrations .  Î¼   is the biomass growth rate,  v g , the flux of glucose,  v Q  the flux of glutamine,  v L  the flux of lactate,  v A  the flux of alanine and  v NH 4  the flux of ammonia. Fluxes are calculated with the middle point Euler approximation (black solid line) and the backward Euler approximation (green dotted line). In addition, fluxes calculated with the backward Euler approximation and filtered with a standard moving average of order 2 are also depicted (blue solid line).,'],\n",
       " 730: ['1471-2105-8-421-6-3',\n",
       "  'GFIG',\n",
       "  'Extracellular fluxes and growth rate calculated from the measured concentrations .  Î¼   is the biomass growth rate,  v g , the flux of glucose,  v Q  the flux of glutamine,  v L  the flux of lactate,  v A  the flux of alanine and  v NH 4  the flux of ammonia. Fluxes are calculated with the middle point Euler approximation (black solid line) and the backward Euler approximation (green dotted line). In addition, fluxes calculated with the backward Euler approximation and filtered with a standard moving average of order 2 are also depicted (blue solid line).,'],\n",
       " 731: ['1471-2105-8-421-6-4',\n",
       "  'GFIG',\n",
       "  'Extracellular fluxes and growth rate calculated from the measured concentrations .  Î¼   is the biomass growth rate,  v g , the flux of glucose,  v Q  the flux of glutamine,  v L  the flux of lactate,  v A  the flux of alanine and  v NH 4  the flux of ammonia. Fluxes are calculated with the middle point Euler approximation (black solid line) and the backward Euler approximation (green dotted line). In addition, fluxes calculated with the backward Euler approximation and filtered with a standard moving average of order 2 are also depicted (blue solid line).,'],\n",
       " 732: ['1471-2105-8-421-6-5',\n",
       "  'GFIG',\n",
       "  'Extracellular fluxes and growth rate calculated from the measured concentrations .  Î¼   is the biomass growth rate,  v g , the flux of glucose,  v Q  the flux of glutamine,  v L  the flux of lactate,  v A  the flux of alanine and  v NH 4  the flux of ammonia. Fluxes are calculated with the middle point Euler approximation (black solid line) and the backward Euler approximation (green dotted line). In addition, fluxes calculated with the backward Euler approximation and filtered with a standard moving average of order 2 are also depicted (blue solid line).,'],\n",
       " 733: ['1471-2105-8-421-6-6',\n",
       "  'GFIG',\n",
       "  'Extracellular fluxes and growth rate calculated from the measured concentrations .  Î¼   is the biomass growth rate,  v g , the flux of glucose,  v Q  the flux of glutamine,  v L  the flux of lactate,  v A  the flux of alanine and  v NH 4  the flux of ammonia. Fluxes are calculated with the middle point Euler approximation (black solid line) and the backward Euler approximation (green dotted line). In addition, fluxes calculated with the backward Euler approximation and filtered with a standard moving average of order 2 are also depicted (blue solid line).,'],\n",
       " 734: ['1471-2105-8-470-4-1',\n",
       "  'D3DR',\n",
       "  'The residues that correspond to several of the highest peaks in Figure 3 as van der Waals spheres for (a)  Pf -Ago and (b)  Aa -Ago.,'],\n",
       " 735: ['1471-2105-8-470-4-2',\n",
       "  'D3DR',\n",
       "  'The residues that correspond to several of the highest peaks in Figure 3 as van der Waals spheres for (a)  Pf -Ago and (b)  Aa -Ago.,'],\n",
       " 736: ['1471-2105-8-476-2-1',\n",
       "  'GFIG',\n",
       "  'Conditional Independence Graphs . Conditional independence graphs for the estimated log-linear models for the  itpr1  gene. For each graph, the predictive probability score (7) is reported as a goodness of fit measure. Note the strong mutual interaction between exons three, four and five.,'],\n",
       " 737: ['1471-2105-8-476-2-2',\n",
       "  'GFIG',\n",
       "  'Conditional Independence Graphs . Conditional independence graphs for the estimated log-linear models for the  itpr1  gene. For each graph, the predictive probability score (7) is reported as a goodness of fit measure. Note the strong mutual interaction between exons three, four and five.,'],\n",
       " 738: ['1471-2105-8-476-2-3',\n",
       "  'GHDR',\n",
       "  'Conditional Independence Graphs . Conditional independence graphs for the estimated log-linear models for the  itpr1  gene. For each graph, the predictive probability score (7) is reported as a goodness of fit measure. Note the strong mutual interaction between exons three, four and five.,'],\n",
       " 739: ['1471-2105-8-476-2-4',\n",
       "  'GFIG',\n",
       "  'Conditional Independence Graphs . Conditional independence graphs for the estimated log-linear models for the  itpr1  gene. For each graph, the predictive probability score (7) is reported as a goodness of fit measure. Note the strong mutual interaction between exons three, four and five.,'],\n",
       " 740: ['1471-2105-8-476-2-5',\n",
       "  'GHDR',\n",
       "  'Conditional Independence Graphs . Conditional independence graphs for the estimated log-linear models for the  itpr1  gene. For each graph, the predictive probability score (7) is reported as a goodness of fit measure. Note the strong mutual interaction between exons three, four and five.,'],\n",
       " 741: ['1471-2105-8-487-1-1',\n",
       "  'GSCR',\n",
       "  'Highlighting and annotating chemical terms in an online journal article . Screenshots showing the effect of the ChemGM.user.js userscript on the  Chemistry Central Journal  web page (full URL: [47]) for Majumder et al. [48] . (a) When the userscript is running a toolbar is added to the top of every webpage. Clicking the highlight button in the toolbar causes the contents of the webpage to be analysed for chemical terms. (b) shows the original text of the abstract. (c) After a minute or so, any chemical terms recognised are highlighted in yellow, and are annotated with hypertext links to their entries in PubChem (if available) and a 2D depiction of the image.,'],\n",
       " 742: ['1471-2105-8-487-1-2',\n",
       "  'GSCR',\n",
       "  'Highlighting and annotating chemical terms in an online journal article . Screenshots showing the effect of the ChemGM.user.js userscript on the  Chemistry Central Journal  web page (full URL: [47]) for Majumder et al. [48] . (a) When the userscript is running a toolbar is added to the top of every webpage. Clicking the highlight button in the toolbar causes the contents of the webpage to be analysed for chemical terms. (b) shows the original text of the abstract. (c) After a minute or so, any chemical terms recognised are highlighted in yellow, and are annotated with hypertext links to their entries in PubChem (if available) and a 2D depiction of the image.,'],\n",
       " 743: ['1471-2105-8-487-1-3',\n",
       "  'GSCR',\n",
       "  'Highlighting and annotating chemical terms in an online journal article . Screenshots showing the effect of the ChemGM.user.js userscript on the  Chemistry Central Journal  web page (full URL: [47]) for Majumder et al. [48] . (a) When the userscript is running a toolbar is added to the top of every webpage. Clicking the highlight button in the toolbar causes the contents of the webpage to be analysed for chemical terms. (b) shows the original text of the abstract. (c) After a minute or so, any chemical terms recognised are highlighted in yellow, and are annotated with hypertext links to their entries in PubChem (if available) and a 2D depiction of the image.,'],\n",
       " 744: ['1471-2105-8-67-4-1',\n",
       "  'GFIG',\n",
       "  'The results from applying the KIGP to one of the training sets for the non-linear case in the simulated example 1, where (a) and (b) are for the simulation with an LK; (c) and (d) are for the simulation with an GK; (e) and (f) for the simulation with an PK. All the legends are same as those in Fig. 3. (a) The NLF plot of each gene for the simulation with an LK; with the cutoff value for NLF (dotted line),\" none of the true preset significant genes was found (2 false negatives). Three false positive genes were misclassified as significant. (b) The contours of the posterior predictive probability of the class \"\"1\"\" for the simulation with an LK (given the two true preset significant genes). For this set of training samples\", the testing MR is 0.5 (the Bayesian bound is 0.055). (c) Same as (a) except it is for the simulation with an GK. (d) Same as (b) except it is for the simulation with an GK. The testing MR is 0.063. (e) Same as (a) except it is for the simulation with an PK. (f) Same as (b) except it is for the simulation with an PK. The testing MR is 0.060.'],\n",
       " 745: ['1471-2105-8-67-4-2',\n",
       "  'GFIG',\n",
       "  'The results from applying the KIGP to one of the training sets for the non-linear case in the simulated example 1, where (a) and (b) are for the simulation with an LK; (c) and (d) are for the simulation with an GK; (e) and (f) for the simulation with an PK. All the legends are same as those in Fig. 3. (a) The NLF plot of each gene for the simulation with an LK; with the cutoff value for NLF (dotted line),\" none of the true preset significant genes was found (2 false negatives). Three false positive genes were misclassified as significant. (b) The contours of the posterior predictive probability of the class \"\"1\"\" for the simulation with an LK (given the two true preset significant genes). For this set of training samples\", the testing MR is 0.5 (the Bayesian bound is 0.055). (c) Same as (a) except it is for the simulation with an GK. (d) Same as (b) except it is for the simulation with an GK. The testing MR is 0.063. (e) Same as (a) except it is for the simulation with an PK. (f) Same as (b) except it is for the simulation with an PK. The testing MR is 0.060.'],\n",
       " 746: ['1471-2105-8-67-4-3',\n",
       "  'GFIG',\n",
       "  'The results from applying the KIGP to one of the training sets for the non-linear case in the simulated example 1, where (a) and (b) are for the simulation with an LK; (c) and (d) are for the simulation with an GK; (e) and (f) for the simulation with an PK. All the legends are same as those in Fig. 3. (a) The NLF plot of each gene for the simulation with an LK; with the cutoff value for NLF (dotted line),\" none of the true preset significant genes was found (2 false negatives). Three false positive genes were misclassified as significant. (b) The contours of the posterior predictive probability of the class \"\"1\"\" for the simulation with an LK (given the two true preset significant genes). For this set of training samples\", the testing MR is 0.5 (the Bayesian bound is 0.055). (c) Same as (a) except it is for the simulation with an GK. (d) Same as (b) except it is for the simulation with an GK. The testing MR is 0.063. (e) Same as (a) except it is for the simulation with an PK. (f) Same as (b) except it is for the simulation with an PK. The testing MR is 0.060.'],\n",
       " 747: ['1471-2105-8-67-4-4',\n",
       "  'GFIG',\n",
       "  'The results from applying the KIGP to one of the training sets for the non-linear case in the simulated example 1, where (a) and (b) are for the simulation with an LK; (c) and (d) are for the simulation with an GK; (e) and (f) for the simulation with an PK. All the legends are same as those in Fig. 3. (a) The NLF plot of each gene for the simulation with an LK; with the cutoff value for NLF (dotted line),\" none of the true preset significant genes was found (2 false negatives). Three false positive genes were misclassified as significant. (b) The contours of the posterior predictive probability of the class \"\"1\"\" for the simulation with an LK (given the two true preset significant genes). For this set of training samples\", the testing MR is 0.5 (the Bayesian bound is 0.055). (c) Same as (a) except it is for the simulation with an GK. (d) Same as (b) except it is for the simulation with an GK. The testing MR is 0.063. (e) Same as (a) except it is for the simulation with an PK. (f) Same as (b) except it is for the simulation with an PK. The testing MR is 0.060.'],\n",
       " 748: ['1471-2105-8-67-4-5',\n",
       "  'GFIG',\n",
       "  'The results from applying the KIGP to one of the training sets for the non-linear case in the simulated example 1, where (a) and (b) are for the simulation with an LK; (c) and (d) are for the simulation with an GK; (e) and (f) for the simulation with an PK. All the legends are same as those in Fig. 3. (a) The NLF plot of each gene for the simulation with an LK; with the cutoff value for NLF (dotted line),\" none of the true preset significant genes was found (2 false negatives). Three false positive genes were misclassified as significant. (b) The contours of the posterior predictive probability of the class \"\"1\"\" for the simulation with an LK (given the two true preset significant genes). For this set of training samples\", the testing MR is 0.5 (the Bayesian bound is 0.055). (c) Same as (a) except it is for the simulation with an GK. (d) Same as (b) except it is for the simulation with an GK. The testing MR is 0.063. (e) Same as (a) except it is for the simulation with an PK. (f) Same as (b) except it is for the simulation with an PK. The testing MR is 0.060.'],\n",
       " 749: ['1471-2105-8-67-4-6',\n",
       "  'GFIG',\n",
       "  'The results from applying the KIGP to one of the training sets for the non-linear case in the simulated example 1, where (a) and (b) are for the simulation with an LK; (c) and (d) are for the simulation with an GK; (e) and (f) for the simulation with an PK. All the legends are same as those in Fig. 3. (a) The NLF plot of each gene for the simulation with an LK; with the cutoff value for NLF (dotted line),\" none of the true preset significant genes was found (2 false negatives). Three false positive genes were misclassified as significant. (b) The contours of the posterior predictive probability of the class \"\"1\"\" for the simulation with an LK (given the two true preset significant genes). For this set of training samples\", the testing MR is 0.5 (the Bayesian bound is 0.055). (c) Same as (a) except it is for the simulation with an GK. (d) Same as (b) except it is for the simulation with an GK. The testing MR is 0.063. (e) Same as (a) except it is for the simulation with an PK. (f) Same as (b) except it is for the simulation with an PK. The testing MR is 0.060.'],\n",
       " 750: ['1471-2105-8-85-3-1',\n",
       "  'GFIG',\n",
       "  'Effect of amplification efficiency over the quantifications performed by different methods . (A)  In-silico  generated PCR data with initial template amount  T 0  = 0.001 and different intrinsic amplification efficiencies ( Ei ) ranging from 0.65 to 0.972 analysed by different methods (see below). Data points represent the base 2 logarithm of the ratio between  T 0  estimations from each simulated reaction and efficiency 0.8 ones vs. the amplification efficiency bias as mean  Â±  SEM of triplicates. (B) Analysis of experimental results by different methods (see below). Bars represent the error of quantifications as mean  Â±  SEM for triplicates. Bars marked with (*) are under-estimations, conversely, the rest of the bars are over-estimations. Method 6 under-estimated  T 0  by 737%, note that it is out of scale in the graph. Data was analysed with the  CT  method assuming constant amplification efficiency equal to 1 {1}; with the  CT  method assuming constant amplification efficiency equal to 0.8 for  in-silico  data, or 0.855 for experimental data {2}; assuming constant amplification efficiency that was estimated from two threshold values {3} [15]; using the assumption-free analysis proposed by Ramakers et.al. {4} [13] ; using the standardized determination of PCR efficiency from single reaction proposed by Tichopad et.al. {5} [16]; using the sigmoid model proposed by Liu et. al. {6} [17] ; and with our model based real-time PCR analysis method (MoBPA) {7}.,'],\n",
       " 751: ['1471-2105-8-85-3-2',\n",
       "  'GFIG',\n",
       "  'Effect of amplification efficiency over the quantifications performed by different methods . (A)  In-silico  generated PCR data with initial template amount  T 0  = 0.001 and different intrinsic amplification efficiencies ( Ei ) ranging from 0.65 to 0.972 analysed by different methods (see below). Data points represent the base 2 logarithm of the ratio between  T 0  estimations from each simulated reaction and efficiency 0.8 ones vs. the amplification efficiency bias as mean  Â±  SEM of triplicates. (B) Analysis of experimental results by different methods (see below). Bars represent the error of quantifications as mean  Â±  SEM for triplicates. Bars marked with (*) are under-estimations, conversely, the rest of the bars are over-estimations. Method 6 under-estimated  T 0  by 737%, note that it is out of scale in the graph. Data was analysed with the  CT  method assuming constant amplification efficiency equal to 1 {1}; with the  CT  method assuming constant amplification efficiency equal to 0.8 for  in-silico  data, or 0.855 for experimental data {2}; assuming constant amplification efficiency that was estimated from two threshold values {3} [15]; using the assumption-free analysis proposed by Ramakers et.al. {4} [13] ; using the standardized determination of PCR efficiency from single reaction proposed by Tichopad et.al. {5} [16]; using the sigmoid model proposed by Liu et. al. {6} [17] ; and with our model based real-time PCR analysis method (MoBPA) {7}.,'],\n",
       " 752: ['1471-2105-8-S2-S12-1-1',\n",
       "  'GFIG',\n",
       "  'Performance of different classifiers . Performance of different classifier variants under cross-validation. Left panel: boxplots of maximum F1 measure. Left to right: 8 original kernels from [1], extended kernel set, combined subcellular location and functional prediction, pairwise learning of the kernel weights (in the combined model). Right panel: precision,\" recall and F1 for the pairwise classifier used as \"\"building block\"\" in the HMM under variation of the threshold     which controls the label multiplicity.\",'],\n",
       " 753: ['1471-2105-8-S2-S12-1-2',\n",
       "  'GFIG',\n",
       "  'Performance of different classifiers . Performance of different classifier variants under cross-validation. Left panel: boxplots of maximum F1 measure. Left to right: 8 original kernels from [1], extended kernel set, combined subcellular location and functional prediction, pairwise learning of the kernel weights (in the combined model). Right panel: precision,\" recall and F1 for the pairwise classifier used as \"\"building block\"\" in the HMM under variation of the threshold     which controls the label multiplicity.\",'],\n",
       " 754: ['1471-2105-8-S3-S6-7-1',\n",
       "  'GSCR',\n",
       "  'DartFlow . The user interface of DartFlow.,'],\n",
       " 755: ['1471-2105-8-S3-S6-7-2',\n",
       "  'GSCR',\n",
       "  'DartFlow . The user interface of DartFlow.,'],\n",
       " 756: ['1471-2105-8-S4-S5-2-1',\n",
       "  'GFIG',\n",
       "  'Assessing accuracy and significance of ATP-binding descriptors .  A . Precision-recall curve for ATP-binding descriptors derived from protein structures with bound ATP or ADP tested against Swiss-Prot,\" shown as curve with red circles. Each circled point corresponds to a different E-value cutoff. The Prosite patterns for \"\"ATP-binding\"\" and \"\"ADP-binding\"\" are included as well (green crosses). Overall\", Prosite achieves a recall of 31% with a precision of 62% (blue cross). For all E-values, our method performs better than Prosite.  B . Distribution of E-values for the ATP-binding descriptors. To assess the significance of hits, the descriptors were tested both against Swiss-Prot (black line) and a shuffled Swiss-Prot version (red line). The cumulative number of hits below a certain E-value threshold is shown. The inlet shows a magnification of the lower right corner. Below an E-value of 1 (dotted vertical line), ~53,000 hits are found in Swiss-Prot whereas only ~1,200 hits are found in the shuffled Swiss-Prot.'],\n",
       " 757: ['1471-2105-8-S4-S5-2-2',\n",
       "  'GFIG',\n",
       "  'Assessing accuracy and significance of ATP-binding descriptors .  A . Precision-recall curve for ATP-binding descriptors derived from protein structures with bound ATP or ADP tested against Swiss-Prot,\" shown as curve with red circles. Each circled point corresponds to a different E-value cutoff. The Prosite patterns for \"\"ATP-binding\"\" and \"\"ADP-binding\"\" are included as well (green crosses). Overall\", Prosite achieves a recall of 31% with a precision of 62% (blue cross). For all E-values, our method performs better than Prosite.  B . Distribution of E-values for the ATP-binding descriptors. To assess the significance of hits, the descriptors were tested both against Swiss-Prot (black line) and a shuffled Swiss-Prot version (red line). The cumulative number of hits below a certain E-value threshold is shown. The inlet shows a magnification of the lower right corner. Below an E-value of 1 (dotted vertical line), ~53,000 hits are found in Swiss-Prot whereas only ~1,200 hits are found in the shuffled Swiss-Prot.'],\n",
       " 758: ['1471-2105-8-S7-S20-11-1',\n",
       "  'DSEM',\n",
       "  'Antibody blockades in the presence of antigen excess and chaotropic agents . An antibody (anti-biotin) molecule is introduced to the nanopore device to produce the characteristic signal shown in panel 1. In present of the antigen (biotin) there is a strong interaction: even 100 fold excess of biotin does not change signal considerably (panel 2). The signal changes greatly in presence of urea (panel 3), in relatively small concentrations insufficient to result in protein denaturing. Signal durations are shown for one second.'],\n",
       " 759: ['1471-2105-8-S7-S20-11-2',\n",
       "  'DSEM',\n",
       "  'Antibody blockades in the presence of antigen excess and chaotropic agents . An antibody (anti-biotin) molecule is introduced to the nanopore device to produce the characteristic signal shown in panel 1. In present of the antigen (biotin) there is a strong interaction: even 100 fold excess of biotin does not change signal considerably (panel 2). The signal changes greatly in presence of urea (panel 3), in relatively small concentrations insufficient to result in protein denaturing. Signal durations are shown for one second.'],\n",
       " 760: ['1471-2105-8-S7-S20-11-3',\n",
       "  'DSEM',\n",
       "  'Antibody blockades in the presence of antigen excess and chaotropic agents . An antibody (anti-biotin) molecule is introduced to the nanopore device to produce the characteristic signal shown in panel 1. In present of the antigen (biotin) there is a strong interaction: even 100 fold excess of biotin does not change signal considerably (panel 2). The signal changes greatly in presence of urea (panel 3), in relatively small concentrations insufficient to result in protein denaturing. Signal durations are shown for one second.'],\n",
       " 761: ['1471-2105-9-111-2-1',\n",
       "  'GFIG',\n",
       "  'Shared and Data-specific variation for leukemia data . Shared (top) and data-specific (bottom) variation retained with CCA (solid line) and PCA (dashed line) as a function of the reduced dimensionality for the leukemia data. The values obtained by random projections (dash-dotted line and dotted confidence intervals) have been included for reference. The suggested dimensionality for the CCA-projection is marked with a tick.,'],\n",
       " 762: ['1471-2105-9-111-2-2',\n",
       "  'GFIG',\n",
       "  'Shared and Data-specific variation for leukemia data . Shared (top) and data-specific (bottom) variation retained with CCA (solid line) and PCA (dashed line) as a function of the reduced dimensionality for the leukemia data. The values obtained by random projections (dash-dotted line and dotted confidence intervals) have been included for reference. The suggested dimensionality for the CCA-projection is marked with a tick.,'],\n",
       " 763: ['1471-2105-9-111-3-1',\n",
       "  'GFIG',\n",
       "  'Shared and Data-specific variation for cell-cycle data . Shared (top) and data-specific (bottom) variation retained with CCA (solid line) and PCA (dashed line) as a function of reduced dimensionality for the cell-cycle data. The values obtained by random projections (dash-dotted line and dotted confidence intervals) have been included for reference. The suggested dimensionality for the CCA-projection is marked with a tick.,'],\n",
       " 764: ['1471-2105-9-111-3-2',\n",
       "  'GFIG',\n",
       "  'Shared and Data-specific variation for cell-cycle data . Shared (top) and data-specific (bottom) variation retained with CCA (solid line) and PCA (dashed line) as a function of reduced dimensionality for the cell-cycle data. The values obtained by random projections (dash-dotted line and dotted confidence intervals) have been included for reference. The suggested dimensionality for the CCA-projection is marked with a tick.,'],\n",
       " 765: ['1471-2105-9-111-4-1',\n",
       "  'GFIG',\n",
       "  'Shared and Data-specific variation for stress data . Shared (top) and data-specific (bottom) variation retained with CCA (solid line) and PCA (dashed line) as a function of the reduced dimensionality for the stress data. The values obtained by random projections (dash-dotted line and dotted confidence intervals) have been included for reference. The suggested dimensionality for the CCA-projection is marked with a tick.,'],\n",
       " 766: ['1471-2105-9-111-4-2',\n",
       "  'GFIG',\n",
       "  'Shared and Data-specific variation for stress data . Shared (top) and data-specific (bottom) variation retained with CCA (solid line) and PCA (dashed line) as a function of the reduced dimensionality for the stress data. The values obtained by random projections (dash-dotted line and dotted confidence intervals) have been included for reference. The suggested dimensionality for the CCA-projection is marked with a tick.,'],\n",
       " 767: ['1471-2105-9-17-3-1',\n",
       "  'D3DR',\n",
       "  'Overlap (purple spheres) of ET Rank template residues (red spheres) with SITE records (green spheres) provided by the PDB, in the context of the surface trace cluster (red sticks) from which the template residues were chosen. (a) Rieske iron-sulfur protein (PDB 1RIE); (b) Casein kinase II (PDB 1QF8, chain A).,'],\n",
       " 768: ['1471-2105-9-17-3-2',\n",
       "  'D3DR',\n",
       "  'Overlap (purple spheres) of ET Rank template residues (red spheres) with SITE records (green spheres) provided by the PDB, in the context of the surface trace cluster (red sticks) from which the template residues were chosen. (a) Rieske iron-sulfur protein (PDB 1RIE); (b) Casein kinase II (PDB 1QF8, chain A).,'],\n",
       " 769: ['1471-2105-9-220-6-1',\n",
       "  'GFIG',\n",
       "  'Location and expression prediction of the MSE2 in  D. melanogaster . Panel  a : Overlap between the best TrainedMinusT6 model-predicted MSE2, the MSE2 region proposed by Janssens et al. [9] and the annotated MSE2 [15] . The predicted MSE2 was identified as the window with the lowest RMS error amongst all positions and window sizes. Panel  b : Predicted mRNA concentration using the MSE2 proposed by Janssens et al. [9], the predicted MSE2 location and the annotated MSE2 location, respectively.'],\n",
       " 770: ['1471-2105-9-220-6-2',\n",
       "  'GFIG',\n",
       "  'Location and expression prediction of the MSE2 in  D. melanogaster . Panel  a : Overlap between the best TrainedMinusT6 model-predicted MSE2, the MSE2 region proposed by Janssens et al. [9] and the annotated MSE2 [15] . The predicted MSE2 was identified as the window with the lowest RMS error amongst all positions and window sizes. Panel  b : Predicted mRNA concentration using the MSE2 proposed by Janssens et al. [9], the predicted MSE2 location and the annotated MSE2 location, respectively.'],\n",
       " 771: ['1471-2105-9-232-2-1',\n",
       "  'GGEN',\n",
       "  \"Unique consensus sequences at aTIS compared to non-aTIS (AUG) in the 5'-UTR region of mRNAs . Figures 2A and 2B where generated with an adaptation of the WebLogo application [32]. The overall height of the nucleotide stack indicates the sequence conservation at that position, while the height of nucleotide symbols within the stack indicates the relative frequency of each base at that position. The start site is indicated at positions 1,2, and 3.  (A) Distinct consensus nucleotide sequences near confirmed alternative translation initiation sites in mammalian mRNAs . The relative abundance of nucleotides (A, T, C, G) at aTISs is shown for a window of -10 to +10 bases at the initiation codon, with the aTIS start codon in positions 1,2, and 3. Conservation around all of the alternative start sites aTISs is illustrated. Note the strong conservation of (G/C) at the -6 position and C at the -7 position.  (B) Consensus nucleotides near AUG translation initiation sites in mammalian mRNAs . Graphical representation of relative nucleotide abundances at AUG sites is shown for bases in the region of -10 to +10 bases relative to the initiation codon, with the AUG codon in positions 1,2, and 3. Conservation at the -3 and +4 locations are consistent with traditional Kozak consensus sequence. These features are distinguished from that of thte aTIS sequences which show conservation at positions -6 and -7 (Figure 2A).,\"],\n",
       " 772: ['1471-2105-9-232-2-2',\n",
       "  'GGEN',\n",
       "  \"Unique consensus sequences at aTIS compared to non-aTIS (AUG) in the 5'-UTR region of mRNAs . Figures 2A and 2B where generated with an adaptation of the WebLogo application [32]. The overall height of the nucleotide stack indicates the sequence conservation at that position, while the height of nucleotide symbols within the stack indicates the relative frequency of each base at that position. The start site is indicated at positions 1,2, and 3.  (A) Distinct consensus nucleotide sequences near confirmed alternative translation initiation sites in mammalian mRNAs . The relative abundance of nucleotides (A, T, C, G) at aTISs is shown for a window of -10 to +10 bases at the initiation codon, with the aTIS start codon in positions 1,2, and 3. Conservation around all of the alternative start sites aTISs is illustrated. Note the strong conservation of (G/C) at the -6 position and C at the -7 position.  (B) Consensus nucleotides near AUG translation initiation sites in mammalian mRNAs . Graphical representation of relative nucleotide abundances at AUG sites is shown for bases in the region of -10 to +10 bases relative to the initiation codon, with the AUG codon in positions 1,2, and 3. Conservation at the -3 and +4 locations are consistent with traditional Kozak consensus sequence. These features are distinguished from that of thte aTIS sequences which show conservation at positions -6 and -7 (Figure 2A).,\"],\n",
       " 773: ['1471-2105-9-232-2-3',\n",
       "  'GGEN',\n",
       "  \"Unique consensus sequences at aTIS compared to non-aTIS (AUG) in the 5'-UTR region of mRNAs . Figures 2A and 2B where generated with an adaptation of the WebLogo application [32]. The overall height of the nucleotide stack indicates the sequence conservation at that position, while the height of nucleotide symbols within the stack indicates the relative frequency of each base at that position. The start site is indicated at positions 1,2, and 3.  (A) Distinct consensus nucleotide sequences near confirmed alternative translation initiation sites in mammalian mRNAs . The relative abundance of nucleotides (A, T, C, G) at aTISs is shown for a window of -10 to +10 bases at the initiation codon, with the aTIS start codon in positions 1,2, and 3. Conservation around all of the alternative start sites aTISs is illustrated. Note the strong conservation of (G/C) at the -6 position and C at the -7 position.  (B) Consensus nucleotides near AUG translation initiation sites in mammalian mRNAs . Graphical representation of relative nucleotide abundances at AUG sites is shown for bases in the region of -10 to +10 bases relative to the initiation codon, with the AUG codon in positions 1,2, and 3. Conservation at the -3 and +4 locations are consistent with traditional Kozak consensus sequence. These features are distinguished from that of thte aTIS sequences which show conservation at positions -6 and -7 (Figure 2A).,\"],\n",
       " 774: ['1471-2105-9-232-2-4',\n",
       "  'GGEN',\n",
       "  \"Unique consensus sequences at aTIS compared to non-aTIS (AUG) in the 5'-UTR region of mRNAs . Figures 2A and 2B where generated with an adaptation of the WebLogo application [32]. The overall height of the nucleotide stack indicates the sequence conservation at that position, while the height of nucleotide symbols within the stack indicates the relative frequency of each base at that position. The start site is indicated at positions 1,2, and 3.  (A) Distinct consensus nucleotide sequences near confirmed alternative translation initiation sites in mammalian mRNAs . The relative abundance of nucleotides (A, T, C, G) at aTISs is shown for a window of -10 to +10 bases at the initiation codon, with the aTIS start codon in positions 1,2, and 3. Conservation around all of the alternative start sites aTISs is illustrated. Note the strong conservation of (G/C) at the -6 position and C at the -7 position.  (B) Consensus nucleotides near AUG translation initiation sites in mammalian mRNAs . Graphical representation of relative nucleotide abundances at AUG sites is shown for bases in the region of -10 to +10 bases relative to the initiation codon, with the AUG codon in positions 1,2, and 3. Conservation at the -3 and +4 locations are consistent with traditional Kozak consensus sequence. These features are distinguished from that of thte aTIS sequences which show conservation at positions -6 and -7 (Figure 2A).,\"],\n",
       " 775: ['1471-2105-9-275-2-1',\n",
       "  'GFIG',\n",
       "  'Data preprocessing result . Spectrogram ID 2 is used as an example of data preprocessing procedure. (A) Original spectrogram without any processing. The maximum  m / z  ratio is 11922.91 and the minimum  m / z  ratio is 800. (B) Original spectrogram and adjusted baseline. (C) Noise reduction using Gaussian kernel smoothing. (D) Normalization using the area under the curve (AUC).,'],\n",
       " 776: ['1471-2105-9-275-2-2',\n",
       "  'GFIG',\n",
       "  'Data preprocessing result . Spectrogram ID 2 is used as an example of data preprocessing procedure. (A) Original spectrogram without any processing. The maximum  m / z  ratio is 11922.91 and the minimum  m / z  ratio is 800. (B) Original spectrogram and adjusted baseline. (C) Noise reduction using Gaussian kernel smoothing. (D) Normalization using the area under the curve (AUC).,'],\n",
       " 777: ['1471-2105-9-275-2-3',\n",
       "  'GFIG',\n",
       "  'Data preprocessing result . Spectrogram ID 2 is used as an example of data preprocessing procedure. (A) Original spectrogram without any processing. The maximum  m / z  ratio is 11922.91 and the minimum  m / z  ratio is 800. (B) Original spectrogram and adjusted baseline. (C) Noise reduction using Gaussian kernel smoothing. (D) Normalization using the area under the curve (AUC).,'],\n",
       " 778: ['1471-2105-9-275-2-4',\n",
       "  'GFIG',\n",
       "  'Data preprocessing result . Spectrogram ID 2 is used as an example of data preprocessing procedure. (A) Original spectrogram without any processing. The maximum  m / z  ratio is 11922.91 and the minimum  m / z  ratio is 800. (B) Original spectrogram and adjusted baseline. (C) Noise reduction using Gaussian kernel smoothing. (D) Normalization using the area under the curve (AUC).,'],\n",
       " 779: ['1471-2105-9-294-2-1',\n",
       "  'GSYS',\n",
       "  'Toy example to illustrate limitations of a two-step approach . The schematic demonstrates why clustering or biclustering followed by gene-enrichment analysis may miss patterns that can be found by our approach. The left panel shows two hypothetical experiments side by side using the same type of visualization as in Fig. 1. Genes with a particular functional annotation are highlighted; others are represented by dashed lines. It can be seen that the natural clusters in the data set do not show enrichment and would, hence not be considered significant by a two-step approach. In contrast, for our approach the histogram of neighbors (right panel) differs from what would be expected for a random subset.,'],\n",
       " 780: ['1471-2105-9-294-2-2',\n",
       "  'GFIG',\n",
       "  'Toy example to illustrate limitations of a two-step approach . The schematic demonstrates why clustering or biclustering followed by gene-enrichment analysis may miss patterns that can be found by our approach. The left panel shows two hypothetical experiments side by side using the same type of visualization as in Fig. 1. Genes with a particular functional annotation are highlighted; others are represented by dashed lines. It can be seen that the natural clusters in the data set do not show enrichment and would, hence not be considered significant by a two-step approach. In contrast, for our approach the histogram of neighbors (right panel) differs from what would be expected for a random subset.,'],\n",
       " 781: ['1471-2105-9-294-9-1',\n",
       "  'GFIG',\n",
       "  'Experimental and randomized histogram and profile for the function hmm.mfs_1 for the Baev data set . Top panel: A density measure that is evaluated for each data point is given by the number of neighbors that are close according to a product similarity measure. The blue bars in the histograms represent the sum of all these densities. Those vector pairs are expected to be most relevant that exhibit a large absolute value of differential expression and a small angle between vectors. The white bars show the randomly distributed vectors that are expected to have some neighbors as well. Subsets of genes that have the same number of elements as the protein function under consideration were randomly selected. The histogram was constructed for this random subset. The process was repeated multiple times and averaged over 20 runs. Bottom panel: Gene expression profile for the function hmm.mfs_1. Profiles were obtained for all those genes in the functional group of hmm.mfs_1 that have a larger number of points in the neighborhood than any of the points in the histograms for the random subset (tails in the histograms). The individual genes that form this group are indicated in the inserted figure legends. The numbers on the x-axis symbolize the individual two-component systems. The order of two-component systems (attributes) is identical to Fig. 5.,'],\n",
       " 782: ['1471-2105-9-294-9-2',\n",
       "  'GFIG',\n",
       "  'Experimental and randomized histogram and profile for the function hmm.mfs_1 for the Baev data set . Top panel: A density measure that is evaluated for each data point is given by the number of neighbors that are close according to a product similarity measure. The blue bars in the histograms represent the sum of all these densities. Those vector pairs are expected to be most relevant that exhibit a large absolute value of differential expression and a small angle between vectors. The white bars show the randomly distributed vectors that are expected to have some neighbors as well. Subsets of genes that have the same number of elements as the protein function under consideration were randomly selected. The histogram was constructed for this random subset. The process was repeated multiple times and averaged over 20 runs. Bottom panel: Gene expression profile for the function hmm.mfs_1. Profiles were obtained for all those genes in the functional group of hmm.mfs_1 that have a larger number of points in the neighborhood than any of the points in the histograms for the random subset (tails in the histograms). The individual genes that form this group are indicated in the inserted figure legends. The numbers on the x-axis symbolize the individual two-component systems. The order of two-component systems (attributes) is identical to Fig. 5.,'],\n",
       " 783: ['1471-2105-9-29-5-1',\n",
       "  'GFIG',\n",
       "  'The mesh grids of the DF of M3 and the score distribution of the matches uniquely validated by M1~M3. The blue points in B~E represent the matches uniquely validated by M3, the red points are those of M2 and the green points are those of M1.'],\n",
       " 784: ['1471-2105-9-29-5-2',\n",
       "  'GFIG',\n",
       "  'The mesh grids of the DF of M3 and the score distribution of the matches uniquely validated by M1~M3. The blue points in B~E represent the matches uniquely validated by M3, the red points are those of M2 and the green points are those of M1.'],\n",
       " 785: ['1471-2105-9-29-5-3',\n",
       "  'GFIG',\n",
       "  'The mesh grids of the DF of M3 and the score distribution of the matches uniquely validated by M1~M3. The blue points in B~E represent the matches uniquely validated by M3, the red points are those of M2 and the green points are those of M1.'],\n",
       " 786: ['1471-2105-9-29-5-4',\n",
       "  'GFIG',\n",
       "  'The mesh grids of the DF of M3 and the score distribution of the matches uniquely validated by M1~M3. The blue points in B~E represent the matches uniquely validated by M3, the red points are those of M2 and the green points are those of M1.'],\n",
       " 787: ['1471-2105-9-29-5-5',\n",
       "  'GFIG',\n",
       "  'The mesh grids of the DF of M3 and the score distribution of the matches uniquely validated by M1~M3. The blue points in B~E represent the matches uniquely validated by M3, the red points are those of M2 and the green points are those of M1.'],\n",
       " 788: ['1471-2105-9-425-8-1',\n",
       "  'GFIG',\n",
       "  'Scatterplots of the most dormant gene in each subgroup of the Lung Cancer data set: (a) BRD2 (36209_at) for Adeno, (b) DSP (36133_at) for Normal, (c) CAPN1 (33908_at) for SCLC, (d) SYNE1 (38113_at) for SQ, (e) SFN (33322_i_at) for COID.,'],\n",
       " 789: ['1471-2105-9-425-8-2',\n",
       "  'GFIG',\n",
       "  'Scatterplots of the most dormant gene in each subgroup of the Lung Cancer data set: (a) BRD2 (36209_at) for Adeno, (b) DSP (36133_at) for Normal, (c) CAPN1 (33908_at) for SCLC, (d) SYNE1 (38113_at) for SQ, (e) SFN (33322_i_at) for COID.,'],\n",
       " 790: ['1471-2105-9-425-8-3',\n",
       "  'GFIG',\n",
       "  'Scatterplots of the most dormant gene in each subgroup of the Lung Cancer data set: (a) BRD2 (36209_at) for Adeno, (b) DSP (36133_at) for Normal, (c) CAPN1 (33908_at) for SCLC, (d) SYNE1 (38113_at) for SQ, (e) SFN (33322_i_at) for COID.,'],\n",
       " 791: ['1471-2105-9-425-8-4',\n",
       "  'GFIG',\n",
       "  'Scatterplots of the most dormant gene in each subgroup of the Lung Cancer data set: (a) BRD2 (36209_at) for Adeno, (b) DSP (36133_at) for Normal, (c) CAPN1 (33908_at) for SCLC, (d) SYNE1 (38113_at) for SQ, (e) SFN (33322_i_at) for COID.,'],\n",
       " 792: ['1471-2105-9-425-8-5',\n",
       "  'GFIG',\n",
       "  'Scatterplots of the most dormant gene in each subgroup of the Lung Cancer data set: (a) BRD2 (36209_at) for Adeno, (b) DSP (36133_at) for Normal, (c) CAPN1 (33908_at) for SCLC, (d) SYNE1 (38113_at) for SQ, (e) SFN (33322_i_at) for COID.,'],\n",
       " 793: ['1471-2105-9-441-6-1',\n",
       "  'GFIG',\n",
       "  'The success rate of pyDock and pyDockTET for predicting crystal and modelled domain assemblies . (a) The success rate of pyDock and pyDockTET in selecting at least one acceptable solution for a sub-set where domain structures have been modelled based on homologue templates, considering only those cases with at least one acceptable solution generated by ZDOCK. (b) The success rate of pyDock and pyDockTET in selecting at least one acceptable solution for the same sub-set when coordinates are taken from crystal structures, considering only those cases with at least one acceptable solution generated by ZDOCK. The solid lines with square markers are the results of pyDockTET, the dash lines with circle markers are the results of pyDock, the dotted lines with diamond markers are the results of random prediction.,'],\n",
       " 794: ['1471-2105-9-441-6-2',\n",
       "  'GFIG',\n",
       "  'The success rate of pyDock and pyDockTET for predicting crystal and modelled domain assemblies . (a) The success rate of pyDock and pyDockTET in selecting at least one acceptable solution for a sub-set where domain structures have been modelled based on homologue templates, considering only those cases with at least one acceptable solution generated by ZDOCK. (b) The success rate of pyDock and pyDockTET in selecting at least one acceptable solution for the same sub-set when coordinates are taken from crystal structures, considering only those cases with at least one acceptable solution generated by ZDOCK. The solid lines with square markers are the results of pyDockTET, the dash lines with circle markers are the results of pyDock, the dotted lines with diamond markers are the results of random prediction.,'],\n",
       " 795: ['1471-2105-9-467-3-1',\n",
       "  'GFIG',\n",
       "  'Sensitivity curves for MI3 versus control methods in learning two-parent models from the synthetic dataset.  (a) Average absolute sensitivity of the 4 methods to recover the known network. (b) Average relative sensitivity of the 4 methods to recover the known network given that only two parents are possible for each dependent node. Vertical dashed lines marked sample size of 350 used in Figure  2 , which is similar to the experimental sample size used for the MYC study.'],\n",
       " 796: ['1471-2105-9-467-3-2',\n",
       "  'GFIG',\n",
       "  'Sensitivity curves for MI3 versus control methods in learning two-parent models from the synthetic dataset.  (a) Average absolute sensitivity of the 4 methods to recover the known network. (b) Average relative sensitivity of the 4 methods to recover the known network given that only two parents are possible for each dependent node. Vertical dashed lines marked sample size of 350 used in Figure  2 , which is similar to the experimental sample size used for the MYC study.'],\n",
       " 797: ['1471-2105-9-486-1-1',\n",
       "  'GFIG',\n",
       "  'Model-based clustering of bimodal gene expression identifies cohesive clusters in 19 tissue types . Heat map representation of posterior pairwise probabilities for classification of tissue phenotype. Left column: classification with 1265 bimodal genes. Right column: classification with 300 bimodal genes translated into extracellular matrix or plasma membrane proteins. Top row: Model-based clustering, identifies all tissues distinctly. Middle and bottom rows: Kmeans and hierarchical clustering classify samples into three/four tissue types: brain, cardiac and skeletal muscle and remaining tissues. Blue, green, yellow, orange and red regions of color bar indicate ovary, stomach, small intestine, pancreas and thymus tissue samples respectively. Tissues in the heat map were ordered according to decreasing sample size from left to right.,'],\n",
       " 798: ['1471-2105-9-486-1-2',\n",
       "  'GFIG',\n",
       "  'Model-based clustering of bimodal gene expression identifies cohesive clusters in 19 tissue types . Heat map representation of posterior pairwise probabilities for classification of tissue phenotype. Left column: classification with 1265 bimodal genes. Right column: classification with 300 bimodal genes translated into extracellular matrix or plasma membrane proteins. Top row: Model-based clustering, identifies all tissues distinctly. Middle and bottom rows: Kmeans and hierarchical clustering classify samples into three/four tissue types: brain, cardiac and skeletal muscle and remaining tissues. Blue, green, yellow, orange and red regions of color bar indicate ovary, stomach, small intestine, pancreas and thymus tissue samples respectively. Tissues in the heat map were ordered according to decreasing sample size from left to right.,'],\n",
       " 799: ['1471-2105-9-486-1-3',\n",
       "  'GFIG',\n",
       "  'Model-based clustering of bimodal gene expression identifies cohesive clusters in 19 tissue types . Heat map representation of posterior pairwise probabilities for classification of tissue phenotype. Left column: classification with 1265 bimodal genes. Right column: classification with 300 bimodal genes translated into extracellular matrix or plasma membrane proteins. Top row: Model-based clustering, identifies all tissues distinctly. Middle and bottom rows: Kmeans and hierarchical clustering classify samples into three/four tissue types: brain, cardiac and skeletal muscle and remaining tissues. Blue, green, yellow, orange and red regions of color bar indicate ovary, stomach, small intestine, pancreas and thymus tissue samples respectively. Tissues in the heat map were ordered according to decreasing sample size from left to right.,'],\n",
       " 800: ['1471-2105-9-486-1-4',\n",
       "  'GFIG',\n",
       "  'Model-based clustering of bimodal gene expression identifies cohesive clusters in 19 tissue types . Heat map representation of posterior pairwise probabilities for classification of tissue phenotype. Left column: classification with 1265 bimodal genes. Right column: classification with 300 bimodal genes translated into extracellular matrix or plasma membrane proteins. Top row: Model-based clustering, identifies all tissues distinctly. Middle and bottom rows: Kmeans and hierarchical clustering classify samples into three/four tissue types: brain, cardiac and skeletal muscle and remaining tissues. Blue, green, yellow, orange and red regions of color bar indicate ovary, stomach, small intestine, pancreas and thymus tissue samples respectively. Tissues in the heat map were ordered according to decreasing sample size from left to right.,'],\n",
       " 801: ['1471-2105-9-486-1-5',\n",
       "  'GFIG',\n",
       "  'Model-based clustering of bimodal gene expression identifies cohesive clusters in 19 tissue types . Heat map representation of posterior pairwise probabilities for classification of tissue phenotype. Left column: classification with 1265 bimodal genes. Right column: classification with 300 bimodal genes translated into extracellular matrix or plasma membrane proteins. Top row: Model-based clustering, identifies all tissues distinctly. Middle and bottom rows: Kmeans and hierarchical clustering classify samples into three/four tissue types: brain, cardiac and skeletal muscle and remaining tissues. Blue, green, yellow, orange and red regions of color bar indicate ovary, stomach, small intestine, pancreas and thymus tissue samples respectively. Tissues in the heat map were ordered according to decreasing sample size from left to right.,'],\n",
       " 802: ['1471-2105-9-486-1-6',\n",
       "  'GFIG',\n",
       "  'Model-based clustering of bimodal gene expression identifies cohesive clusters in 19 tissue types . Heat map representation of posterior pairwise probabilities for classification of tissue phenotype. Left column: classification with 1265 bimodal genes. Right column: classification with 300 bimodal genes translated into extracellular matrix or plasma membrane proteins. Top row: Model-based clustering, identifies all tissues distinctly. Middle and bottom rows: Kmeans and hierarchical clustering classify samples into three/four tissue types: brain, cardiac and skeletal muscle and remaining tissues. Blue, green, yellow, orange and red regions of color bar indicate ovary, stomach, small intestine, pancreas and thymus tissue samples respectively. Tissues in the heat map were ordered according to decreasing sample size from left to right.,'],\n",
       " 803: ['1471-2105-9-486-1-7',\n",
       "  'GFIG',\n",
       "  'Model-based clustering of bimodal gene expression identifies cohesive clusters in 19 tissue types . Heat map representation of posterior pairwise probabilities for classification of tissue phenotype. Left column: classification with 1265 bimodal genes. Right column: classification with 300 bimodal genes translated into extracellular matrix or plasma membrane proteins. Top row: Model-based clustering, identifies all tissues distinctly. Middle and bottom rows: Kmeans and hierarchical clustering classify samples into three/four tissue types: brain, cardiac and skeletal muscle and remaining tissues. Blue, green, yellow, orange and red regions of color bar indicate ovary, stomach, small intestine, pancreas and thymus tissue samples respectively. Tissues in the heat map were ordered according to decreasing sample size from left to right.,'],\n",
       " 804: ['1471-2105-9-487-4-1',\n",
       "  'GFIG',\n",
       "  'Average energy difference between the ground state and the next lowest energy state for different values of designability N S  for the hexagonal (a) and triangular (b) shapes.  Although there is a strong visible trend towards a higher energy gap as the conformations become more designable, there are exceptions particularly for the most designable conformations (corresponding to the largest N s ), having in both cases average energy gaps below the maximum.,'],\n",
       " 805: ['1471-2105-9-487-4-2',\n",
       "  'GFIG',\n",
       "  'Average energy difference between the ground state and the next lowest energy state for different values of designability N S  for the hexagonal (a) and triangular (b) shapes.  Although there is a strong visible trend towards a higher energy gap as the conformations become more designable, there are exceptions particularly for the most designable conformations (corresponding to the largest N s ), having in both cases average energy gaps below the maximum.,'],\n",
       " 806: ['1471-2105-9-489-1-1',\n",
       "  'GFIG',\n",
       "  'Average ROC curves for 13 Latin Square experiments . The performance of ranking true and false positives for pairs of N = 1 experiments are depicted. The first experiment from 13 2   Latin Square experiments was selected for analysis. For each of the 13 comparisons, an ROC curve was generated. Shown is the average of all 13 ROC curves. Figure 1A shows the full-scale performance for all false positives. Figure 1B is a zoomed in view of 1A with the x and y-axes zoomed to show detail of restrictive cutoffs with few false positives. Figure 1C is a box plot of the number of TP detected at an arbitrary cut off level of 4 FP (vertical dashed line in 1B).'],\n",
       " 807: ['1471-2105-9-489-1-2',\n",
       "  'GFIG',\n",
       "  'Average ROC curves for 13 Latin Square experiments . The performance of ranking true and false positives for pairs of N = 1 experiments are depicted. The first experiment from 13 2   Latin Square experiments was selected for analysis. For each of the 13 comparisons, an ROC curve was generated. Shown is the average of all 13 ROC curves. Figure 1A shows the full-scale performance for all false positives. Figure 1B is a zoomed in view of 1A with the x and y-axes zoomed to show detail of restrictive cutoffs with few false positives. Figure 1C is a box plot of the number of TP detected at an arbitrary cut off level of 4 FP (vertical dashed line in 1B).'],\n",
       " 808: ['1471-2105-9-489-1-3',\n",
       "  'GFIG',\n",
       "  'Average ROC curves for 13 Latin Square experiments . The performance of ranking true and false positives for pairs of N = 1 experiments are depicted. The first experiment from 13 2   Latin Square experiments was selected for analysis. For each of the 13 comparisons, an ROC curve was generated. Shown is the average of all 13 ROC curves. Figure 1A shows the full-scale performance for all false positives. Figure 1B is a zoomed in view of 1A with the x and y-axes zoomed to show detail of restrictive cutoffs with few false positives. Figure 1C is a box plot of the number of TP detected at an arbitrary cut off level of 4 FP (vertical dashed line in 1B).'],\n",
       " 809: ['1471-2105-9-530-5-1',\n",
       "  'GTAB',\n",
       "  'Uploading Your Own Data Set . (A,\" B) Data file structures. Exemplary tab-delimited text data files for replication timing (A) and transcription (B). (C) A screenshot of the \"\"Data Upload\"\" page.\"'],\n",
       " 810: ['1471-2105-9-530-5-2',\n",
       "  'GSCR',\n",
       "  'Uploading Your Own Data Set . (A,\" B) Data file structures. Exemplary tab-delimited text data files for replication timing (A) and transcription (B). (C) A screenshot of the \"\"Data Upload\"\" page.\"'],\n",
       " 811: ['1471-2105-9-530-5-3',\n",
       "  'GTAB',\n",
       "  'Uploading Your Own Data Set . (A,\" B) Data file structures. Exemplary tab-delimited text data files for replication timing (A) and transcription (B). (C) A screenshot of the \"\"Data Upload\"\" page.\"'],\n",
       " 812: ['1471-2105-9-530-5-4',\n",
       "  'GSCR',\n",
       "  'Uploading Your Own Data Set . (A,\" B) Data file structures. Exemplary tab-delimited text data files for replication timing (A) and transcription (B). (C) A screenshot of the \"\"Data Upload\"\" page.\"'],\n",
       " 813: ['1471-2105-9-533-2-1',\n",
       "  'GFIG',\n",
       "  \"Example of exploitation of the GPAT results : (A) Venn diagram showing the genes with a single occupancy by Pol II (red) or TBP (green) respectively or a co-occupancy (yellow). (B) Distribution of Pol II (blue) and TBP (red) binding sites relative to the 5' end of the matched transcript. The distribution patterns of both Pol II and TBP, but not GST, cluster within +/- 1 kb around the 5' end of the matched transcripts. (C) Distribution of the expression level in each gene category. The highest expression level is observed for genes where both Pol II and TBP were detected at the promoter. Furthermore, genes bound only by Pol II, but not TBP show a high level of expression, suggesting the possibility of TBP independent genes.\"],\n",
       " 814: ['1471-2105-9-533-2-2',\n",
       "  'GFIG',\n",
       "  \"Example of exploitation of the GPAT results : (A) Venn diagram showing the genes with a single occupancy by Pol II (red) or TBP (green) respectively or a co-occupancy (yellow). (B) Distribution of Pol II (blue) and TBP (red) binding sites relative to the 5' end of the matched transcript. The distribution patterns of both Pol II and TBP, but not GST, cluster within +/- 1 kb around the 5' end of the matched transcripts. (C) Distribution of the expression level in each gene category. The highest expression level is observed for genes where both Pol II and TBP were detected at the promoter. Furthermore, genes bound only by Pol II, but not TBP show a high level of expression, suggesting the possibility of TBP independent genes.\"],\n",
       " 815: ['1471-2105-9-533-2-3',\n",
       "  'GFIG',\n",
       "  \"Example of exploitation of the GPAT results : (A) Venn diagram showing the genes with a single occupancy by Pol II (red) or TBP (green) respectively or a co-occupancy (yellow). (B) Distribution of Pol II (blue) and TBP (red) binding sites relative to the 5' end of the matched transcript. The distribution patterns of both Pol II and TBP, but not GST, cluster within +/- 1 kb around the 5' end of the matched transcripts. (C) Distribution of the expression level in each gene category. The highest expression level is observed for genes where both Pol II and TBP were detected at the promoter. Furthermore, genes bound only by Pol II, but not TBP show a high level of expression, suggesting the possibility of TBP independent genes.\"],\n",
       " 816: ['1471-2105-9-58-4-1',\n",
       "  'GFIG',\n",
       "  'Error rate distribution analysis for signatures related to breast cancer metastasis . ( A ) Using 295 breast cancer samples (published by van de Vijver  et al .), SET can reduce the gene number from 70 to 49 without losing prediction power (error rate 0.325 and 0.315, respectively). ( B ) For 151 samples with lymph-node-negative pathology status, the top 12 genes had a similar discrimination power to that of 70 genes (error rate 0.272 and 0.291, respectively; p value  <  0.001). ( C ) For 144 samples with lymph-node-positive pathology status, the top 8 genes had a similar discrimination power to that of 70 genes (error rate 0.319 and 0.396, respectively; p value  <  0.001).,'],\n",
       " 817: ['1471-2105-9-58-4-2',\n",
       "  'GFIG',\n",
       "  'Error rate distribution analysis for signatures related to breast cancer metastasis . ( A ) Using 295 breast cancer samples (published by van de Vijver  et al .), SET can reduce the gene number from 70 to 49 without losing prediction power (error rate 0.325 and 0.315, respectively). ( B ) For 151 samples with lymph-node-negative pathology status, the top 12 genes had a similar discrimination power to that of 70 genes (error rate 0.272 and 0.291, respectively; p value  <  0.001). ( C ) For 144 samples with lymph-node-positive pathology status, the top 8 genes had a similar discrimination power to that of 70 genes (error rate 0.319 and 0.396, respectively; p value  <  0.001).,'],\n",
       " 818: ['1471-2105-9-58-4-3',\n",
       "  'GFIG',\n",
       "  'Error rate distribution analysis for signatures related to breast cancer metastasis . ( A ) Using 295 breast cancer samples (published by van de Vijver  et al .), SET can reduce the gene number from 70 to 49 without losing prediction power (error rate 0.325 and 0.315, respectively). ( B ) For 151 samples with lymph-node-negative pathology status, the top 12 genes had a similar discrimination power to that of 70 genes (error rate 0.272 and 0.291, respectively; p value  <  0.001). ( C ) For 144 samples with lymph-node-positive pathology status, the top 8 genes had a similar discrimination power to that of 70 genes (error rate 0.319 and 0.396, respectively; p value  <  0.001).,'],\n",
       " 819: ['1471-2105-9-S6-S8-1-1',\n",
       "  'GFIG',\n",
       "  'The number of selected genes by performing REDISC and RELIC with different parameters.,'],\n",
       " 820: ['1471-2105-9-S6-S8-1-2',\n",
       "  'GFIG',\n",
       "  'The number of selected genes by performing REDISC and RELIC with different parameters.,'],\n",
       " 821: ['1471-2121-10-10-3-1',\n",
       "  'GFIG',\n",
       "  'Fructosamine assay of the products of BSA incubation with different saccharides and Western blots of D-ribose-glycated BSA . Incubation conditions were as in Figure 1, except that aliquots were taken from incubation mixtures for NBT assays at different time intervals (A). Western blotting of D-rib-glycated BSA with anti-AGEs (B); and BSA in the absence of sugar (control) (C).'],\n",
       " 822: ['1471-2121-10-10-3-2',\n",
       "  'GGEL',\n",
       "  'Fructosamine assay of the products of BSA incubation with different saccharides and Western blots of D-ribose-glycated BSA . Incubation conditions were as in Figure 1, except that aliquots were taken from incubation mixtures for NBT assays at different time intervals (A). Western blotting of D-rib-glycated BSA with anti-AGEs (B); and BSA in the absence of sugar (control) (C).'],\n",
       " 823: ['1471-2121-10-10-3-3',\n",
       "  'GGEL',\n",
       "  'Fructosamine assay of the products of BSA incubation with different saccharides and Western blots of D-ribose-glycated BSA . Incubation conditions were as in Figure 1, except that aliquots were taken from incubation mixtures for NBT assays at different time intervals (A). Western blotting of D-rib-glycated BSA with anti-AGEs (B); and BSA in the absence of sugar (control) (C).'],\n",
       " 824: ['1471-2121-10-27-2-1',\n",
       "  'GGEL',\n",
       "  'AhR agonist TCDD and AhR antagonist RSV regulated MMP-9 expression and activity in AGS cells . After different concentrations (as shown above) of TCDD treatment for 24 hours, (A) RT-PCR analysis of MMP-9 mRNA expression in a concentration-response. (B) Gelatin zymography analysis of MMP-9 protein activity in a concentration-response. After co-treatment with TCDD (1 nM) and RSV (at different concentrations as shown above) for 24 hours, (C) mRNA expression of MMP-9 was detected by RT-PCR. (D) Protein activity of MMP-9 was detected by gelatin zymography.,'],\n",
       " 825: ['1471-2121-10-27-2-2',\n",
       "  'GGEL',\n",
       "  'AhR agonist TCDD and AhR antagonist RSV regulated MMP-9 expression and activity in AGS cells . After different concentrations (as shown above) of TCDD treatment for 24 hours, (A) RT-PCR analysis of MMP-9 mRNA expression in a concentration-response. (B) Gelatin zymography analysis of MMP-9 protein activity in a concentration-response. After co-treatment with TCDD (1 nM) and RSV (at different concentrations as shown above) for 24 hours, (C) mRNA expression of MMP-9 was detected by RT-PCR. (D) Protein activity of MMP-9 was detected by gelatin zymography.,'],\n",
       " 826: ['1471-2121-10-27-2-3',\n",
       "  'GGEL',\n",
       "  'AhR agonist TCDD and AhR antagonist RSV regulated MMP-9 expression and activity in AGS cells . After different concentrations (as shown above) of TCDD treatment for 24 hours, (A) RT-PCR analysis of MMP-9 mRNA expression in a concentration-response. (B) Gelatin zymography analysis of MMP-9 protein activity in a concentration-response. After co-treatment with TCDD (1 nM) and RSV (at different concentrations as shown above) for 24 hours, (C) mRNA expression of MMP-9 was detected by RT-PCR. (D) Protein activity of MMP-9 was detected by gelatin zymography.,'],\n",
       " 827: ['1471-2121-10-27-2-4',\n",
       "  'GGEL',\n",
       "  'AhR agonist TCDD and AhR antagonist RSV regulated MMP-9 expression and activity in AGS cells . After different concentrations (as shown above) of TCDD treatment for 24 hours, (A) RT-PCR analysis of MMP-9 mRNA expression in a concentration-response. (B) Gelatin zymography analysis of MMP-9 protein activity in a concentration-response. After co-treatment with TCDD (1 nM) and RSV (at different concentrations as shown above) for 24 hours, (C) mRNA expression of MMP-9 was detected by RT-PCR. (D) Protein activity of MMP-9 was detected by gelatin zymography.,'],\n",
       " 828: ['1471-2121-10-42-1-1',\n",
       "  'GFIG',\n",
       "  'Mass spectrometric neutral N-glycan profile of human embryonic stem cells (hESC) .  A . MALDI-TOF MS spectrum of neutral N-glycan fraction isolated from a hESC sample.  B . Average of relative signal intensities from 40 most abundant neutral N-glycans of four finnish hESC lines (blue columns), embryoid bodies derived from the hESC lines (EB, red columns), and stage 3 differentiated cells (st.3, white columns). The columns indicate the mean abundance of each glycan signal (% of the total detected glycan signals). Error bars represent the standard error of mean. Proposed monosaccharide compositions are indicated on the x-axis and proposed structures for the major N-glycans are shown as schematic drawings. Gray circle/H: hexose, green circle: mannose, yellow circle: galactose, blue circle: glucose, gray square/N: N-acetylhexosamine, blue square: N-acetylglucosamine, red triangle/F: fucose/deoxyhexose, violet diamond/S: N-acetylneuraminic acid, light blue diamond/G: N-glycolylneuraminic acid. Asterisks indicate known polyhexose contamination that was not included in panel B.,'],\n",
       " 829: ['1471-2121-10-42-1-2',\n",
       "  'GFIG',\n",
       "  'Mass spectrometric neutral N-glycan profile of human embryonic stem cells (hESC) .  A . MALDI-TOF MS spectrum of neutral N-glycan fraction isolated from a hESC sample.  B . Average of relative signal intensities from 40 most abundant neutral N-glycans of four finnish hESC lines (blue columns), embryoid bodies derived from the hESC lines (EB, red columns), and stage 3 differentiated cells (st.3, white columns). The columns indicate the mean abundance of each glycan signal (% of the total detected glycan signals). Error bars represent the standard error of mean. Proposed monosaccharide compositions are indicated on the x-axis and proposed structures for the major N-glycans are shown as schematic drawings. Gray circle/H: hexose, green circle: mannose, yellow circle: galactose, blue circle: glucose, gray square/N: N-acetylhexosamine, blue square: N-acetylglucosamine, red triangle/F: fucose/deoxyhexose, violet diamond/S: N-acetylneuraminic acid, light blue diamond/G: N-glycolylneuraminic acid. Asterisks indicate known polyhexose contamination that was not included in panel B.,'],\n",
       " 830: ['1471-2121-10-43-5-1',\n",
       "  'DMFL',\n",
       "  'Bub2 overexpression causes a defect in cell wall formation . Wild type cells with GAL1-BUB2 (KSY 129) were grown in YPGR for eight (A. and C.) or fifteen (B. and C.) hours. A. and B. FM4-64 staining to visualize the plasma membrane. Arrow points to closed bud necks. Arrowhead points to abnormal plasma membrane at the bud neck. C. Calcofluor staining shows lack of septum between buds in chains. D. Calcofluor staining shows abnormal septum formation after longer incubation in YPGR (arrowhead). Images are a single plane from a Z series and represent open or closed bud necks as determined from examination of all planes of the series. Scale bar = 5  Î¼ m.,'],\n",
       " 831: ['1471-2121-10-43-5-2',\n",
       "  'DMFL',\n",
       "  'Bub2 overexpression causes a defect in cell wall formation . Wild type cells with GAL1-BUB2 (KSY 129) were grown in YPGR for eight (A. and C.) or fifteen (B. and C.) hours. A. and B. FM4-64 staining to visualize the plasma membrane. Arrow points to closed bud necks. Arrowhead points to abnormal plasma membrane at the bud neck. C. Calcofluor staining shows lack of septum between buds in chains. D. Calcofluor staining shows abnormal septum formation after longer incubation in YPGR (arrowhead). Images are a single plane from a Z series and represent open or closed bud necks as determined from examination of all planes of the series. Scale bar = 5  Î¼ m.,'],\n",
       " 832: ['1471-2121-10-43-5-3',\n",
       "  'DMFL',\n",
       "  'Bub2 overexpression causes a defect in cell wall formation . Wild type cells with GAL1-BUB2 (KSY 129) were grown in YPGR for eight (A. and C.) or fifteen (B. and C.) hours. A. and B. FM4-64 staining to visualize the plasma membrane. Arrow points to closed bud necks. Arrowhead points to abnormal plasma membrane at the bud neck. C. Calcofluor staining shows lack of septum between buds in chains. D. Calcofluor staining shows abnormal septum formation after longer incubation in YPGR (arrowhead). Images are a single plane from a Z series and represent open or closed bud necks as determined from examination of all planes of the series. Scale bar = 5  Î¼ m.,'],\n",
       " 833: ['1471-2121-10-43-5-4',\n",
       "  'DMFL',\n",
       "  'Bub2 overexpression causes a defect in cell wall formation . Wild type cells with GAL1-BUB2 (KSY 129) were grown in YPGR for eight (A. and C.) or fifteen (B. and C.) hours. A. and B. FM4-64 staining to visualize the plasma membrane. Arrow points to closed bud necks. Arrowhead points to abnormal plasma membrane at the bud neck. C. Calcofluor staining shows lack of septum between buds in chains. D. Calcofluor staining shows abnormal septum formation after longer incubation in YPGR (arrowhead). Images are a single plane from a Z series and represent open or closed bud necks as determined from examination of all planes of the series. Scale bar = 5  Î¼ m.,'],\n",
       " 834: ['1471-2121-10-63-4-1',\n",
       "  'DMFL',\n",
       "  'GX1-rmhTNF Î±  selectively concentrates at gastric cancer cells .  99 Tc m -GX1-rmhTNF Î±  remained in tumor tissue from 2 h to 18 h after injection (A, C), while no such accumulation was seen in the  99 Tc m -rmhTNF Î±  group (B, D). Tumors are indicated by arrows. (E) Positive staining of CD31 in tumor vasculature (200  ). (F) FITC-labeled GX1-rmhTNF Î±  bound specifically to tumor vasculature (200  ). (G) GX1-rmhTNF Î±  colocalized with CD31 in tumor vasculature (200  ). Arrow indicates a positively stained microvessel.'],\n",
       " 835: ['1471-2121-10-63-4-2',\n",
       "  'DMFL',\n",
       "  'GX1-rmhTNF Î±  selectively concentrates at gastric cancer cells .  99 Tc m -GX1-rmhTNF Î±  remained in tumor tissue from 2 h to 18 h after injection (A, C), while no such accumulation was seen in the  99 Tc m -rmhTNF Î±  group (B, D). Tumors are indicated by arrows. (E) Positive staining of CD31 in tumor vasculature (200  ). (F) FITC-labeled GX1-rmhTNF Î±  bound specifically to tumor vasculature (200  ). (G) GX1-rmhTNF Î±  colocalized with CD31 in tumor vasculature (200  ). Arrow indicates a positively stained microvessel.'],\n",
       " 836: ['1471-2121-10-63-4-3',\n",
       "  'DMFL',\n",
       "  'GX1-rmhTNF Î±  selectively concentrates at gastric cancer cells .  99 Tc m -GX1-rmhTNF Î±  remained in tumor tissue from 2 h to 18 h after injection (A, C), while no such accumulation was seen in the  99 Tc m -rmhTNF Î±  group (B, D). Tumors are indicated by arrows. (E) Positive staining of CD31 in tumor vasculature (200  ). (F) FITC-labeled GX1-rmhTNF Î±  bound specifically to tumor vasculature (200  ). (G) GX1-rmhTNF Î±  colocalized with CD31 in tumor vasculature (200  ). Arrow indicates a positively stained microvessel.'],\n",
       " 837: ['1471-2121-10-63-4-4',\n",
       "  'DMFL',\n",
       "  'GX1-rmhTNF Î±  selectively concentrates at gastric cancer cells .  99 Tc m -GX1-rmhTNF Î±  remained in tumor tissue from 2 h to 18 h after injection (A, C), while no such accumulation was seen in the  99 Tc m -rmhTNF Î±  group (B, D). Tumors are indicated by arrows. (E) Positive staining of CD31 in tumor vasculature (200  ). (F) FITC-labeled GX1-rmhTNF Î±  bound specifically to tumor vasculature (200  ). (G) GX1-rmhTNF Î±  colocalized with CD31 in tumor vasculature (200  ). Arrow indicates a positively stained microvessel.'],\n",
       " 838: ['1471-2121-10-63-4-5',\n",
       "  'DMFL',\n",
       "  'GX1-rmhTNF Î±  selectively concentrates at gastric cancer cells .  99 Tc m -GX1-rmhTNF Î±  remained in tumor tissue from 2 h to 18 h after injection (A, C), while no such accumulation was seen in the  99 Tc m -rmhTNF Î±  group (B, D). Tumors are indicated by arrows. (E) Positive staining of CD31 in tumor vasculature (200  ). (F) FITC-labeled GX1-rmhTNF Î±  bound specifically to tumor vasculature (200  ). (G) GX1-rmhTNF Î±  colocalized with CD31 in tumor vasculature (200  ). Arrow indicates a positively stained microvessel.'],\n",
       " 839: ['1471-2121-10-63-4-6',\n",
       "  'DMFL',\n",
       "  'GX1-rmhTNF Î±  selectively concentrates at gastric cancer cells .  99 Tc m -GX1-rmhTNF Î±  remained in tumor tissue from 2 h to 18 h after injection (A, C), while no such accumulation was seen in the  99 Tc m -rmhTNF Î±  group (B, D). Tumors are indicated by arrows. (E) Positive staining of CD31 in tumor vasculature (200  ). (F) FITC-labeled GX1-rmhTNF Î±  bound specifically to tumor vasculature (200  ). (G) GX1-rmhTNF Î±  colocalized with CD31 in tumor vasculature (200  ). Arrow indicates a positively stained microvessel.'],\n",
       " 840: ['1471-2121-10-65-6-1',\n",
       "  'GFIG',\n",
       "  'Calcium measurements . Calcium transients in Ag-np treated U251 and IMR-90 cells: Cells were stained with calcium binding fluorophore and subsequently activated with Ag-np ( A ). Data obtained from experiments where the U251 cells were treated with Ag-np for 4 hours and then stained with the dye ( B ). Graph denotes the calcium concentrations in Ag-np treated IMR-90 cells at 4, 24 and 48 hours. ( C ) U251 cells treated with Ag-np. * p  <  0.05 (n = 4).'],\n",
       " 841: ['1471-2121-10-65-6-2',\n",
       "  'GFIG',\n",
       "  'Calcium measurements . Calcium transients in Ag-np treated U251 and IMR-90 cells: Cells were stained with calcium binding fluorophore and subsequently activated with Ag-np ( A ). Data obtained from experiments where the U251 cells were treated with Ag-np for 4 hours and then stained with the dye ( B ). Graph denotes the calcium concentrations in Ag-np treated IMR-90 cells at 4, 24 and 48 hours. ( C ) U251 cells treated with Ag-np. * p  <  0.05 (n = 4).'],\n",
       " 842: ['1471-2121-10-65-6-3',\n",
       "  'GFIG',\n",
       "  'Calcium measurements . Calcium transients in Ag-np treated U251 and IMR-90 cells: Cells were stained with calcium binding fluorophore and subsequently activated with Ag-np ( A ). Data obtained from experiments where the U251 cells were treated with Ag-np for 4 hours and then stained with the dye ( B ). Graph denotes the calcium concentrations in Ag-np treated IMR-90 cells at 4, 24 and 48 hours. ( C ) U251 cells treated with Ag-np. * p  <  0.05 (n = 4).'],\n",
       " 843: ['1471-2121-11-60-6-10',\n",
       "  'DMFL',\n",
       "  'Effects of Crb2a extracellular domain-containing constructs on endogenous Crumbs proteins . (A-D\"\\') Confocal z-projections of 6 d rods labeled with GFP (green), panCrb (blue) and anti-HA (red) antibodies. (A,\" A\\') Wild-type rods. (B-B\"\"\\') Crb2a FL  transgenic rods. Inset in B shows fine processes emerging from the inner segment. Bracket in B\\' indicates a region of strong panCrb labeling that is weakly labeled by anti-HA antibodies. (C-C\"\"\\') Crb2a Extra_TM  transgenic rods. (D-D\"\"\\') Crb2a Extra_Secr  transgenic rods. Scale bar\", 5  Î¼ m.'],\n",
       " 844: ['1471-2121-11-60-6-11',\n",
       "  'DMFL',\n",
       "  'Effects of Crb2a extracellular domain-containing constructs on endogenous Crumbs proteins . (A-D\"\\') Confocal z-projections of 6 d rods labeled with GFP (green), panCrb (blue) and anti-HA (red) antibodies. (A,\" A\\') Wild-type rods. (B-B\"\"\\') Crb2a FL  transgenic rods. Inset in B shows fine processes emerging from the inner segment. Bracket in B\\' indicates a region of strong panCrb labeling that is weakly labeled by anti-HA antibodies. (C-C\"\"\\') Crb2a Extra_TM  transgenic rods. (D-D\"\"\\') Crb2a Extra_Secr  transgenic rods. Scale bar\", 5  Î¼ m.'],\n",
       " 845: ['1471-2121-11-60-6-12',\n",
       "  'DMFL',\n",
       "  'Effects of Crb2a extracellular domain-containing constructs on endogenous Crumbs proteins . (A-D\"\\') Confocal z-projections of 6 d rods labeled with GFP (green), panCrb (blue) and anti-HA (red) antibodies. (A,\" A\\') Wild-type rods. (B-B\"\"\\') Crb2a FL  transgenic rods. Inset in B shows fine processes emerging from the inner segment. Bracket in B\\' indicates a region of strong panCrb labeling that is weakly labeled by anti-HA antibodies. (C-C\"\"\\') Crb2a Extra_TM  transgenic rods. (D-D\"\"\\') Crb2a Extra_Secr  transgenic rods. Scale bar\", 5  Î¼ m.'],\n",
       " 846: ['1471-2121-11-60-6-13',\n",
       "  'DMFL',\n",
       "  'Effects of Crb2a extracellular domain-containing constructs on endogenous Crumbs proteins . (A-D\"\\') Confocal z-projections of 6 d rods labeled with GFP (green), panCrb (blue) and anti-HA (red) antibodies. (A,\" A\\') Wild-type rods. (B-B\"\"\\') Crb2a FL  transgenic rods. Inset in B shows fine processes emerging from the inner segment. Bracket in B\\' indicates a region of strong panCrb labeling that is weakly labeled by anti-HA antibodies. (C-C\"\"\\') Crb2a Extra_TM  transgenic rods. (D-D\"\"\\') Crb2a Extra_Secr  transgenic rods. Scale bar\", 5  Î¼ m.'],\n",
       " 847: ['1471-2121-11-60-6-14',\n",
       "  'DMFL',\n",
       "  'Effects of Crb2a extracellular domain-containing constructs on endogenous Crumbs proteins . (A-D\"\\') Confocal z-projections of 6 d rods labeled with GFP (green), panCrb (blue) and anti-HA (red) antibodies. (A,\" A\\') Wild-type rods. (B-B\"\"\\') Crb2a FL  transgenic rods. Inset in B shows fine processes emerging from the inner segment. Bracket in B\\' indicates a region of strong panCrb labeling that is weakly labeled by anti-HA antibodies. (C-C\"\"\\') Crb2a Extra_TM  transgenic rods. (D-D\"\"\\') Crb2a Extra_Secr  transgenic rods. Scale bar\", 5  Î¼ m.'],\n",
       " 848: ['1471-2121-11-60-6-15',\n",
       "  'DMFL',\n",
       "  'Effects of Crb2a extracellular domain-containing constructs on endogenous Crumbs proteins . (A-D\"\\') Confocal z-projections of 6 d rods labeled with GFP (green), panCrb (blue) and anti-HA (red) antibodies. (A,\" A\\') Wild-type rods. (B-B\"\"\\') Crb2a FL  transgenic rods. Inset in B shows fine processes emerging from the inner segment. Bracket in B\\' indicates a region of strong panCrb labeling that is weakly labeled by anti-HA antibodies. (C-C\"\"\\') Crb2a Extra_TM  transgenic rods. (D-D\"\"\\') Crb2a Extra_Secr  transgenic rods. Scale bar\", 5  Î¼ m.'],\n",
       " 849: ['1471-2121-11-60-6-16',\n",
       "  'DMFL',\n",
       "  'Effects of Crb2a extracellular domain-containing constructs on endogenous Crumbs proteins . (A-D\"\\') Confocal z-projections of 6 d rods labeled with GFP (green), panCrb (blue) and anti-HA (red) antibodies. (A,\" A\\') Wild-type rods. (B-B\"\"\\') Crb2a FL  transgenic rods. Inset in B shows fine processes emerging from the inner segment. Bracket in B\\' indicates a region of strong panCrb labeling that is weakly labeled by anti-HA antibodies. (C-C\"\"\\') Crb2a Extra_TM  transgenic rods. (D-D\"\"\\') Crb2a Extra_Secr  transgenic rods. Scale bar\", 5  Î¼ m.'],\n",
       " 850: ['1471-2121-11-60-6-1',\n",
       "  'GFIG',\n",
       "  'Effects of Crb2a extracellular domain-containing constructs on endogenous Crumbs proteins . (A-D\"\\') Confocal z-projections of 6 d rods labeled with GFP (green), panCrb (blue) and anti-HA (red) antibodies. (A,\" A\\') Wild-type rods. (B-B\"\"\\') Crb2a FL  transgenic rods. Inset in B shows fine processes emerging from the inner segment. Bracket in B\\' indicates a region of strong panCrb labeling that is weakly labeled by anti-HA antibodies. (C-C\"\"\\') Crb2a Extra_TM  transgenic rods. (D-D\"\"\\') Crb2a Extra_Secr  transgenic rods. Scale bar\", 5  Î¼ m.'],\n",
       " 851: ['1471-2121-11-60-6-2',\n",
       "  'DMFL',\n",
       "  'Effects of Crb2a extracellular domain-containing constructs on endogenous Crumbs proteins . (A-D\"\\') Confocal z-projections of 6 d rods labeled with GFP (green), panCrb (blue) and anti-HA (red) antibodies. (A,\" A\\') Wild-type rods. (B-B\"\"\\') Crb2a FL  transgenic rods. Inset in B shows fine processes emerging from the inner segment. Bracket in B\\' indicates a region of strong panCrb labeling that is weakly labeled by anti-HA antibodies. (C-C\"\"\\') Crb2a Extra_TM  transgenic rods. (D-D\"\"\\') Crb2a Extra_Secr  transgenic rods. Scale bar\", 5  Î¼ m.'],\n",
       " 852: ['1471-2121-11-60-6-3',\n",
       "  'GGEN',\n",
       "  'Effects of Crb2a extracellular domain-containing constructs on endogenous Crumbs proteins . (A-D\"\\') Confocal z-projections of 6 d rods labeled with GFP (green), panCrb (blue) and anti-HA (red) antibodies. (A,\" A\\') Wild-type rods. (B-B\"\"\\') Crb2a FL  transgenic rods. Inset in B shows fine processes emerging from the inner segment. Bracket in B\\' indicates a region of strong panCrb labeling that is weakly labeled by anti-HA antibodies. (C-C\"\"\\') Crb2a Extra_TM  transgenic rods. (D-D\"\"\\') Crb2a Extra_Secr  transgenic rods. Scale bar\", 5  Î¼ m.'],\n",
       " 853: ['1471-2121-11-60-6-4',\n",
       "  'DMFL',\n",
       "  'Effects of Crb2a extracellular domain-containing constructs on endogenous Crumbs proteins . (A-D\"\\') Confocal z-projections of 6 d rods labeled with GFP (green), panCrb (blue) and anti-HA (red) antibodies. (A,\" A\\') Wild-type rods. (B-B\"\"\\') Crb2a FL  transgenic rods. Inset in B shows fine processes emerging from the inner segment. Bracket in B\\' indicates a region of strong panCrb labeling that is weakly labeled by anti-HA antibodies. (C-C\"\"\\') Crb2a Extra_TM  transgenic rods. (D-D\"\"\\') Crb2a Extra_Secr  transgenic rods. Scale bar\", 5  Î¼ m.'],\n",
       " 854: ['1471-2121-11-60-6-5',\n",
       "  'DMFL',\n",
       "  'Effects of Crb2a extracellular domain-containing constructs on endogenous Crumbs proteins . (A-D\"\\') Confocal z-projections of 6 d rods labeled with GFP (green), panCrb (blue) and anti-HA (red) antibodies. (A,\" A\\') Wild-type rods. (B-B\"\"\\') Crb2a FL  transgenic rods. Inset in B shows fine processes emerging from the inner segment. Bracket in B\\' indicates a region of strong panCrb labeling that is weakly labeled by anti-HA antibodies. (C-C\"\"\\') Crb2a Extra_TM  transgenic rods. (D-D\"\"\\') Crb2a Extra_Secr  transgenic rods. Scale bar\", 5  Î¼ m.'],\n",
       " 855: ['1471-2121-11-60-6-6',\n",
       "  'DMFL',\n",
       "  'Effects of Crb2a extracellular domain-containing constructs on endogenous Crumbs proteins . (A-D\"\\') Confocal z-projections of 6 d rods labeled with GFP (green), panCrb (blue) and anti-HA (red) antibodies. (A,\" A\\') Wild-type rods. (B-B\"\"\\') Crb2a FL  transgenic rods. Inset in B shows fine processes emerging from the inner segment. Bracket in B\\' indicates a region of strong panCrb labeling that is weakly labeled by anti-HA antibodies. (C-C\"\"\\') Crb2a Extra_TM  transgenic rods. (D-D\"\"\\') Crb2a Extra_Secr  transgenic rods. Scale bar\", 5  Î¼ m.'],\n",
       " 856: ['1471-2121-11-60-6-7',\n",
       "  'DMFL',\n",
       "  'Effects of Crb2a extracellular domain-containing constructs on endogenous Crumbs proteins . (A-D\"\\') Confocal z-projections of 6 d rods labeled with GFP (green), panCrb (blue) and anti-HA (red) antibodies. (A,\" A\\') Wild-type rods. (B-B\"\"\\') Crb2a FL  transgenic rods. Inset in B shows fine processes emerging from the inner segment. Bracket in B\\' indicates a region of strong panCrb labeling that is weakly labeled by anti-HA antibodies. (C-C\"\"\\') Crb2a Extra_TM  transgenic rods. (D-D\"\"\\') Crb2a Extra_Secr  transgenic rods. Scale bar\", 5  Î¼ m.'],\n",
       " 857: ['1471-2121-11-60-6-8',\n",
       "  'DMFL',\n",
       "  'Effects of Crb2a extracellular domain-containing constructs on endogenous Crumbs proteins . (A-D\"\\') Confocal z-projections of 6 d rods labeled with GFP (green), panCrb (blue) and anti-HA (red) antibodies. (A,\" A\\') Wild-type rods. (B-B\"\"\\') Crb2a FL  transgenic rods. Inset in B shows fine processes emerging from the inner segment. Bracket in B\\' indicates a region of strong panCrb labeling that is weakly labeled by anti-HA antibodies. (C-C\"\"\\') Crb2a Extra_TM  transgenic rods. (D-D\"\"\\') Crb2a Extra_Secr  transgenic rods. Scale bar\", 5  Î¼ m.'],\n",
       " 858: ['1471-2121-11-60-6-9',\n",
       "  'DMFL',\n",
       "  'Effects of Crb2a extracellular domain-containing constructs on endogenous Crumbs proteins . (A-D\"\\') Confocal z-projections of 6 d rods labeled with GFP (green), panCrb (blue) and anti-HA (red) antibodies. (A,\" A\\') Wild-type rods. (B-B\"\"\\') Crb2a FL  transgenic rods. Inset in B shows fine processes emerging from the inner segment. Bracket in B\\' indicates a region of strong panCrb labeling that is weakly labeled by anti-HA antibodies. (C-C\"\"\\') Crb2a Extra_TM  transgenic rods. (D-D\"\"\\') Crb2a Extra_Secr  transgenic rods. Scale bar\", 5  Î¼ m.'],\n",
       " 859: ['1471-2121-11-93-3-1',\n",
       "  'GFIG',\n",
       "  'Venn diagram and Volcano plots representing the summary of microarray results . Figure A represents venn diagram, where each circle shows the main effects of Avadis analysis: A16 Vs NA16, A16 Vs Colony and A16 Vs Tumour. Numbers inside each compartment represents the number of transcripts that are significant for that effect. The intersections of the sets represent genes with  P  <  0.05 and Fold change 2 for each of the effects involved in the intersection. Note the expansion of transcripts that are significant and high from A16 Vs NA16 to A16 Vs Tumour. B, C and D represent Volcano plot comparison of gene expression between A16 Vs NA16, A16 Vs Colony and A16 Vs Tumour. The  x  axis indicates the differential expression profiles, plotting the fold-induction ratios in a log scale. The  y  axis indicates the statistical significance of the difference in expression ( P  value from a  t  test). Yellow color represents genes with  P  <  0.05 and Fold change 2. Squares within circles show potentially interesting genes from a biological standpoint. The increase in the fold change was observed from A16 to tumor cells and the genes that were obtained within the significant range were analysed through affymetrix database for different process based on functional annotations as shown in the Additional File  5  Figure S4.'],\n",
       " 860: ['1471-2121-11-93-3-2',\n",
       "  'GFIG',\n",
       "  'Venn diagram and Volcano plots representing the summary of microarray results . Figure A represents venn diagram, where each circle shows the main effects of Avadis analysis: A16 Vs NA16, A16 Vs Colony and A16 Vs Tumour. Numbers inside each compartment represents the number of transcripts that are significant for that effect. The intersections of the sets represent genes with  P  <  0.05 and Fold change 2 for each of the effects involved in the intersection. Note the expansion of transcripts that are significant and high from A16 Vs NA16 to A16 Vs Tumour. B, C and D represent Volcano plot comparison of gene expression between A16 Vs NA16, A16 Vs Colony and A16 Vs Tumour. The  x  axis indicates the differential expression profiles, plotting the fold-induction ratios in a log scale. The  y  axis indicates the statistical significance of the difference in expression ( P  value from a  t  test). Yellow color represents genes with  P  <  0.05 and Fold change 2. Squares within circles show potentially interesting genes from a biological standpoint. The increase in the fold change was observed from A16 to tumor cells and the genes that were obtained within the significant range were analysed through affymetrix database for different process based on functional annotations as shown in the Additional File  5  Figure S4.'],\n",
       " 861: ['1471-2121-11-93-3-3',\n",
       "  'GFIG',\n",
       "  'Venn diagram and Volcano plots representing the summary of microarray results . Figure A represents venn diagram, where each circle shows the main effects of Avadis analysis: A16 Vs NA16, A16 Vs Colony and A16 Vs Tumour. Numbers inside each compartment represents the number of transcripts that are significant for that effect. The intersections of the sets represent genes with  P  <  0.05 and Fold change 2 for each of the effects involved in the intersection. Note the expansion of transcripts that are significant and high from A16 Vs NA16 to A16 Vs Tumour. B, C and D represent Volcano plot comparison of gene expression between A16 Vs NA16, A16 Vs Colony and A16 Vs Tumour. The  x  axis indicates the differential expression profiles, plotting the fold-induction ratios in a log scale. The  y  axis indicates the statistical significance of the difference in expression ( P  value from a  t  test). Yellow color represents genes with  P  <  0.05 and Fold change 2. Squares within circles show potentially interesting genes from a biological standpoint. The increase in the fold change was observed from A16 to tumor cells and the genes that were obtained within the significant range were analysed through affymetrix database for different process based on functional annotations as shown in the Additional File  5  Figure S4.'],\n",
       " 862: ['1471-2121-11-93-3-4',\n",
       "  'GFIG',\n",
       "  'Venn diagram and Volcano plots representing the summary of microarray results . Figure A represents venn diagram, where each circle shows the main effects of Avadis analysis: A16 Vs NA16, A16 Vs Colony and A16 Vs Tumour. Numbers inside each compartment represents the number of transcripts that are significant for that effect. The intersections of the sets represent genes with  P  <  0.05 and Fold change 2 for each of the effects involved in the intersection. Note the expansion of transcripts that are significant and high from A16 Vs NA16 to A16 Vs Tumour. B, C and D represent Volcano plot comparison of gene expression between A16 Vs NA16, A16 Vs Colony and A16 Vs Tumour. The  x  axis indicates the differential expression profiles, plotting the fold-induction ratios in a log scale. The  y  axis indicates the statistical significance of the difference in expression ( P  value from a  t  test). Yellow color represents genes with  P  <  0.05 and Fold change 2. Squares within circles show potentially interesting genes from a biological standpoint. The increase in the fold change was observed from A16 to tumor cells and the genes that were obtained within the significant range were analysed through affymetrix database for different process based on functional annotations as shown in the Additional File  5  Figure S4.'],\n",
       " 863: ['1471-2121-12-10-1-1',\n",
       "  'GFIG',\n",
       "  'Effect of carbon monoxide concentration in mitochondrial swelling . Mitochondrial swelling was followed by the decrease in absorbance at 540 nm at 37 Â° C for 30 minutes.  A . Mitochondrial swelling was assessed in the presence of CO at 0,10,25,50,100, 250 and 500  Î¼ M.  Left panel , representative graphics of experiments, performed in triplicates and repeated, at least, three times.  Right panel , quantitative expression of mitochondrial swelling measured at 28 minutes of incubation. All values are mean  Â±  SD (error bars), n = 3; * p  <  0.05, compared to control mitochondria.  B . Rhodamine 123 fluorescence change ( Î» exc : 485 nm,  Î» em : 535 nm) was used to measure mitochondrial depolarization and to test CO toxicity in isolated liver mitochondria. Quantitative expression of mitochondrial depolarization measured at 15 minutes after CO treatment, all values are mean  Â±  SD (error bars), n = 3; * p  <  0.05, compared to non-treated mitochondria.  C . Mitochondria were pre-treated with CO at 0,10, 50 or 100  Î¼ M for 15 minutes at RT, and measurements were acquired at 37 Â° C after Ca 2+  at 12.5  Î¼ M addition to trigger swelling.  Left panel, r epresentative graphics of experiments, which were done in triplicates and repeated, at least, three times.  Right panel , quantitative expression of mitochondrial swelling measured at 12 minutes of incubation. The effect of 12.5  Î¼ M of Ca 2+  was normalized as 100% of swelling. All values are mean  Â±  SD (error bars), n = 3; * p  <  0.05, compared to control mitochondria; ** p  <  0.05, compared to Ca 2+  12.5  Î¼ M treated mitochondria.'],\n",
       " 864: ['1471-2121-12-10-1-2',\n",
       "  'GFIG',\n",
       "  'Effect of carbon monoxide concentration in mitochondrial swelling . Mitochondrial swelling was followed by the decrease in absorbance at 540 nm at 37 Â° C for 30 minutes.  A . Mitochondrial swelling was assessed in the presence of CO at 0,10,25,50,100, 250 and 500  Î¼ M.  Left panel , representative graphics of experiments, performed in triplicates and repeated, at least, three times.  Right panel , quantitative expression of mitochondrial swelling measured at 28 minutes of incubation. All values are mean  Â±  SD (error bars), n = 3; * p  <  0.05, compared to control mitochondria.  B . Rhodamine 123 fluorescence change ( Î» exc : 485 nm,  Î» em : 535 nm) was used to measure mitochondrial depolarization and to test CO toxicity in isolated liver mitochondria. Quantitative expression of mitochondrial depolarization measured at 15 minutes after CO treatment, all values are mean  Â±  SD (error bars), n = 3; * p  <  0.05, compared to non-treated mitochondria.  C . Mitochondria were pre-treated with CO at 0,10, 50 or 100  Î¼ M for 15 minutes at RT, and measurements were acquired at 37 Â° C after Ca 2+  at 12.5  Î¼ M addition to trigger swelling.  Left panel, r epresentative graphics of experiments, which were done in triplicates and repeated, at least, three times.  Right panel , quantitative expression of mitochondrial swelling measured at 12 minutes of incubation. The effect of 12.5  Î¼ M of Ca 2+  was normalized as 100% of swelling. All values are mean  Â±  SD (error bars), n = 3; * p  <  0.05, compared to control mitochondria; ** p  <  0.05, compared to Ca 2+  12.5  Î¼ M treated mitochondria.'],\n",
       " 865: ['1471-2121-12-10-1-3',\n",
       "  'GFIG',\n",
       "  'Effect of carbon monoxide concentration in mitochondrial swelling . Mitochondrial swelling was followed by the decrease in absorbance at 540 nm at 37 Â° C for 30 minutes.  A . Mitochondrial swelling was assessed in the presence of CO at 0,10,25,50,100, 250 and 500  Î¼ M.  Left panel , representative graphics of experiments, performed in triplicates and repeated, at least, three times.  Right panel , quantitative expression of mitochondrial swelling measured at 28 minutes of incubation. All values are mean  Â±  SD (error bars), n = 3; * p  <  0.05, compared to control mitochondria.  B . Rhodamine 123 fluorescence change ( Î» exc : 485 nm,  Î» em : 535 nm) was used to measure mitochondrial depolarization and to test CO toxicity in isolated liver mitochondria. Quantitative expression of mitochondrial depolarization measured at 15 minutes after CO treatment, all values are mean  Â±  SD (error bars), n = 3; * p  <  0.05, compared to non-treated mitochondria.  C . Mitochondria were pre-treated with CO at 0,10, 50 or 100  Î¼ M for 15 minutes at RT, and measurements were acquired at 37 Â° C after Ca 2+  at 12.5  Î¼ M addition to trigger swelling.  Left panel, r epresentative graphics of experiments, which were done in triplicates and repeated, at least, three times.  Right panel , quantitative expression of mitochondrial swelling measured at 12 minutes of incubation. The effect of 12.5  Î¼ M of Ca 2+  was normalized as 100% of swelling. All values are mean  Â±  SD (error bars), n = 3; * p  <  0.05, compared to control mitochondria; ** p  <  0.05, compared to Ca 2+  12.5  Î¼ M treated mitochondria.'],\n",
       " 866: ['1471-2121-12-10-1-4',\n",
       "  'GFIG',\n",
       "  'Effect of carbon monoxide concentration in mitochondrial swelling . Mitochondrial swelling was followed by the decrease in absorbance at 540 nm at 37 Â° C for 30 minutes.  A . Mitochondrial swelling was assessed in the presence of CO at 0,10,25,50,100, 250 and 500  Î¼ M.  Left panel , representative graphics of experiments, performed in triplicates and repeated, at least, three times.  Right panel , quantitative expression of mitochondrial swelling measured at 28 minutes of incubation. All values are mean  Â±  SD (error bars), n = 3; * p  <  0.05, compared to control mitochondria.  B . Rhodamine 123 fluorescence change ( Î» exc : 485 nm,  Î» em : 535 nm) was used to measure mitochondrial depolarization and to test CO toxicity in isolated liver mitochondria. Quantitative expression of mitochondrial depolarization measured at 15 minutes after CO treatment, all values are mean  Â±  SD (error bars), n = 3; * p  <  0.05, compared to non-treated mitochondria.  C . Mitochondria were pre-treated with CO at 0,10, 50 or 100  Î¼ M for 15 minutes at RT, and measurements were acquired at 37 Â° C after Ca 2+  at 12.5  Î¼ M addition to trigger swelling.  Left panel, r epresentative graphics of experiments, which were done in triplicates and repeated, at least, three times.  Right panel , quantitative expression of mitochondrial swelling measured at 12 minutes of incubation. The effect of 12.5  Î¼ M of Ca 2+  was normalized as 100% of swelling. All values are mean  Â±  SD (error bars), n = 3; * p  <  0.05, compared to control mitochondria; ** p  <  0.05, compared to Ca 2+  12.5  Î¼ M treated mitochondria.'],\n",
       " 867: ['1471-2121-12-10-1-5',\n",
       "  'GFIG',\n",
       "  'Effect of carbon monoxide concentration in mitochondrial swelling . Mitochondrial swelling was followed by the decrease in absorbance at 540 nm at 37 Â° C for 30 minutes.  A . Mitochondrial swelling was assessed in the presence of CO at 0,10,25,50,100, 250 and 500  Î¼ M.  Left panel , representative graphics of experiments, performed in triplicates and repeated, at least, three times.  Right panel , quantitative expression of mitochondrial swelling measured at 28 minutes of incubation. All values are mean  Â±  SD (error bars), n = 3; * p  <  0.05, compared to control mitochondria.  B . Rhodamine 123 fluorescence change ( Î» exc : 485 nm,  Î» em : 535 nm) was used to measure mitochondrial depolarization and to test CO toxicity in isolated liver mitochondria. Quantitative expression of mitochondrial depolarization measured at 15 minutes after CO treatment, all values are mean  Â±  SD (error bars), n = 3; * p  <  0.05, compared to non-treated mitochondria.  C . Mitochondria were pre-treated with CO at 0,10, 50 or 100  Î¼ M for 15 minutes at RT, and measurements were acquired at 37 Â° C after Ca 2+  at 12.5  Î¼ M addition to trigger swelling.  Left panel, r epresentative graphics of experiments, which were done in triplicates and repeated, at least, three times.  Right panel , quantitative expression of mitochondrial swelling measured at 12 minutes of incubation. The effect of 12.5  Î¼ M of Ca 2+  was normalized as 100% of swelling. All values are mean  Â±  SD (error bars), n = 3; * p  <  0.05, compared to control mitochondria; ** p  <  0.05, compared to Ca 2+  12.5  Î¼ M treated mitochondria.'],\n",
       " 868: ['1471-2121-2-16-8-1',\n",
       "  'GGEL',\n",
       "  \"Effect of AGE-modified type I collagen on the expression of iNOS in osteoblast-like cells. UMR106 and MC3T3E1 cells were cultured on control or AGE-modified type I collagen matrices at 37 Â° C for 24 hours. After this incubation, cell monolayers were lysated in Laemmli's buffer and electrophoresed under reducing conditions on an 8 % SDS-PAGE. Western blotting was performed, and specific bands were detected using a rabbit polyclonal antibody against iNOS. A representative blot from three independent experiments is presented (top figure). Images were scanned and analyzed by densitometry. The relative intensity of each band is shown (bottom figure) as a percentage of the corresponding basal condition (unmodified collagen), and represents the mean  Â±  SEM. Differences between collagen vs. AGE-collagen are as follows:  * p < 0.05;  ** p < 0.002\"],\n",
       " 869: ['1471-2121-2-16-8-2',\n",
       "  'GFIG',\n",
       "  \"Effect of AGE-modified type I collagen on the expression of iNOS in osteoblast-like cells. UMR106 and MC3T3E1 cells were cultured on control or AGE-modified type I collagen matrices at 37 Â° C for 24 hours. After this incubation, cell monolayers were lysated in Laemmli's buffer and electrophoresed under reducing conditions on an 8 % SDS-PAGE. Western blotting was performed, and specific bands were detected using a rabbit polyclonal antibody against iNOS. A representative blot from three independent experiments is presented (top figure). Images were scanned and analyzed by densitometry. The relative intensity of each band is shown (bottom figure) as a percentage of the corresponding basal condition (unmodified collagen), and represents the mean  Â±  SEM. Differences between collagen vs. AGE-collagen are as follows:  * p < 0.05;  ** p < 0.002\"],\n",
       " 870: ['1471-2121-3-7-2-1',\n",
       "  'DMEL',\n",
       "  'Scanning electron microscopic (SEM) observations showing the surface topology of control-, and di-h-HALI-treated LSEC.  ( A ) SEM micrograph of a control LSEC shows the presence of numerous fenestrae grouped in sieve plates (arrow). The bulging area contains the nucleus (N). Scale bar, 2  Î¼ m. ( B ) High-power SEM micrograph of the fenestrated cytoplasm obtained after 60 minutes exposure to 100 nM di-h-HALI. Note a typical cytoplasmic unfenestrated area (asterisk), surrounded by circular rows of very small fenestrae (arrow), suggesting nascent fenestrae fanning out into the surrounding fenestrated cytoplasm. Scale bar, 250 nm. ( C ) Shows a SEM micrograph of a LSEC treated with 100 nM di-h-HALI for 120 minutes, revealing a substantially increased number of fenestrae (large arrow). Thin nonfenestrated cytoplasmic arms (arrowheads) divide the cytoplasm into large sieve plates. In the fenestrated cytoplasm, small cytoplasmic unfenestrated areas devoid of connected fenestrae rows could be observed (small arrow), nucleus (N). Scale bar, 2  Î¼ m.'],\n",
       " 871: ['1471-2121-3-7-2-2',\n",
       "  'DMTR',\n",
       "  'Scanning electron microscopic (SEM) observations showing the surface topology of control-, and di-h-HALI-treated LSEC.  ( A ) SEM micrograph of a control LSEC shows the presence of numerous fenestrae grouped in sieve plates (arrow). The bulging area contains the nucleus (N). Scale bar, 2  Î¼ m. ( B ) High-power SEM micrograph of the fenestrated cytoplasm obtained after 60 minutes exposure to 100 nM di-h-HALI. Note a typical cytoplasmic unfenestrated area (asterisk), surrounded by circular rows of very small fenestrae (arrow), suggesting nascent fenestrae fanning out into the surrounding fenestrated cytoplasm. Scale bar, 250 nm. ( C ) Shows a SEM micrograph of a LSEC treated with 100 nM di-h-HALI for 120 minutes, revealing a substantially increased number of fenestrae (large arrow). Thin nonfenestrated cytoplasmic arms (arrowheads) divide the cytoplasm into large sieve plates. In the fenestrated cytoplasm, small cytoplasmic unfenestrated areas devoid of connected fenestrae rows could be observed (small arrow), nucleus (N). Scale bar, 2  Î¼ m.'],\n",
       " 872: ['1471-2121-3-7-2-3',\n",
       "  'DMEL',\n",
       "  'Scanning electron microscopic (SEM) observations showing the surface topology of control-, and di-h-HALI-treated LSEC.  ( A ) SEM micrograph of a control LSEC shows the presence of numerous fenestrae grouped in sieve plates (arrow). The bulging area contains the nucleus (N). Scale bar, 2  Î¼ m. ( B ) High-power SEM micrograph of the fenestrated cytoplasm obtained after 60 minutes exposure to 100 nM di-h-HALI. Note a typical cytoplasmic unfenestrated area (asterisk), surrounded by circular rows of very small fenestrae (arrow), suggesting nascent fenestrae fanning out into the surrounding fenestrated cytoplasm. Scale bar, 250 nm. ( C ) Shows a SEM micrograph of a LSEC treated with 100 nM di-h-HALI for 120 minutes, revealing a substantially increased number of fenestrae (large arrow). Thin nonfenestrated cytoplasmic arms (arrowheads) divide the cytoplasm into large sieve plates. In the fenestrated cytoplasm, small cytoplasmic unfenestrated areas devoid of connected fenestrae rows could be observed (small arrow), nucleus (N). Scale bar, 2  Î¼ m.'],\n",
       " 873: ['1471-2121-4-2-5-1',\n",
       "  'DMEL',\n",
       "  'TRITC-phalloidin staining demonstrates that altered pseudopodia are enriched for filamentous actin. Cells were electroporated with control buffer (A), or electroporated with C3 (25  Î¼ g/ml, B, C) and plated on VCAM-1 that had been immobilized on glass coverslips.'],\n",
       " 874: ['1471-2121-4-2-5-2',\n",
       "  'DMEL',\n",
       "  'TRITC-phalloidin staining demonstrates that altered pseudopodia are enriched for filamentous actin. Cells were electroporated with control buffer (A), or electroporated with C3 (25  Î¼ g/ml, B, C) and plated on VCAM-1 that had been immobilized on glass coverslips.'],\n",
       " 875: ['1471-2121-5-12-9-1',\n",
       "  'DMTR',\n",
       "  'Normal distribution of the phosphorylated RNA pol II in fibroblast nuclei of AD-EDMD patient 99-3 . Fibroblast culture cells from control (a) or patient (b) were immunolabelled with mab V/22 recognizing RNA pol II (a\",\"b\"\"). Note the normal distribution of RNA pol II all over the nucleus in control as well as in patient fibroblasts (compare a\"\" and b\"\"). The corresponding phase-contrast images (a\",b) and Hoechst fluorescences (a\\',b\\') are shown. Bar: 10  Âµ m.'],\n",
       " 876: ['1471-2121-5-12-9-2',\n",
       "  'DMFL',\n",
       "  'Normal distribution of the phosphorylated RNA pol II in fibroblast nuclei of AD-EDMD patient 99-3 . Fibroblast culture cells from control (a) or patient (b) were immunolabelled with mab V/22 recognizing RNA pol II (a\",\"b\"\"). Note the normal distribution of RNA pol II all over the nucleus in control as well as in patient fibroblasts (compare a\"\" and b\"\"). The corresponding phase-contrast images (a\",b) and Hoechst fluorescences (a\\',b\\') are shown. Bar: 10  Âµ m.'],\n",
       " 877: ['1471-2121-5-12-9-3',\n",
       "  'DMFL',\n",
       "  'Normal distribution of the phosphorylated RNA pol II in fibroblast nuclei of AD-EDMD patient 99-3 . Fibroblast culture cells from control (a) or patient (b) were immunolabelled with mab V/22 recognizing RNA pol II (a\",\"b\"\"). Note the normal distribution of RNA pol II all over the nucleus in control as well as in patient fibroblasts (compare a\"\" and b\"\"). The corresponding phase-contrast images (a\",b) and Hoechst fluorescences (a\\',b\\') are shown. Bar: 10  Âµ m.'],\n",
       " 878: ['1471-2121-5-12-9-4',\n",
       "  'DMTR',\n",
       "  'Normal distribution of the phosphorylated RNA pol II in fibroblast nuclei of AD-EDMD patient 99-3 . Fibroblast culture cells from control (a) or patient (b) were immunolabelled with mab V/22 recognizing RNA pol II (a\",\"b\"\"). Note the normal distribution of RNA pol II all over the nucleus in control as well as in patient fibroblasts (compare a\"\" and b\"\"). The corresponding phase-contrast images (a\",b) and Hoechst fluorescences (a\\',b\\') are shown. Bar: 10  Âµ m.'],\n",
       " 879: ['1471-2121-5-12-9-5',\n",
       "  'DMFL',\n",
       "  'Normal distribution of the phosphorylated RNA pol II in fibroblast nuclei of AD-EDMD patient 99-3 . Fibroblast culture cells from control (a) or patient (b) were immunolabelled with mab V/22 recognizing RNA pol II (a\",\"b\"\"). Note the normal distribution of RNA pol II all over the nucleus in control as well as in patient fibroblasts (compare a\"\" and b\"\"). The corresponding phase-contrast images (a\",b) and Hoechst fluorescences (a\\',b\\') are shown. Bar: 10  Âµ m.'],\n",
       " 880: ['1471-2121-5-12-9-6',\n",
       "  'DMFL',\n",
       "  'Normal distribution of the phosphorylated RNA pol II in fibroblast nuclei of AD-EDMD patient 99-3 . Fibroblast culture cells from control (a) or patient (b) were immunolabelled with mab V/22 recognizing RNA pol II (a\",\"b\"\"). Note the normal distribution of RNA pol II all over the nucleus in control as well as in patient fibroblasts (compare a\"\" and b\"\"). The corresponding phase-contrast images (a\",b) and Hoechst fluorescences (a\\',b\\') are shown. Bar: 10  Âµ m.'],\n",
       " 881: ['1471-2121-5-32-10-1',\n",
       "  'DMFL',\n",
       "  'IGF-II increases colony formation of L-cells in soft agar.  Forced expression of the rat pro-Igf2 gene (pro-Igf2 c ) in L-cells increases the number of soft-agar colonies by 4 to 5 fold as compared to the L-cell parent, as shown in this representative photomicrograph at 7 days (10  ).'],\n",
       " 882: ['1471-2121-5-32-10-2',\n",
       "  'DMTR',\n",
       "  'IGF-II increases colony formation of L-cells in soft agar.  Forced expression of the rat pro-Igf2 gene (pro-Igf2 c ) in L-cells increases the number of soft-agar colonies by 4 to 5 fold as compared to the L-cell parent, as shown in this representative photomicrograph at 7 days (10  ).'],\n",
       " 883: ['1471-2121-5-35-11-1',\n",
       "  'GFIG',\n",
       "  'Age-size structure of a growing culture. Panel (a) is the age-size structure for a perfectly deterministic population growing exponentially in mass during the division cycle. The dots on the exponentially increasing line are placed at equal age intervals shown by their representation at the bottom of the panel. The representation of the dots at the left of panel (a) indicates that there is a greater concentration of smaller cells than younger cells. In panel (b) the age-size structure for a population with variation in size and interdivision times is illustrated. The cloud of points (indicated by a few points as representative of the population) is one possible age-size structure. In panel (c) the newborn cells are indicated by the filled circles, the dividing cells by open circles, and the cells in the act of initiation of DNA synthesis by + signs. It can be seen that the larger cells at birth will, on average, reach the size required for initiation of DNA replication more quickly than smaller cells. This is because the larger cells are closer to the initiation size (represented by I on the right side of panel (c)). The B and D distributions at the right of panel (c) indicate the size distributions of newborn (B) cells and dividing (D) cells. The B, D, and I distributions at the top of panel (c) illustrate the age distributions for newborn, dividing, and initiating cells. The size distribution of initiating cells is drawn with a narrower distribution. Variations in mass increase during the period after initiation lead to the widening of the size distribution at division. Panel (d) is a replotting of the pattern in panel (c) with the bottom time scale defined by setting the time of initiation of DNA synthesisas age 0.0. Cells before initiation have a negative age value, and cells after initiation have a positive age value. Initiation takes place, by definition in this panel, at age 0.0. There is some variation in the size of cells at initiation, but it is proposed that this variation is less than the variation at other events of the cell cycle. The narrowing of the age-size structure at the time of initiation is a graphic representation of the size-homeostasis mechanism. No matter what size cells are present at birth or division, these cells are returned to their proper age-size relationship at the instant of initiation of DNA synthesis. Larger cells at division produce larger newborn cells which then reach initiation size earlier than smaller cells which were produced by the division of smaller dividing cells. This is a restatement of the idea that larger cells get to initiation earlier because larger cells have less of a negative age value at cell birth. At the top and right panels of (c) and (d) are representation of the presumed variation of the sizes and ages of cells at particular events. The size at birth is always a little more widely distributed than the size at division due to a slight inequality of partition of mass at division. The size at initiation of DNA replication is drawn with a relatively small variability.'],\n",
       " 884: ['1471-2121-5-35-11-2',\n",
       "  'GFIG',\n",
       "  'Age-size structure of a growing culture. Panel (a) is the age-size structure for a perfectly deterministic population growing exponentially in mass during the division cycle. The dots on the exponentially increasing line are placed at equal age intervals shown by their representation at the bottom of the panel. The representation of the dots at the left of panel (a) indicates that there is a greater concentration of smaller cells than younger cells. In panel (b) the age-size structure for a population with variation in size and interdivision times is illustrated. The cloud of points (indicated by a few points as representative of the population) is one possible age-size structure. In panel (c) the newborn cells are indicated by the filled circles, the dividing cells by open circles, and the cells in the act of initiation of DNA synthesis by + signs. It can be seen that the larger cells at birth will, on average, reach the size required for initiation of DNA replication more quickly than smaller cells. This is because the larger cells are closer to the initiation size (represented by I on the right side of panel (c)). The B and D distributions at the right of panel (c) indicate the size distributions of newborn (B) cells and dividing (D) cells. The B, D, and I distributions at the top of panel (c) illustrate the age distributions for newborn, dividing, and initiating cells. The size distribution of initiating cells is drawn with a narrower distribution. Variations in mass increase during the period after initiation lead to the widening of the size distribution at division. Panel (d) is a replotting of the pattern in panel (c) with the bottom time scale defined by setting the time of initiation of DNA synthesisas age 0.0. Cells before initiation have a negative age value, and cells after initiation have a positive age value. Initiation takes place, by definition in this panel, at age 0.0. There is some variation in the size of cells at initiation, but it is proposed that this variation is less than the variation at other events of the cell cycle. The narrowing of the age-size structure at the time of initiation is a graphic representation of the size-homeostasis mechanism. No matter what size cells are present at birth or division, these cells are returned to their proper age-size relationship at the instant of initiation of DNA synthesis. Larger cells at division produce larger newborn cells which then reach initiation size earlier than smaller cells which were produced by the division of smaller dividing cells. This is a restatement of the idea that larger cells get to initiation earlier because larger cells have less of a negative age value at cell birth. At the top and right panels of (c) and (d) are representation of the presumed variation of the sizes and ages of cells at particular events. The size at birth is always a little more widely distributed than the size at division due to a slight inequality of partition of mass at division. The size at initiation of DNA replication is drawn with a relatively small variability.'],\n",
       " 885: ['1471-2121-5-35-11-3',\n",
       "  'GFIG',\n",
       "  'Age-size structure of a growing culture. Panel (a) is the age-size structure for a perfectly deterministic population growing exponentially in mass during the division cycle. The dots on the exponentially increasing line are placed at equal age intervals shown by their representation at the bottom of the panel. The representation of the dots at the left of panel (a) indicates that there is a greater concentration of smaller cells than younger cells. In panel (b) the age-size structure for a population with variation in size and interdivision times is illustrated. The cloud of points (indicated by a few points as representative of the population) is one possible age-size structure. In panel (c) the newborn cells are indicated by the filled circles, the dividing cells by open circles, and the cells in the act of initiation of DNA synthesis by + signs. It can be seen that the larger cells at birth will, on average, reach the size required for initiation of DNA replication more quickly than smaller cells. This is because the larger cells are closer to the initiation size (represented by I on the right side of panel (c)). The B and D distributions at the right of panel (c) indicate the size distributions of newborn (B) cells and dividing (D) cells. The B, D, and I distributions at the top of panel (c) illustrate the age distributions for newborn, dividing, and initiating cells. The size distribution of initiating cells is drawn with a narrower distribution. Variations in mass increase during the period after initiation lead to the widening of the size distribution at division. Panel (d) is a replotting of the pattern in panel (c) with the bottom time scale defined by setting the time of initiation of DNA synthesisas age 0.0. Cells before initiation have a negative age value, and cells after initiation have a positive age value. Initiation takes place, by definition in this panel, at age 0.0. There is some variation in the size of cells at initiation, but it is proposed that this variation is less than the variation at other events of the cell cycle. The narrowing of the age-size structure at the time of initiation is a graphic representation of the size-homeostasis mechanism. No matter what size cells are present at birth or division, these cells are returned to their proper age-size relationship at the instant of initiation of DNA synthesis. Larger cells at division produce larger newborn cells which then reach initiation size earlier than smaller cells which were produced by the division of smaller dividing cells. This is a restatement of the idea that larger cells get to initiation earlier because larger cells have less of a negative age value at cell birth. At the top and right panels of (c) and (d) are representation of the presumed variation of the sizes and ages of cells at particular events. The size at birth is always a little more widely distributed than the size at division due to a slight inequality of partition of mass at division. The size at initiation of DNA replication is drawn with a relatively small variability.'],\n",
       " 886: ['1471-2121-5-35-11-4',\n",
       "  'GFIG',\n",
       "  'Age-size structure of a growing culture. Panel (a) is the age-size structure for a perfectly deterministic population growing exponentially in mass during the division cycle. The dots on the exponentially increasing line are placed at equal age intervals shown by their representation at the bottom of the panel. The representation of the dots at the left of panel (a) indicates that there is a greater concentration of smaller cells than younger cells. In panel (b) the age-size structure for a population with variation in size and interdivision times is illustrated. The cloud of points (indicated by a few points as representative of the population) is one possible age-size structure. In panel (c) the newborn cells are indicated by the filled circles, the dividing cells by open circles, and the cells in the act of initiation of DNA synthesis by + signs. It can be seen that the larger cells at birth will, on average, reach the size required for initiation of DNA replication more quickly than smaller cells. This is because the larger cells are closer to the initiation size (represented by I on the right side of panel (c)). The B and D distributions at the right of panel (c) indicate the size distributions of newborn (B) cells and dividing (D) cells. The B, D, and I distributions at the top of panel (c) illustrate the age distributions for newborn, dividing, and initiating cells. The size distribution of initiating cells is drawn with a narrower distribution. Variations in mass increase during the period after initiation lead to the widening of the size distribution at division. Panel (d) is a replotting of the pattern in panel (c) with the bottom time scale defined by setting the time of initiation of DNA synthesisas age 0.0. Cells before initiation have a negative age value, and cells after initiation have a positive age value. Initiation takes place, by definition in this panel, at age 0.0. There is some variation in the size of cells at initiation, but it is proposed that this variation is less than the variation at other events of the cell cycle. The narrowing of the age-size structure at the time of initiation is a graphic representation of the size-homeostasis mechanism. No matter what size cells are present at birth or division, these cells are returned to their proper age-size relationship at the instant of initiation of DNA synthesis. Larger cells at division produce larger newborn cells which then reach initiation size earlier than smaller cells which were produced by the division of smaller dividing cells. This is a restatement of the idea that larger cells get to initiation earlier because larger cells have less of a negative age value at cell birth. At the top and right panels of (c) and (d) are representation of the presumed variation of the sizes and ages of cells at particular events. The size at birth is always a little more widely distributed than the size at division due to a slight inequality of partition of mass at division. The size at initiation of DNA replication is drawn with a relatively small variability.'],\n",
       " 887: ['1471-2121-5-40-2-1',\n",
       "  'GGEL',\n",
       "  'The cells delay with low levels of Cdt1. Wild type spores carrying myc tagged Cdt1 were germinated and irradiated as described in the legend to Figure 1A. Samples were taken for protein extracts and flow cytometry at the times indicated. Total protein extracts were prepared and the amount of Cdt1-myc was investigated by SDS-PAGE and immunoblot analyses against total Cdc2, which served as loading control, and Cdt1-myc (top panel). Quantification of the fraction of cells with a 1C DNA content is also shown in the bottom panel (filled symbols: control; open symbols: UV).,'],\n",
       " 888: ['1471-2121-5-40-2-2',\n",
       "  'GFIG',\n",
       "  'The cells delay with low levels of Cdt1. Wild type spores carrying myc tagged Cdt1 were germinated and irradiated as described in the legend to Figure 1A. Samples were taken for protein extracts and flow cytometry at the times indicated. Total protein extracts were prepared and the amount of Cdt1-myc was investigated by SDS-PAGE and immunoblot analyses against total Cdc2, which served as loading control, and Cdt1-myc (top panel). Quantification of the fraction of cells with a 1C DNA content is also shown in the bottom panel (filled symbols: control; open symbols: UV).,'],\n",
       " 889: ['1471-2121-6-12-5-1',\n",
       "  'GFIG',\n",
       "  'Light scattering and [Ca 2+ ] e  oscillations of  regA -  cells. Light scattering and [Ca 2+ ] e  were recorded as outlined in Methods. (a, b) Regular light scattering oscillations with a phase length of roughly 4 ï¿½ï¿½ 5 min but with strongly reduced amplitude as compared to wild type oscillations (see also [3]). (c) Irregular light scattering changes. (d) Oscillations of cAMP levels in the  regA -  strain were less pronounced than in the wild type; the graph shows examples of one cAMP oscillation each, determined during one spike of light scattering oscillations.,'],\n",
       " 890: ['1471-2121-6-12-5-2',\n",
       "  'GMAT',\n",
       "  'Light scattering and [Ca 2+ ] e  oscillations of  regA -  cells. Light scattering and [Ca 2+ ] e  were recorded as outlined in Methods. (a, b) Regular light scattering oscillations with a phase length of roughly 4 ï¿½ï¿½ 5 min but with strongly reduced amplitude as compared to wild type oscillations (see also [3]). (c) Irregular light scattering changes. (d) Oscillations of cAMP levels in the  regA -  strain were less pronounced than in the wild type; the graph shows examples of one cAMP oscillation each, determined during one spike of light scattering oscillations.,'],\n",
       " 891: ['1471-2121-6-12-5-3',\n",
       "  'GFIG',\n",
       "  'Light scattering and [Ca 2+ ] e  oscillations of  regA -  cells. Light scattering and [Ca 2+ ] e  were recorded as outlined in Methods. (a, b) Regular light scattering oscillations with a phase length of roughly 4 ï¿½ï¿½ 5 min but with strongly reduced amplitude as compared to wild type oscillations (see also [3]). (c) Irregular light scattering changes. (d) Oscillations of cAMP levels in the  regA -  strain were less pronounced than in the wild type; the graph shows examples of one cAMP oscillation each, determined during one spike of light scattering oscillations.,'],\n",
       " 892: ['1471-2121-6-12-5-4',\n",
       "  'GFIG',\n",
       "  'Light scattering and [Ca 2+ ] e  oscillations of  regA -  cells. Light scattering and [Ca 2+ ] e  were recorded as outlined in Methods. (a, b) Regular light scattering oscillations with a phase length of roughly 4 ï¿½ï¿½ 5 min but with strongly reduced amplitude as compared to wild type oscillations (see also [3]). (c) Irregular light scattering changes. (d) Oscillations of cAMP levels in the  regA -  strain were less pronounced than in the wild type; the graph shows examples of one cAMP oscillation each, determined during one spike of light scattering oscillations.,'],\n",
       " 893: ['1471-2121-6-12-5-5',\n",
       "  'GFIG',\n",
       "  'Light scattering and [Ca 2+ ] e  oscillations of  regA -  cells. Light scattering and [Ca 2+ ] e  were recorded as outlined in Methods. (a, b) Regular light scattering oscillations with a phase length of roughly 4 ï¿½ï¿½ 5 min but with strongly reduced amplitude as compared to wild type oscillations (see also [3]). (c) Irregular light scattering changes. (d) Oscillations of cAMP levels in the  regA -  strain were less pronounced than in the wild type; the graph shows examples of one cAMP oscillation each, determined during one spike of light scattering oscillations.,'],\n",
       " 894: ['1471-2121-6-13-10-1',\n",
       "  'GFIG',\n",
       "  'cAMP and arachidonic acid elicit Ca 2+ -release from internal stores. (A) [Ca 2+ ] e  was recorded at t 7  in  iplA -  cells with permeabilized plasma membranes. Amoebae were challenged with 1  Î¼ M cAMP and 3  Î¼ M AA, respectively. (B) The response of permeabilized wild type stimulated with 1  Î¼ M cAMP at t 6  is shown for comparison.'],\n",
       " 895: ['1471-2121-6-13-10-2',\n",
       "  'GFIG',\n",
       "  'cAMP and arachidonic acid elicit Ca 2+ -release from internal stores. (A) [Ca 2+ ] e  was recorded at t 7  in  iplA -  cells with permeabilized plasma membranes. Amoebae were challenged with 1  Î¼ M cAMP and 3  Î¼ M AA, respectively. (B) The response of permeabilized wild type stimulated with 1  Î¼ M cAMP at t 6  is shown for comparison.'],\n",
       " 896: ['1471-2121-6-13-7-1',\n",
       "  'GFIG',\n",
       "  'Recordings of Ca 2+ -fluxes in  iplA -  and wild type amoebae. [Ca 2+ ] e  was measured in cell suspensions with a Ca 2+ -sensitive electrode. (A) Treatment of amoebae with 5 mM EGTA for 30 min activated capacitative Ca 2+ -influx (one out of 12/6 determinations in 4/3 independent experiments is shown for  iplA -  and wild type, respectively). (B) Capacitative influx was blocked by the addition of 1 mM NaN 3  (one out of 5/4 determinations in 3/3 independent experiments). Measurements were done at t 3 .'],\n",
       " 897: ['1471-2121-6-13-7-2',\n",
       "  'GFIG',\n",
       "  'Recordings of Ca 2+ -fluxes in  iplA -  and wild type amoebae. [Ca 2+ ] e  was measured in cell suspensions with a Ca 2+ -sensitive electrode. (A) Treatment of amoebae with 5 mM EGTA for 30 min activated capacitative Ca 2+ -influx (one out of 12/6 determinations in 4/3 independent experiments is shown for  iplA -  and wild type, respectively). (B) Capacitative influx was blocked by the addition of 1 mM NaN 3  (one out of 5/4 determinations in 3/3 independent experiments). Measurements were done at t 3 .'],\n",
       " 898: ['1471-2121-6-13-7-3',\n",
       "  'GFIG',\n",
       "  'Recordings of Ca 2+ -fluxes in  iplA -  and wild type amoebae. [Ca 2+ ] e  was measured in cell suspensions with a Ca 2+ -sensitive electrode. (A) Treatment of amoebae with 5 mM EGTA for 30 min activated capacitative Ca 2+ -influx (one out of 12/6 determinations in 4/3 independent experiments is shown for  iplA -  and wild type, respectively). (B) Capacitative influx was blocked by the addition of 1 mM NaN 3  (one out of 5/4 determinations in 3/3 independent experiments). Measurements were done at t 3 .'],\n",
       " 899: ['1471-2121-6-13-7-4',\n",
       "  'GFIG',\n",
       "  'Recordings of Ca 2+ -fluxes in  iplA -  and wild type amoebae. [Ca 2+ ] e  was measured in cell suspensions with a Ca 2+ -sensitive electrode. (A) Treatment of amoebae with 5 mM EGTA for 30 min activated capacitative Ca 2+ -influx (one out of 12/6 determinations in 4/3 independent experiments is shown for  iplA -  and wild type, respectively). (B) Capacitative influx was blocked by the addition of 1 mM NaN 3  (one out of 5/4 determinations in 3/3 independent experiments). Measurements were done at t 3 .'],\n",
       " 900: ['1471-2121-6-17-5-1',\n",
       "  'DMFL',\n",
       "  'EB1-  N2-GFP aggregates are aggresomes. Cells expressing EB1-  N2-GFP were fixed and co-immunostained for GFP (green, panels A, D and G) and either the 20s proteosomal subunit (red, panel B), ubiquitin (red, panel E) or vimentin (red, panel H). Merged images are shown in panels C, F and I. All three aggresome markers were present in EB1-  N2-GFP aggregates. Bar = 10  Î¼ m.'],\n",
       " 901: ['1471-2121-6-17-5-2',\n",
       "  'DMFL',\n",
       "  'EB1-  N2-GFP aggregates are aggresomes. Cells expressing EB1-  N2-GFP were fixed and co-immunostained for GFP (green, panels A, D and G) and either the 20s proteosomal subunit (red, panel B), ubiquitin (red, panel E) or vimentin (red, panel H). Merged images are shown in panels C, F and I. All three aggresome markers were present in EB1-  N2-GFP aggregates. Bar = 10  Î¼ m.'],\n",
       " 902: ['1471-2121-6-17-5-3',\n",
       "  'DMFL',\n",
       "  'EB1-  N2-GFP aggregates are aggresomes. Cells expressing EB1-  N2-GFP were fixed and co-immunostained for GFP (green, panels A, D and G) and either the 20s proteosomal subunit (red, panel B), ubiquitin (red, panel E) or vimentin (red, panel H). Merged images are shown in panels C, F and I. All three aggresome markers were present in EB1-  N2-GFP aggregates. Bar = 10  Î¼ m.'],\n",
       " 903: ['1471-2121-6-17-5-4',\n",
       "  'DMFL',\n",
       "  'EB1-  N2-GFP aggregates are aggresomes. Cells expressing EB1-  N2-GFP were fixed and co-immunostained for GFP (green, panels A, D and G) and either the 20s proteosomal subunit (red, panel B), ubiquitin (red, panel E) or vimentin (red, panel H). Merged images are shown in panels C, F and I. All three aggresome markers were present in EB1-  N2-GFP aggregates. Bar = 10  Î¼ m.'],\n",
       " 904: ['1471-2121-6-17-5-5',\n",
       "  'DMFL',\n",
       "  'EB1-  N2-GFP aggregates are aggresomes. Cells expressing EB1-  N2-GFP were fixed and co-immunostained for GFP (green, panels A, D and G) and either the 20s proteosomal subunit (red, panel B), ubiquitin (red, panel E) or vimentin (red, panel H). Merged images are shown in panels C, F and I. All three aggresome markers were present in EB1-  N2-GFP aggregates. Bar = 10  Î¼ m.'],\n",
       " 905: ['1471-2121-6-17-5-6',\n",
       "  'DMFL',\n",
       "  'EB1-  N2-GFP aggregates are aggresomes. Cells expressing EB1-  N2-GFP were fixed and co-immunostained for GFP (green, panels A, D and G) and either the 20s proteosomal subunit (red, panel B), ubiquitin (red, panel E) or vimentin (red, panel H). Merged images are shown in panels C, F and I. All three aggresome markers were present in EB1-  N2-GFP aggregates. Bar = 10  Î¼ m.'],\n",
       " 906: ['1471-2121-6-17-5-7',\n",
       "  'DMFL',\n",
       "  'EB1-  N2-GFP aggregates are aggresomes. Cells expressing EB1-  N2-GFP were fixed and co-immunostained for GFP (green, panels A, D and G) and either the 20s proteosomal subunit (red, panel B), ubiquitin (red, panel E) or vimentin (red, panel H). Merged images are shown in panels C, F and I. All three aggresome markers were present in EB1-  N2-GFP aggregates. Bar = 10  Î¼ m.'],\n",
       " 907: ['1471-2121-6-17-5-8',\n",
       "  'DMFL',\n",
       "  'EB1-  N2-GFP aggregates are aggresomes. Cells expressing EB1-  N2-GFP were fixed and co-immunostained for GFP (green, panels A, D and G) and either the 20s proteosomal subunit (red, panel B), ubiquitin (red, panel E) or vimentin (red, panel H). Merged images are shown in panels C, F and I. All three aggresome markers were present in EB1-  N2-GFP aggregates. Bar = 10  Î¼ m.'],\n",
       " 908: ['1471-2121-6-17-5-9',\n",
       "  'DMFL',\n",
       "  'EB1-  N2-GFP aggregates are aggresomes. Cells expressing EB1-  N2-GFP were fixed and co-immunostained for GFP (green, panels A, D and G) and either the 20s proteosomal subunit (red, panel B), ubiquitin (red, panel E) or vimentin (red, panel H). Merged images are shown in panels C, F and I. All three aggresome markers were present in EB1-  N2-GFP aggregates. Bar = 10  Î¼ m.'],\n",
       " 909: ['1471-2121-7-16-4-1',\n",
       "  'GGEL',\n",
       "  'ALK1-induced PP1 Î±  negatively regulates transcriptional activity downstream of TGF- Î² /ALK1. ( A ) TGF- Î²  kinetics and upregulation of PP1 isoforms on mRNA level. MEECs were stimulated with TGF- Î²  or adenovirally infected with LacZ, caALK1, caALK5 or Smad7 at MOI of 500. Forty hours later, including starvation overnight, the cells were lysed, RNA was isolated and cDNA was prepared. PCR-amplified products for PP1 Î± ,  Î²  and  Î³  and Id1 are indicated on the right of the figure.  Î² -actin was included as a loading control. ( B ) MEECs were transfected with siRNA-PP1 Î±  using oligofectamine in medium without serum. Twelve hours later the transfection medium was changed to medium with 10% FCS. Forty-eight hours later, the cells were lysed. Whole cell lysate was sonicated, fractionated on SDS-PAGE and subjected to immunoblotting. The filter was incubated with antibodies against PP1 Î±  and actin to measure loading of proteins in each sample. ( C )  Left panel . MEECs were transfected with BRE-luc in the absence or presence of PP1 Î± . Sixteen hours later the cells (where indicated) were transfected with siRNA-PP1 Î±  using oligofectamine. Forty-eight hours later, luciferase activity was measured 6 h after stimulation with TGF- Î²  or not. Values are corrected for transfection efficiency as measured by  Î² -galactosidase activity. A representative experiment using triplicate samples is shown.  Right panel . MEECs were transfected with (CAGA) 12 -luc in the absence or presence of PP1 Î± . After 16 h, the cells was transfected with siRNA-PP1 Î±  (where indicated) using oligofectamine. Thirty-two hours later, cells were incubated with TGF- Î²  for an additional 16 h, whereafter luciferase activity was measured. Values are corrected for transfection efficiency as measured by  Î² -galactosidase activity. A representative experiment using triplicate samples is shown.,'],\n",
       " 910: ['1471-2121-7-16-4-2',\n",
       "  'GFIG',\n",
       "  'ALK1-induced PP1 Î±  negatively regulates transcriptional activity downstream of TGF- Î² /ALK1. ( A ) TGF- Î²  kinetics and upregulation of PP1 isoforms on mRNA level. MEECs were stimulated with TGF- Î²  or adenovirally infected with LacZ, caALK1, caALK5 or Smad7 at MOI of 500. Forty hours later, including starvation overnight, the cells were lysed, RNA was isolated and cDNA was prepared. PCR-amplified products for PP1 Î± ,  Î²  and  Î³  and Id1 are indicated on the right of the figure.  Î² -actin was included as a loading control. ( B ) MEECs were transfected with siRNA-PP1 Î±  using oligofectamine in medium without serum. Twelve hours later the transfection medium was changed to medium with 10% FCS. Forty-eight hours later, the cells were lysed. Whole cell lysate was sonicated, fractionated on SDS-PAGE and subjected to immunoblotting. The filter was incubated with antibodies against PP1 Î±  and actin to measure loading of proteins in each sample. ( C )  Left panel . MEECs were transfected with BRE-luc in the absence or presence of PP1 Î± . Sixteen hours later the cells (where indicated) were transfected with siRNA-PP1 Î±  using oligofectamine. Forty-eight hours later, luciferase activity was measured 6 h after stimulation with TGF- Î²  or not. Values are corrected for transfection efficiency as measured by  Î² -galactosidase activity. A representative experiment using triplicate samples is shown.  Right panel . MEECs were transfected with (CAGA) 12 -luc in the absence or presence of PP1 Î± . After 16 h, the cells was transfected with siRNA-PP1 Î±  (where indicated) using oligofectamine. Thirty-two hours later, cells were incubated with TGF- Î²  for an additional 16 h, whereafter luciferase activity was measured. Values are corrected for transfection efficiency as measured by  Î² -galactosidase activity. A representative experiment using triplicate samples is shown.,'],\n",
       " 911: ['1471-2121-7-16-4-3',\n",
       "  'GFIG',\n",
       "  'ALK1-induced PP1 Î±  negatively regulates transcriptional activity downstream of TGF- Î² /ALK1. ( A ) TGF- Î²  kinetics and upregulation of PP1 isoforms on mRNA level. MEECs were stimulated with TGF- Î²  or adenovirally infected with LacZ, caALK1, caALK5 or Smad7 at MOI of 500. Forty hours later, including starvation overnight, the cells were lysed, RNA was isolated and cDNA was prepared. PCR-amplified products for PP1 Î± ,  Î²  and  Î³  and Id1 are indicated on the right of the figure.  Î² -actin was included as a loading control. ( B ) MEECs were transfected with siRNA-PP1 Î±  using oligofectamine in medium without serum. Twelve hours later the transfection medium was changed to medium with 10% FCS. Forty-eight hours later, the cells were lysed. Whole cell lysate was sonicated, fractionated on SDS-PAGE and subjected to immunoblotting. The filter was incubated with antibodies against PP1 Î±  and actin to measure loading of proteins in each sample. ( C )  Left panel . MEECs were transfected with BRE-luc in the absence or presence of PP1 Î± . Sixteen hours later the cells (where indicated) were transfected with siRNA-PP1 Î±  using oligofectamine. Forty-eight hours later, luciferase activity was measured 6 h after stimulation with TGF- Î²  or not. Values are corrected for transfection efficiency as measured by  Î² -galactosidase activity. A representative experiment using triplicate samples is shown.  Right panel . MEECs were transfected with (CAGA) 12 -luc in the absence or presence of PP1 Î± . After 16 h, the cells was transfected with siRNA-PP1 Î±  (where indicated) using oligofectamine. Thirty-two hours later, cells were incubated with TGF- Î²  for an additional 16 h, whereafter luciferase activity was measured. Values are corrected for transfection efficiency as measured by  Î² -galactosidase activity. A representative experiment using triplicate samples is shown.,'],\n",
       " 912: ['1471-2121-7-16-4-4',\n",
       "  'GFIG',\n",
       "  'ALK1-induced PP1 Î±  negatively regulates transcriptional activity downstream of TGF- Î² /ALK1. ( A ) TGF- Î²  kinetics and upregulation of PP1 isoforms on mRNA level. MEECs were stimulated with TGF- Î²  or adenovirally infected with LacZ, caALK1, caALK5 or Smad7 at MOI of 500. Forty hours later, including starvation overnight, the cells were lysed, RNA was isolated and cDNA was prepared. PCR-amplified products for PP1 Î± ,  Î²  and  Î³  and Id1 are indicated on the right of the figure.  Î² -actin was included as a loading control. ( B ) MEECs were transfected with siRNA-PP1 Î±  using oligofectamine in medium without serum. Twelve hours later the transfection medium was changed to medium with 10% FCS. Forty-eight hours later, the cells were lysed. Whole cell lysate was sonicated, fractionated on SDS-PAGE and subjected to immunoblotting. The filter was incubated with antibodies against PP1 Î±  and actin to measure loading of proteins in each sample. ( C )  Left panel . MEECs were transfected with BRE-luc in the absence or presence of PP1 Î± . Sixteen hours later the cells (where indicated) were transfected with siRNA-PP1 Î±  using oligofectamine. Forty-eight hours later, luciferase activity was measured 6 h after stimulation with TGF- Î²  or not. Values are corrected for transfection efficiency as measured by  Î² -galactosidase activity. A representative experiment using triplicate samples is shown.  Right panel . MEECs were transfected with (CAGA) 12 -luc in the absence or presence of PP1 Î± . After 16 h, the cells was transfected with siRNA-PP1 Î±  (where indicated) using oligofectamine. Thirty-two hours later, cells were incubated with TGF- Î²  for an additional 16 h, whereafter luciferase activity was measured. Values are corrected for transfection efficiency as measured by  Î² -galactosidase activity. A representative experiment using triplicate samples is shown.,'],\n",
       " 913: ['1471-2121-7-28-2-1',\n",
       "  'DMFL',\n",
       "  'L cells were incubated simultaneously with Wg (green) and Cy3-conjugated transferrin (red) for 1 hour at 4 Â° C ( a-c ) or 37 Â° C ( d-f ). Arrow-heads indicate positions of Wg and transferrin co-localization.,'],\n",
       " 914: ['1471-2121-7-28-2-2',\n",
       "  'DMFL',\n",
       "  'L cells were incubated simultaneously with Wg (green) and Cy3-conjugated transferrin (red) for 1 hour at 4 Â° C ( a-c ) or 37 Â° C ( d-f ). Arrow-heads indicate positions of Wg and transferrin co-localization.,'],\n",
       " 915: ['1471-2121-7-28-2-3',\n",
       "  'DMFL',\n",
       "  'L cells were incubated simultaneously with Wg (green) and Cy3-conjugated transferrin (red) for 1 hour at 4 Â° C ( a-c ) or 37 Â° C ( d-f ). Arrow-heads indicate positions of Wg and transferrin co-localization.,'],\n",
       " 916: ['1471-2121-7-28-2-4',\n",
       "  'DMFL',\n",
       "  'L cells were incubated simultaneously with Wg (green) and Cy3-conjugated transferrin (red) for 1 hour at 4 Â° C ( a-c ) or 37 Â° C ( d-f ). Arrow-heads indicate positions of Wg and transferrin co-localization.,'],\n",
       " 917: ['1471-2121-7-28-2-5',\n",
       "  'DMFL',\n",
       "  'L cells were incubated simultaneously with Wg (green) and Cy3-conjugated transferrin (red) for 1 hour at 4 Â° C ( a-c ) or 37 Â° C ( d-f ). Arrow-heads indicate positions of Wg and transferrin co-localization.,'],\n",
       " 918: ['1471-2121-7-28-2-6',\n",
       "  'DMFL',\n",
       "  'L cells were incubated simultaneously with Wg (green) and Cy3-conjugated transferrin (red) for 1 hour at 4 Â° C ( a-c ) or 37 Â° C ( d-f ). Arrow-heads indicate positions of Wg and transferrin co-localization.,'],\n",
       " 919: ['1471-2121-7-38-4-1',\n",
       "  'GFIG',\n",
       "  'Expression of NE marker proteins and desmin in 10 tissues of adult mouse . Transcript levels of selected lamina proteins were measured by quantitative real-time PCR in ten adult C129 mouse organs or tissues. All signal values were normalized to peptidylprolyl isomerase A (Ppia), which was the most consistent reference transcript among four reference transcripts tested across all mouse tissues. The Y-axis indicates the expression level of a gene relative to the highest value (normalized to 100%) in the same tissue. The error bars indicate the standard deviation. The raw data used for preparation of this figure was derived from  Additional File 2 .'],\n",
       " 920: ['1471-2121-7-38-4-2',\n",
       "  'GFIG',\n",
       "  'Expression of NE marker proteins and desmin in 10 tissues of adult mouse . Transcript levels of selected lamina proteins were measured by quantitative real-time PCR in ten adult C129 mouse organs or tissues. All signal values were normalized to peptidylprolyl isomerase A (Ppia), which was the most consistent reference transcript among four reference transcripts tested across all mouse tissues. The Y-axis indicates the expression level of a gene relative to the highest value (normalized to 100%) in the same tissue. The error bars indicate the standard deviation. The raw data used for preparation of this figure was derived from  Additional File 2 .'],\n",
       " 921: ['1471-2121-7-38-4-3',\n",
       "  'GFIG',\n",
       "  'Expression of NE marker proteins and desmin in 10 tissues of adult mouse . Transcript levels of selected lamina proteins were measured by quantitative real-time PCR in ten adult C129 mouse organs or tissues. All signal values were normalized to peptidylprolyl isomerase A (Ppia), which was the most consistent reference transcript among four reference transcripts tested across all mouse tissues. The Y-axis indicates the expression level of a gene relative to the highest value (normalized to 100%) in the same tissue. The error bars indicate the standard deviation. The raw data used for preparation of this figure was derived from  Additional File 2 .'],\n",
       " 922: ['1471-2121-7-38-4-4',\n",
       "  'GFIG',\n",
       "  'Expression of NE marker proteins and desmin in 10 tissues of adult mouse . Transcript levels of selected lamina proteins were measured by quantitative real-time PCR in ten adult C129 mouse organs or tissues. All signal values were normalized to peptidylprolyl isomerase A (Ppia), which was the most consistent reference transcript among four reference transcripts tested across all mouse tissues. The Y-axis indicates the expression level of a gene relative to the highest value (normalized to 100%) in the same tissue. The error bars indicate the standard deviation. The raw data used for preparation of this figure was derived from  Additional File 2 .'],\n",
       " 923: ['1471-2121-7-38-4-5',\n",
       "  'GFIG',\n",
       "  'Expression of NE marker proteins and desmin in 10 tissues of adult mouse . Transcript levels of selected lamina proteins were measured by quantitative real-time PCR in ten adult C129 mouse organs or tissues. All signal values were normalized to peptidylprolyl isomerase A (Ppia), which was the most consistent reference transcript among four reference transcripts tested across all mouse tissues. The Y-axis indicates the expression level of a gene relative to the highest value (normalized to 100%) in the same tissue. The error bars indicate the standard deviation. The raw data used for preparation of this figure was derived from  Additional File 2 .'],\n",
       " 924: ['1471-2121-7-38-4-6',\n",
       "  'GFIG',\n",
       "  'Expression of NE marker proteins and desmin in 10 tissues of adult mouse . Transcript levels of selected lamina proteins were measured by quantitative real-time PCR in ten adult C129 mouse organs or tissues. All signal values were normalized to peptidylprolyl isomerase A (Ppia), which was the most consistent reference transcript among four reference transcripts tested across all mouse tissues. The Y-axis indicates the expression level of a gene relative to the highest value (normalized to 100%) in the same tissue. The error bars indicate the standard deviation. The raw data used for preparation of this figure was derived from  Additional File 2 .'],\n",
       " 925: ['1471-2121-7-38-4-7',\n",
       "  'GFIG',\n",
       "  'Expression of NE marker proteins and desmin in 10 tissues of adult mouse . Transcript levels of selected lamina proteins were measured by quantitative real-time PCR in ten adult C129 mouse organs or tissues. All signal values were normalized to peptidylprolyl isomerase A (Ppia), which was the most consistent reference transcript among four reference transcripts tested across all mouse tissues. The Y-axis indicates the expression level of a gene relative to the highest value (normalized to 100%) in the same tissue. The error bars indicate the standard deviation. The raw data used for preparation of this figure was derived from  Additional File 2 .'],\n",
       " 926: ['1471-2121-7-38-4-8',\n",
       "  'GFIG',\n",
       "  'Expression of NE marker proteins and desmin in 10 tissues of adult mouse . Transcript levels of selected lamina proteins were measured by quantitative real-time PCR in ten adult C129 mouse organs or tissues. All signal values were normalized to peptidylprolyl isomerase A (Ppia), which was the most consistent reference transcript among four reference transcripts tested across all mouse tissues. The Y-axis indicates the expression level of a gene relative to the highest value (normalized to 100%) in the same tissue. The error bars indicate the standard deviation. The raw data used for preparation of this figure was derived from  Additional File 2 .'],\n",
       " 927: ['1471-2121-7-38-4-9',\n",
       "  'GFIG',\n",
       "  'Expression of NE marker proteins and desmin in 10 tissues of adult mouse . Transcript levels of selected lamina proteins were measured by quantitative real-time PCR in ten adult C129 mouse organs or tissues. All signal values were normalized to peptidylprolyl isomerase A (Ppia), which was the most consistent reference transcript among four reference transcripts tested across all mouse tissues. The Y-axis indicates the expression level of a gene relative to the highest value (normalized to 100%) in the same tissue. The error bars indicate the standard deviation. The raw data used for preparation of this figure was derived from  Additional File 2 .'],\n",
       " 928: ['1471-2121-8-22-2-1',\n",
       "  'DMFL',\n",
       "  'Intracellular localization of wild type and glycosylation site mutated PPT1 in HeLa cells . HeLa cells were transfected with the wild type pCMV5-PPT1 plasmid or with plasmids carrying mutations S199A, S214A, S214A and S199A + S214A. The cells were double-stained using the polyclonal antibody for PPT1 (red) and the monoclonal antibody for LAMP-1, lysosomal membrane protein (green). Colocalization is shown in yellow.'],\n",
       " 929: ['1471-2121-8-22-2-2',\n",
       "  'DMFL',\n",
       "  'Intracellular localization of wild type and glycosylation site mutated PPT1 in HeLa cells . HeLa cells were transfected with the wild type pCMV5-PPT1 plasmid or with plasmids carrying mutations S199A, S214A, S214A and S199A + S214A. The cells were double-stained using the polyclonal antibody for PPT1 (red) and the monoclonal antibody for LAMP-1, lysosomal membrane protein (green). Colocalization is shown in yellow.'],\n",
       " 930: ['1471-2121-8-22-2-3',\n",
       "  'DMFL',\n",
       "  'Intracellular localization of wild type and glycosylation site mutated PPT1 in HeLa cells . HeLa cells were transfected with the wild type pCMV5-PPT1 plasmid or with plasmids carrying mutations S199A, S214A, S214A and S199A + S214A. The cells were double-stained using the polyclonal antibody for PPT1 (red) and the monoclonal antibody for LAMP-1, lysosomal membrane protein (green). Colocalization is shown in yellow.'],\n",
       " 931: ['1471-2121-8-22-2-4',\n",
       "  'DMFL',\n",
       "  'Intracellular localization of wild type and glycosylation site mutated PPT1 in HeLa cells . HeLa cells were transfected with the wild type pCMV5-PPT1 plasmid or with plasmids carrying mutations S199A, S214A, S214A and S199A + S214A. The cells were double-stained using the polyclonal antibody for PPT1 (red) and the monoclonal antibody for LAMP-1, lysosomal membrane protein (green). Colocalization is shown in yellow.'],\n",
       " 932: ['1471-2121-8-22-7-1',\n",
       "  'DMFL',\n",
       "  'Intracellular localization of wild type and mutant PPT1 in non-neuronal and neuronal cells . HeLa cells  (A-C)  transfected with pCMV5-PPT1 plasmids and neurons  (D-F)  infected with PPT1-SFV bearing the indicated mutations were double-stained for PPT1 using the GST-PPT1 antibody (green), for lysosomes using the Lamp-1 antibody (red), for synaptic vesicles using the SV2 antibody (red) and for ER using the PDI antibody (red). Colocalization is shown in yellow. Scale bar = 20  Î¼ m.,'],\n",
       " 933: ['1471-2121-8-22-7-2',\n",
       "  'DMFL',\n",
       "  'Intracellular localization of wild type and mutant PPT1 in non-neuronal and neuronal cells . HeLa cells  (A-C)  transfected with pCMV5-PPT1 plasmids and neurons  (D-F)  infected with PPT1-SFV bearing the indicated mutations were double-stained for PPT1 using the GST-PPT1 antibody (green), for lysosomes using the Lamp-1 antibody (red), for synaptic vesicles using the SV2 antibody (red) and for ER using the PDI antibody (red). Colocalization is shown in yellow. Scale bar = 20  Î¼ m.,'],\n",
       " 934: ['1471-2121-8-22-7-3',\n",
       "  'DMFL',\n",
       "  'Intracellular localization of wild type and mutant PPT1 in non-neuronal and neuronal cells . HeLa cells  (A-C)  transfected with pCMV5-PPT1 plasmids and neurons  (D-F)  infected with PPT1-SFV bearing the indicated mutations were double-stained for PPT1 using the GST-PPT1 antibody (green), for lysosomes using the Lamp-1 antibody (red), for synaptic vesicles using the SV2 antibody (red) and for ER using the PDI antibody (red). Colocalization is shown in yellow. Scale bar = 20  Î¼ m.,'],\n",
       " 935: ['1471-2121-8-22-7-4',\n",
       "  'DMFL',\n",
       "  'Intracellular localization of wild type and mutant PPT1 in non-neuronal and neuronal cells . HeLa cells  (A-C)  transfected with pCMV5-PPT1 plasmids and neurons  (D-F)  infected with PPT1-SFV bearing the indicated mutations were double-stained for PPT1 using the GST-PPT1 antibody (green), for lysosomes using the Lamp-1 antibody (red), for synaptic vesicles using the SV2 antibody (red) and for ER using the PDI antibody (red). Colocalization is shown in yellow. Scale bar = 20  Î¼ m.,'],\n",
       " 936: ['1471-2121-8-22-7-5',\n",
       "  'DMFL',\n",
       "  'Intracellular localization of wild type and mutant PPT1 in non-neuronal and neuronal cells . HeLa cells  (A-C)  transfected with pCMV5-PPT1 plasmids and neurons  (D-F)  infected with PPT1-SFV bearing the indicated mutations were double-stained for PPT1 using the GST-PPT1 antibody (green), for lysosomes using the Lamp-1 antibody (red), for synaptic vesicles using the SV2 antibody (red) and for ER using the PDI antibody (red). Colocalization is shown in yellow. Scale bar = 20  Î¼ m.,'],\n",
       " 937: ['1471-2121-8-22-7-6',\n",
       "  'DMFL',\n",
       "  'Intracellular localization of wild type and mutant PPT1 in non-neuronal and neuronal cells . HeLa cells  (A-C)  transfected with pCMV5-PPT1 plasmids and neurons  (D-F)  infected with PPT1-SFV bearing the indicated mutations were double-stained for PPT1 using the GST-PPT1 antibody (green), for lysosomes using the Lamp-1 antibody (red), for synaptic vesicles using the SV2 antibody (red) and for ER using the PDI antibody (red). Colocalization is shown in yellow. Scale bar = 20  Î¼ m.,'],\n",
       " 938: ['1471-2121-8-26-1-1',\n",
       "  'GFIG',\n",
       "  'Induction of ATM activation and H2AX phosphorylation in A549 cells following their exposure to IM16 CS . Bivariate distributions (scattergrams; DNA content, DI  vs.  ATM-S1981 P  IF, or  vs .  Î³ H2AX IF, respectively) of the A549 cells, mock treated (Ctrl) or exposed 20 min to CS (CS), as described in the Materials and Methods. After exposure to CS or mock treatment the cells were incubated for an additional hour, then fixed and subjected to immunocytochemical detection of ATM-S1981 P  or  Î³ H2AX, their DNA counterstained with DAPI and the intensity of IF- and DAPI-fluorescence measured by iCys. The inset shows the cellular DNA content histogram representing cells from these cultures. Based on differences in DI the cells can be subdivided into G 1 , S and G 2 M compartments, as shown in the left panel. Mitotic (M) A549 cells constitutively express higher level of ATM-S1981 P  and  Î³ H2AX [14,38,70,71] and their position on the scattergrams is marked within dashed oval boundaries. The dashed line represents the upper threshold of ATM-S1981 P or  Î³ H2AX IF level for 97% of interphase cells.,'],\n",
       " 939: ['1471-2121-8-26-1-2',\n",
       "  'GFIG',\n",
       "  'Induction of ATM activation and H2AX phosphorylation in A549 cells following their exposure to IM16 CS . Bivariate distributions (scattergrams; DNA content, DI  vs.  ATM-S1981 P  IF, or  vs .  Î³ H2AX IF, respectively) of the A549 cells, mock treated (Ctrl) or exposed 20 min to CS (CS), as described in the Materials and Methods. After exposure to CS or mock treatment the cells were incubated for an additional hour, then fixed and subjected to immunocytochemical detection of ATM-S1981 P  or  Î³ H2AX, their DNA counterstained with DAPI and the intensity of IF- and DAPI-fluorescence measured by iCys. The inset shows the cellular DNA content histogram representing cells from these cultures. Based on differences in DI the cells can be subdivided into G 1 , S and G 2 M compartments, as shown in the left panel. Mitotic (M) A549 cells constitutively express higher level of ATM-S1981 P  and  Î³ H2AX [14,38,70,71] and their position on the scattergrams is marked within dashed oval boundaries. The dashed line represents the upper threshold of ATM-S1981 P or  Î³ H2AX IF level for 97% of interphase cells.,'],\n",
       " 940: ['1471-2121-8-26-1-3',\n",
       "  'GFIG',\n",
       "  'Induction of ATM activation and H2AX phosphorylation in A549 cells following their exposure to IM16 CS . Bivariate distributions (scattergrams; DNA content, DI  vs.  ATM-S1981 P  IF, or  vs .  Î³ H2AX IF, respectively) of the A549 cells, mock treated (Ctrl) or exposed 20 min to CS (CS), as described in the Materials and Methods. After exposure to CS or mock treatment the cells were incubated for an additional hour, then fixed and subjected to immunocytochemical detection of ATM-S1981 P  or  Î³ H2AX, their DNA counterstained with DAPI and the intensity of IF- and DAPI-fluorescence measured by iCys. The inset shows the cellular DNA content histogram representing cells from these cultures. Based on differences in DI the cells can be subdivided into G 1 , S and G 2 M compartments, as shown in the left panel. Mitotic (M) A549 cells constitutively express higher level of ATM-S1981 P  and  Î³ H2AX [14,38,70,71] and their position on the scattergrams is marked within dashed oval boundaries. The dashed line represents the upper threshold of ATM-S1981 P or  Î³ H2AX IF level for 97% of interphase cells.,'],\n",
       " 941: ['1471-2121-8-26-1-4',\n",
       "  'GFIG',\n",
       "  'Induction of ATM activation and H2AX phosphorylation in A549 cells following their exposure to IM16 CS . Bivariate distributions (scattergrams; DNA content, DI  vs.  ATM-S1981 P  IF, or  vs .  Î³ H2AX IF, respectively) of the A549 cells, mock treated (Ctrl) or exposed 20 min to CS (CS), as described in the Materials and Methods. After exposure to CS or mock treatment the cells were incubated for an additional hour, then fixed and subjected to immunocytochemical detection of ATM-S1981 P  or  Î³ H2AX, their DNA counterstained with DAPI and the intensity of IF- and DAPI-fluorescence measured by iCys. The inset shows the cellular DNA content histogram representing cells from these cultures. Based on differences in DI the cells can be subdivided into G 1 , S and G 2 M compartments, as shown in the left panel. Mitotic (M) A549 cells constitutively express higher level of ATM-S1981 P  and  Î³ H2AX [14,38,70,71] and their position on the scattergrams is marked within dashed oval boundaries. The dashed line represents the upper threshold of ATM-S1981 P or  Î³ H2AX IF level for 97% of interphase cells.,'],\n",
       " 942: ['1471-2121-8-26-3-1',\n",
       "  'GFIG',\n",
       "  'Induction of ATM activation in A549 cells by CS as revealed by Western blotting . The cells were exposed to 5,10, 15 or 20 min of IM16 smoke (CS) and harvested at 30 min post-exposure. The extent of ATM activation (ATM-S1981 P ) was determined by Western blotting and compared with the respective samples of the mock treated cells (M) and with expression of ATM detected by antibody that was not phospho-specific.,'],\n",
       " 943: ['1471-2121-8-26-3-2',\n",
       "  'GSCR',\n",
       "  'Induction of ATM activation in A549 cells by CS as revealed by Western blotting . The cells were exposed to 5,10, 15 or 20 min of IM16 smoke (CS) and harvested at 30 min post-exposure. The extent of ATM activation (ATM-S1981 P ) was determined by Western blotting and compared with the respective samples of the mock treated cells (M) and with expression of ATM detected by antibody that was not phospho-specific.,'],\n",
       " 944: ['1471-2121-8-40-5-1',\n",
       "  'GHDR',\n",
       "  'Result on synthesized 3D image segmentation . (a) The cross-sectional views of binary mask of the synthesized image. (b) The cross-sectional views of synthesized image with added noise. (c) The cross-sectional views of the overlaid boundaries of segmentation on synthesized noisy 3D image. (d) The cross-sectional views of randomly color-coded segmentation result.,'],\n",
       " 945: ['1471-2121-8-40-5-2',\n",
       "  'GFIG',\n",
       "  'Result on synthesized 3D image segmentation . (a) The cross-sectional views of binary mask of the synthesized image. (b) The cross-sectional views of synthesized image with added noise. (c) The cross-sectional views of the overlaid boundaries of segmentation on synthesized noisy 3D image. (d) The cross-sectional views of randomly color-coded segmentation result.,'],\n",
       " 946: ['1471-2121-8-40-5-3',\n",
       "  'GFIG',\n",
       "  'Result on synthesized 3D image segmentation . (a) The cross-sectional views of binary mask of the synthesized image. (b) The cross-sectional views of synthesized image with added noise. (c) The cross-sectional views of the overlaid boundaries of segmentation on synthesized noisy 3D image. (d) The cross-sectional views of randomly color-coded segmentation result.,'],\n",
       " 947: ['1471-2121-8-40-5-4',\n",
       "  'GFIG',\n",
       "  'Result on synthesized 3D image segmentation . (a) The cross-sectional views of binary mask of the synthesized image. (b) The cross-sectional views of synthesized image with added noise. (c) The cross-sectional views of the overlaid boundaries of segmentation on synthesized noisy 3D image. (d) The cross-sectional views of randomly color-coded segmentation result.,'],\n",
       " 948: ['1471-2121-8-8-8-1',\n",
       "  'DMEL',\n",
       "  'CLIC1 in T84 cells . Cells were grown to confluence on permeable supports, fixed, stained for CLIC1, and a stack of Z images at 0.2  Î¼ m intervals collected by confocal microscopy.  A:  image from the apex of the cells.  4 Î¼ :  image taken 4  Î¼ m below image A.  8 Î¼ :  image take 8  Î¼ m below image A.  Z:  vertical section generated from a 2  Î¼ m thick slice through the center of the stack of images. The scale bars represent 5  Î¼ m.'],\n",
       " 949: ['1471-2121-8-8-8-2',\n",
       "  'DMFL',\n",
       "  'CLIC1 in T84 cells . Cells were grown to confluence on permeable supports, fixed, stained for CLIC1, and a stack of Z images at 0.2  Î¼ m intervals collected by confocal microscopy.  A:  image from the apex of the cells.  4 Î¼ :  image taken 4  Î¼ m below image A.  8 Î¼ :  image take 8  Î¼ m below image A.  Z:  vertical section generated from a 2  Î¼ m thick slice through the center of the stack of images. The scale bars represent 5  Î¼ m.'],\n",
       " 950: ['1471-2121-8-8-8-3',\n",
       "  'DMFL',\n",
       "  'CLIC1 in T84 cells . Cells were grown to confluence on permeable supports, fixed, stained for CLIC1, and a stack of Z images at 0.2  Î¼ m intervals collected by confocal microscopy.  A:  image from the apex of the cells.  4 Î¼ :  image taken 4  Î¼ m below image A.  8 Î¼ :  image take 8  Î¼ m below image A.  Z:  vertical section generated from a 2  Î¼ m thick slice through the center of the stack of images. The scale bars represent 5  Î¼ m.'],\n",
       " 951: ['1471-2121-8-8-8-4',\n",
       "  'DMFL',\n",
       "  'CLIC1 in T84 cells . Cells were grown to confluence on permeable supports, fixed, stained for CLIC1, and a stack of Z images at 0.2  Î¼ m intervals collected by confocal microscopy.  A:  image from the apex of the cells.  4 Î¼ :  image taken 4  Î¼ m below image A.  8 Î¼ :  image take 8  Î¼ m below image A.  Z:  vertical section generated from a 2  Î¼ m thick slice through the center of the stack of images. The scale bars represent 5  Î¼ m.'],\n",
       " 952: ['1471-2121-9-10-10-1',\n",
       "  'DMFL',\n",
       "  \"GFP-fusion proteins maintain their distinct localization in cells lacking endogenous filamin and  Î± -actinin . The GFP probes were expressed in a cell line lacking both endogenous filamin and a-actinin and the localization examined. In each case, the results obtained were similar to the localization found when probes were expressed in wild-type cells. GFP-FLN maintains its cortical localization in the double mutant while GFP- Î± A is found in new protrusions but not in the cortex (Panels A and C). The same results were found for the ABD's of both proteins (Panels B and D).\"],\n",
       " 953: ['1471-2121-9-10-10-2',\n",
       "  'DMFL',\n",
       "  \"GFP-fusion proteins maintain their distinct localization in cells lacking endogenous filamin and  Î± -actinin . The GFP probes were expressed in a cell line lacking both endogenous filamin and a-actinin and the localization examined. In each case, the results obtained were similar to the localization found when probes were expressed in wild-type cells. GFP-FLN maintains its cortical localization in the double mutant while GFP- Î± A is found in new protrusions but not in the cortex (Panels A and C). The same results were found for the ABD's of both proteins (Panels B and D).\"],\n",
       " 954: ['1471-2121-9-10-10-3',\n",
       "  'DMTR',\n",
       "  \"GFP-fusion proteins maintain their distinct localization in cells lacking endogenous filamin and  Î± -actinin . The GFP probes were expressed in a cell line lacking both endogenous filamin and a-actinin and the localization examined. In each case, the results obtained were similar to the localization found when probes were expressed in wild-type cells. GFP-FLN maintains its cortical localization in the double mutant while GFP- Î± A is found in new protrusions but not in the cortex (Panels A and C). The same results were found for the ABD's of both proteins (Panels B and D).\"],\n",
       " 955: ['1471-2121-9-10-10-4',\n",
       "  'DMFL',\n",
       "  \"GFP-fusion proteins maintain their distinct localization in cells lacking endogenous filamin and  Î± -actinin . The GFP probes were expressed in a cell line lacking both endogenous filamin and a-actinin and the localization examined. In each case, the results obtained were similar to the localization found when probes were expressed in wild-type cells. GFP-FLN maintains its cortical localization in the double mutant while GFP- Î± A is found in new protrusions but not in the cortex (Panels A and C). The same results were found for the ABD's of both proteins (Panels B and D).\"],\n",
       " 956: ['1471-2121-9-10-8-1',\n",
       "  'DMFL',\n",
       "  'Immunostaining of polarized cells . AX2 wild type cells were fixed and stained with affinity purified antibodies to either filamin or a-actinin (green). The cells were then counterstained with rhodamine phalloidin (red) to visualize total F-actin. In some cells, filamin was missing from the front of the cell, in a region that was clearly stained with phalloidin (A-C).  Î± -actinin was localized to the leading edge protrusions but relatively absent from the rear of the cell. Both localization patters mirror the results found with the GFP probes.,'],\n",
       " 957: ['1471-2121-9-10-8-2',\n",
       "  'DMFL',\n",
       "  'Immunostaining of polarized cells . AX2 wild type cells were fixed and stained with affinity purified antibodies to either filamin or a-actinin (green). The cells were then counterstained with rhodamine phalloidin (red) to visualize total F-actin. In some cells, filamin was missing from the front of the cell, in a region that was clearly stained with phalloidin (A-C).  Î± -actinin was localized to the leading edge protrusions but relatively absent from the rear of the cell. Both localization patters mirror the results found with the GFP probes.,'],\n",
       " 958: ['1471-2121-9-10-8-3',\n",
       "  'DMFL',\n",
       "  'Immunostaining of polarized cells . AX2 wild type cells were fixed and stained with affinity purified antibodies to either filamin or a-actinin (green). The cells were then counterstained with rhodamine phalloidin (red) to visualize total F-actin. In some cells, filamin was missing from the front of the cell, in a region that was clearly stained with phalloidin (A-C).  Î± -actinin was localized to the leading edge protrusions but relatively absent from the rear of the cell. Both localization patters mirror the results found with the GFP probes.,'],\n",
       " 959: ['1471-2121-9-10-8-4',\n",
       "  'DMFL',\n",
       "  'Immunostaining of polarized cells . AX2 wild type cells were fixed and stained with affinity purified antibodies to either filamin or a-actinin (green). The cells were then counterstained with rhodamine phalloidin (red) to visualize total F-actin. In some cells, filamin was missing from the front of the cell, in a region that was clearly stained with phalloidin (A-C).  Î± -actinin was localized to the leading edge protrusions but relatively absent from the rear of the cell. Both localization patters mirror the results found with the GFP probes.,'],\n",
       " 960: ['1471-2121-9-10-8-5',\n",
       "  'DMFL',\n",
       "  'Immunostaining of polarized cells . AX2 wild type cells were fixed and stained with affinity purified antibodies to either filamin or a-actinin (green). The cells were then counterstained with rhodamine phalloidin (red) to visualize total F-actin. In some cells, filamin was missing from the front of the cell, in a region that was clearly stained with phalloidin (A-C).  Î± -actinin was localized to the leading edge protrusions but relatively absent from the rear of the cell. Both localization patters mirror the results found with the GFP probes.,'],\n",
       " 961: ['1471-2121-9-10-8-6',\n",
       "  'DMFL',\n",
       "  'Immunostaining of polarized cells . AX2 wild type cells were fixed and stained with affinity purified antibodies to either filamin or a-actinin (green). The cells were then counterstained with rhodamine phalloidin (red) to visualize total F-actin. In some cells, filamin was missing from the front of the cell, in a region that was clearly stained with phalloidin (A-C).  Î± -actinin was localized to the leading edge protrusions but relatively absent from the rear of the cell. Both localization patters mirror the results found with the GFP probes.,'],\n",
       " 962: ['1471-2121-9-1-3-1',\n",
       "  'DMEL',\n",
       "  'Abnormalities observed during meiosis of Mer 3  spermatocytes: a spermatid containing two nuclei of equal size (arrows) and two Nebenkerns (A), tripolar spindle in a spermatocyte going through the second meiotic division (B), and four-polar spindle in a spermatocyte undergoing the second meiotic division (C-E). Panel C shows the phase contrast image, panel D the DAPI-stained nuclei, and panel E the merged image. Arrow points to the central spindle midzone [29]. For comparison, wild-type meiotic cells in telophase were shown (F).'],\n",
       " 963: ['1471-2121-9-1-3-2',\n",
       "  'DMEL',\n",
       "  'Abnormalities observed during meiosis of Mer 3  spermatocytes: a spermatid containing two nuclei of equal size (arrows) and two Nebenkerns (A), tripolar spindle in a spermatocyte going through the second meiotic division (B), and four-polar spindle in a spermatocyte undergoing the second meiotic division (C-E). Panel C shows the phase contrast image, panel D the DAPI-stained nuclei, and panel E the merged image. Arrow points to the central spindle midzone [29]. For comparison, wild-type meiotic cells in telophase were shown (F).'],\n",
       " 964: ['1471-2121-9-1-3-3',\n",
       "  'DMEL',\n",
       "  'Abnormalities observed during meiosis of Mer 3  spermatocytes: a spermatid containing two nuclei of equal size (arrows) and two Nebenkerns (A), tripolar spindle in a spermatocyte going through the second meiotic division (B), and four-polar spindle in a spermatocyte undergoing the second meiotic division (C-E). Panel C shows the phase contrast image, panel D the DAPI-stained nuclei, and panel E the merged image. Arrow points to the central spindle midzone [29]. For comparison, wild-type meiotic cells in telophase were shown (F).'],\n",
       " 965: ['1471-2121-9-1-3-4',\n",
       "  'DMFL',\n",
       "  'Abnormalities observed during meiosis of Mer 3  spermatocytes: a spermatid containing two nuclei of equal size (arrows) and two Nebenkerns (A), tripolar spindle in a spermatocyte going through the second meiotic division (B), and four-polar spindle in a spermatocyte undergoing the second meiotic division (C-E). Panel C shows the phase contrast image, panel D the DAPI-stained nuclei, and panel E the merged image. Arrow points to the central spindle midzone [29]. For comparison, wild-type meiotic cells in telophase were shown (F).'],\n",
       " 966: ['1471-2121-9-1-3-5',\n",
       "  'DMLI',\n",
       "  'Abnormalities observed during meiosis of Mer 3  spermatocytes: a spermatid containing two nuclei of equal size (arrows) and two Nebenkerns (A), tripolar spindle in a spermatocyte going through the second meiotic division (B), and four-polar spindle in a spermatocyte undergoing the second meiotic division (C-E). Panel C shows the phase contrast image, panel D the DAPI-stained nuclei, and panel E the merged image. Arrow points to the central spindle midzone [29]. For comparison, wild-type meiotic cells in telophase were shown (F).'],\n",
       " 967: ['1471-2121-9-1-3-6',\n",
       "  'DMLI',\n",
       "  'Abnormalities observed during meiosis of Mer 3  spermatocytes: a spermatid containing two nuclei of equal size (arrows) and two Nebenkerns (A), tripolar spindle in a spermatocyte going through the second meiotic division (B), and four-polar spindle in a spermatocyte undergoing the second meiotic division (C-E). Panel C shows the phase contrast image, panel D the DAPI-stained nuclei, and panel E the merged image. Arrow points to the central spindle midzone [29]. For comparison, wild-type meiotic cells in telophase were shown (F).'],\n",
       " 968: ['1471-2121-9-1-4-1',\n",
       "  'DMFL',\n",
       "  'Difference in nuclear grouping during cyst polarization between the control and Merlin mutant spermatids. Dissected testes from the control  FM6  (A),  Mer 3  (B-E), and  Mer 4  (F-H) males were stained with DAPI and examined as described before. Note that while the spermatid nuclei were grouped in one area (arrow) of the control cyst (A), the spermatid nuclei in the  Mer 3  (B-E) and  Mer 4  (F-G) cysts were seen as two diffuse groups (arrows). In some cases, the spermatid nuclei were scattered in the mutant cyst (H). Panel C shows a phase-contrast image of a  Mer 3  cyst, panel D displays the same cyst stained with DAPI, and panel E represents a merged image. The cyst shown in panels F and G was obtained from a male carrying the  Mer 4  mutation and a GFP marker as described in Materials and Methods.,'],\n",
       " 969: ['1471-2121-9-1-4-2',\n",
       "  'DMFL',\n",
       "  'Difference in nuclear grouping during cyst polarization between the control and Merlin mutant spermatids. Dissected testes from the control  FM6  (A),  Mer 3  (B-E), and  Mer 4  (F-H) males were stained with DAPI and examined as described before. Note that while the spermatid nuclei were grouped in one area (arrow) of the control cyst (A), the spermatid nuclei in the  Mer 3  (B-E) and  Mer 4  (F-G) cysts were seen as two diffuse groups (arrows). In some cases, the spermatid nuclei were scattered in the mutant cyst (H). Panel C shows a phase-contrast image of a  Mer 3  cyst, panel D displays the same cyst stained with DAPI, and panel E represents a merged image. The cyst shown in panels F and G was obtained from a male carrying the  Mer 4  mutation and a GFP marker as described in Materials and Methods.,'],\n",
       " 970: ['1471-2121-9-1-4-3',\n",
       "  'DMLI',\n",
       "  'Difference in nuclear grouping during cyst polarization between the control and Merlin mutant spermatids. Dissected testes from the control  FM6  (A),  Mer 3  (B-E), and  Mer 4  (F-H) males were stained with DAPI and examined as described before. Note that while the spermatid nuclei were grouped in one area (arrow) of the control cyst (A), the spermatid nuclei in the  Mer 3  (B-E) and  Mer 4  (F-G) cysts were seen as two diffuse groups (arrows). In some cases, the spermatid nuclei were scattered in the mutant cyst (H). Panel C shows a phase-contrast image of a  Mer 3  cyst, panel D displays the same cyst stained with DAPI, and panel E represents a merged image. The cyst shown in panels F and G was obtained from a male carrying the  Mer 4  mutation and a GFP marker as described in Materials and Methods.,'],\n",
       " 971: ['1471-2121-9-1-4-4',\n",
       "  'DMFL',\n",
       "  'Difference in nuclear grouping during cyst polarization between the control and Merlin mutant spermatids. Dissected testes from the control  FM6  (A),  Mer 3  (B-E), and  Mer 4  (F-H) males were stained with DAPI and examined as described before. Note that while the spermatid nuclei were grouped in one area (arrow) of the control cyst (A), the spermatid nuclei in the  Mer 3  (B-E) and  Mer 4  (F-G) cysts were seen as two diffuse groups (arrows). In some cases, the spermatid nuclei were scattered in the mutant cyst (H). Panel C shows a phase-contrast image of a  Mer 3  cyst, panel D displays the same cyst stained with DAPI, and panel E represents a merged image. The cyst shown in panels F and G was obtained from a male carrying the  Mer 4  mutation and a GFP marker as described in Materials and Methods.,'],\n",
       " 972: ['1471-2121-9-1-4-5',\n",
       "  'DMFL',\n",
       "  'Difference in nuclear grouping during cyst polarization between the control and Merlin mutant spermatids. Dissected testes from the control  FM6  (A),  Mer 3  (B-E), and  Mer 4  (F-H) males were stained with DAPI and examined as described before. Note that while the spermatid nuclei were grouped in one area (arrow) of the control cyst (A), the spermatid nuclei in the  Mer 3  (B-E) and  Mer 4  (F-G) cysts were seen as two diffuse groups (arrows). In some cases, the spermatid nuclei were scattered in the mutant cyst (H). Panel C shows a phase-contrast image of a  Mer 3  cyst, panel D displays the same cyst stained with DAPI, and panel E represents a merged image. The cyst shown in panels F and G was obtained from a male carrying the  Mer 4  mutation and a GFP marker as described in Materials and Methods.,'],\n",
       " 973: ['1471-2121-9-1-4-6',\n",
       "  'DMFL',\n",
       "  'Difference in nuclear grouping during cyst polarization between the control and Merlin mutant spermatids. Dissected testes from the control  FM6  (A),  Mer 3  (B-E), and  Mer 4  (F-H) males were stained with DAPI and examined as described before. Note that while the spermatid nuclei were grouped in one area (arrow) of the control cyst (A), the spermatid nuclei in the  Mer 3  (B-E) and  Mer 4  (F-G) cysts were seen as two diffuse groups (arrows). In some cases, the spermatid nuclei were scattered in the mutant cyst (H). Panel C shows a phase-contrast image of a  Mer 3  cyst, panel D displays the same cyst stained with DAPI, and panel E represents a merged image. The cyst shown in panels F and G was obtained from a male carrying the  Mer 4  mutation and a GFP marker as described in Materials and Methods.,'],\n",
       " 974: ['1471-2121-9-1-4-7',\n",
       "  'DMFL',\n",
       "  'Difference in nuclear grouping during cyst polarization between the control and Merlin mutant spermatids. Dissected testes from the control  FM6  (A),  Mer 3  (B-E), and  Mer 4  (F-H) males were stained with DAPI and examined as described before. Note that while the spermatid nuclei were grouped in one area (arrow) of the control cyst (A), the spermatid nuclei in the  Mer 3  (B-E) and  Mer 4  (F-G) cysts were seen as two diffuse groups (arrows). In some cases, the spermatid nuclei were scattered in the mutant cyst (H). Panel C shows a phase-contrast image of a  Mer 3  cyst, panel D displays the same cyst stained with DAPI, and panel E represents a merged image. The cyst shown in panels F and G was obtained from a male carrying the  Mer 4  mutation and a GFP marker as described in Materials and Methods.,'],\n",
       " 975: ['1471-2121-9-1-4-8',\n",
       "  'DMFL',\n",
       "  'Difference in nuclear grouping during cyst polarization between the control and Merlin mutant spermatids. Dissected testes from the control  FM6  (A),  Mer 3  (B-E), and  Mer 4  (F-H) males were stained with DAPI and examined as described before. Note that while the spermatid nuclei were grouped in one area (arrow) of the control cyst (A), the spermatid nuclei in the  Mer 3  (B-E) and  Mer 4  (F-G) cysts were seen as two diffuse groups (arrows). In some cases, the spermatid nuclei were scattered in the mutant cyst (H). Panel C shows a phase-contrast image of a  Mer 3  cyst, panel D displays the same cyst stained with DAPI, and panel E represents a merged image. The cyst shown in panels F and G was obtained from a male carrying the  Mer 4  mutation and a GFP marker as described in Materials and Methods.,'],\n",
       " 976: ['1471-2121-9-1-5-1',\n",
       "  'DMEL',\n",
       "  'Spermatid individualization in the control (A-D) and  Mer 3  (E-I) cysts. Panels A and E illustrate the phase-contrast view of a control or  Mer 3  cyst, respectively. Panels B and F show the location of the DAPI-stained sperm nuclei. Panels C and G display the sites (arrowheads) and orientation of the actin cones as visualized by FITC-conjugated phalloidin staining. Panels D and H represent merged images. Panel I is an enlarged view of the rectangular area denoted in panel F.'],\n",
       " 977: ['1471-2121-9-1-5-2',\n",
       "  'DMEL',\n",
       "  'Spermatid individualization in the control (A-D) and  Mer 3  (E-I) cysts. Panels A and E illustrate the phase-contrast view of a control or  Mer 3  cyst, respectively. Panels B and F show the location of the DAPI-stained sperm nuclei. Panels C and G display the sites (arrowheads) and orientation of the actin cones as visualized by FITC-conjugated phalloidin staining. Panels D and H represent merged images. Panel I is an enlarged view of the rectangular area denoted in panel F.'],\n",
       " 978: ['1471-2121-9-1-5-3',\n",
       "  'DMFL',\n",
       "  'Spermatid individualization in the control (A-D) and  Mer 3  (E-I) cysts. Panels A and E illustrate the phase-contrast view of a control or  Mer 3  cyst, respectively. Panels B and F show the location of the DAPI-stained sperm nuclei. Panels C and G display the sites (arrowheads) and orientation of the actin cones as visualized by FITC-conjugated phalloidin staining. Panels D and H represent merged images. Panel I is an enlarged view of the rectangular area denoted in panel F.'],\n",
       " 979: ['1471-2121-9-1-5-4',\n",
       "  'DMFL',\n",
       "  'Spermatid individualization in the control (A-D) and  Mer 3  (E-I) cysts. Panels A and E illustrate the phase-contrast view of a control or  Mer 3  cyst, respectively. Panels B and F show the location of the DAPI-stained sperm nuclei. Panels C and G display the sites (arrowheads) and orientation of the actin cones as visualized by FITC-conjugated phalloidin staining. Panels D and H represent merged images. Panel I is an enlarged view of the rectangular area denoted in panel F.'],\n",
       " 980: ['1471-2121-9-1-5-5',\n",
       "  'DMFL',\n",
       "  'Spermatid individualization in the control (A-D) and  Mer 3  (E-I) cysts. Panels A and E illustrate the phase-contrast view of a control or  Mer 3  cyst, respectively. Panels B and F show the location of the DAPI-stained sperm nuclei. Panels C and G display the sites (arrowheads) and orientation of the actin cones as visualized by FITC-conjugated phalloidin staining. Panels D and H represent merged images. Panel I is an enlarged view of the rectangular area denoted in panel F.'],\n",
       " 981: ['1471-2121-9-1-5-6',\n",
       "  'DMFL',\n",
       "  'Spermatid individualization in the control (A-D) and  Mer 3  (E-I) cysts. Panels A and E illustrate the phase-contrast view of a control or  Mer 3  cyst, respectively. Panels B and F show the location of the DAPI-stained sperm nuclei. Panels C and G display the sites (arrowheads) and orientation of the actin cones as visualized by FITC-conjugated phalloidin staining. Panels D and H represent merged images. Panel I is an enlarged view of the rectangular area denoted in panel F.'],\n",
       " 982: ['1471-2121-9-1-5-7',\n",
       "  'DMFL',\n",
       "  'Spermatid individualization in the control (A-D) and  Mer 3  (E-I) cysts. Panels A and E illustrate the phase-contrast view of a control or  Mer 3  cyst, respectively. Panels B and F show the location of the DAPI-stained sperm nuclei. Panels C and G display the sites (arrowheads) and orientation of the actin cones as visualized by FITC-conjugated phalloidin staining. Panels D and H represent merged images. Panel I is an enlarged view of the rectangular area denoted in panel F.'],\n",
       " 983: ['1471-2121-9-1-5-8',\n",
       "  'DMEL',\n",
       "  'Spermatid individualization in the control (A-D) and  Mer 3  (E-I) cysts. Panels A and E illustrate the phase-contrast view of a control or  Mer 3  cyst, respectively. Panels B and F show the location of the DAPI-stained sperm nuclei. Panels C and G display the sites (arrowheads) and orientation of the actin cones as visualized by FITC-conjugated phalloidin staining. Panels D and H represent merged images. Panel I is an enlarged view of the rectangular area denoted in panel F.'],\n",
       " 984: ['1471-2121-9-1-5-9',\n",
       "  'DMFL',\n",
       "  'Spermatid individualization in the control (A-D) and  Mer 3  (E-I) cysts. Panels A and E illustrate the phase-contrast view of a control or  Mer 3  cyst, respectively. Panels B and F show the location of the DAPI-stained sperm nuclei. Panels C and G display the sites (arrowheads) and orientation of the actin cones as visualized by FITC-conjugated phalloidin staining. Panels D and H represent merged images. Panel I is an enlarged view of the rectangular area denoted in panel F.'],\n",
       " 985: ['1471-2121-9-26-1-10',\n",
       "  'DMLI',\n",
       "  'Straw blood cells in air and in solution.  Straw cells originating from mouse, human and other animals.  A . Straw cells derived from mouse epithelia, lymphocyte, liver and lung cells.  B  Human filamentous straw cells,  C . Human smooth straw cells,  D . Pig lung straw cells,  E . Straw cells in solution; visible dark dots have diameters near 1  Î¼ m (arrows).  F . Turnover of straw cells incubated at 37 Â° C  in vitro  (   Human blood,    Rabbit sera).  G . Carton presentation of a straw cell and its components.  H . Synchronized rabbit straw cells in air. Straw cells losing filamentous extensions were incubated in sterile water at day-0.  I . Emergence of new filaments at day-1.  J . Emergence of rabbit straw cell network in 5 days (dark arrow), emergence of aggregated cell bodies in solution (white arrow).  K . Emergence of rabbit regular cells in 6 days, 12 days ( L ) and 14 days ( M ). Space bar equals to 10  Î¼ m.'],\n",
       " 986: ['1471-2121-9-26-1-11',\n",
       "  'DMTR',\n",
       "  'Straw blood cells in air and in solution.  Straw cells originating from mouse, human and other animals.  A . Straw cells derived from mouse epithelia, lymphocyte, liver and lung cells.  B  Human filamentous straw cells,  C . Human smooth straw cells,  D . Pig lung straw cells,  E . Straw cells in solution; visible dark dots have diameters near 1  Î¼ m (arrows).  F . Turnover of straw cells incubated at 37 Â° C  in vitro  (   Human blood,    Rabbit sera).  G . Carton presentation of a straw cell and its components.  H . Synchronized rabbit straw cells in air. Straw cells losing filamentous extensions were incubated in sterile water at day-0.  I . Emergence of new filaments at day-1.  J . Emergence of rabbit straw cell network in 5 days (dark arrow), emergence of aggregated cell bodies in solution (white arrow).  K . Emergence of rabbit regular cells in 6 days, 12 days ( L ) and 14 days ( M ). Space bar equals to 10  Î¼ m.'],\n",
       " 987: ['1471-2121-9-26-1-12',\n",
       "  'DMEL',\n",
       "  'Straw blood cells in air and in solution.  Straw cells originating from mouse, human and other animals.  A . Straw cells derived from mouse epithelia, lymphocyte, liver and lung cells.  B  Human filamentous straw cells,  C . Human smooth straw cells,  D . Pig lung straw cells,  E . Straw cells in solution; visible dark dots have diameters near 1  Î¼ m (arrows).  F . Turnover of straw cells incubated at 37 Â° C  in vitro  (   Human blood,    Rabbit sera).  G . Carton presentation of a straw cell and its components.  H . Synchronized rabbit straw cells in air. Straw cells losing filamentous extensions were incubated in sterile water at day-0.  I . Emergence of new filaments at day-1.  J . Emergence of rabbit straw cell network in 5 days (dark arrow), emergence of aggregated cell bodies in solution (white arrow).  K . Emergence of rabbit regular cells in 6 days, 12 days ( L ) and 14 days ( M ). Space bar equals to 10  Î¼ m.'],\n",
       " 988: ['1471-2121-9-26-1-13',\n",
       "  'DMTR',\n",
       "  'Straw blood cells in air and in solution.  Straw cells originating from mouse, human and other animals.  A . Straw cells derived from mouse epithelia, lymphocyte, liver and lung cells.  B  Human filamentous straw cells,  C . Human smooth straw cells,  D . Pig lung straw cells,  E . Straw cells in solution; visible dark dots have diameters near 1  Î¼ m (arrows).  F . Turnover of straw cells incubated at 37 Â° C  in vitro  (   Human blood,    Rabbit sera).  G . Carton presentation of a straw cell and its components.  H . Synchronized rabbit straw cells in air. Straw cells losing filamentous extensions were incubated in sterile water at day-0.  I . Emergence of new filaments at day-1.  J . Emergence of rabbit straw cell network in 5 days (dark arrow), emergence of aggregated cell bodies in solution (white arrow).  K . Emergence of rabbit regular cells in 6 days, 12 days ( L ) and 14 days ( M ). Space bar equals to 10  Î¼ m.'],\n",
       " 989: ['1471-2121-9-26-1-14',\n",
       "  'DMEL',\n",
       "  'Straw blood cells in air and in solution.  Straw cells originating from mouse, human and other animals.  A . Straw cells derived from mouse epithelia, lymphocyte, liver and lung cells.  B  Human filamentous straw cells,  C . Human smooth straw cells,  D . Pig lung straw cells,  E . Straw cells in solution; visible dark dots have diameters near 1  Î¼ m (arrows).  F . Turnover of straw cells incubated at 37 Â° C  in vitro  (   Human blood,    Rabbit sera).  G . Carton presentation of a straw cell and its components.  H . Synchronized rabbit straw cells in air. Straw cells losing filamentous extensions were incubated in sterile water at day-0.  I . Emergence of new filaments at day-1.  J . Emergence of rabbit straw cell network in 5 days (dark arrow), emergence of aggregated cell bodies in solution (white arrow).  K . Emergence of rabbit regular cells in 6 days, 12 days ( L ) and 14 days ( M ). Space bar equals to 10  Î¼ m.'],\n",
       " 990: ['1471-2121-9-26-1-15',\n",
       "  'DMEL',\n",
       "  'Straw blood cells in air and in solution.  Straw cells originating from mouse, human and other animals.  A . Straw cells derived from mouse epithelia, lymphocyte, liver and lung cells.  B  Human filamentous straw cells,  C . Human smooth straw cells,  D . Pig lung straw cells,  E . Straw cells in solution; visible dark dots have diameters near 1  Î¼ m (arrows).  F . Turnover of straw cells incubated at 37 Â° C  in vitro  (   Human blood,    Rabbit sera).  G . Carton presentation of a straw cell and its components.  H . Synchronized rabbit straw cells in air. Straw cells losing filamentous extensions were incubated in sterile water at day-0.  I . Emergence of new filaments at day-1.  J . Emergence of rabbit straw cell network in 5 days (dark arrow), emergence of aggregated cell bodies in solution (white arrow).  K . Emergence of rabbit regular cells in 6 days, 12 days ( L ) and 14 days ( M ). Space bar equals to 10  Î¼ m.'],\n",
       " 991: ['1471-2121-9-26-1-1',\n",
       "  'DMEL',\n",
       "  'Straw blood cells in air and in solution.  Straw cells originating from mouse, human and other animals.  A . Straw cells derived from mouse epithelia, lymphocyte, liver and lung cells.  B  Human filamentous straw cells,  C . Human smooth straw cells,  D . Pig lung straw cells,  E . Straw cells in solution; visible dark dots have diameters near 1  Î¼ m (arrows).  F . Turnover of straw cells incubated at 37 Â° C  in vitro  (   Human blood,    Rabbit sera).  G . Carton presentation of a straw cell and its components.  H . Synchronized rabbit straw cells in air. Straw cells losing filamentous extensions were incubated in sterile water at day-0.  I . Emergence of new filaments at day-1.  J . Emergence of rabbit straw cell network in 5 days (dark arrow), emergence of aggregated cell bodies in solution (white arrow).  K . Emergence of rabbit regular cells in 6 days, 12 days ( L ) and 14 days ( M ). Space bar equals to 10  Î¼ m.'],\n",
       " 992: ['1471-2121-9-26-1-2',\n",
       "  'DMEL',\n",
       "  'Straw blood cells in air and in solution.  Straw cells originating from mouse, human and other animals.  A . Straw cells derived from mouse epithelia, lymphocyte, liver and lung cells.  B  Human filamentous straw cells,  C . Human smooth straw cells,  D . Pig lung straw cells,  E . Straw cells in solution; visible dark dots have diameters near 1  Î¼ m (arrows).  F . Turnover of straw cells incubated at 37 Â° C  in vitro  (   Human blood,    Rabbit sera).  G . Carton presentation of a straw cell and its components.  H . Synchronized rabbit straw cells in air. Straw cells losing filamentous extensions were incubated in sterile water at day-0.  I . Emergence of new filaments at day-1.  J . Emergence of rabbit straw cell network in 5 days (dark arrow), emergence of aggregated cell bodies in solution (white arrow).  K . Emergence of rabbit regular cells in 6 days, 12 days ( L ) and 14 days ( M ). Space bar equals to 10  Î¼ m.'],\n",
       " 993: ['1471-2121-9-26-1-3',\n",
       "  'DMLI',\n",
       "  'Straw blood cells in air and in solution.  Straw cells originating from mouse, human and other animals.  A . Straw cells derived from mouse epithelia, lymphocyte, liver and lung cells.  B  Human filamentous straw cells,  C . Human smooth straw cells,  D . Pig lung straw cells,  E . Straw cells in solution; visible dark dots have diameters near 1  Î¼ m (arrows).  F . Turnover of straw cells incubated at 37 Â° C  in vitro  (   Human blood,    Rabbit sera).  G . Carton presentation of a straw cell and its components.  H . Synchronized rabbit straw cells in air. Straw cells losing filamentous extensions were incubated in sterile water at day-0.  I . Emergence of new filaments at day-1.  J . Emergence of rabbit straw cell network in 5 days (dark arrow), emergence of aggregated cell bodies in solution (white arrow).  K . Emergence of rabbit regular cells in 6 days, 12 days ( L ) and 14 days ( M ). Space bar equals to 10  Î¼ m.'],\n",
       " 994: ['1471-2121-9-26-1-4',\n",
       "  'DMTR',\n",
       "  'Straw blood cells in air and in solution.  Straw cells originating from mouse, human and other animals.  A . Straw cells derived from mouse epithelia, lymphocyte, liver and lung cells.  B  Human filamentous straw cells,  C . Human smooth straw cells,  D . Pig lung straw cells,  E . Straw cells in solution; visible dark dots have diameters near 1  Î¼ m (arrows).  F . Turnover of straw cells incubated at 37 Â° C  in vitro  (   Human blood,    Rabbit sera).  G . Carton presentation of a straw cell and its components.  H . Synchronized rabbit straw cells in air. Straw cells losing filamentous extensions were incubated in sterile water at day-0.  I . Emergence of new filaments at day-1.  J . Emergence of rabbit straw cell network in 5 days (dark arrow), emergence of aggregated cell bodies in solution (white arrow).  K . Emergence of rabbit regular cells in 6 days, 12 days ( L ) and 14 days ( M ). Space bar equals to 10  Î¼ m.'],\n",
       " 995: ['1471-2121-9-26-1-5',\n",
       "  'DMLI',\n",
       "  'Straw blood cells in air and in solution.  Straw cells originating from mouse, human and other animals.  A . Straw cells derived from mouse epithelia, lymphocyte, liver and lung cells.  B  Human filamentous straw cells,  C . Human smooth straw cells,  D . Pig lung straw cells,  E . Straw cells in solution; visible dark dots have diameters near 1  Î¼ m (arrows).  F . Turnover of straw cells incubated at 37 Â° C  in vitro  (   Human blood,    Rabbit sera).  G . Carton presentation of a straw cell and its components.  H . Synchronized rabbit straw cells in air. Straw cells losing filamentous extensions were incubated in sterile water at day-0.  I . Emergence of new filaments at day-1.  J . Emergence of rabbit straw cell network in 5 days (dark arrow), emergence of aggregated cell bodies in solution (white arrow).  K . Emergence of rabbit regular cells in 6 days, 12 days ( L ) and 14 days ( M ). Space bar equals to 10  Î¼ m.'],\n",
       " 996: ['1471-2121-9-26-1-6',\n",
       "  'GFIG',\n",
       "  'Straw blood cells in air and in solution.  Straw cells originating from mouse, human and other animals.  A . Straw cells derived from mouse epithelia, lymphocyte, liver and lung cells.  B  Human filamentous straw cells,  C . Human smooth straw cells,  D . Pig lung straw cells,  E . Straw cells in solution; visible dark dots have diameters near 1  Î¼ m (arrows).  F . Turnover of straw cells incubated at 37 Â° C  in vitro  (   Human blood,    Rabbit sera).  G . Carton presentation of a straw cell and its components.  H . Synchronized rabbit straw cells in air. Straw cells losing filamentous extensions were incubated in sterile water at day-0.  I . Emergence of new filaments at day-1.  J . Emergence of rabbit straw cell network in 5 days (dark arrow), emergence of aggregated cell bodies in solution (white arrow).  K . Emergence of rabbit regular cells in 6 days, 12 days ( L ) and 14 days ( M ). Space bar equals to 10  Î¼ m.'],\n",
       " 997: ['1471-2121-9-26-1-7',\n",
       "  'DMTR',\n",
       "  'Straw blood cells in air and in solution.  Straw cells originating from mouse, human and other animals.  A . Straw cells derived from mouse epithelia, lymphocyte, liver and lung cells.  B  Human filamentous straw cells,  C . Human smooth straw cells,  D . Pig lung straw cells,  E . Straw cells in solution; visible dark dots have diameters near 1  Î¼ m (arrows).  F . Turnover of straw cells incubated at 37 Â° C  in vitro  (   Human blood,    Rabbit sera).  G . Carton presentation of a straw cell and its components.  H . Synchronized rabbit straw cells in air. Straw cells losing filamentous extensions were incubated in sterile water at day-0.  I . Emergence of new filaments at day-1.  J . Emergence of rabbit straw cell network in 5 days (dark arrow), emergence of aggregated cell bodies in solution (white arrow).  K . Emergence of rabbit regular cells in 6 days, 12 days ( L ) and 14 days ( M ). Space bar equals to 10  Î¼ m.'],\n",
       " 998: ['1471-2121-9-26-1-8',\n",
       "  'DMTR',\n",
       "  'Straw blood cells in air and in solution.  Straw cells originating from mouse, human and other animals.  A . Straw cells derived from mouse epithelia, lymphocyte, liver and lung cells.  B  Human filamentous straw cells,  C . Human smooth straw cells,  D . Pig lung straw cells,  E . Straw cells in solution; visible dark dots have diameters near 1  Î¼ m (arrows).  F . Turnover of straw cells incubated at 37 Â° C  in vitro  (   Human blood,    Rabbit sera).  G . Carton presentation of a straw cell and its components.  H . Synchronized rabbit straw cells in air. Straw cells losing filamentous extensions were incubated in sterile water at day-0.  I . Emergence of new filaments at day-1.  J . Emergence of rabbit straw cell network in 5 days (dark arrow), emergence of aggregated cell bodies in solution (white arrow).  K . Emergence of rabbit regular cells in 6 days, 12 days ( L ) and 14 days ( M ). Space bar equals to 10  Î¼ m.'],\n",
       " 999: ['1471-2121-9-26-1-9',\n",
       "  'GFIG',\n",
       "  'Straw blood cells in air and in solution.  Straw cells originating from mouse, human and other animals.  A . Straw cells derived from mouse epithelia, lymphocyte, liver and lung cells.  B  Human filamentous straw cells,  C . Human smooth straw cells,  D . Pig lung straw cells,  E . Straw cells in solution; visible dark dots have diameters near 1  Î¼ m (arrows).  F . Turnover of straw cells incubated at 37 Â° C  in vitro  (   Human blood,    Rabbit sera).  G . Carton presentation of a straw cell and its components.  H . Synchronized rabbit straw cells in air. Straw cells losing filamentous extensions were incubated in sterile water at day-0.  I . Emergence of new filaments at day-1.  J . Emergence of rabbit straw cell network in 5 days (dark arrow), emergence of aggregated cell bodies in solution (white arrow).  K . Emergence of rabbit regular cells in 6 days, 12 days ( L ) and 14 days ( M ). Space bar equals to 10  Î¼ m.'],\n",
       " ...}"
      ]
     },
     "execution_count": 3,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "labels_dict"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "1"
      ]
     },
     "execution_count": 6,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "col_idx = data_df.columns.get_loc(\"CLASS\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.6.9"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 4
}
